Talking glossary
Candidate gene
Knockout
Haplotype
Genetic marker
Locus

Wikipedia:
GWAS
Manhattan plot
Microarray
Intrinscially disordered protein
Sequence alignment
CATH database
SCOP


Medline:
SNPs
GWAS

 Copied below
 SCOP Random coil PFam  Haplotype native state uniprot 



https://medlineplus.gov/genetics/understanding/genomicresearch/encode/

https://www.genome.gov/genetics-glossary/BRCA1-BRCA2
https://www.genome.gov/genetics-glossary/Exon
https://www.genome.gov/genetics-glossary/Centimorgan
https://www.genome.gov/genetics-glossary/Copy-Number-Variation
https://www.genome.gov/genetics-glossary/Crossing-Over
https://www.genome.gov/genetics-glossary/Cytogenetics
https://www.genome.gov/genetics-glossary/Gene-Environment-Interaction
https://www.genome.gov/genetics-glossary/Genetic-Map
https://www.genome.gov/genetics-glossary/homologous-recombination
https://www.genome.gov/genetics-glossary/hybridization


https://www.genome.gov/genetics-glossary/Microsatellite

https://www.genome.gov/genetics-glossary/Personalized-Medicine

https://www.genome.gov/genetics-glossary/Physical-Map

https://www.genome.gov/genetics-glossary/Polygenic-Trait

https://www.genome.gov/genetics-glossary/Tandem-Repeat

https://www.genome.gov/genetics-glossary/hapmap
https://www.genome.gov/genetics-glossary/LOD-Score
https://www.genome.gov/genetics-glossary/Mapping

https://www.genome.gov/genetics-glossary/Linkage





 

################################################################################################################################################################################
################################################################################################################################################################################
################################################################################################################################################################################
 
 




################################################################################################################################################################################
################################################################################################################################################################################
################################################################################################################################################################################
 

## Random coil (protein conformation)

Adapted from Wikipedia https://en.wikipedia.org/wiki/Random_coil

A random coil is a polymer conformation where the monomer subunits are oriented randomly while still being bonded to adjacent units. It is not one specific shape, but a statistical distribution of shapes for all the chains in a population of macromolecules. The conformation's name is derived from the idea that, in the absence of specific, stabilizing interactions, a polymer backbone will "sample" all possible conformations randomly. Many linear, unbranched homopolymers — in solution, or above their melting temperatures — assume (approximate) random coils.


<!-- Contents -->
  <!-- 1	Random walk model: The Gaussian chain -->
  <!-- 2	Real polymers -->
  <!-- 3	Spectroscopy -->
  <!-- 4	See also -->
  <!-- 5	References -->
  <!-- 6	External links -->
  
  
  <!-- Random walk model: The Gaussian chain -->
  <!-- Main article: Ideal chain -->
  
  <!-- Short random chain -->
  <!-- There are an enormous number of different ways in which a chain can be curled around in a relatively compact shape, like an unraveling ball of twine with much open space, and comparatively few ways it can be more or less stretched out. So, if each conformation has an equal probability or statistical weight, chains are much more likely to be ball-like than they are to be extended — a purely entropic effect. In an ensemble of chains, most of them will, therefore, be loosely balled up. This is the kind of shape any one of them will have most of the time. -->
  
  <!-- Consider a linear polymer to be a freely-jointed chain with N subunits, each of length {\displaystyle \scriptstyle \ell }\scriptstyle \ell , that occupy zero volume, so that no part of the chain excludes another from any location. One can regard the segments of each such chain in an ensemble as performing a random walk (or "random flight") in three dimensions, limited only by the constraint that each segment must be joined to its neighbors. This is the ideal chain mathematical model. It is clear that the maximum, fully extended length L of the chain is {\displaystyle \scriptstyle N\,\times \,\ell }\scriptstyle N\,\times \,\ell . If we assume that each possible chain conformation has an equal statistical weight, it can be shown that the probability P(r) of a polymer chain in the population to have distance r between the ends will obey a characteristic distribution described by the formula -->
  
  <!-- {\displaystyle P(r)=4\pi r^{2}\left({\frac {3}{2\;\pi \langle r^{2}\rangle }}\right)^{3/2}\;e^{-\,{\frac {3r^{2}}{2\langle r^{2}\rangle }}}}{\displaystyle P(r)=4\pi r^{2}\left({\frac {3}{2\;\pi \langle r^{2}\rangle }}\right)^{3/2}\;e^{-\,{\frac {3r^{2}}{2\langle r^{2}\rangle }}}} -->
  <!-- The average (root mean square) end-to-end distance for the chain, {\displaystyle \scriptstyle {\sqrt {\langle r^{2}\rangle }}}\scriptstyle {\sqrt  {\langle r^{2}\rangle }}, turns out to be {\displaystyle \scriptstyle \ell }\scriptstyle \ell  times the square root of N — in other words, the average distance scales with N0.5. -->
  
  <!-- Note that although this model is termed a "Gaussian chain", the distribution function is not a gaussian (normal) distribution. The end-to-end distance probability distribution function of a Gaussian chain is non-zero only for r > 0. [1] -->
  
  <!-- Real polymers -->
  <!-- A real polymer is not freely-jointed. A -C-C- single bond has a fixed tetrahedral angle of 109.5 degrees. The value of L is well-defined for, say, a fully extended polyethylene or nylon, but it is less than N x l because of the zig-zag backbone. There is, however, free rotation about many chain bonds. The model above can be enhanced. A longer, "effective" unit length can be defined such that the chain can be regarded as freely-jointed, along with a smaller N, such that the constraint L = N x l is still obeyed. It, too, gives a Gaussian distribution. However, specific cases can also be precisely calculated. The average end-to-end distance for freely-rotating (not freely-jointed) polymethylene (polyethylene with each -C-C- considered as a subunit) is l times the square root of 2N, an increase by a factor of about 1.4. Unlike the zero volume assumed in a random walk calculation, all real polymers' segments occupy space because of the van der Waals radii of their atoms, including bulky substituent groups that interfere with bond rotations. This can also be taken into account in calculations. All such effects increase the mean end-to-end distance. -->

<!-- Because their polymerization is stochastically driven, chain lengths in any real population of synthetic polymers will obey a statistical distribution. In that case, we should take N to be an average value. Also, many polymers have random branching. -->

<!-- Even with corrections for local constraints, the random walk model ignores steric interference between chains, and between distal parts of the same chain. A chain often cannot move from a given conformation to a closely related one by a small displacement because one part of it would have to pass through another part, or through a neighbor. We may still hope that the ideal-chain, random-coil model will be at least a qualitative indication of the shapes and dimensions of real polymers in solution, and in the amorphous state, as long as there are only weak physicochemical interactions between the monomers. This model, and the Flory-Huggins Solution Theory,[2][3] for which Paul Flory received the Nobel Prize in Chemistry in 1974, ostensibly apply only to ideal, dilute solutions. But there is reason to believe (e.g., neutron diffraction studies) that excluded volume effects may cancel out, so that, under certain conditions, chain dimensions in amorphous polymers have approximately the ideal, calculated size [4] When separate chains interact cooperatively, as in forming crystalline regions in solid thermoplastics, a different mathematical approach must be used. -->

<!-- Stiffer polymers such as helical polypeptides, Kevlar, and double-stranded DNA can be treated by the worm-like chain model. -->

<!-- Even copolymers with monomers of unequal length will distribute in random coils if the subunits lack any specific interactions. The parts of branched polymers may also assume random coils. -->

<!-- Below their melting temperatures, most thermoplastic polymers (polyethylene, nylon, etc.) have amorphous regions in which the chains approximate random coils, alternating with regions that are crystalline. The amorphous regions contribute elasticity and the crystalline regions contribute strength and rigidity. -->

More complex polymers such as proteins, with various interacting chemical groups attached to their backbones, self-assemble into well-defined structures. But segments of proteins, and polypeptides that lack secondary structure, are often assumed to exhibit a random-coil conformation in which the only fixed relationship is the joining of adjacent amino acid residues by a peptide bond. This is not actually the case, since the ensemble will be energy weighted due to interactions between amino acid side-chains, with lower-energy conformations being present more frequently. In addition, even arbitrary sequences of amino acids tend to exhibit some hydrogen bonding and secondary structure. For this reason, the term "statistical coil" is occasionally preferred. The conformational entropy of the random-coil stabilizes the unfolded protein state and represents main free energy contribution that opposes to protein folding.

Spectroscopy
A random-coil conformation can be detected using spectroscopic techniques. The arrangement of the planar amide bonds results in a distinctive signal in circular dichroism. The chemical shift of amino acids in a random-coil conformation is well known in nuclear magnetic resonance (NMR). Deviations from these signatures often indicates the presence of some secondary structure, rather than complete random coil. Furthermore, there are signals in multidimensional NMR experiments that indicate that stable, non-local amino acid interactions are absent for polypeptides in a random-coil conformation. Likewise, in the images produced by crystallography experiments, segments of random coil result simply in a reduction in "electron density" or contrast. A randomly coiled state for any polypeptide chain can be attained by denaturing the system. However, there is evidence that proteins are never truly random coils, even when denatured (Shortle & Ackerman).



Flory, P.J. (1953) Principles of Polymer Chemistry, Cornell Univ. Press, ISBN 0-8014-0134-8
Flory, P.J. (1969) Statistical Mechanics of Chain Molecules, Wiley, ISBN 0-470-26495-0; reissued 1989, ISBN 1-56990-019-1
"Conformations, Solutions, and Molecular Weight" from "Polymer Science & Technology" courtesy of Prentice Hall Professional publications [1]

############################################################
############################################################
############################################################
############################################################
############################################################

## PFam


Pfam is a database of protein families that includes their annotations and multiple sequence alignments generated using bioinformatics tools known as hidden Markov models (HMM; 1, 2, 3_. The most recent version, Pfam 34.0, was released in March 2021 and contains 19,179 families (4).


Uses
The general purpose of the Pfam database is to provide a complete and accurate classification of protein families and domains.[5] Originally, the rationale behind creating the database was to have a semi-automated method of curating information on known protein families to improve the efficiency of annotating genomes.[6] The Pfam classification of protein families has been widely adopted by biologists because of its wide coverage of proteins and sensible naming conventions.[7]

It is used by experimental biologists researching specific proteins, by structural biologists to identify new targets for structure determination, by computational biologists to organise sequences and by evolutionary biologists tracing the origins of proteins.[8] Early genome projects, such as human and fly used Pfam extensively for functional annotation of genomic data.[9][10][11]

The Pfam website allows users to submit protein or DNA sequences to search for matches to families in the database. If DNA is submitted, a six-frame translation is performed, then each frame is searched.[12] Rather than performing a typical BLAST search, Pfam uses profile hidden Markov models, which give greater weight to matches at conserved sites, allowing better remote homology detection, making them more suitable for annotating genomes of organisms with no well-annotated close relatives.[13]

Pfam has also been used in the creation of other resources such as iPfam, which catalogs domain-domain interactions within and between proteins, based on information in structure databases and mapping of Pfam domains onto these structures.[14]

Features
For each family in Pfam one can:
  
  View a description of the family
Look at multiple alignments
View protein domain architectures
Examine species distribution
Follow links to other databases
View known protein structures
Entries can be of several types: family, domain, repeat or motif. Family is the default class, which simply indicates that members are related. Domains are defined as an autonomous structural unit or reusable sequence unit that can be found in multiple protein contexts. Repeats are not usually stable in isolation, but rather are usually required to form tandem repeats in order to form a domain or extended structure. Motifs are usually shorter sequence units found outside of globular domains.[9]

The descriptions of Pfam families are managed by the general public using Wikipedia (see History).

As of release 29.0, 76.1% of protein sequences in UniprotKB matched to at least one Pfam domain.[15]

Creation of new entries
New families come from a range of sources, primarily the PDB and analysis of complete proteomes to find genes with no Pfam hit.[16]

For each family, a representative subset of sequences are aligned into a high-quality seed alignment. Sequences for the seed alignment are taken primarily from pfamseq (a non-redundant database of reference proteomes) with some supplementation from UniprotKB.[15] This seed alignment is then used to build a profile hidden Markov model using HMMER. This HMM is then searched against sequence databases, and all hits that reach a curated gathering threshold are classified as members of the protein family. The resulting collection of members is then aligned to the profile HMM to generate a full alignment.

For each family, a manually curated gathering threshold is assigned that maximises the number of true matches to the family while excluding any false positive matches. False positives are estimated by observing overlaps between Pfam family hits that are not from the same clan. This threshold is used to assess whether a match to a family HMM should be included in the protein family. Upon each update of Pfam, gathering thresholds are reassessed to prevent overlaps between new and existing families.[16]

Domains of unknown function
Domains of unknown function (DUFs) represent a growing fraction of the Pfam database. The families are so named because they have been found to be conserved across species, but perform an unknown role. Each newly added DUF is named in order of addition. Names of these entries are updated as their functions are identified. Normally when the function of at least one protein belonging to a DUF has been determined, the function of the entire DUF is updated and the family is renamed. Some named families are still domains of unknown function, that are named after a representative protein, e.g. YbbR. Numbers of DUFs are expected to continue increasing as conserved sequences of unknown function continue to be identified in sequence data. It is expected that DUFs will eventually outnumber families of known function.[16]

Clans
Over time both sequence and residue coverage have increased, and as families have grown, more evolutionary relationships have been discovered, allowing the grouping of families into clans.[8] Clans were first introduced to the Pfam database in 2005. They are groupings of related families that share a single evolutionary origin, as confirmed by structural, functional, sequence and HMM comparisons.[5] As of release 29.0, approximately one third of protein families belonged to a clan.[15] This portion has grown to around three-fourths by 2019 (version 32.0).[17]

To identify possible clan relationships, Pfam curators use the Simple Comparison Of Outputs Program (SCOOP) as well as information from the ECOD database.[17] ECOD is a semi-automated hierarchical database of protein families with known structures, with families that map readily to Pfam entries and homology levels that usually map to Pfam clans.[18]

History
Pfam was founded in 1995 by Erik Sonhammer, Sean Eddy and Richard Durbin as a collection of commonly occurring protein domains that could be used to annotate the protein coding genes of multicellular animals.[6] One of its major aims at inception was to aid in the annotation of the C. elegans genome.[6] The project was partly driven by the assertion in ‘One thousand families for the molecular biologist’ by Cyrus Chothia that there were around 1500 different families of proteins and that the majority of proteins fell into just 1000 of these.[5][19] Counter to this assertion, the Pfam database currently contains 16,306 entries corresponding to unique protein domains and families. However, many of these families contain structural and functional similarities indicating a shared evolutionary origin (see Clans).[5]

A major point of difference between Pfam and other databases at the time of its inception was the use of two alignment types for entries: a smaller, manually checked seed alignment, as well as a full alignment built by aligning sequences to a profile hidden Markov model built from the seed alignment. This smaller seed alignment was easier to update as new releases of sequence databases came out, and thus represented a promising solution to the dilemma of how to keep the database up to date as genome sequencing became more efficient and more data needed to be processed over time. A further improvement to the speed at which the database could be updated came in version 24.0, with the introduction of HMMER3, which is ~100 times faster than HMMER2 and more sensitive.[8]

Because the entries in Pfam-A do not cover all known proteins, an automatically generated supplement was provided called Pfam-B. Pfam-B contained a large number of small families derived from clusters produced by an algorithm called ADDA.[20] Although of lower quality, Pfam-B families could be useful when no Pfam-A families were found. Pfam-B was discontinued as of release 28.0,[21] then reintroduced in release 33.1 using a new clustering algorithm, MMSeqs2.[22]

Pfam was originally hosted on three mirror sites around the world to preserve redundancy. However between 2012 and 2014, the Pfam resource was moved to EMBL-EBI, which allowed for hosting of the website from one domain (xfam.org), using duplicate independent data centres. This allowed for better centralisation of updates, and grouping with other Xfam projects such as Rfam, TreeFam, iPfam and others, whilst retaining critical resilience provided by hosting from multiple centres.[23]

Pfam has undergone a substantial reorganisation over the last two years to further reduce manual effort involved in curation and allow for more frequent updates.[15]

Community curation
Curation of such a large database presented issues in terms of keeping up with the volume of new families and updated information that needed to be added. To speed up releases of the database, the developers started a number of initiatives to allow greater community involvement in managing the database.

A critical step in improving the pace of updating and improving entries was to open up the functional annotation of Pfam domains to the Wikipedia community in release 26.0.[16] For entries that already had a Wikipedia entry, this was linked into the Pfam page, and for those that did not, the community were invited to create one and inform the curators, in order for it to be linked in. It is anticipated that while community involvement will greatly improve the level of annotation of these families, some will remain insufficiently notable for inclusion in Wikipedia, in which case they will retain their original Pfam description. Some Wikipedia articles cover multiple families, such as the Zinc finger article. An automated procedure for generating articles based on InterPro and Pfam data has also been implemented, which populates a page with information and links to databases as well as available images, then once an article has been reviewed by a curator it is moved from the Sandbox to Wikipedia proper. In order to guard against vandalism of articles, each Wikipedia revision is reviewed by curators before it is displayed on the Pfam website. Almost all cases of vandalism have been corrected by the community before they reach curators, however.[16]

Pfam is run by an international consortium of three groups. In the earlier releases of Pfam, family entries could only be modified at the Cambridge, UK site, limiting the ability of consortium members to contribute to site curation. In release 26.0, developers moved to a new system that allowed registered users anywhere in the world to add or modify Pfam families.[16]



References
Finn RD, Tate J, Mistry J, Coggill PC, Sammut SJ, Hotz HR, Ceric G, Forslund K, Eddy SR, Sonnhammer EL, Bateman A (2008). "The Pfam protein families database". Nucleic Acids Res. 36 (Database issue): D281–8. doi:10.1093/nar/gkm960. PMC 2238907. PMID 18039703.
Finn, R. D.; Mistry, J.; Schuster-Böckler, B.; Griffiths-Jones, S.; Hollich, V.; Lassmann, T.; Moxon, S.; Marshall, M.; Khanna, A.; Durbin, R.; Eddy, S. R.; Sonnhammer, E. L.; Bateman, A. (January 2006). "Pfam: clans, web tools and services" (Free full text). Nucleic Acids Research. 34 (Database issue): D247–D251. doi:10.1093/nar/gkj149. ISSN 0305-1048. PMC 1347511. PMID 16381856.
Bateman, A.; Coin, L.; Durbin, R.; Finn, R. D.; Hollich, V.; Griffiths-Jones, S.; Khanna, A.; Marshall, M.; Moxon, S.; Sonnhammer, E. L.; Studholme, D. J.; Yeats, C.; Eddy, S. R. (2004). "The Pfam protein families database". Nucleic Acids Research. 32 (Database issue): 138D–1141. doi:10.1093/nar/gkh121. ISSN 0305-1048. PMC 308855. PMID 14681378. open access
"Pfam 34.0 is released". Xfam Blog. Retrieved 2 July 2021.
Sammut, Stephen; Finn, Robert D.; Bateman, Alex (2008). "Pfam 10 years on: 10 000 families and still growing". Briefings in Bioinformatics. 9 (3): 210–219. doi:10.1093/bib/bbn010. PMID 18344544.
Sonnhammer, Erik L.L.; Eddy, Sean R.; Durbin, Richard (1997). "Pfam: A Comprehensive Database of Protein Domain Families Based on Seed Alignments". Proteins. 28 (3): 405–420. doi:10.1002/(sici)1097-0134(199707)28:3<405::aid-prot10>3.0.co;2-l. PMID 9223186.
Xu, Qifang; Dunbrack, Roland L. (2012). "Assignment of protein sequences to existing domain and family classification systems: Pfam and the PDB". Bioinformatics. 28 (21): 2763–2772. doi:10.1093/bioinformatics/bts533. PMC 3476341. PMID 22942020.
Finn, R. D.; Mistry, J.; Tate, J.; Coggill, P.; Heger, A.; Pollington, J. E.; Gavin, O. L.; Gunasekaran, P.; Ceric, G.; Forslund, K.; Holm, L.; Sonnhammer, E. L. L.; Eddy, S. R.; Bateman, A. (2009). "The Pfam protein families database". Nucleic Acids Research. 38 (Database): D211–D222. doi:10.1093/nar/gkp985. ISSN 0305-1048. PMC 2808889. PMID 19920124.
Bateman A, Birney E, Cerruti L, Durbin R, Etwiller L, Eddy SR, Griffiths-Jones S, Howe KL, Marshall M, Sonnhammer EL (2002). "The Pfam protein families database". Nucleic Acids Res. 30 (1): 276–80. doi:10.1093/nar/30.1.276. PMC 99071. PMID 11752314.
Adams MD, Celniker SE, Holt RA, Evans CA, Gocayne JD, et al. (2000). "The genome sequence of Drosophila melanogaster". Science. 287 (5461): 2185–95. Bibcode:2000Sci...287.2185.. CiteSeerX 10.1.1.549.8639. doi:10.1126/science.287.5461.2185. PMID 10731132.
Lander, Eric S.; Linton, Lauren M.; Birren, Bruce; Nusbaum, Chad; Zody, Michael C.; et al. (2001). "Initial sequencing and analysis of the human genome". Nature. 409 (6822): 860–921. doi:10.1038/35057062. ISSN 0028-0836. PMID 11237011.
Finn, Robert D.; Bateman, Alex; Clements, Jody; Coggill, Penelope; Eberhardt, Ruth Y.; Eddy, Sean R.; Heger, Andreas; Hetherington, Kirstie; Holm, Liisa; Mistry, Jaina; Sonnhammer, Erik L. L.; Tate, John; Punta, Marco (2014). "Pfam: the protein families database". Nucleic Acids Research. 42 (D1): D222–D230. doi:10.1093/nar/gkt1223. ISSN 0305-1048. PMC 3965110. PMID 24288371.
Sonnhammer EL, Eddy SR, Birney E, Bateman A, Durbin R (1998). "Pfam: multiple sequence alignments and HMM-profiles of protein domains". Nucleic Acids Res. 26 (1): 320–2. doi:10.1093/nar/26.1.320. PMC 147209. PMID 9399864.
Finn, R. D.; Marshall, M.; Bateman, A. (2004). "iPfam: visualization of protein-protein interactions in PDB at domain and amino acid resolutions". Bioinformatics. 21 (3): 410–412. doi:10.1093/bioinformatics/bti011. ISSN 1367-4803. PMID 15353450.
Finn, Robert D.; Coggill, Penelope; Eberhardt, Ruth Y.; Eddy, Sean R.; Mistry, Jaina; Mitchell, Alex L.; Potter, Simon C.; Punta, Marco; Qureshi, Matloob; Sangrador-Vegas, Amaia; Salazar, Gustavo A.; Tate, John; Bateman, Alex (2016). "The Pfam protein families database: towards a more sustainable future". Nucleic Acids Research. 44 (D1): D279–D285. doi:10.1093/nar/gkv1344. ISSN 0305-1048. PMC 4702930. PMID 26673716.
Punta, M.; Coggill, P. C.; Eberhardt, R. Y.; Mistry, J.; Tate, J.; Boursnell, C.; Pang, N.; Forslund, K.; Ceric, G.; Clements, J.; Heger, A.; Holm, L.; Sonnhammer, E. L. L.; Eddy, S. R.; Bateman, A.; Finn, R. D. (2011). "The Pfam protein families database". Nucleic Acids Research. 40 (D1): D290–D301. doi:10.1093/nar/gkr1065. ISSN 0305-1048. PMC 3245129. PMID 22127870.
El-Gebali, Sara; Mistry, Jaina; Bateman, Alex; Eddy, Sean R; Luciani, Aurélien; Potter, Simon C; Qureshi, Matloob; Richardson, Lorna J; Salazar, Gustavo A; Smart, Alfredo; Sonnhammer, Erik L L; Hirsh, Layla; Paladin, Lisanna; Piovesan, Damiano; Tosatto, Silvio C E; Finn, Robert D (8 January 2019). "The Pfam protein families database in 2019". Nucleic Acids Research. 47 (D1): D427–D432. doi:10.1093/nar/gky995. PMC 6324024. PMID 30357350.
"Evolutionary Classification of Protein Domains". prodata.swmed.edu. Retrieved 18 May 2019.
Chothia, Cyrus (1992). "One thousand families for the molecular biologist". Nature. 357 (6379): 543–544. Bibcode:1992Natur.357..543C. doi:10.1038/357543a0. ISSN 0028-0836. PMID 1608464. S2CID 4355476.
Heger, A.; Wilton, C. A.; Sivakumar, A.; Holm, L. (January 2005). "ADDA: a domain database with global coverage of the protein universe" (Free full text). Nucleic Acids Research. 33 (Database issue): D188–D191. doi:10.1093/nar/gki096. ISSN 0305-1048. PMC 540050. PMID 15608174.
"Pfam 28.0 release notes". Retrieved 30 June 2015.
"A new Pfam-B is released". Xfam Blog. 30 June 2020.
"Moving to xfam.org". Retrieved 25 November 2016.
Dunbrack, Roland. "PDBfam". Fox Chase Cancer Center. Retrieved 9 March 2013.
Xu, Qifang; Dunbrack, Roland (2012). "Assignment of protein sequences to existing domain and family classification systems: Pfam and the PDB". Bioinformatics. 28 (21): 2763–72. doi:10.1093/bioinformatics/bts533. PMC 3476341. PMID 22942020.


############################################################
############################################################
############################################################
############################################################
############################################################
############################################################
############################################################

## Haplotype

Haplotype
Adapted from Wikipedia, https://en.wikipedia.org/wiki/Haplotype


DNA molecule 1 differs from DNA molecule 2 at a single base-pair location (a C/A polymorphism).
A haplotype (haploid genotype) is a group of alleles in an organism that are inherited together from a single parent.[1][2]

Many organisms contain genetic material (DNA) which is inherited from two parents. Normally these organisms have their DNA organized in two sets of pairwise similar chromosomes. The offspring gets one chromosome in each pair from each parent. A set of pairs of chromosomes is called diploid and a set of only one half of each pair is called haploid. The haploid genotype (haplotype) is a genotype that considers the singular chromosomes rather than the pairs of chromosomes. It can be all the chromosomes from one of the parents or a minor part of a chromosome, for example a sequence of 9000 base pairs.

However, there are other uses of this term. First, it is used to mean a collection of specific alleles (that is, specific DNA sequences) in a cluster of tightly linked genes on a chromosome that are likely to be inherited together—that is, they are likely to be conserved as a sequence that survives the descent of many generations of reproduction.[3][4] A second use is to mean a set of linked single-nucleotide polymorphism (SNP) alleles that tend to always occur together (i.e., that are associated statistically). It is thought that identifying these statistical associations and a few alleles of a specific haplotype sequence can facilitate identifying all other such polymorphic sites that are nearby on the chromosome. Such information is critical for investigating the genetics of common diseases; which in fact have been investigated in humans by the International HapMap Project.[5][6] Thirdly, many human genetic testing companies use the term in a third way: to refer to an individual collection of specific mutations within a given genetic segment; (see short tandem repeat mutation).

The term 'haplogroup' refers to the SNP/unique-event polymorphism (UEP) mutations that represent the clade to which a collection of particular human haplotypes belong. (Clade here refers to a set of haplotypes sharing a common ancestor.)[7] A haplogroup is a group of similar haplotypes that share a common ancestor with a single-nucleotide polymorphism mutation.[8][9] Mitochondrial DNA passes along a maternal lineage that can date back thousands of years.[8]



Haplotype resolution
An organism's genotype may not define its haplotype uniquely. For example, consider a diploid organism and two bi-allelic loci (such as SNPs) on the same chromosome. Assume the first locus has alleles A or T and the second locus G or C. Both loci, then, have three possible genotypes: (AA, AT, and TT) and (GG, GC, and CC), respectively. For a given individual, there are nine possible configurations (haplotypes) at these two loci (shown in the Punnett square below). For individuals who are homozygous at one or both loci, the haplotypes are unambiguous - meaning that there is not any differentiation of haplotype T1T2 vs haplotype T2T1; where T1 and T2 are labeled to show that they are the same locus, but labeled as such to show it doesn't matter which order you consider them in, the end result is two T loci. For individuals heterozygous at both loci, the gametic phase is ambiguous - in these cases, you don't know which haplotype you have, e.g., TA vs AT.

AA	AT	TT
GG	AG AG	AG TG	TG TG
GC	AG AC	AG TC
or
AC TG	TG TC
CC	AC AC	AC TC	TC TC
The only unequivocal method of resolving phase ambiguity is by sequencing. However, it is possible to estimate the probability of a particular haplotype when phase is ambiguous using a sample of individuals.

Given the genotypes for a number of individuals, the haplotypes can be inferred by haplotype resolution or haplotype phasing techniques. These methods work by applying the observation that certain haplotypes are common in certain genomic regions. Therefore, given a set of possible haplotype resolutions, these methods choose those that use fewer different haplotypes overall. The specifics of these methods vary - some are based on combinatorial approaches (e.g., parsimony), whereas others use likelihood functions based on different models and assumptions such as the Hardy–Weinberg principle, the coalescent theory model, or perfect phylogeny. The parameters in these models are then estimated using algorithms such as the expectation-maximization algorithm (EM), Markov chain Monte Carlo (MCMC), or hidden Markov models (HMM).

Microfluidic whole genome haplotyping is a technique for the physical separation of individual chromosomes from a metaphase cell followed by direct resolution of the haplotype for each allele.

Y-DNA haplotypes from genealogical DNA tests
Main article: Genealogical DNA test
Unlike other chromosomes, Y chromosomes generally do not come in pairs. Every human male (excepting those with XYY syndrome) has only one copy of that chromosome. This means that there is not any chance variation of which copy is inherited, and also (for most of the chromosome) not any shuffling between copies by recombination; so, unlike autosomal haplotypes, there is effectively not any randomisation of the Y-chromosome haplotype between generations. A human male should largely share the same Y chromosome as his father, give or take a few mutations; thus Y chromosomes tend to pass largely intact from father to son, with a small but accumulating number of mutations that can serve to differentiate male lineages. In particular, the Y-DNA represented as the numbered results of a Y-DNA genealogical DNA test should match, except for mutations.

UEP results (SNP results)
Unique-event polymorphisms (UEPs) such as SNPs represent haplogroups. STRs represent haplotypes. The results that comprise the full Y-DNA haplotype from the Y chromosome DNA test can be divided into two parts: the results for UEPs, sometimes loosely called the SNP results as most UEPs are single-nucleotide polymorphisms, and the results for microsatellite short tandem repeat sequences (Y-STRs).

The UEP results represent the inheritance of events it is believed can be assumed to have happened only once in all human history. These can be used to identify the individual's Y-DNA haplogroup, his place in the "family tree" of the whole of humanity. Different Y-DNA haplogroups identify genetic populations that are often distinctly associated with particular geographic regions; their appearance in more recent populations located in different regions represents the migrations tens of thousands of years ago of the direct patrilineal ancestors of current individuals.

Y-STR haplotypes
Genetic results also include the Y-STR haplotype, the set of results from the Y-STR markers tested.

Unlike the UEPs, the Y-STRs mutate much more easily, which allows them to be used to distinguish recent genealogy. But it also means that, rather than the population of descendants of a genetic event all sharing the same result, the Y-STR haplotypes are likely to have spread apart, to form a cluster of more or less similar results. Typically, this cluster will have a definite most probable center, the modal haplotype (presumably similar to the haplotype of the original founding event), and also a haplotype diversity — the degree to which it has become spread out. The further in the past the defining event occurred, and the more that subsequent population growth occurred early, the greater the haplotype diversity will be for a particular number of descendants. However, if the haplotype diversity is smaller for a particular number of descendants, this may indicate a more recent common ancestor, or a recent population expansion.

It is important to note that, unlike for UEPs, two individuals with a similar Y-STR haplotype may not necessarily share a similar ancestry. Y-STR events are not unique. Instead, the clusters of Y-STR haplotype results inherited from different events and different histories tend to overlap.

In most cases, it is a long time since the haplogroups' defining events, so typically the cluster of Y-STR haplotype results associated with descendants of that event has become rather broad. These results will tend to significantly overlap the (similarly broad) clusters of Y-STR haplotypes associated with other haplogroups. This makes it impossible for researchers to predict with absolute certainty to which Y-DNA haplogroup a Y-STR haplotype would point. If the UEPs are not tested, the Y-STRs may be used only to predict probabilities for haplogroup ancestry, but not certainties.

A similar scenario exists in trying to evaluate whether shared surnames indicate shared genetic ancestry. A cluster of similar Y-STR haplotypes may indicate a shared common ancestor, with an identifiable modal haplotype, but only if the cluster is sufficiently distinct from what may have happened by chance from different individuals who historically adopted the same name independently. Many names were adopted from common occupations, for instance, or were associated with habitation of particular sites. More extensive haplotype typing is needed to establish genetic genealogy. Commercial DNA-testing companies now offer their customers testing of more numerous sets of markers to improve definition of their genetic ancestry. The number of sets of markers tested has increased from 12 during the early years to 111 more recently.

Establishing plausible relatedness between different surnames data-mined from a database is significantly more difficult. The researcher must establish that the very nearest member of the population in question, chosen purposely from the population for that reason, would be unlikely to match by accident. This is more than establishing that a randomly selected member of the population is unlikely to have such a close match by accident. Because of the difficulty, establishing relatedness between different surnames as in such a scenario is likely to be impossible, except in special cases where there is specific information to drastically limit the size of the population of candidates under consideration.

Diversity
Haplotype diversity is a measure of the uniqueness of a particular haplotype in a given population. The haplotype diversity (H) is computed as:[10]
{\displaystyle H={\frac {N}{N-1}}(1-\sum _{i}x_{i}^{2})}H={\frac  {N}{N-1}}(1-\sum _{{i}}x_{i}^{2})
where {\displaystyle x_{i}}x_{i} is the (relative) haplotype frequency of each haplotype in the sample and {\displaystyle N}N is the sample size. Haplotype diversity is given for each sample.



References
 By C. Barry Cox, Peter D. Moore, Richard Ladle. Wiley-Blackwell, 2016. ISBN 978-1-118-96858-1 p106. Biogeography: An Ecological and Evolutionary Approach
 Editorial Board, V&S Publishers, 2012, ISBN 9381588643 p137.Concise Dictionary of Science
 BiologyPages/H/Haplotypes.html Kimball's Biology Pages (Creative Commons Attribution 3.0)
"haplotype / haplotypes | Learn Science at Scitable". www.nature.com.
The International HapMap Consortium (2003). "The International HapMap Project" (PDF). Nature. 426 (6968): 789–796. Bibcode:2003Natur.426..789G. doi:10.1038/nature02168. hdl:2027.42/62838. PMID 14685227. S2CID 4387110.
The International HapMap Consortium (2005). "A haplotype map of the human genome". Nature. 437 (7063): 1299–1320. Bibcode:2005Natur.437.1299T. doi:10.1038/nature04226. PMC 1880871. PMID 16255080.
"Facts & Genes. Volume 7, Issue 3". Archived from the original on May 9, 2008.
Arora, Devender; Singh, Ajeet; Sharma, Vikrant; Bhaduria, Harvendra Singh; Patel, Ram Bahadur (2015). "Hgs Db: Haplogroups Database to understand migration and molecular risk assessment". Bioinformation. 11 (6): 272–5. doi:10.6026/97320630011272. PMC 4512000. PMID 26229286.
International Society of Genetic Genealogy 2015 Genetics Glossary
Masatoshi Nei and Fumio Tajima, "DNA polymorphism detectable by restriction endonucleases", Genetics 97:145 (1981)
Becker T.; Knapp M. (2004). "Maximum-likelihood estimation of haplotype frequencies in nuclear families". Genetic Epidemiology. 27 (1): 21–32. doi:10.1002/gepi.10323. PMID 15185400.
Li S.S.; Khalid N.; Carlson C.; Zhao L.P. (2003). "Estimating haplotype frequencies and standard errors for multiple single nucleotide polymorphisms". Biostatistics. 4 (4): 513–522. doi:10.1093/biostatistics/4.4.513. PMID 14557108.
Roach J.C.; Glusman G.; Hubley R.; Montsaroff S.Z.; Holloway A.K.; Mauldin D.E.; Srivastava D.; Garg V.; Pollard K.S.; Galas D.J.; Hood L.; Smit A.F.A. (2011). "Chromosomal Haplotypes by Genetic Phasing of Human Families". American Journal of Human Genetics. 89 (3): 382–397. doi:10.1016/j.ajhg.2011.07.023. PMC 3169815. PMID 21855840.
Barrett J.C.; Fry B.; Maller J.; Daly M.J. (2005). "Haploview: analysis and visualization of LD and haplotype maps". Bioinformatics. 21 (2): 263–265. doi:10.1093/bioinformatics/bth457. PMID 15297300.
Delaneau O, Zagury JF, Marchini J (2013). "Improved whole chromosome phasing for disease and population genetic studies". Nature Methods. 10 (1): 5–6. doi:10.1038/nmeth.2307. PMID 23269371. S2CID 205421216.
Purcell S.; Daly M. J.; Sham P. C. (2007). "WHAP: haplotype-based association analysis". Bioinformatics. 23 (2): 255–256. doi:10.1093/bioinformatics/btl580. PMID 17118959.

############################################################
############################################################
############################################################
############################################################
############################################################
############################################################
############################################################
############################################################


## Uniprot

UniProt
From Wikipedia, the free encyclopedia
(Redirected from TrEMBL)
Jump to navigationJump to search
UniProt
UPlogo1.png
Content
Description	UniProt is the Universal Protein resource, a central repository of protein data created by combining the Swiss-Prot, TrEMBL and PIR-PSD databases.
Data types
captured	Protein annotation
Organisms	All
Contact
Research center	EMBL-EBI, UK; SIB, Switzerland; PIR, US.
Primary citation	UniProt Consortium[1]
Access
Data format	Custom flat file, FASTA, GFF, RDF, XML.
Website	www.uniprot.org
www.uniprot.org/news/
  Download URL	www.uniprot.org/downloads & for downloading complete data sets ftp.uniprot.org
Web service URL	Yes – JAVA API see info here & REST see info here
Tools
Web	Advanced search, BLAST, ClustalO, bulk retrieval/download, ID mapping
Miscellaneous
License	Creative Commons Attribution-NoDerivs
Versioning	Yes
Data release
frequency	8 weeks
Curation policy	Yes – manual and automatic. Rules for automatic annotation generated by database curators and computational algorithms.
Bookmarkable
entities	Yes – both individual protein entries and searches
UniProt is a freely accessible database of protein sequence and functional information, many entries being derived from genome sequencing projects. It contains a large amount of information about the biological function of proteins derived from the research literature. It is maintained by the UniProt consortium, which consists of several European bioinformatics organisations and a foundation from Washington, DC, United States.




The UniProt consortium
The UniProt consortium comprises the European Bioinformatics Institute (EBI), the Swiss Institute of Bioinformatics (SIB), and the Protein Information Resource (PIR). EBI, located at the Wellcome Trust Genome Campus in Hinxton, UK, hosts a large resource of bioinformatics databases and services. SIB, located in Geneva, Switzerland, maintains the ExPASy (Expert Protein Analysis System) servers that are a central resource for proteomics tools and databases. PIR, hosted by the National Biomedical Research Foundation (NBRF) at the Georgetown University Medical Center in Washington, DC, US, is heir to the oldest protein sequence database, Margaret Dayhoff's Atlas of Protein Sequence and Structure, first published in 1965.[2] In 2002, EBI, SIB, and PIR joined forces as the UniProt consortium.[3]

The roots of UniProt databases
Each consortium member is heavily involved in protein database maintenance and annotation. Until recently, EBI and SIB together produced the Swiss-Prot and TrEMBL databases, while PIR produced the Protein Sequence Database (PIR-PSD).[4][5][6] These databases coexisted with differing protein sequence coverage and annotation priorities.

Swiss-Prot was created in 1986 by Amos Bairoch during his PhD and developed by the Swiss Institute of Bioinformatics and subsequently developed by Rolf Apweiler at the European Bioinformatics Institute.[7][8][9] Swiss-Prot aimed to provide reliable protein sequences associated with a high level of annotation (such as the description of the function of a protein, its domain structure, post-translational modifications, variants, etc.), a minimal level of redundancy and high level of integration with other databases. Recognizing that sequence data were being generated at a pace exceeding Swiss-Prot's ability to keep up, TrEMBL (Translated EMBL Nucleotide Sequence Data Library) was created to provide automated annotations for those proteins not in Swiss-Prot. Meanwhile, PIR maintained the PIR-PSD and related databases, including iProClass, a database of protein sequences and curated families.

The consortium members pooled their overlapping resources and expertise, and launched UniProt in December 2003.[10]

Organization of UniProt databases
UniProt provides four core databases: UniProtKB (with sub-parts Swiss-Prot and TrEMBL), UniParc, UniRef.

UniProtKB
UniProt Knowledgebase (UniProtKB) is a protein database partially curated by experts, consisting of two sections: UniProtKB/Swiss-Prot (containing reviewed, manually annotated entries) and UniProtKB/TrEMBL (containing unreviewed, automatically annotated entries).[11] As of 19 March 2014, release "2014_03" of UniProtKB/Swiss-Prot contains 542,782 sequence entries (comprising 193,019,802 amino acids abstracted from 226,896 references) and release "2014_03" of UniProtKB/TrEMBL contains 54,247,468 sequence entries (comprising 17,207,833,179 amino acids).[12][13]

UniProtKB/Swiss-Prot
UniProtKB/Swiss-Prot is a manually annotated, non-redundant protein sequence database. It combines information extracted from scientific literature and biocurator-evaluated computational analysis. The aim of UniProtKB/Swiss-Prot is to provide all known relevant information about a particular protein. Annotation is regularly reviewed to keep up with current scientific findings. The manual annotation of an entry involves detailed analysis of the protein sequence and of the scientific literature.[14]

Sequences from the same gene and the same species are merged into the same database entry. Differences between sequences are identified, and their cause documented (for example alternative splicing, natural variation, incorrect initiation sites, incorrect exon boundaries, frameshifts, unidentified conflicts). A range of sequence analysis tools is used in the annotation of UniProtKB/Swiss-Prot entries. Computer-predictions are manually evaluated, and relevant results selected for inclusion in the entry. These predictions include post-translational modifications, transmembrane domains and topology, signal peptides, domain identification, and protein family classification.[14][15]

Relevant publications are identified by searching databases such as PubMed. The full text of each paper is read, and information is extracted and added to the entry. Annotation arising from the scientific literature includes, but is not limited to:[10][14][15]

Protein and gene names
Function
Enzyme-specific information such as catalytic activity, cofactors and catalytic residues
Subcellular location
Protein-protein interactions
Pattern of expression
Locations and roles of significant domains and sites
Ion-, substrate- and cofactor-binding sites
Protein variant forms produced by natural genetic variation, RNA editing, alternative splicing, proteolytic processing, and post-translational modification
Annotated entries undergo quality assurance before inclusion into UniProtKB/Swiss-Prot. When new data becomes available, entries are updated.

UniProtKB/TrEMBL
UniProtKB/TrEMBL contains high-quality computationally analyzed records, which are enriched with automatic annotation. It was introduced in response to increased dataflow resulting from genome projects, as the time- and labour-consuming manual annotation process of UniProtKB/Swiss-Prot could not be broadened to include all available protein sequences.[10] The translations of annotated coding sequences in the EMBL-Bank/GenBank/DDBJ nucleotide sequence database are automatically processed and entered in UniProtKB/TrEMBL. UniProtKB/TrEMBL also contains sequences from PDB, and from gene prediction, including Ensembl, RefSeq and CCDS.[16] Since 22 July 2021 it also includes predicted with AlphaFold tertiary structures.[17]

UniParc
UniProt Archive (UniParc) is a comprehensive and non-redundant database, which contains all the protein sequences from the main, publicly available protein sequence databases.[18] Proteins may exist in several different source databases, and in multiple copies in the same database. In order to avoid redundancy, UniParc stores each unique sequence only once. Identical sequences are merged, regardless of whether they are from the same or different species. Each sequence is given a stable and unique identifier (UPI), making it possible to identify the same protein from different source databases. UniParc contains only protein sequences, with no annotation. Database cross-references in UniParc entries allow further information about the protein to be retrieved from the source databases. When sequences in the source databases change, these changes are tracked by UniParc and history of all changes is archived.

Source databases
Currently UniParc contains protein sequences from the following publicly available databases:
  
  INSDC EMBL-Bank/DDBJ/GenBank nucleotide sequence databases
Ensembl
European Patent Office (EPO)
FlyBase: the primary repository of genetic and molecular data for the insect family Drosophilidae (FlyBase)
H-Invitational Database (H-Inv)
International Protein Index (IPI)
Japan Patent Office (JPO)
Protein Information Resource (PIR-PSD)
Protein Data Bank (PDB)
Protein Research Foundation (PRF)[19]
RefSeq
Saccharomyces Genome Database (SGD)
The Arabidopsis Information Resource (TAIR)
TROME[20]
US Patent Office (USPTO)
UniProtKB/Swiss-Prot, UniProtKB/Swiss-Prot protein isoforms, UniProtKB/TrEMBL
Vertebrate and Genome Annotation Database (VEGA)
WormBase
UniRef
The UniProt Reference Clusters (UniRef) consist of three databases of clustered sets of protein sequences from UniProtKB and selected UniParc records.[21] The UniRef100 database combines identical sequences and sequence fragments (from any organism) into a single UniRef entry. The sequence of a representative protein, the accession numbers of all the merged entries and links to the corresponding UniProtKB and UniParc records are displayed. UniRef100 sequences are clustered using the CD-HIT algorithm to build UniRef90 and UniRef50.[21][22] Each cluster is composed of sequences that have at least 90% or 50% sequence identity, respectively, to the longest sequence. Clustering sequences significantly reduces database size, enabling faster sequence searches.

UniRef is available from the UniProt FTP site.

Funding
UniProt is funded by grants from the National Human Genome Research Institute, the National Institutes of Health (NIH), the European Commission, the Swiss Federal Government through the Federal Office of Education and Science, NCI-caBIG, and the US Department of Defense.[11]

References
UniProt, Consortium. (January 2015). "UniProt: a hub for protein information". Nucleic Acids Research. 43 (Database issue): D204–12. doi:10.1093/nar/gku989. PMC 4384041. PMID 25348405.
Dayhoff, Margaret O. (1965). Atlas of protein sequence and structure. Silver Spring, Md: National Biomedical Research Foundation.
"2002 Release: NHGRI Funds Global Protein Database". National Human Genome Research Institute (NHGRI). Retrieved 14 April 2018.
O'Donovan, C.; Martin, M. J.; Gattiker, A.; Gasteiger, E.; Bairoch, A.; Apweiler, R. (2002). "High-quality protein knowledge resource: SWISS-PROT and TrEMBL". Briefings in Bioinformatics. 3 (3): 275–284. doi:10.1093/bib/3.3.275. PMID 12230036.
 Wu, C. H.; Yeh, L. S.; Huang, H.; Arminski, L.; Castro-Alvear, J.; Chen, Y.; Hu, Z.; Kourtesis, P.; Ledley, R. S.; Suzek, B. E.; Vinayaka, C. R.; Zhang, J.; Barker, W. C. (2003). "The Protein Information Resource". Nucleic Acids Research. 31 (1): 345–347. doi:10.1093/nar/gkg040. PMC 165487. PMID 12520019.
 Boeckmann, B.; Bairoch, A.; Apweiler, R.; Blatter, M. C.; Estreicher, A.; Gasteiger, E.; Martin, M. J.; Michoud, K.; O'Donovan, C.; Phan, I.; Pilbout, S.; Schneider, M. (2003). "The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003". Nucleic Acids Research. 31 (1): 365–370. doi:10.1093/nar/gkg095. PMC 165542. PMID 12520024.
Bairoch, A.; Apweiler, R. (1996). "The SWISS-PROT protein sequence data bank and its new supplement TREMBL". Nucleic Acids Research. 24 (1): 21–25. doi:10.1093/nar/24.1.21. PMC 145613. PMID 8594581.
Bairoch, A. (2000). "Serendipity in bioinformatics, the tribulations of a Swiss bioinformatician through exciting times!". Bioinformatics. 16 (1): 48–64. doi:10.1093/bioinformatics/16.1.48. PMID 10812477.
Séverine Altairac, "Naissance d’une banque de données: Interview du prof. Amos Bairoch". Protéines à la Une, August 2006. ISSN 1660-9824.
Apweiler, R.; Bairoch, A.; Wu, C. H. (2004). "Protein sequence databases". Current Opinion in Chemical Biology. 8 (1): 76–80. doi:10.1016/j.cbpa.2003.12.004. PMID 15036160.
Uniprot, C. (2009). "The Universal Protein Resource (UniProt) in 2010". Nucleic Acids Research. 38 (Database issue): D142–D148. doi:10.1093/nar/gkp846. PMC 2808944. PMID 19843607.
"UniProtKB/Swiss-Prot Release 2018_03 statistics". web.expasy.org. Retrieved 14 April 2018.
EMBL-EBI. "Current Release Statistics < Uniprot < EMBL-EBI". www.ebi.ac.uk. Retrieved 14 April 2018.
"How do we manually annotate a UniProtKB entry?". www.uniprot.org. Retrieved 14 April 2018.
Apweiler, R.; Bairoch, A.; Wu, C. H.; Barker, W. C.; Boeckmann, B.; Ferro, S.; Gasteiger, E.; Huang, H.; Lopez, R.; Magrane, M.; Martin, M. J.; Natale, D. A.; o’Donovan, C.; Redaschi, N.; Yeh, L. S. (2004). "UniProt: The Universal Protein knowledgebase". Nucleic Acids Research. 32 (90001): 115D–1119. doi:10.1093/nar/gkh131. PMC 308865. PMID 14681372.
"Where do the UniProtKB protein sequences come from?". www.uniprot.org. Retrieved 14 April 2018.
"Putting the power of AlphaFold into the world's hands". Deepmind. Retrieved 24 July 2021.
Leinonen, R.; Diez, F. G.; Binns, D.; Fleischmann, W.; Lopez, R.; Apweiler, R. (2004). "UniProt archive". Bioinformatics. 20 (17): 3236–3237. doi:10.1093/bioinformatics/bth191. PMID 15044231.
"Protein Research Foundation".
ftp://ftp.isrec.isb-sib.ch/pub/databases/trome
Suzek, B. E.; Huang, H.; McGarvey, P.; Mazumder, R.; Wu, C. H. (2007). "UniRef: Comprehensive and non-redundant UniProt reference clusters". Bioinformatics. 23 (10): 1282–1288. doi:10.1093/bioinformatics/btm098. PMID 17379688.
Li, W.; Jaroszewski, L.; Godzik, A. (2001). "Clustering of highly homologous sequences to reduce the size of large protein databases". Bioinformatics. 17 (3): 282–283. doi:10.1093/bioinformatics/17.3.282. PMID 11294794.



############################################################
############################################################
############################################################

## Native state (protein)


Native state
From Wikipedia, the free encyclopedia
Jump to navigationJump to search
For the sovereign entities of British India, see Native states.
For native states in metallurgy, see Native state (metallurgy).

This article does not cite any sources. Please help improve this article by adding citations to reliable sources. Unsourced material may be challenged and removed.
Find sources: "Native state" – news · newspapers · books · scholar · JSTOR (December 2009) (Learn how and when to remove this template message)
In biochemistry, the native state of a protein or nucleic acid is its properly folded and/or assembled form, which is operative and functional. The native state of a biomolecule may possess all four levels of biomolecular structure, with the secondary through quaternary structure being formed from weak interactions along the covalently-bonded backbone. This is in contrast to the denatured state, in which these weak interactions are disrupted, leading to the loss of these forms of structure and retaining only the biomolecule's primary structure.


Contents
1	Biochemistry
1.1	Proteins
1.2	Nucleic acids
2	External links
Biochemistry
Proteins
Main article: Protein structure
While all protein molecules begin as simple unbranched chains of amino acids, once completed they assume highly specific three-dimensional shapes. That ultimate shape, known as tertiary structure, is the folded shape that possesses a minimum of free energy. It is a protein's tertiary, folded structure that makes it capable of performing its biological function. In fact, shape changes in proteins are the primary cause of several neurodegenerative diseases, including those caused by prions and amyloid (i.e. mad cow disease, kuru, Creutzfeldt–Jakob disease).

Many enzymes and other non-structural proteins have more than one native state, and they operate or undergo regulation by transitioning between these states. However, "native state" is used almost exclusively in the singular, typically to distinguish properly folded proteins from denatured or unfolded ones. In other contexts, the folded shape of a protein is most often referred to as its native "conformation" or "structure."

Folded and unfolded proteins are often easily distinguished by virtue of their water solubilities, as many proteins become insoluble on denaturation. Proteins in the native state will have defined secondary structure, which can be detected spectroscopically, by circular dichroism and by nuclear magnetic resonance (NMR).

The native state of a protein can be distinguished from a molten globule, by among other things, distances measured by NMR. Amino acids widely separated in a protein's sequence may touch or lie very close to one another within a stably folded protein. In a molten globule, on the other hand, their time-averaged distances are liable to be greater.

Learning how native state proteins can be manufactured is important, as attempts to create proteins from scratch have resulted in molten globules and not true native state products. Therefore, an understanding of the native state is crucial in protein engineering.

Nucleic acids
Nucleic acids attain their native state through base pairing and, to a lesser extent, other interactions such as coaxial stacking. Biological DNA usually exists as long linear double helices bound to proteins in chromatin, and biological RNA such as tRNA often form complex native configurations approaching the complexity of folded proteins. Additionally, artificial nucleic acid structures used in DNA nanotechnology are designed to have specific native configurations in which multiple nucleic acid strands are assembled into a single complex. In some cases native state of biological DNA performs their functions without being controlled by any other regulatory units.
