Talking glossary
Locus

Wikipedia:


Medline:

Copied below
SCOP Random coil PFam  Haplotype native state uniprot 



https://medlineplus.gov/genetics/understanding/genomicresearch/encode/

https://www.genome.gov/genetics-glossary/BRCA1-BRCA2
https://www.genome.gov/genetics-glossary/Exon
https://www.genome.gov/genetics-glossary/Centimorgan
https://www.genome.gov/genetics-glossary/Copy-Number-Variation
https://www.genome.gov/genetics-glossary/Crossing-Over
https://www.genome.gov/genetics-glossary/Cytogenetics
https://www.genome.gov/genetics-glossary/Gene-Environment-Interaction
https://www.genome.gov/genetics-glossary/Genetic-Map
https://www.genome.gov/genetics-glossary/homologous-recombination
https://www.genome.gov/genetics-glossary/hybridization


https://www.genome.gov/genetics-glossary/Microsatellite
https://www.genome.gov/genetics-glossary/Personalized-Medicine
https://www.genome.gov/genetics-glossary/Physical-Map
https://www.genome.gov/genetics-glossary/Polygenic-Trait

https://www.genome.gov/genetics-glossary/Tandem-Repeat
https://www.genome.gov/genetics-glossary/hapmap
https://www.genome.gov/genetics-glossary/LOD-Score
https://www.genome.gov/genetics-glossary/Mapping
https://www.genome.gov/genetics-glossary/Linkage





 

################################################################################################################################################################################
################################################################################################################################################################################
################################################################################################################################################################################
 
 




################################################################################################################################################################################
################################################################################################################################################################################
################################################################################################################################################################################
 

## Random coil (protein conformation)

Adapted from Wikipedia https://en.wikipedia.org/wiki/Random_coil

A random coil is a polymer conformation where the monomer subunits are oriented randomly while still being bonded to adjacent units. It is not one specific shape, but a statistical distribution of shapes for all the chains in a population of macromolecules. The conformation's name is derived from the idea that, in the absence of specific, stabilizing interactions, a polymer backbone will "sample" all possible conformations randomly. Many linear, unbranched homopolymers — in solution, or above their melting temperatures — assume (approximate) random coils.


<!-- Contents -->
  <!-- 1	Random walk model: The Gaussian chain -->
  <!-- 2	Real polymers -->
  <!-- 3	Spectroscopy -->
  <!-- 4	See also -->
  <!-- 5	References -->
  <!-- 6	External links -->
  
  
  <!-- Random walk model: The Gaussian chain -->
  <!-- Main article: Ideal chain -->
  
  <!-- Short random chain -->
  <!-- There are an enormous number of different ways in which a chain can be curled around in a relatively compact shape, like an unraveling ball of twine with much open space, and comparatively few ways it can be more or less stretched out. So, if each conformation has an equal probability or statistical weight, chains are much more likely to be ball-like than they are to be extended — a purely entropic effect. In an ensemble of chains, most of them will, therefore, be loosely balled up. This is the kind of shape any one of them will have most of the time. -->
  
  <!-- Consider a linear polymer to be a freely-jointed chain with N subunits, each of length {\displaystyle \scriptstyle \ell }\scriptstyle \ell , that occupy zero volume, so that no part of the chain excludes another from any location. One can regard the segments of each such chain in an ensemble as performing a random walk (or "random flight") in three dimensions, limited only by the constraint that each segment must be joined to its neighbors. This is the ideal chain mathematical model. It is clear that the maximum, fully extended length L of the chain is {\displaystyle \scriptstyle N\,\times \,\ell }\scriptstyle N\,\times \,\ell . If we assume that each possible chain conformation has an equal statistical weight, it can be shown that the probability P(r) of a polymer chain in the population to have distance r between the ends will obey a characteristic distribution described by the formula -->
  
  <!-- {\displaystyle P(r)=4\pi r^{2}\left({\frac {3}{2\;\pi \langle r^{2}\rangle }}\right)^{3/2}\;e^{-\,{\frac {3r^{2}}{2\langle r^{2}\rangle }}}}{\displaystyle P(r)=4\pi r^{2}\left({\frac {3}{2\;\pi \langle r^{2}\rangle }}\right)^{3/2}\;e^{-\,{\frac {3r^{2}}{2\langle r^{2}\rangle }}}} -->
  <!-- The average (root mean square) end-to-end distance for the chain, {\displaystyle \scriptstyle {\sqrt {\langle r^{2}\rangle }}}\scriptstyle {\sqrt  {\langle r^{2}\rangle }}, turns out to be {\displaystyle \scriptstyle \ell }\scriptstyle \ell  times the square root of N — in other words, the average distance scales with N0.5. -->
  
  <!-- Note that although this model is termed a "Gaussian chain", the distribution function is not a gaussian (normal) distribution. The end-to-end distance probability distribution function of a Gaussian chain is non-zero only for r > 0. [1] -->
  
  <!-- Real polymers -->
  <!-- A real polymer is not freely-jointed. A -C-C- single bond has a fixed tetrahedral angle of 109.5 degrees. The value of L is well-defined for, say, a fully extended polyethylene or nylon, but it is less than N x l because of the zig-zag backbone. There is, however, free rotation about many chain bonds. The model above can be enhanced. A longer, "effective" unit length can be defined such that the chain can be regarded as freely-jointed, along with a smaller N, such that the constraint L = N x l is still obeyed. It, too, gives a Gaussian distribution. However, specific cases can also be precisely calculated. The average end-to-end distance for freely-rotating (not freely-jointed) polymethylene (polyethylene with each -C-C- considered as a subunit) is l times the square root of 2N, an increase by a factor of about 1.4. Unlike the zero volume assumed in a random walk calculation, all real polymers' segments occupy space because of the van der Waals radii of their atoms, including bulky substituent groups that interfere with bond rotations. This can also be taken into account in calculations. All such effects increase the mean end-to-end distance. -->

<!-- Because their polymerization is stochastically driven, chain lengths in any real population of synthetic polymers will obey a statistical distribution. In that case, we should take N to be an average value. Also, many polymers have random branching. -->

<!-- Even with corrections for local constraints, the random walk model ignores steric interference between chains, and between distal parts of the same chain. A chain often cannot move from a given conformation to a closely related one by a small displacement because one part of it would have to pass through another part, or through a neighbor. We may still hope that the ideal-chain, random-coil model will be at least a qualitative indication of the shapes and dimensions of real polymers in solution, and in the amorphous state, as long as there are only weak physicochemical interactions between the monomers. This model, and the Flory-Huggins Solution Theory,[2][3] for which Paul Flory received the Nobel Prize in Chemistry in 1974, ostensibly apply only to ideal, dilute solutions. But there is reason to believe (e.g., neutron diffraction studies) that excluded volume effects may cancel out, so that, under certain conditions, chain dimensions in amorphous polymers have approximately the ideal, calculated size [4] When separate chains interact cooperatively, as in forming crystalline regions in solid thermoplastics, a different mathematical approach must be used. -->

<!-- Stiffer polymers such as helical polypeptides, Kevlar, and double-stranded DNA can be treated by the worm-like chain model. -->

<!-- Even copolymers with monomers of unequal length will distribute in random coils if the subunits lack any specific interactions. The parts of branched polymers may also assume random coils. -->

<!-- Below their melting temperatures, most thermoplastic polymers (polyethylene, nylon, etc.) have amorphous regions in which the chains approximate random coils, alternating with regions that are crystalline. The amorphous regions contribute elasticity and the crystalline regions contribute strength and rigidity. -->

More complex polymers such as proteins, with various interacting chemical groups attached to their backbones, self-assemble into well-defined structures. But segments of proteins, and polypeptides that lack secondary structure, are often assumed to exhibit a random-coil conformation in which the only fixed relationship is the joining of adjacent amino acid residues by a peptide bond. This is not actually the case, since the ensemble will be energy weighted due to interactions between amino acid side-chains, with lower-energy conformations being present more frequently. In addition, even arbitrary sequences of amino acids tend to exhibit some hydrogen bonding and secondary structure. For this reason, the term "statistical coil" is occasionally preferred. The conformational entropy of the random-coil stabilizes the unfolded protein state and represents main free energy contribution that opposes to protein folding.

Spectroscopy
A random-coil conformation can be detected using spectroscopic techniques. The arrangement of the planar amide bonds results in a distinctive signal in circular dichroism. The chemical shift of amino acids in a random-coil conformation is well known in nuclear magnetic resonance (NMR). Deviations from these signatures often indicates the presence of some secondary structure, rather than complete random coil. Furthermore, there are signals in multidimensional NMR experiments that indicate that stable, non-local amino acid interactions are absent for polypeptides in a random-coil conformation. Likewise, in the images produced by crystallography experiments, segments of random coil result simply in a reduction in "electron density" or contrast. A randomly coiled state for any polypeptide chain can be attained by denaturing the system. However, there is evidence that proteins are never truly random coils, even when denatured (Shortle & Ackerman).



Flory, P.J. (1953) Principles of Polymer Chemistry, Cornell Univ. Press, ISBN 0-8014-0134-8
Flory, P.J. (1969) Statistical Mechanics of Chain Molecules, Wiley, ISBN 0-470-26495-0; reissued 1989, ISBN 1-56990-019-1
"Conformations, Solutions, and Molecular Weight" from "Polymer Science & Technology" courtesy of Prentice Hall Professional publications [1]



############################################################
############################################################
############################################################
############################################################
############################################################
############################################################
############################################################

## Haplotype

Haplotype
Adapted from Wikipedia, https://en.wikipedia.org/wiki/Haplotype


DNA molecule 1 differs from DNA molecule 2 at a single base-pair location (a C/A polymorphism).
A haplotype (haploid genotype) is a group of alleles in an organism that are inherited together from a single parent.[1][2]

Many organisms contain genetic material (DNA) which is inherited from two parents. Normally these organisms have their DNA organized in two sets of pairwise similar chromosomes. The offspring gets one chromosome in each pair from each parent. A set of pairs of chromosomes is called diploid and a set of only one half of each pair is called haploid. The haploid genotype (haplotype) is a genotype that considers the singular chromosomes rather than the pairs of chromosomes. It can be all the chromosomes from one of the parents or a minor part of a chromosome, for example a sequence of 9000 base pairs.

However, there are other uses of this term. First, it is used to mean a collection of specific alleles (that is, specific DNA sequences) in a cluster of tightly linked genes on a chromosome that are likely to be inherited together—that is, they are likely to be conserved as a sequence that survives the descent of many generations of reproduction.[3][4] A second use is to mean a set of linked single-nucleotide polymorphism (SNP) alleles that tend to always occur together (i.e., that are associated statistically). It is thought that identifying these statistical associations and a few alleles of a specific haplotype sequence can facilitate identifying all other such polymorphic sites that are nearby on the chromosome. Such information is critical for investigating the genetics of common diseases; which in fact have been investigated in humans by the International HapMap Project.[5][6] Thirdly, many human genetic testing companies use the term in a third way: to refer to an individual collection of specific mutations within a given genetic segment; (see short tandem repeat mutation).

The term 'haplogroup' refers to the SNP/unique-event polymorphism (UEP) mutations that represent the clade to which a collection of particular human haplotypes belong. (Clade here refers to a set of haplotypes sharing a common ancestor.)[7] A haplogroup is a group of similar haplotypes that share a common ancestor with a single-nucleotide polymorphism mutation.[8][9] Mitochondrial DNA passes along a maternal lineage that can date back thousands of years.[8]



Haplotype resolution
An organism's genotype may not define its haplotype uniquely. For example, consider a diploid organism and two bi-allelic loci (such as SNPs) on the same chromosome. Assume the first locus has alleles A or T and the second locus G or C. Both loci, then, have three possible genotypes: (AA, AT, and TT) and (GG, GC, and CC), respectively. For a given individual, there are nine possible configurations (haplotypes) at these two loci (shown in the Punnett square below). For individuals who are homozygous at one or both loci, the haplotypes are unambiguous - meaning that there is not any differentiation of haplotype T1T2 vs haplotype T2T1; where T1 and T2 are labeled to show that they are the same locus, but labeled as such to show it doesn't matter which order you consider them in, the end result is two T loci. For individuals heterozygous at both loci, the gametic phase is ambiguous - in these cases, you don't know which haplotype you have, e.g., TA vs AT.

AA	AT	TT
GG	AG AG	AG TG	TG TG
GC	AG AC	AG TC
or
AC TG	TG TC
CC	AC AC	AC TC	TC TC
The only unequivocal method of resolving phase ambiguity is by sequencing. However, it is possible to estimate the probability of a particular haplotype when phase is ambiguous using a sample of individuals.

Given the genotypes for a number of individuals, the haplotypes can be inferred by haplotype resolution or haplotype phasing techniques. These methods work by applying the observation that certain haplotypes are common in certain genomic regions. Therefore, given a set of possible haplotype resolutions, these methods choose those that use fewer different haplotypes overall. The specifics of these methods vary - some are based on combinatorial approaches (e.g., parsimony), whereas others use likelihood functions based on different models and assumptions such as the Hardy–Weinberg principle, the coalescent theory model, or perfect phylogeny. The parameters in these models are then estimated using algorithms such as the expectation-maximization algorithm (EM), Markov chain Monte Carlo (MCMC), or hidden Markov models (HMM).

Microfluidic whole genome haplotyping is a technique for the physical separation of individual chromosomes from a metaphase cell followed by direct resolution of the haplotype for each allele.

Y-DNA haplotypes from genealogical DNA tests
Main article: Genealogical DNA test
Unlike other chromosomes, Y chromosomes generally do not come in pairs. Every human male (excepting those with XYY syndrome) has only one copy of that chromosome. This means that there is not any chance variation of which copy is inherited, and also (for most of the chromosome) not any shuffling between copies by recombination; so, unlike autosomal haplotypes, there is effectively not any randomisation of the Y-chromosome haplotype between generations. A human male should largely share the same Y chromosome as his father, give or take a few mutations; thus Y chromosomes tend to pass largely intact from father to son, with a small but accumulating number of mutations that can serve to differentiate male lineages. In particular, the Y-DNA represented as the numbered results of a Y-DNA genealogical DNA test should match, except for mutations.

UEP results (SNP results)
Unique-event polymorphisms (UEPs) such as SNPs represent haplogroups. STRs represent haplotypes. The results that comprise the full Y-DNA haplotype from the Y chromosome DNA test can be divided into two parts: the results for UEPs, sometimes loosely called the SNP results as most UEPs are single-nucleotide polymorphisms, and the results for microsatellite short tandem repeat sequences (Y-STRs).

The UEP results represent the inheritance of events it is believed can be assumed to have happened only once in all human history. These can be used to identify the individual's Y-DNA haplogroup, his place in the "family tree" of the whole of humanity. Different Y-DNA haplogroups identify genetic populations that are often distinctly associated with particular geographic regions; their appearance in more recent populations located in different regions represents the migrations tens of thousands of years ago of the direct patrilineal ancestors of current individuals.

Y-STR haplotypes
Genetic results also include the Y-STR haplotype, the set of results from the Y-STR markers tested.

Unlike the UEPs, the Y-STRs mutate much more easily, which allows them to be used to distinguish recent genealogy. But it also means that, rather than the population of descendants of a genetic event all sharing the same result, the Y-STR haplotypes are likely to have spread apart, to form a cluster of more or less similar results. Typically, this cluster will have a definite most probable center, the modal haplotype (presumably similar to the haplotype of the original founding event), and also a haplotype diversity — the degree to which it has become spread out. The further in the past the defining event occurred, and the more that subsequent population growth occurred early, the greater the haplotype diversity will be for a particular number of descendants. However, if the haplotype diversity is smaller for a particular number of descendants, this may indicate a more recent common ancestor, or a recent population expansion.

It is important to note that, unlike for UEPs, two individuals with a similar Y-STR haplotype may not necessarily share a similar ancestry. Y-STR events are not unique. Instead, the clusters of Y-STR haplotype results inherited from different events and different histories tend to overlap.

In most cases, it is a long time since the haplogroups' defining events, so typically the cluster of Y-STR haplotype results associated with descendants of that event has become rather broad. These results will tend to significantly overlap the (similarly broad) clusters of Y-STR haplotypes associated with other haplogroups. This makes it impossible for researchers to predict with absolute certainty to which Y-DNA haplogroup a Y-STR haplotype would point. If the UEPs are not tested, the Y-STRs may be used only to predict probabilities for haplogroup ancestry, but not certainties.

A similar scenario exists in trying to evaluate whether shared surnames indicate shared genetic ancestry. A cluster of similar Y-STR haplotypes may indicate a shared common ancestor, with an identifiable modal haplotype, but only if the cluster is sufficiently distinct from what may have happened by chance from different individuals who historically adopted the same name independently. Many names were adopted from common occupations, for instance, or were associated with habitation of particular sites. More extensive haplotype typing is needed to establish genetic genealogy. Commercial DNA-testing companies now offer their customers testing of more numerous sets of markers to improve definition of their genetic ancestry. The number of sets of markers tested has increased from 12 during the early years to 111 more recently.

Establishing plausible relatedness between different surnames data-mined from a database is significantly more difficult. The researcher must establish that the very nearest member of the population in question, chosen purposely from the population for that reason, would be unlikely to match by accident. This is more than establishing that a randomly selected member of the population is unlikely to have such a close match by accident. Because of the difficulty, establishing relatedness between different surnames as in such a scenario is likely to be impossible, except in special cases where there is specific information to drastically limit the size of the population of candidates under consideration.

Diversity
Haplotype diversity is a measure of the uniqueness of a particular haplotype in a given population. The haplotype diversity (H) is computed as:[10]
{\displaystyle H={\frac {N}{N-1}}(1-\sum _{i}x_{i}^{2})}H={\frac  {N}{N-1}}(1-\sum _{{i}}x_{i}^{2})
where {\displaystyle x_{i}}x_{i} is the (relative) haplotype frequency of each haplotype in the sample and {\displaystyle N}N is the sample size. Haplotype diversity is given for each sample.



References
 By C. Barry Cox, Peter D. Moore, Richard Ladle. Wiley-Blackwell, 2016. ISBN 978-1-118-96858-1 p106. Biogeography: An Ecological and Evolutionary Approach
 Editorial Board, V&S Publishers, 2012, ISBN 9381588643 p137.Concise Dictionary of Science
 BiologyPages/H/Haplotypes.html Kimball's Biology Pages (Creative Commons Attribution 3.0)
"haplotype / haplotypes | Learn Science at Scitable". www.nature.com.
The International HapMap Consortium (2003). "The International HapMap Project" (PDF). Nature. 426 (6968): 789–796. Bibcode:2003Natur.426..789G. doi:10.1038/nature02168. hdl:2027.42/62838. PMID 14685227. S2CID 4387110.
The International HapMap Consortium (2005). "A haplotype map of the human genome". Nature. 437 (7063): 1299–1320. Bibcode:2005Natur.437.1299T. doi:10.1038/nature04226. PMC 1880871. PMID 16255080.
"Facts & Genes. Volume 7, Issue 3". Archived from the original on May 9, 2008.
Arora, Devender; Singh, Ajeet; Sharma, Vikrant; Bhaduria, Harvendra Singh; Patel, Ram Bahadur (2015). "Hgs Db: Haplogroups Database to understand migration and molecular risk assessment". Bioinformation. 11 (6): 272–5. doi:10.6026/97320630011272. PMC 4512000. PMID 26229286.
International Society of Genetic Genealogy 2015 Genetics Glossary
Masatoshi Nei and Fumio Tajima, "DNA polymorphism detectable by restriction endonucleases", Genetics 97:145 (1981)
Becker T.; Knapp M. (2004). "Maximum-likelihood estimation of haplotype frequencies in nuclear families". Genetic Epidemiology. 27 (1): 21–32. doi:10.1002/gepi.10323. PMID 15185400.
Li S.S.; Khalid N.; Carlson C.; Zhao L.P. (2003). "Estimating haplotype frequencies and standard errors for multiple single nucleotide polymorphisms". Biostatistics. 4 (4): 513–522. doi:10.1093/biostatistics/4.4.513. PMID 14557108.
Roach J.C.; Glusman G.; Hubley R.; Montsaroff S.Z.; Holloway A.K.; Mauldin D.E.; Srivastava D.; Garg V.; Pollard K.S.; Galas D.J.; Hood L.; Smit A.F.A. (2011). "Chromosomal Haplotypes by Genetic Phasing of Human Families". American Journal of Human Genetics. 89 (3): 382–397. doi:10.1016/j.ajhg.2011.07.023. PMC 3169815. PMID 21855840.
Barrett J.C.; Fry B.; Maller J.; Daly M.J. (2005). "Haploview: analysis and visualization of LD and haplotype maps". Bioinformatics. 21 (2): 263–265. doi:10.1093/bioinformatics/bth457. PMID 15297300.
Delaneau O, Zagury JF, Marchini J (2013). "Improved whole chromosome phasing for disease and population genetic studies". Nature Methods. 10 (1): 5–6. doi:10.1038/nmeth.2307. PMID 23269371. S2CID 205421216.
Purcell S.; Daly M. J.; Sham P. C. (2007). "WHAP: haplotype-based association analysis". Bioinformatics. 23 (2): 255–256. doi:10.1093/bioinformatics/btl580. PMID 17118959.

############################################################
############################################################
############################################################
############################################################
############################################################
############################################################
############################################################
############################################################





############################################################
############################################################
############################################################

## Native state (protein)


Native state
From Wikipedia, the free encyclopedia
Jump to navigationJump to search
For the sovereign entities of British India, see Native states.
For native states in metallurgy, see Native state (metallurgy).

This article does not cite any sources. Please help improve this article by adding citations to reliable sources. Unsourced material may be challenged and removed.
Find sources: "Native state" – news · newspapers · books · scholar · JSTOR (December 2009) (Learn how and when to remove this template message)
In biochemistry, the native state of a protein or nucleic acid is its properly folded and/or assembled form, which is operative and functional. The native state of a biomolecule may possess all four levels of biomolecular structure, with the secondary through quaternary structure being formed from weak interactions along the covalently-bonded backbone. This is in contrast to the denatured state, in which these weak interactions are disrupted, leading to the loss of these forms of structure and retaining only the biomolecule's primary structure.


Contents
1	Biochemistry
1.1	Proteins
1.2	Nucleic acids
2	External links
Biochemistry
Proteins
Main article: Protein structure
While all protein molecules begin as simple unbranched chains of amino acids, once completed they assume highly specific three-dimensional shapes. That ultimate shape, known as tertiary structure, is the folded shape that possesses a minimum of free energy. It is a protein's tertiary, folded structure that makes it capable of performing its biological function. In fact, shape changes in proteins are the primary cause of several neurodegenerative diseases, including those caused by prions and amyloid (i.e. mad cow disease, kuru, Creutzfeldt–Jakob disease).

Many enzymes and other non-structural proteins have more than one native state, and they operate or undergo regulation by transitioning between these states. However, "native state" is used almost exclusively in the singular, typically to distinguish properly folded proteins from denatured or unfolded ones. In other contexts, the folded shape of a protein is most often referred to as its native "conformation" or "structure."

Folded and unfolded proteins are often easily distinguished by virtue of their water solubilities, as many proteins become insoluble on denaturation. Proteins in the native state will have defined secondary structure, which can be detected spectroscopically, by circular dichroism and by nuclear magnetic resonance (NMR).

The native state of a protein can be distinguished from a molten globule, by among other things, distances measured by NMR. Amino acids widely separated in a protein's sequence may touch or lie very close to one another within a stably folded protein. In a molten globule, on the other hand, their time-averaged distances are liable to be greater.

Learning how native state proteins can be manufactured is important, as attempts to create proteins from scratch have resulted in molten globules and not true native state products. Therefore, an understanding of the native state is crucial in protein engineering.

Nucleic acids

Nucleic acids attain their native state through base pairing and, to a lesser extent, other interactions such as coaxial stacking. Biological DNA usually exists as long linear double helices bound to proteins in chromatin, and biological RNA such as tRNA often form complex native configurations approaching the complexity of folded proteins. Additionally, artificial nucleic acid structures used in DNA nanotechnology are designed to have specific native configurations in which multiple nucleic acid strands are assembled into a single complex. In some cases native state of biological DNA performs their functions without being controlled by any other regulatory units.
