## GWAS (Genome-wide Association Mapping)


Adapated from  Wikipedia:

https://en.wikipedia.org/wiki/Genome-wide_association_study

In genomics, a **genome-wide association study** (GWAS), also known as whole genome association study (WGAS), is an **observational study** of a genome-wide set of genetic **variants(()) in different individuals to see if any variant is associated with a trait. GWAS typically focus on associations between **single-nucleotide polymorphisms (SNPs)** and traits like major human diseases, but can equally be applied to any other genetic variants, phenotypes, or organisms.  

```{r}
## Load packages and data
#install.packages("qqman")
library(qqman)
data(gwasResults)

manhattan(gwasResults,
          annotatePval = 0.005, 
          main = "Manhattan Plot", 
          ylab = "Significance",
          annotateTop = TRUE)

```
**Fig. 1: Manhattan plot of a GWAS**: An illustration of a **Manhattan plot** depicting several strongly associated risk loci. Each dot represents a SNP, with the X-axis showing genomic location and Y-axis showing association level (significance). 

When applied to human health data, GWAS compare the DNA of participants having varying phenotypes for a particular trait or disease. These participants may be people with a disease (**"cases"** in epidemiology lingo) and similar people without the disease (controls); a study designed like this is a type of **case-control** study.   Alternatively, study participants  may be people with different phenotypes for a particular trait along a continuous range, such as example blood pressure. 

This approach is known as **phenotype-first**, in which the participants are classified first by their clinical manifestation(s) or phenotypes, as opposed to **genotype-first**. Each person gives a sample of DNA, from which typically hundreds of thousands to millions of genetic variants are read, typically using what are called using **SNP arrays**, a type of DNA **micro-array** which contains **genetic probes** for each SNP in the study laid out in a grid microscipic spots  If a sample from a subject in the study contains the SNP, it will bind to the probe in spot containing the SNP.  Any bound samples can them be made to give off a visual signal indicating succesful binding and the presence of specific SNPs.  If one type of the variant (one allele) is more frequent in people with the disease, the variant is said to be **associated** with the disease. The associated SNPs are then considered to mark a region of the human genome - but not a particular causal mutation -  that may influence the risk of disease.

GWAS investigate the *entire* genome, in contrast to methods that specifically test a small number of pre-specified genetic regions. Hence, GWAS is a non-**candidate gene**-driven approach, in contrast to gene-specific candidate-driven studies. GWAS identify SNPs and other variants in DNA associated with a disease, but they cannot on their own specify which genes are causal.

The first successful GWAS published in 2002 studied heart attach (myocardial infarction; 4). This study design was then implemented in the landmark GWA 2005 study investigating patients with age-related macular degeneration, and found two SNPs with significantly altered allele frequency compared to healthy controls (5). As of 2017, over 3,000 human GWAS have examined over 1,800 diseases and traits, and thousands of SNP associations have been found (6). Except in the case of rare genetic diseases, these associations are *very* weak; but while they may not explain much of the risk, they do provide insight into genes and pathways that can be important.


Contents
1.	Background
1.	Methods
1.	Results
1.	Clinical applications
  1. Atrial fibrillation
  1. Schizophrenia
1.	Agricultural applications
  1. Plant growth stages and yield components
  1. Plant pathogens
1	Limitations
1.	Fine-mapping
1.	See also
1.	References
1.	External links


### Background

<!-- GWAS typically identify common **variants** with small **effect sizes** (lower right) (7] -->

Any two human genomes differ in millions of different ways. There are small variations in the individual nucleotides of the genomes (**SNPs**) as well as many larger variations, such as **deletions**, **insertions** and **copy number variations** (CNVs). Any of these may cause alterations in an individual's traits, or **phenotype**, which can be anything from disease risk to physical properties such as height (8).  

Around the year 2000, prior to the introduction of GWAS, the primary method of investigation was through inheritance studies of genetic linkage in families. This approach had proven highly useful towards relatively simple, typically **Mendellian-inherited** **monogenic** (single-gene) disorders and phenotypes (9, 8, 10). However, for common and complex diseases the results of genetic linkage studies proved hard to reproduce in new, independent samples or different populations (8, 10). A suggested alternative to linkage studies was the **genetic association study**. This study type asks if the allele of a genetic variant is found more often than expected in individuals with the phenotype of interest (e.g. with the disease being studied). 

<!-- Early calculations on statistical power indicated that this approach could be better than linkage studies at detecting weak genetic effects (11] -->

In addition to the conceptual framework several additional factors enabled GWAS. One was the advent of **biobanks**, which are repositories of human genetic material that greatly reduced the cost and difficulty of collecting sufficient numbers of biological specimens for study (12).  Another was the **International HapMap Project**, which, from 2003 identified a majority of the common SNPs interrogated in GWAS (13). The **haploblock** structure identified by HapMap project also allowed the focus on the subset of SNPs that would describe most of the variation. Also the development of the methods to genotype all these SNPs using **genotyping arrays** was an important prerequisite (14).

### GWAS Methods

<!-- Example calculation illustrating the methodology of a case-control GWA study. The allele count of each measured SNP is evaluated—in this case with a chi-squared test—to identify variants associated with the trait in question. The numbers in this example are taken from a 2007 study of coronary artery disease (CAD) that showed that the individuals with the G-allele of SNP1 (rs1333049) were overrepresented amongst CAD-patients (15] -->

<!-- Illustration of a simulated genotype by phenotype regression for a single SNP. Each dot represents an individual. A GWAS of a continuous trait essentially consists of repeating this analysis at each SNP. -->
<!-- The most common approach of GWAS is the case-control setup, which compares two large groups of individuals, one healthy control group and one case group affected by a disease. All individuals in each group are genotyped for the majority of common known SNPs. The exact number of SNPs depends on the genotyping technology, but are typically one million or more (7] For each of these SNPs it is then investigated if the allele frequency is significantly altered between the case and the control group (16] In such setups, the fundamental unit for reporting effect sizes is the odds ratio. The odds ratio is the ratio of two odds, which in the context of GWAS are the odds of case for individuals having a specific allele and the odds of case for individuals who do not have that same allele. -->

Here's a hypothetical example of a GWAS comparing a group with a disease to a control group.  Suppose that there are two alleles, T and C. The number of individuals in the **case** group having allele T is represented by 'A' and the number of individuals in the control group having allele T is represented by 'B'. Similarly, the number of individuals in the case group having allele C is represented by 'X' and the number of individuals in the control group having allele C is represented by 'Y'. In this case the odds ratio for allele T is A:B (meaning 'A to B', in standard odds terminology) divided by X:Y, which in mathematical notation is simply (A/B)/(X/Y).

When the allele frequency in the case group is much higher than in the control group, the odds ratio is higher than 1, and vice versa for lower allele frequency. Additionally, a P-value for the significance of the odds ratio is typically calculated using a simple chi-squared test. Finding odds ratios that are significantly different from 1 is the objective of the GWA study because this shows that a SNP is associated with disease (16] Because so many variants are tested, it is standard practice to require the p-value to be lower than 5×10−8 to consider a variant significant.

There are several variations to this case-control approach. A common alternative to case-control GWAS is the analysis of quantitative phenotypic data, e.g. height or biomarker concentrations or even gene expression. Likewise, alternative statistics designed for dominance or recessive penetrance patterns can be used (16] Calculations are typically done using bioinformatics software such as SNPTEST and PLINK, which also include support for many of these alternative statistics (15, 17).  GWAS focuses on the effect of individual SNPs. However, it is also possible that complex interactions among two or more SNPs (epistasis) might contribute to complex diseases. Due to the potentially exponential number of interactions, detecting statistically significant interactions in GWAS data is both computationally and statistically challenging. This task has been tackled in existing publications that use algorithms inspired from data mining (18) Moreover, the researchers try to integrate GWA data with other biological data such as protein-protein interaction network to extract more informative results (19, 20).

<!-- A key step in the majority of GWAS is the imputation of genotypes at SNPs not on the genotype chip used in the study (21] This process greatly increases the number of SNPs that can be tested for association, increases the power of the study, and facilitates meta-analysis of GWAS across distinct cohorts. Genotype imputation is carried out by statistical methods that combine the GWAS data together with a reference panel of haplotypes. These methods take advantage of sharing of haplotypes between individuals over short stretches of sequence to impute alleles. Existing software packages for genotype imputation include IMPUTE2,[22] Minimac, Beagle[23] and MaCH (24] -->


In addition to the calculation of association, it is common to take into account any variables that could potentially confound the results. Sex and age are common examples of confounding variables. Moreover, it is also known that many genetic variations are associated with the geographical and historical populations in which the mutations first arose (25). Because of this association, studies must take account of the geographic and ethnic background of participants by controlling for what is called population stratification. If they fail to do so, these studies can produce false positive results (26)

After odds ratios and P-values have been calculated for all SNPs, a common approach is to create a Manhattan plot. In the context of GWAS, this plot shows the negative logarithm of the P-value as a function of genomic location. Thus the SNPs with the most significant association stand out on the plot, usually as stacks of points because of haploblock structure. Importantly, the P-value threshold for significance is corrected for multiple testing issues. The exact threshold varies by study,[27] but the conventional threshold is 5×10−8 to be significant in the face of hundreds of thousands to millions of tested SNPs (7, 16, 28] GWAS typically perform the first analysis in a discovery cohort, followed by validation of the most significant SNPs in an independent validation cohort (29]

### Results

<!-- Regional association plot, showing individual SNPs in the LDL receptor region and their association to LDL-cholesterol levels. This type of plot is similar to the Manhattan plot in the lead section, but for a more limited section of the genome. The haploblock structure is visualized with colour scale and the association level is given by the left Y-axis. The dot representing the rs73015013 SNP (in the top-middle) has a high Y-axis location because this SNP explains some of the variation in LDL-cholesterol (30] -->

<!-- Relationship between the minor allele frequency and the effect size of genome wide significant variants in a GWAS of height. -->
<!-- Attempts have been made at creating comprehensive catalogues of SNPs that have been identified from GWAS (31] As of 2009, SNPs associated with diseases are numbered in the thousands (32] -->

An early GWAS, conducted in 2005, compared 96 patients with the eye disorder age-related macular degeneration (ARMD) with 50 healthy controls (33). It identified two SNPs with significantly altered allele frequencies between the two groups. These SNPs were located in the gene encoding complement factor H, which was an unexpected finding in the research of ARMD. These findings prompted further functional research towards therapeutical manipulation of the complement system in ARMD (34). Another landmark publication in the history of GWAS was the Wellcome Trust Case Control Consortium (WTCCC) study, the largest GWAS ever conducted at the time of its publication in 2007. The WTCCC included 14,000 cases of seven common diseases (~2,000 individuals for each of coronary heart disease, type 1 diabetes, type 2 diabetes, rheumatoid arthritis, Crohn's disease, bipolar disorder, and hypertension) and 3,000 shared controls (15). This study was successful in uncovering many new disease genes underlying these diseases (15, 35).

Since these first landmark GWAS, there have been two general trends (36).  One has been towards larger and larger sample sizes. In 2018, several genome-wide association studies are reaching a total sample size of over 1 million participants, including a study of insomnia containing 1.3 million individuals (38). The reason is the drive towards reliably detecting risk-SNPs that have ever smaller effects and lower allele frequencies. Another trend has been towards the use of more narrowly defined continous phenotypes, such as concentrations of blood lipids, proinsulin or similar biomarkers (39, 40). These are called intermediate phenotypes, and their analyses may be of value to functional research into biomarkers (41).  

<!-- A variation of GWAS uses participants that are first-degree relatives of people with a disease. This type of study has been named genome-wide association study by proxy (GWAX) (42] -->

A central point of debate on GWAS has been that most of the SNP variations found by GWAS are associated with only a small increased risk of the disease, and have only a small predictive value.

<!-- The median odds ratio is 1.33 per risk-SNP, with only a few showing odds ratios above 3.0 (1, 43] -->

The effects sizes are considered small because they do not explain much of the heritable variation. This heritable variation is estimated from heritability studies based on monozygotic twins (44). For example, it is known that 80-90% of variance in height can be explained by hereditary differences, but GWAS only account for a minority of this variance (44).  This is sometimes referred to as the problem of the **missing heritability** of GWAS.

### Clinical applications

**THIS SECTION IS OPTIONAL**

A challenge for future successful GWAS is to apply the findings in a way that accelerates drug and diagnostics development, including better integration of genetic studies into the drug-development process and a focus on the role of genetic variation in maintaining health as a blueprint for designing new drugs and diagnostics (45). Several studies have looked into the use of risk-SNP markers as a means of directly improving the accuracy of prognosis. Some have found that the accuracy of prognosis improves (46),  while others report only minor benefits from this use (47).  Generally, a problem with this direct approach is the small magnitudes of the effects observed. A small effect ultimately translates into a poor separation of cases and controls and thus only a small improvement of prognosis accuracy. An alternative application is therefore the potential for GWAS to elucidate pathophysiology (48).

One such success is related to identifying the genetic variant associated with response to anti-hepatitis C virus treatment. For genotype 1 hepatitis C treated with Pegylated interferon-alpha-2a or Pegylated interferon-alpha-2b combined with ribavirin, a GWAS (49) has shown that SNPs near the human IL28B gene, encoding interferon lambda 3, are associated with significant differences in response to the treatment. A later report demonstrated that the same genetic variants are also associated with the natural clearance of the genotype 1 hepatitis C virus (50).  These major findings facilitated the development of personalized medicine and allowed physicians to customize medical decisions based on the patient's genotype (51).

The goal of elucidating pathophysiology has also led to increased interest in the association between risk-SNPs and the gene *expression* of nearby genes, the so-called **expression quantitative trait loci (eQTL)** studies (52).  The reason is that GWAS studies identify risk-SNPs, but not risk-genes, and specification of genes is one step closer towards actionable drug targets. As a result, major GWAS by 2011 typically included extensive eQTL analysis (53, 54, 55).  One of the strongest eQTL effects observed for a GWA-identified risk SNP is the SORT1 locus (39). Functional follow up studies of this locus using small interfering RNA and gene knock-out mice have shed light on the metabolism of low-density lipoproteins, which have important clinical implications for cardiovascular disease (39, 56, 57).  

#### Atrial fibrillation

**This section is optional**

For example, a meta-analysis accomplished in 2018 revealed the discovery of 70 new loci associated with atrial fibrillation. It has been identified different variants associated with transcription factor coding-genes, such as TBX3 and TBX5, NKX2-5 o PITX2, which are involved in cardiac conduction regulation, in ionic channel modulation and cardiac development. It was also identified new genes involved in tachycardia (CASQ2) or associated with alteration of cardiac muscle cell communication (PKP2) (58) 

#### Schizophrenia

**This section is optional**

While there is some research using a High-Precision Protein Interaction Prediction (HiPPIP) computational model that discovered 504 new protein-protein interactions (PPIs) associated with genes linked to schizophrenia (59,60) the evidence supporting the genetic basis of schizophrenia is actually controversial and may suffer from some of the limitation of this method of study (61]

### Agricultural applications

#### Plant growth stages and yield components

**This section is optional**

GWAS act as an important tool in plant breeding. With large genotyping and phenotyping data, GWAS are powerful in analyzing complex inheritance modes of traits that are important yield components such as number of grains per spike, weight of each grain and plant structure. In a study on GWAS in spring wheat, GWAS have revealed a strong correlation of grain production with booting data, biomass and number of grains per spike (62).

#### Plant pathogens

**This section is optional**

The emergences of plant pathogens have posed serious threats to plant health and biodiversity. Under this consideration, identification of wild types that have the natural resistance to certain pathogens could be of vital importance. Furthermore, we need to predict which alleles are associated with the resistance. GWAS is a powerful tool to detect the relationships of certain variants and the resistance to the plant pathogen, which is beneficial for developing new pathogen-resisted cultivars (63).

### Limitations


#### Limitations - common errors

**This section is optional**

GWAS have several issues and limitations that can be taken care of through proper quality control and study setup. Lack of well defined case and control groups, insufficient sample size, control for **multiple testing** and control for **population stratification** are common problems (2). Particularly the statistical issue of multiple testing wherein it has been noted that "the GWA approach can be problematic because the massive number of statistical tests performed presents an unprecedented potential for false-positive results" (2). Ignoring these correctible issues has been cited as contributing to a general sense of problems with the GWAS methodology (64). In addition to easily correctible problems such as these, some more subtle but important issues have surfaced. A high-profile GWA study that investigated individuals with very long life spans to identify SNPs associated with longevity is an example of this (65). The publication came under scrutiny because of a discrepancy between the type of genotyping array in the case and control group, which caused several SNPs to be falsely highlighted as associated with longevity (66).  The study was subsequently retracted (67), but a modified manuscript was later published (68).


#### Limitations - fundamental problems (READ THIS ECTION)

**READ THIS SECTION** 

The fundamental assumptions of GWAS and have attracted fundamental criticism, mainly because of their assumption that common genetic variation plays a large role in explaining the heritable variation of common disease (69). Indeed, it has been estimated that for most conditions the SNP heritability attributable to common SNPs is <0.05 (70). This aspect of GWAS has attracted the criticism that, although it could not have been known prospectively, GWAS were ultimately not worth the expenditure (48). GWAS also face criticism that the broad variation of individual responses or compensatory mechanisms to a disease state cancel out and mask potential genes or causal variants associated with the disease (71).  Additionally, GWAS identify candidate risk variants for the population from which their analysis is performed, and with most GWAS stemming from European databases, there is a lack of translation of the identified risk variants to other non-European populations (72] Alternative strategies suggested involve linkage analysis (73, 74] More recently, the rapidly decreasing price of complete genome sequencing have also provided a realistic alternative to genotyping array-based GWAS. It can be discussed if the use of this new technique is still referred to as a GWA study, but high-throughput sequencing does have potential to side-step some of the shortcomings of non-sequencing GWA (75]


## Fine-mapping

**THIS SECTION IS OPTIONAL**

Genotyping arrays designed for GWAS rely on linkage disequilibrium to provide coverage of the entire genome by genotyping a subset of variants. Because of this, the reported associated variants are unlikely to be the actual causal variants. Associated regions can contain hundreds of variants spanning large regions and encompassing many different genes, making the biological interpretation of GWAS loci more difficult. Fine-mapping is a process to refine these lists of associated variants to a credible set most likely to include the causal variant.

Fine-mapping requires all variants in the associated region to have been genotyped or imputed (dense coverage), very stringent quality control resulting in high-quality genotypes, and large sample sizes sufficient in separating out highly correlated signals. There are several different methods to perform fine-mapping, and all methods produce a posterior probability that a variant in that locus is causal. Because the requirements are often difficult to satisfy, there are still limited examples of these methods being more generally applied.

<!-- See also -->


<!-- Association mapping -->
<!-- Epidemiology -->
<!-- Gene–environment interaction -->
<!-- Genomics -->
<!-- Linkage disequilibrium -->
<!-- Molecular epidemiology -->
<!-- Polygenic score -->
