## Protein Data Bank

Adapated from  Wikipedia

The Protein Data Bank (PDB)[1] is a database for the three-dimensional structural data of large biological molecules, such as proteins and nucleic acids. The data, typically obtained by **X-ray crystallography**, NMR spectroscopy, or, increasingly, cryo-electron microscopy, and submitted by biologists and biochemists from around the world, are freely accessible on the Internet via the websites of its member organisations (PDBe,[2] PDBj,[3] RCSB,[4] and BMRB[5]). 

The PDB is a key in areas of structural biology, such as structural genomics. Most major scientific journals and some funding agencies now require scientists to submit their structure data to the PDB. Many other databases use protein structures deposited in the PDB. For example, SCOP and CATH classify protein structures, while PDBsum provides a graphic overview of PDB entries using information from other sources, such as Gene ontology.[6, 7]

Most structures are determined by X-ray diffraction, but about 10% of structures are determined by protein NMR. When using X-ray diffraction, approximations of the coordinates of the atoms of the protein are obtained, whereas using NMR, the distance between pairs of atoms of the protein is estimated. The final conformation of the protein is obtained from NMR by solving a distance geometry problem. After 2013, a growing number of proteins are determined by cryo-electron microscopy. Clicking on the numbers in the linked external table displays examples of structures determined by that method.

Historically, the number of structures in the PDB has grown at an approximately exponential rate, with 100 registered structures in 1982, 1,000 structures in 1993, 10,000 in 1999, and 100,000 in 2014.[19, 20]

### History

** OPTIONAL **

Two forces converged to initiate the PDB: a small but growing collection of sets of protein structure data determined by X-ray diffraction; and the newly available (1968) molecular graphics display, the Brookhaven RAster Display (BRAD), to visualize these protein structures in 3-D. In 1969, with the sponsorship of Walter Hamilton at the Brookhaven National Laboratory, Edgar Meyer (Texas A&M University) began to write software to store atomic coordinate files in a common format to make them available for geometric and graphical evaluation. By 1971, one of Meyer's programs, SEARCH, enabled researchers to remotely access information from the database to study protein structures offline.[8] SEARCH was instrumental in enabling networking, thus marking the functional beginning of the PDB.

The Protein Data Bank was announced in October 1971 in Nature New Biology[9] as a joint venture between Cambridge Crystallographic Data Centre, UK and Brookhaven National Laboratory, US.

Upon Hamilton's death in 1973, Tom Koeztle took over direction of the PDB for the subsequent 20 years. In January 1994, Joel Sussman of Israel's Weizmann Institute of Science was appointed head of the PDB. In October 1998,[10] the PDB was transferred to the Research Collaboratory for Structural Bioinformatics (RCSB);[11] the transfer was completed in June 1999. The new director was Helen M. Berman of Rutgers University (one of the managing institutions of the RCSB, the other being the San Diego Supercomputer Center at UC San Diego).[12] In 2003, with the formation of the wwPDB, the PDB became an international organization. The founding members are PDBe (Europe),[2] RCSB (USA), and PDBj (Japan).[3] The BMRB[5] joined in 2006. Each of the four members of wwPDB can act as deposition, data processing and distribution centers for PDB data. The data processing refers to the fact that wwPDB staff review and annotate each submitted entry.[13] The data are then automatically checked for plausibility (the source code[14] for this validation software has been made available to the public at no charge).

For PDB structures determined by X-ray diffraction that have a structure factor file, their electron density map may be viewed. The data of such structures is stored on the "electron density server".[17, 18]



### File format

**OPTIONAL** 
The file format initially used by the PDB was called the PDB file format. The original format was restricted by the width of computer punch cards to 80 characters per line. Around 1996, the "macromolecular Crystallographic Information file" format, mmCIF, which is an extension of the CIF format was phased in. mmCIF became the standard format for the PDB archive in 2014.[21] In 2019, the wwPDB announced that depositions for crystallographic methods would only be accepted in mmCIF format.[22]

An XML version of PDB, called PDBML, was described in 2005.[23] The structure files can be downloaded in any of these three formats, though an increasing number of structures do not fit the legacy PDB format. Individual files are easily downloaded into graphics packages from Internet URLs:

<!-- For PDB format files, use, e.g., http://www.pdb.org/pdb/files/4hhb.pdb.gz or http://pdbe.org/download/4hhb -->
<!-- For PDBML (XML) files, use, e.g., http://www.pdb.org/pdb/files/4hhb.xml.gz or http://pdbe.org/pdbml/4hhb -->
<!-- The "4hhb" is the PDB identifier. Each structure published in PDB receives a four-character alphanumeric identifier, its PDB ID. (This is not a unique identifier for biomolecules, because several structures for the same molecule—in different environments or conformations—may be contained in PDB with different PDB IDs.) -->

<!-- ### Viewing the data -->

<!-- The structure files may be viewed using one of several free and open source computer programs, including Jmol, Pymol, VMD, and Rasmol. Other non-free, shareware programs include ICM-Browser,[24] MDL Chime, UCSF Chimera, Swiss-PDB Viewer,[25] StarBiochem[26] (a Java-based interactive molecular viewer with integrated search of protein databank), Sirius, and VisProt3DS[27] (a tool for Protein Visualization in 3D stereoscopic view in anaglyph and other modes), and Discovery Studio. The RCSB PDB website contains an extensive list of both free and commercial molecule visualization programs and web browser plugins. -->

<!-- See also -->
<!-- Crystallographic database -->
<!-- Protein structure -->
<!-- Protein structure prediction -->
<!-- Protein structure database -->
<!-- PDBREPORT lists all anomalies (also errors) in PDB structures -->
<!-- PDBsum—extracts data from other databases about PDB structures -->
<!-- Proteopedia—a collaborative 3D encyclopedia of proteins and other molecules -->
<!-- References -->
<!--  wwPDB, Consortium (2019). "Protein Data Bank: the single global archive for 3D macromolecular structure data". Nucleic Acids Res. 47 (D1): 520–528. doi:10.1093/nar/gky949. PMC 6324056. PMID 30357364. -->
<!--  "PDBe home < Node < EMBL-EBI". pdbe.org. -->
<!--  "Protein Data Bank Japan - PDB Japan - PDBj". pdbj.org. -->
<!--  Bank, RCSB Protein Data. "RCSB PDB: Homepage". rcsb.org. -->
<!--  "Biological Magnetic Resonance Bank". bmrb.wisc.edu. -->
<!--  Berman, H. M. (January 2008). "The Protein Data Bank: a historical perspective" (PDF). Acta Crystallographica Section A. A64 (1): 88–95. doi:10.1107/S0108767307035623. PMID 18156675. -->
<!--  Laskowski RA, Hutchinson EG, Michie AD, Wallace AC, Jones ML, Thornton JM (December 1997). "PDBsum: a Web-based database of summaries and analyses of all PDB structures". Trends Biochem. Sci. 22 (12): 488–90. doi:10.1016/S0968-0004(97)01140-7. PMID 9433130. -->
<!--  Meyer EF (1997). "The first years of the Protein Data Bank". Protein Science. Cambridge University Press. 6 (7): 1591–1597. doi:10.1002/pro.5560060724. PMC 2143743. PMID 9232661. -->
<!--  "Protein Data Bank". Nature New Biology. 1971. doi:10.1038/newbio233223b0. -->
<!--  Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, Shindyalov IN, Bourne PE (January 2000). "The Protein Data Bank". Nucleic Acids Res. 28 (1): 235–242. doi:10.1093/nar/28.1.235. PMC 102472. PMID 10592235. -->
<!--  "Research Collaboratory for Structural Bioinformatics". RCSB.org. Research Collaboratory for Structural Bioinformatics. Archived from the original on 2007-02-05. -->
<!--  "RCSB PDB Newsletter Archive". RCSB Protein Data Bank. -->
<!--  Curry E, Freitas A, O'Riáin S (2010). "The Role of Community-Driven Data Curation for Enterprises". In D. Wood (ed.). Linking Enterprise Data. Boston: Springer US. pp. 25–47. ISBN 978-1-441-97664-2. -->
<!--  "PDB Validation Suite". sw-tools.pdb.org. -->
<!--  Burley SK, Berman HM, Bhikadiya C, Bi C, Chen L, Costanzo LD, et al. (wwPDB consortium) (January 2019). "Protein Data Bank: the single global archive for 3D macromolecular structure data". Nucleic Acids Research. 47 (D1): D520–D528. doi:10.1093/nar/gky949. PMC 6324056. PMID 30357364. -->
<!--  "PDB Current Holdings Breakdown". RCSB. -->
<!--  "The Uppsala Electron Density Server". Uppsala University. Retrieved 2013-04-06. -->
<!--  Kleywegt GJ, Harris MR, Zou JY, Taylor TC, Wählby A, Jones TA (Dec 2004). "The Uppsala Electron-Density Server". Acta Crystallogr D. 60 (Pt 12 Pt 1): 2240–2249. doi:10.1107/S0907444904013253. PMID 15572777. -->
<!--  Anon (2014). "Hard data: It has been no small feat for the Protein Data Bank to stay relevant for 100,000 structures". Nature. 509 (7500): 260. doi:10.1038/509260a. PMID 24834514. -->
<!--  "Content Growth Report". RCSB PDB. Archived from the original on 2007-04-28. Retrieved 2013-04-06. -->
<!--  "wwPDB: File Formats and the PDB". wwpdb.org. Retrieved April 1, 2020. -->
<!--  wwPDB.org. "wwPDB: 2019 News". wwpdb.org. -->
<!--  Westbrook J, Ito N, Nakamura H, Henrick K, Berman HM (April 2005). "PDBML: the representation of archival macromolecular structure data in XML". Bioinformatics. 21 (7): 988–992. doi:10.1093/bioinformatics/bti082. PMID 15509603. -->
<!--  "ICM-Browser". Molsoft L.L.C. Retrieved 2013-04-06. -->
<!--  "Swiss PDB Viewer". Swiss Institute of Bioinformatics. Retrieved 2013-04-06. -->
<!--  "STAR: Biochem - Home". web.mit.edu. -->
<!--  "VisProt3DS". Molecular Systems Ltd. Retrieved 2013-04-06. -->
<!-- External links -->
<!-- 	Wikidata has the property: -->
<!-- Disc Plain blue dark.svg PDB structure ID (P638) (see uses) -->
<!-- The Worldwide Protein Data Bank (wwPDB)—parent site to regional hosts (below) -->
<!-- RCSB Protein Data Bank (USA) -->
<!-- PDBe (Europe) -->
<!-- PDBj (Japan) -->
<!-- BMRB, Biological Magnetic Resonance Data Bank (USA) -->
<!-- wwPDB Documentation—documentation on both the PDB and PDBML file formats -->
<!-- Looking at Structures—The RCSB's introduction to crystallography -->
<!-- PDBsum Home Page—Extracts data from other databases about PDB structures. -->
<!-- Nucleic Acid Database, NDB—a PDB mirror especially for searching for nucleic acids -->
<!-- Introductory PDB tutorial sponsored by PDB -->