<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title> 2 Clade | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</title>
  <meta name="description" content="A collection of open-access and/or free resources on biology and environmental science." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content=" 2 Clade | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content=" 2 Clade | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  
  <meta name="twitter:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

<meta name="author" content="Compiled and edited by Nathan Brouwer" />


<meta name="date" content="2021-09-03" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="when-things-are-homolgous-and-analogous.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Biology for everyone</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#alignments"><i class="fa fa-check"></i><b>1.1</b> Alignments</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#talking-glossary-allele-0.5-min"><i class="fa fa-check"></i><b>1.2</b> Talking Glossary: Allele (0.5 min)</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#talking-glossary-bioinformatics"><i class="fa fa-check"></i><b>1.3</b> Talking glossary: Bioinformatics</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="clade.html"><a href="clade.html"><i class="fa fa-check"></i><b>2</b> Clade</a>
<ul>
<li class="chapter" data-level="2.1" data-path="clade.html"><a href="clade.html#clades-and-phylogenetic-trees"><i class="fa fa-check"></i><b>2.1</b> Clades and phylogenetic trees</a></li>
<li class="chapter" data-level="2.2" data-path="clade.html"><a href="clade.html#terminology"><i class="fa fa-check"></i><b>2.2</b> Terminology</a></li>
<li class="chapter" data-level="2.3" data-path="clade.html"><a href="clade.html#in-popular-culture"><i class="fa fa-check"></i><b>2.3</b> In popular culture</a></li>
<li class="chapter" data-level="2.4" data-path="clade.html"><a href="clade.html#references"><i class="fa fa-check"></i><b>2.4</b> References</a></li>
<li class="chapter" data-level="2.5" data-path="clade.html"><a href="clade.html#talking-glossary-cloning-0.5-min"><i class="fa fa-check"></i><b>2.5</b> Talking Glossary: Cloning (0.5 min)</a></li>
<li class="chapter" data-level="2.6" data-path="clade.html"><a href="clade.html#talking-glossary-codominance-0.5-min"><i class="fa fa-check"></i><b>2.6</b> Talking Glossary: Codominance (0.5 min)</a></li>
<li class="chapter" data-level="2.7" data-path="clade.html"><a href="clade.html#comparative-genomics"><i class="fa fa-check"></i><b>2.7</b> Comparative genomics</a></li>
<li class="chapter" data-level="2.8" data-path="clade.html"><a href="clade.html#talking-glossary-contig-0.75-min"><i class="fa fa-check"></i><b>2.8</b> Talking Glossary: Contig (0.75 min)</a></li>
<li class="chapter" data-level="2.9" data-path="clade.html"><a href="clade.html#talking-glossary---foundations-1-review-crossing-over"><i class="fa fa-check"></i><b>2.9</b> Talking Glossary - Foundations 1 Review: Crossing over</a></li>
<li class="chapter" data-level="2.10" data-path="clade.html"><a href="clade.html#talking-glossary-deletion-mutation-1.5-min"><i class="fa fa-check"></i><b>2.10</b> Talking Glossary: Deletion mutation (1.5 min)</a></li>
<li class="chapter" data-level="2.11" data-path="clade.html"><a href="clade.html#talking-glossary-dna-sequencing-0.75-min"><i class="fa fa-check"></i><b>2.11</b> Talking Glossary: DNA sequencing (0.75 min)</a></li>
<li class="chapter" data-level="2.12" data-path="clade.html"><a href="clade.html#sex-chromosome-dosage-compensation"><i class="fa fa-check"></i><b>2.12</b> Sex-Chromosome Dosage compensation</a>
<ul>
<li class="chapter" data-level="2.12.1" data-path="clade.html"><a href="clade.html#random-inactivation-of-one-x-in-xx-individuals"><i class="fa fa-check"></i><b>2.12.1</b> Random inactivation of one X in XX individuals</a></li>
<li class="chapter" data-level="2.12.2" data-path="clade.html"><a href="clade.html#two-fold-increased-transcription-of-a-single-x-in-xy-individuals"><i class="fa fa-check"></i><b>2.12.2</b> Two-fold increased transcription of a single X in XY individuals</a></li>
</ul></li>
<li class="chapter" data-level="2.13" data-path="clade.html"><a href="clade.html#talking-glossary-duplication-0.75-min"><i class="fa fa-check"></i><b>2.13</b> Talking Glossary: Duplication (0.75 min)</a></li>
<li class="chapter" data-level="2.14" data-path="clade.html"><a href="clade.html#talking-glossary-epigenetics"><i class="fa fa-check"></i><b>2.14</b> Talking Glossary: Epigenetics</a></li>
<li class="chapter" data-level="2.15" data-path="clade.html"><a href="clade.html#talking-glossary-founder-effect-a-type-of-genetic-bottleneck-1-min"><i class="fa fa-check"></i><b>2.15</b> Talking glossary: Founder effect (A type of genetic bottleneck; 1 min)</a></li>
<li class="chapter" data-level="2.16" data-path="clade.html"><a href="clade.html#talking-glossary-frameshift-mutation-1.5-min"><i class="fa fa-check"></i><b>2.16</b> Talking Glossary: Frameshift mutation (1.5 min)</a></li>
<li class="chapter" data-level="2.17" data-path="clade.html"><a href="clade.html#talking-glossary-u03-gel-electrophoresis-3-min"><i class="fa fa-check"></i><b>2.17</b> Talking Glossary U03: Gel Electrophoresis (“3 min)</a></li>
<li class="chapter" data-level="2.18" data-path="clade.html"><a href="clade.html#talking-glossary-gene-amplification-1.5-min"><i class="fa fa-check"></i><b>2.18</b> Talking Glossary: Gene amplification (1.5 min)</a></li>
<li class="chapter" data-level="2.19" data-path="clade.html"><a href="clade.html#talking-glossary-gene-pool-0.75-min"><i class="fa fa-check"></i><b>2.19</b> Talking Glossary: Gene Pool (0.75 min)</a></li>
<li class="chapter" data-level="2.20" data-path="clade.html"><a href="clade.html#gene-prediction"><i class="fa fa-check"></i><b>2.20</b> Gene prediction</a></li>
<li class="chapter" data-level="2.21" data-path="clade.html"><a href="clade.html#genbank"><i class="fa fa-check"></i><b>2.21</b> GenBank</a></li>
<li class="chapter" data-level="2.22" data-path="clade.html"><a href="clade.html#genetic-distance"><i class="fa fa-check"></i><b>2.22</b> Genetic distance</a></li>
<li class="chapter" data-level="2.23" data-path="clade.html"><a href="clade.html#talking-glossary-genetic-drift-0.75-min"><i class="fa fa-check"></i><b>2.23</b> Talking Glossary: Genetic drift (0.75 min)</a></li>
<li class="chapter" data-level="2.24" data-path="clade.html"><a href="clade.html#talking-glossary-genetic-engineering-1.5-min"><i class="fa fa-check"></i><b>2.24</b> Talking Glossary: Genetic engineering (1.5 min)</a></li>
<li class="chapter" data-level="2.25" data-path="clade.html"><a href="clade.html#genome-assembly"><i class="fa fa-check"></i><b>2.25</b> Genome assembly</a></li>
<li class="chapter" data-level="2.26" data-path="clade.html"><a href="clade.html#talking-glossary-genomics"><i class="fa fa-check"></i><b>2.26</b> Talking Glossary: Genomics</a></li>
<li class="chapter" data-level="2.27" data-path="clade.html"><a href="clade.html#talking-glossary-genotype"><i class="fa fa-check"></i><b>2.27</b> Talking Glossary: Genotype</a></li>
<li class="chapter" data-level="2.28" data-path="clade.html"><a href="clade.html#talking-glossary-germline-1.25-min"><i class="fa fa-check"></i><b>2.28</b> Talking Glossary: Germline (1.25 min)</a></li>
<li class="chapter" data-level="2.29" data-path="clade.html"><a href="clade.html#guild"><i class="fa fa-check"></i><b>2.29</b> Guild</a></li>
<li class="chapter" data-level="2.30" data-path="clade.html"><a href="clade.html#talking-glossary-haploid-1.25-min"><i class="fa fa-check"></i><b>2.30</b> Talking Glossary: Haploid (1.25 min)</a></li>
<li class="chapter" data-level="2.31" data-path="clade.html"><a href="clade.html#talking-glossary-heterozygous"><i class="fa fa-check"></i><b>2.31</b> Talking Glossary: Heterozygous</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html"><i class="fa fa-check"></i><b>3</b> When things are homolgous AND analogous!</a>
<ul>
<li class="chapter" data-level="3.1" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-homozygous"><i class="fa fa-check"></i><b>3.1</b> Talking Glossary: Homozygous</a></li>
<li class="chapter" data-level="3.2" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-insertion-mutation-1.25-min"><i class="fa fa-check"></i><b>3.2</b> Talking Glossary: Insertion mutation (1.25 min)</a></li>
<li class="chapter" data-level="3.3" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#jove-videos"><i class="fa fa-check"></i><b>3.3</b> JoVe Videos</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#biotechnology"><i class="fa fa-check"></i><b>3.3.1</b> Biotechnology</a></li>
<li class="chapter" data-level="3.3.2" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#macromolecules"><i class="fa fa-check"></i><b>3.3.2</b> Macromolecules</a></li>
<li class="chapter" data-level="3.3.3" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#cellular-respiration"><i class="fa fa-check"></i><b>3.3.3</b> Cellular respiration</a></li>
<li class="chapter" data-level="3.3.4" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#population-and-community-ecology"><i class="fa fa-check"></i><b>3.3.4</b> Population and community ecology</a></li>
<li class="chapter" data-level="3.3.5" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#population-genetics"><i class="fa fa-check"></i><b>3.3.5</b> Population genetics</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-locus-1-min"><i class="fa fa-check"></i><b>3.4</b> Talking Glossary: Locus (1 min)</a></li>
<li class="chapter" data-level="3.5" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-missense-mutation-1.25-min"><i class="fa fa-check"></i><b>3.5</b> Talking Glossary: Missense Mutation (1.25 min)</a></li>
<li class="chapter" data-level="3.6" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-mitochondrial-dna-1.5-min"><i class="fa fa-check"></i><b>3.6</b> Talking Glossary: Mitochondrial DNA (1.5 min)</a></li>
<li class="chapter" data-level="3.7" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-mutagen-0.5-min"><i class="fa fa-check"></i><b>3.7</b> Talking Glossary: Mutagen (0.5 min)</a></li>
<li class="chapter" data-level="3.8" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-mutation-0.5-min"><i class="fa fa-check"></i><b>3.8</b> Talking Glossary: Mutation (0.5 min)</a></li>
<li class="chapter" data-level="3.9" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-non-coding-dna-1-min"><i class="fa fa-check"></i><b>3.9</b> Talking Glossary: Non-coding DNA (1 min)</a></li>
<li class="chapter" data-level="3.10" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-nonsene-mutation"><i class="fa fa-check"></i><b>3.10</b> Talking Glossary: Nonsene mutation</a></li>
<li class="chapter" data-level="3.11" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-open-reading-frame-definition-3-min"><i class="fa fa-check"></i><b>3.11</b> Talking Glossary : Open reading frame definition (3 min)</a></li>
<li class="chapter" data-level="3.12" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-pcr---the-polymerase-chain-reaction-0.5-min"><i class="fa fa-check"></i><b>3.12</b> Talking Glossary: PCR - The Polymerase Chain Reaction (0.5 min)</a></li>
<li class="chapter" data-level="3.13" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-phenotype"><i class="fa fa-check"></i><b>3.13</b> Talking Glossary: Phenotype</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html"><i class="fa fa-check"></i><b>4</b> Phylogenetics vocab</a>
<ul>
<li class="chapter" data-level="4.1" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-plasmid-1-min"><i class="fa fa-check"></i><b>4.1</b> Talking Glossary: Plasmid (1 min)</a></li>
<li class="chapter" data-level="4.2" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-point-mutation-0.5-min"><i class="fa fa-check"></i><b>4.2</b> Talking Glossary: Point mutation (0.5 min)</a></li>
<li class="chapter" data-level="4.3" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-polymorphism-1-min"><i class="fa fa-check"></i><b>4.3</b> Talking Glossary: Polymorphism (1 min)</a></li>
<li class="chapter" data-level="4.4" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-u03-primer-0.5-min"><i class="fa fa-check"></i><b>4.4</b> Talking Glossary U03: Primer (0.5 min)</a></li>
<li class="chapter" data-level="4.5" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-promoter-1.5-min"><i class="fa fa-check"></i><b>4.5</b> Talking Glossary: Promoter (1.5 min)</a></li>
<li class="chapter" data-level="4.6" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-recessive"><i class="fa fa-check"></i><b>4.6</b> Talking Glossary: Recessive</a></li>
<li class="chapter" data-level="4.7" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-recombinant-dna-0.5-min"><i class="fa fa-check"></i><b>4.7</b> Talking Glossary: Recombinant DNA (0.5 min)</a></li>
<li class="chapter" data-level="4.8" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-repressor-1-min"><i class="fa fa-check"></i><b>4.8</b> Talking Glossary: Repressor (1 min)</a></li>
<li class="chapter" data-level="4.9" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-restriction-enzyme-0.5-min"><i class="fa fa-check"></i><b>4.9</b> Talking Glossary: Restriction Enzyme (0.5 min)</a></li>
<li class="chapter" data-level="4.10" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-u03-shotgun-sequencing-1-min"><i class="fa fa-check"></i><b>4.10</b> Talking Glossary U03: Shotgun sequencing (1 min)</a></li>
<li class="chapter" data-level="4.11" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-sickle-cell-disease-4-min"><i class="fa fa-check"></i><b>4.11</b> Talking Glossary: Sickle Cell Disease (4 min)</a></li>
<li class="chapter" data-level="4.12" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-somatic-cells-0.75-min"><i class="fa fa-check"></i><b>4.12</b> Talking glossary: Somatic cells (0.75 min)</a></li>
<li class="chapter" data-level="4.13" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#sequence-annotation"><i class="fa fa-check"></i><b>4.13</b> Sequence annotation</a>
<ul>
<li class="chapter" data-level="4.13.1" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#process"><i class="fa fa-check"></i><b>4.13.1</b> Process</a></li>
</ul></li>
<li class="chapter" data-level="4.14" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-substitution-mutation-1.75-min"><i class="fa fa-check"></i><b>4.14</b> Talking Glossary: Substitution mutation (1.75 min)</a></li>
<li class="chapter" data-level="4.15" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#taxa-taxon-and-clades-a-brief-primer"><i class="fa fa-check"></i><b>4.15</b> Taxa, Taxon, and clades: A Brief Primer</a></li>
<li class="chapter" data-level="4.16" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#what-is-a-taxon"><i class="fa fa-check"></i><b>4.16</b> What is a “taxon”</a></li>
<li class="chapter" data-level="4.17" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#what-is-a-clade"><i class="fa fa-check"></i><b>4.17</b> What is a clade?</a></li>
<li class="chapter" data-level="4.18" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#further-reading"><i class="fa fa-check"></i><b>4.18</b> Further Reading:</a></li>
<li class="chapter" data-level="4.19" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-trait-0.75-min"><i class="fa fa-check"></i><b>4.19</b> Talking Glossary: Trait (0.75 min)</a></li>
<li class="chapter" data-level="4.20" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-vector-2-min"><i class="fa fa-check"></i><b>4.20</b> Talking Glossary: Vector (2 min)</a></li>
<li class="chapter" data-level="4.21" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-x-chromosome-1.25-min"><i class="fa fa-check"></i><b>4.21</b> Talking Glossary: X-chromosome (1.25 min)</a></li>
<li class="chapter" data-level="4.22" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-x-inactivation-lyonization-1.45-min"><i class="fa fa-check"></i><b>4.22</b> Talking Glossary: X-inactivation (“Lyonization”; 1.45 min))</a></li>
<li class="chapter" data-level="4.23" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#additional-glossaries"><i class="fa fa-check"></i><b>4.23</b> Additional Glossaries</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="clade" class="section level1" number="2">
<h1><span class="header-section-number"> 2</span> Clade</h1>
<p>Adapted from Wikipedia
<a href="https://en.wikipedia.org/wiki/Clade" class="uri">https://en.wikipedia.org/wiki/Clade</a></p>
<p>A clade, also known as a monophyletic group, is a group of organisms that are monophyletic —that is, composed of a common ancestor and all its descendants .[4] . The common ancestor may be an individual, a population , a species (extinct or extant ), and so on. Clades are nested, one in another, as each branch in turn splits into smaller branches. These splits reflect evolutionary history as populations diverged and evolved independently. The term “clade” is also used with a similar meaning in other fields besides biology, such as historical linguistics.</p>
<p>A clade is by definition monophyletic , meaning that it contains one ancestor (which can be an organism, a population, or a species) and all its descendants.[note 1] [12] [13] The ancestor can be known or unknown; any and all members of a clade can be extant or extinct.</p>
<div class="figure"><span id="fig:unnamed-chunk-1"></span>
<iframe src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/46/Clade-grade_II.svg/640px-Clade-grade_II.svg.png" width="672" height="400px">
</iframe>
<p class="caption">
Figure 2.1: Figure 1: Cladogram (family tree) of a biological group. The last common ancestor is the vertical line stem at the bottom. The blue and red subgroups are clades; each shows its common ancestor stem at the bottom of the subgroup branch. The green subgroup is not a clade; it is a non-monophyletic group , because it excludes the blue branch, even though it has also descended from a common ancestor. The green subgroup together with the blue one forms a clade again.
</p>
</div>
<p>Over the last few decades, the cladistic approach has revolutionized biological classification and revealed surprising evolutionary relationships among organisms.[5] Taxonomists have worked to make the taxonomic system reflect evolution. Taxonomists therefore try to avoid naming taxa that are not clades; that is, taxa that are not monophyletic . Some of the relationships between organisms that the molecular biology arm of cladistics has revealed are that fungi are closer relatives to animals than they are to plants, archaea are now considered different from bacteria , and multicellular organisms may have evolved from archaea.[6]</p>
<div class="figure"><span id="fig:unnamed-chunk-2"></span>
<iframe src="https://upload.wikimedia.org/wikipedia/commons/1/11/Cladogram_Crocodilia_NL.PNG" width="672" height="400px">
</iframe>
<p class="caption">
Figure 2.2: Gavialidae, Crocodylidae and Alligatoridae are clade names that are here applied to a phylogenetic tree of crocodylians.
</p>
</div>
<p>Many commonly named groups, rodents and insects for example, are clades because, in each case, the group consists of a common ancestor with all its descendant branches. Rodents, for example, are a branch of mammals that split off after the end of the period when the clade Dinosauria stopped being the dominant terrestrial vertebrates 66 million years ago. The original population and all its descendants are a clade. The rodent clade corresponds to the order Rodentia, and insects to the class Insecta. These clades include smaller clades, such as chipmunk or ant , each of which consists of even smaller clades. The clade “rodent” is in turn included in the mammal, vertebrate and animal clades.</p>
<div id="clades-and-phylogenetic-trees" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Clades and phylogenetic trees</h2>
<p>The science that tries to reconstruct phylogenetic trees and thus discover clades is called phylogenetics . The results of phylogenetic analyses are tree-shaped diagrams called phylogenies; they, and all their branches, are phylogenetic hypotheses.[14]</p>
</div>
<div id="terminology" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Terminology</h2>
<p>The relationship between clades can be described in several ways:</p>
<ol style="list-style-type: decimal">
<li>A clade located within a clade is said to be nested within that clade. In the diagram, the hominoid clade, i.e. the apes and humans, is nested within the primate clade.</li>
<li>Two clades are sisters (sister groups sister clades) if they have an immediate common ancestor. In the diagram, lemurs and lorises are sister clades, while humans and tarsiers are not.</li>
</ol>
<div class="figure"><span id="fig:unnamed-chunk-3"></span>
<iframe src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5e/Primate_cladogram.svg/556px-Primate_cladogram.svg.png" width="672" height="400px">
</iframe>
<p class="caption">
Figure 2.3: Cladogram of modern primate groups; all tarsiers are haplorhines, but not all haplorhines are tarsiers; all apes are catarrhines, but not all catarrhines are apes; etc.
</p>
</div>
</div>
<div id="in-popular-culture" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> In popular culture</h2>
<p>An episode of <a href="https://en.wikipedia.org/wiki/Elementary_(TV_series)">Elementary</a> is titled “<a href="https://en.wikipedia.org/wiki/List_of_Elementary_episodes#ep38">Dead Clade Walking</a>” and deals with a case involving a rare fossil.</p>
</div>
<div id="references" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> References</h2>
<ol style="list-style-type: decimal">
<li>Wells, John C. (2008). Longman Pronunciation Dictionary (3rd ed.). Longman. ISBN 978-1-4058-8118-0. “clade”. Merriam-Webster Dictionary. Retrieved 19 April 2020.
Martin, Elizabeth; Hin, Robert (2008). A Dictionary of Biology. Oxford University Press.</li>
<li>Cracraft, Joel; Donoghue, Michael J., eds. (2004). “Introduction”. Assembling the Tree of Life. Oxford University Press. p. 1. ISBN 978-0-19-972960-9.</li>
<li>Palmer, Douglas (2009). Evolution: The Story of Life. Berkeley: University of California Press. p. 13.</li>
<li>Pace, Norman R. (18 May 2006). “Time for a change”. Nature. 441 (7091): 289. Bibcode:2006Natur.441..289P. <a href="doi:10.1038/441289a" class="uri">doi:10.1038/441289a</a>. ISSN 1476-4687. PMID 16710401. S2CID 4431143.</li>
<li>Dupuis, Claude (1984). “Willi Hennig’s impact on taxonomic thought”. Annual Review of Ecology and Systematics. 15: 1–24. <a href="doi:10.1146/annurev.es.15.110184.000245" class="uri">doi:10.1146/annurev.es.15.110184.000245</a>.</li>
<li>Huxley, J. S. (1957). “The three types of evolutionary process”. Nature. 180 (4584): 454–455. Bibcode:1957Natur.180..454H. <a href="doi:10.1038/180454a0" class="uri">doi:10.1038/180454a0</a>. S2CID 4174182.</li>
<li>Huxley, T.H. (1876): Lectures on Evolution. New York Tribune. Extra. no 36. In Collected Essays IV: pp 46-138 original text w/ figures</li>
<li>Brower, Andrew V. Z. (2013). “Willi Hennig at 100”. Cladistics. 30 (2): 224–225. <a href="doi:10.1111/cla.12057" class="uri">doi:10.1111/cla.12057</a>.</li>
<li>”Evolution 101”. page 10. Understanding Evolution website. University of California, Berkeley. Retrieved 26 February 2016.</li>
<li>“International Code of Phylogenetic Nomenclature. Version 4c. Chapter I. Taxa”. 2010. Retrieved 22 September 2012.</li>
<li>Envall, Mats (2008). “On the difference between mono-, holo-, and paraphyletic groups: a consistent distinction of process and pattern”. Biological Journal of the Linnean Society. 94: 217. <a href="doi:10.1111/j.1095-8312.2008.00984.x" class="uri">doi:10.1111/j.1095-8312.2008.00984.x</a>.</li>
<li>Nixon, Kevin C.; Carpenter, James M. (1 September 2000). “On the Other”Phylogenetic Systematics”“. Cladistics. 16 (3): 298–318. <a href="doi:10.1111/j.1096-0031.2000.tb00285.x" class="uri">doi:10.1111/j.1096-0031.2000.tb00285.x</a>. S2CID 73530548.</li>
</ol>

</div>
<div id="talking-glossary-cloning-0.5-min" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> Talking Glossary: Cloning (0.5 min)</h2>
<p>National Human Genome Research Institute
<a href="https://www.genome.gov/genetics-glossary/Cloning" class="uri">https://www.genome.gov/genetics-glossary/Cloning</a></p>
<p><strong>Introduction</strong>: “Cloning is the process of making identical copies of an organism, cell, or DNA sequence. Molecular cloning is a process by which scientists amplify a desired DNA sequence. The target sequence is isolated, inserted into another DNA molecule (known as a vector), and introduced into a suitable host cell. Then, each time the host cell divides, it replicates the foreign DNA sequence along with its own DNA. Cloning also can refer to asexual reproduction.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/cloning.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/cloning.mp3</a></p>
<p><strong>Transcript</strong>: “Cloning is a word we use to describe a molecular process of making millions or billions of copies of a single molecule. It’s different from the uses of the terms”cellular cloning” or “organism cloning” that are used in the reproductive genetics universe. We use molecular cloning to amplify, or make many copies of, genes or proteins or other micro molecules that amplifies the signal and allows us to study these molecules in a laboratory”.</p>
<p>Leslie G. Biesecker, M.D.</p>

</div>
<div id="talking-glossary-codominance-0.5-min" class="section level2" number="2.6">
<h2><span class="header-section-number">2.6</span> Talking Glossary: Codominance (0.5 min)</h2>
<p>Abstract: “Codominance is a relationship between two versions of a gene. Individuals receive one version of a gene, called an allele, from each parent. If the alleles are different, the dominant allele usually will be expressed, while the effect of the other allele, called recessive, is masked. In codominance, however, neither allele is recessive and the phenotypes of both alleles are expressed.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/codominance.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/codominance.mp3</a></p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/codominance.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/codominance.jpg</a>
Transcript</p>
<p>Codominance means that neither allele can mask the expression of the other allele. An example in humans would be the ABO blood group, where alleles A and alleles B are both expressed. So if an individual inherits allele A from their mother and allele B from their father, they have blood type AB.</p>
<p><a href="https://www.genome.gov/staff/Suzanne-Hart-PhD">Suzanne Hart</a>, Ph.D.</p>

</div>
<div id="comparative-genomics" class="section level2" number="2.7">
<h2><span class="header-section-number">2.7</span> Comparative genomics</h2>
<p>Adapted from Wikipedia</p>
<p><a href="https://en.wikipedia.org/wiki/Comparative_genomics" class="uri">https://en.wikipedia.org/wiki/Comparative_genomics</a></p>
<p>Comparative genomics is a field of biological research in which the genomic features of different organisms are compared.[2][3] The genomic features may include the DNA sequence, genes, gene order, regulatory sequences, and other genomic structural landmarks.[3] In this branch of genomics, whole genomes resulting from genome projects are compared to study basic biological similarities and differences as well as evolutionary relationships between organisms.[2][4][5] The major principle of comparative genomics is that common features of two organisms will often be encoded within the DNA that is evolutionarily conserved between them.[6] Therefore, comparative genomic approaches start with making some form of alignment of genome sequences and looking for similar sequences in the aligned genomes and checking to what extent those sequences are conserved. Based on these, genome and molecular evolution are inferred and this may, in turn, be put in the context of, for example, evolution, ecology, or pathogenicity.</p>
<p>Comparative genomics is now a standard component of the analysis of every new genome sequence.[2][8] With the explosion in the number of genome projects due to the advancements in DNA sequencing technologies[9] Comparative genomics has revealed high levels of similarity between closely related organisms, such as humans and chimpanzees, and, more surprisingly, similarity between seemingly distantly related organisms, such as humans and yeast.</p>
<p>The medical field benefits from the study of comparative genomics. Vaccine development in particular has experienced useful advances in technology due to genomic approaches to problems. In an approach known as reverse vaccinology, researchers can discover candidate antigens (proteins in pathogens that can be targeted by the immune system) for vaccine development by analyzing the genome of a pathogen or a family of pathogens.</p>
<p>1* Darling A.E.; Miklós I.; Ragan M.A. (2008). “Dynamics of Genome Rearrangement in Bacterial Populations”. PLOS Genetics. 4 (7): e1000128. <a href="doi:10.1371/journal.pgen.1000128" class="uri">doi:10.1371/journal.pgen.1000128</a>. PMC 2483231. PMID 18650965. open access
1. Touchman, J. (2010). “Comparative Genomics”. Nature Education Knowledge. 3 (10): 13.
1. Xia, X. (2013). Comparative Genomics. SpringerBriefs in Genetics. Heidelberg: Springer. <a href="doi:10.1007/978-3-642-37146-2" class="uri">doi:10.1007/978-3-642-37146-2</a>. ISBN 978-3-642-37145-5. S2CID 5491782.
1. Russel, P.J.; Hertz, P.E.; McMillan, B. (2011). Biology: The Dynamic Science (2nd ed.). Belmont, CA: Brooks/Cole. pp. 409–410.
1. Primrose, S.B.; Twyman, R.M. (2003). Principles of Genome Analysis and Genomics (3rd ed.). Malden, MA: Blackwell Publishing.
1. Hardison, R.C. (2003). “Comparative genomics”. PLOS Biology. 1 (2): e58. <a href="doi:10.1371/journal.pbio.0000058" class="uri">doi:10.1371/journal.pbio.0000058</a>. PMC 261895. PMID 14624258. open access
1. Ellegren, H. (2008). “Comparative genomics and the study of evolution by natural selection”. Molecular Ecology. 17 (21): 4586–4596. <a href="doi:10.1111/j.1365-294X.2008.03954.x" class="uri">doi:10.1111/j.1365-294X.2008.03954.x</a>. PMID 19140982. S2CID 43171654.
1. Koonin, E.V.; Galperin, M.Y. (2003). Sequence - Evolution - Function: Computational approaches in comparative genomics. Dordrecht: Springer Science+Business Media.</p>

</div>
<div id="talking-glossary-contig-0.75-min" class="section level2" number="2.8">
<h2><span class="header-section-number">2.8</span> Talking Glossary: Contig (0.75 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Contig" class="uri">https://www.genome.gov/genetics-glossary/Contig</a></p>
<p><strong>Abstract</strong>: “A contig–from the word”contiguous”–is a series of overlapping DNA sequences used to make a physical map that reconstructs the original DNA sequence of a chromosome or a region of a chromosome. A contig can also refer to one of the DNA sequences used in making such a map.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/contig.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/contig.mp3</a></p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/contig.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/contig.mp3</a></p>
<p><strong>Transcript</strong>: “A chromosome is a very long molecule of DNA. And it is very hard to study it at once, so what researchers do is they break it into smaller pieces and they sequence each one of those individual pieces first, and then they attempt to put it together to reconstruct the original chromosome sequence. A contig is the physical map, which results from putting together several little overlapping bits of DNA into a longer sequence. The contig is the physical map resulting from taking small pieces of DNA that overlap and putting them together into a longer sequence.”</p>
<p><a href="https://www.genome.gov/staff/Belen-Hurle-PhD">Belen Hurle</a>, Ph.D.</p>

</div>
<div id="talking-glossary---foundations-1-review-crossing-over" class="section level2" number="2.9">
<h2><span class="header-section-number">2.9</span> Talking Glossary - Foundations 1 Review: Crossing over</h2>
<p>Abstract: “Crossing over is the swapping of genetic material that occurs in the germ line. During the formation of egg and sperm cells, also known as meiosis, paired chromosomes from each parent align so that similar DNA sequences from the paired chromosomes cross over one another. Crossing over results in a shuffling of genetic material and is an important cause of the genetic variation seen among offspring.”</p>
<p>Audio:
<a href="https://www.genome.gov/sites/default/files/tg/en/narration/crossing_over.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/crossing_over.mp3</a></p>
<p><strong>Transcript</strong>: “Crossing over is a biological occurrence that happens during meiosis when the paired homologs, or chromosomes of the same type, are lined up. In meiosis, they’re lined up on the meiotic plates, [as they’re] sometimes called, and those paired chromosomes then have to have some biological mechanism that sort of keeps them together. And it turns out that there are these things called chiasmata, which are actually where strands of the duplicated homologous chromosomes break and recombine with the same strand of the other homolog. So if you have two Chromosome 1s lined up, one strand of one Chromosome 1 will break and it will reanneal with a similar breakage on the other Chromosome 1. So that then the new chromosome that will happen will have part of, say, the maternal Chromosome 1 and the paternal Chromosome 1, where maternal and paternal means where that person got their Chromosomes 1s from, their one or their two. Therefore, the child that’s formed out of one of those Chromosome 1s now has a piece of his or her grandmother’s Chromosome 1 and a piece of his or her grandfather’s Chromosome 1. And it’s this crossing over that lets recombination across generations of genetic material happen, and it also allows us to use that information to find the locations of genes.”</p>
<p><a href="https://www.genome.gov/staff/Joan-E-Bailey-Wilson-PhD">Joan E. Bailey-Wilson</a>, Ph.D</p>
<p>Photo: <a href="https://i.irp.nih.gov/pi/0010100622.jpg" class="uri">https://i.irp.nih.gov/pi/0010100622.jpg</a></p>

</div>
<div id="talking-glossary-deletion-mutation-1.5-min" class="section level2" number="2.10">
<h2><span class="header-section-number">2.10</span> Talking Glossary: Deletion mutation (1.5 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Deletion" class="uri">https://www.genome.gov/genetics-glossary/Deletion</a></p>
<p><strong>Abstract</strong>: “Deletion is a type of mutation involving the loss of genetic material. It can be small, involving a single missing DNA base pair, or large, involving a piece of a chromosome.”</p>
<p>Note: Among evolutionary biologists, “deletion” usually means deletion of a single base; larger deletions are specified, e.g. “chromosomal deletion.” In order to recognize a deletion, there’s needs to be some frame of reference, such as a consensus sequence, a reference genomic sequence, or other individuals in a multiple sequence alignment (MSA). Deletions of all sizes can be useful for constructing phylogenies.</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/deletion.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/deletion.jpg</a>
Example of single base and chromosomal deletion.</p>
<p><strong>Transcript:</strong> “Deletion really means that something is missing. And as a geneticist talking about deletion it means something is missing of the genetic material. And it can be something small, just a base pair; it can be something larger; it can be part of a gene; it can be even larger; it can be an entire gene; or yet larger again, it can be part of the chromosome. And depending upon what it is, you have to look at it in different ways. You can find a deletion in a chromosome just by doing a cytogenetic or chromosome analysis, or a deletion in a gene you can find out by sequencing the DNA. So when you have a deletion, depending upon the size, it can have different effects. What was the most surprising to me was that just by having a deletion of one base pair, you can have the most severe birth defect, and sometimes by missing an entire chromosome, you don’t even see all that much compared to just having a deletion of a small base pair. Different deletions can lead to different findings, and they can affect just behavior; they can affect how a child, how a person looks; they can affect a very severe problem that the child may die at birth; or they can affect something that just has to do with eye color, hair color, with weight or height of the person.”</p>
<p>Maximilian Muenke, M.D.</p>

</div>
<div id="talking-glossary-dna-sequencing-0.75-min" class="section level2" number="2.11">
<h2><span class="header-section-number">2.11</span> Talking Glossary: DNA sequencing (0.75 min)</h2>
<p><strong>Introduction</strong>: “DNA sequencing is a laboratory technique used to determine the exact sequence of bases (A, C, G, and T) in a DNA molecule. The DNA base sequence carries the information a cell needs to assemble protein and RNA molecules. DNA sequence information is important to scientists investigating the functions of genes. The technology of DNA sequencing was made faster and less expensive as a part of the Human Genome Project.”</p>
<p><strong>Transcript:</strong> “DNA consists of a linear string of nucleotides, or bases, for simplicity, referred to by the first letters of their chemical names–A, T, C and G. The process of deducing the order of nucleotides in DNA is called DNA sequencing. Since the DNA sequence confers information that the cell uses to make RNA molecules and proteins, establishing the sequence of DNA is key for understanding how genomes work. The technology for DNA sequencing was made faster and less expensive as a part of the Human Genome Project. And recent developments have profoundly increased the efficiency of DNA sequencing even further.”</p>
<p>Eric D. Green, M.D., Ph.D.</p>

</div>
<div id="sex-chromosome-dosage-compensation" class="section level2" number="2.12">
<h2><span class="header-section-number">2.12</span> Sex-Chromosome Dosage compensation</h2>
<p><a href="https://en.wikipedia.org/wiki/Sex-Chromosome_Dosage_compensation">Adapted from Wikipedia</a></p>
<p>Dosage compensation is the process by which organisms equalize the expression of genes between individuals with different sex chromosome karyotypes (e.g. XX versus XY). Across species, different “sexes” are often characterized by different types and numbers of sex chromosomes . In order to neutralize the large difference in gene dosage produced by differing numbers of sex chromosomes, various evolutionary branches have acquired various methods to equalize gene expression . Because sex chromosomes contain different numbers of genes , different species of organisms have developed different mechanisms to cope with this inequality. Replicating the actual gene is impossible; thus organisms instead equalize the expression from each gene. For example, in humans , XX individuals silence the transcription of one X chromosome of each pair, and transcribe all information from the other, expressed X chromosome. Thus, human XX individuals have the same number of expressed X-linked genes as do human XY individuals, with both having essentially one X chromosome per cell, from which to transcribe and express genes. This is called X-inactivation. In each XX cell, one of the two X chromosomes is randomly inactivated.</p>
<p>Other lineages have evolved different mechanisms to cope with the differences in gene copy numbers between the sexes that are observed on sex chromosomes. Some lineages have evolved upregulation mechanisms which restores expression of X-specific genes in the heterogametic sex (e.g. XY) to the same levels observed in the ancestor prior to the evolution of the sex chromosome.</p>
<div id="random-inactivation-of-one-x-in-xx-individuals" class="section level3" number="2.12.1">
<h3><span class="header-section-number">2.12.1</span> Random inactivation of one X in XX individuals</h3>
<p>One logical way to equalize gene expression amongst XX and XY that follow a XX/XY sex differentiation scheme would be to decrease or altogether eliminate the expression of one of the X chromosomes in an XX, (homogametic) individual, such that both XX and XY individuals express only one X chromosome. This is the case in many mammalian organisms, including humans and mice.[1 This process involves histone tail modifications, DNA methylation patterns, and reorganization of large-scale chromatin structure. In spite of these extensive modifications, not all genes along the X chromosome are subject to X-inactivation.</p>
</div>
<div id="two-fold-increased-transcription-of-a-single-x-in-xy-individuals" class="section level3" number="2.12.2">
<h3><span class="header-section-number">2.12.2</span> Two-fold increased transcription of a single X in XY individuals</h3>
<p>Another mechanism common for achieving equal X-related genetic expression between XX and XY indviduals involves two-fold increased transcription of a single X chromosome in XY individuals. Thus, heterogametic organisms (XY) with one X chromosome may match the level of expression achieved in homogametic individuals with two active X chromosomes. This mechanism is observed in Drosophila.</p>

</div>
</div>
<div id="talking-glossary-duplication-0.75-min" class="section level2" number="2.13">
<h2><span class="header-section-number">2.13</span> Talking Glossary: Duplication (0.75 min)</h2>
<p><strong>Abstract</strong>: “Duplication is a type of mutation that involves the production of one or more copies of a gene or region of a chromosome. Gene and chromosome duplications occur in all organisms, though they are especially prominent among plants. Gene duplication is an important mechanism by which evolution occurs.”</p>
<p>Note: in introductory biology courses we typically focus on duplication of large portions of chromosomes or duplication of entire chromosmes.</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/duplication.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/duplication.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/duplication.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/duplication.mp3</a></p>
<p>Transcript: “Duplications occur when there is more than one copy of a specific stretch of DNA. This can occur in several different contexts. During a disease process, extra copies of the gene can contribute to a cancer. Genes can also duplicate through evolution, where one copy can continue the original function and the other copy of the gene produces a new function. On occasion, whole chromosomes are duplicated. In humans this causes disease. Throughout evolution, there have been several occasions, both in fish and plants, where whole genomes have been duplicated.”</p>
<p>Lawrence C. Brody, Ph.D.</p>

</div>
<div id="talking-glossary-epigenetics" class="section level2" number="2.14">
<h2><span class="header-section-number">2.14</span> Talking Glossary: Epigenetics</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Epigenetics" class="uri">https://www.genome.gov/genetics-glossary/Epigenetics</a></p>
<p><strong>Abstract</strong>: Epigenetics is an emerging field of science that studies heritable changes caused by the activation and deactivation of genes without any change in the underlying DNA sequence of the organism. The word epigenetics is of Greek origin and literally means over and above (epi) the genome.</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/epigenetics.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/epigenetics.mp3</a></p>
<p><strong>Transcript</strong>: “Epigenetics is the study of changes in gene function that are heritable and that are not attributed to alterations of the DNA sequence. The term epi means above. It’s a Greek prefix. It’s also defined as on top of the basic DNA sequence. In general terms you can think of them like accent marks on words where the DNA is the language and the modifications are the accent marks. Epigenetic marks change the way genes are expressed. The promise of epigenetics is that it tells us about the cell, it’s a way to define the cell that’s different than just looking at gene expression levels. We could look at any kind of cell and it will have specialized epigenetic patterns. There are two types of modifications: DNA methylation and histone modification. DNA methylation goes awry in cancers so if we knew the normal pattern of methylation and then looked at the pattern of methylation in a tumor we could see what changes were taking place and we could see which genes were being affected.”</p>
<p><a href="https://www.genome.gov/staff/Laura-Elnitski-PhD">Laura Elnitski</a></p>
<p>Photograph: <a href="https://i.irp.nih.gov/pi/0012009821.jpg" class="uri">https://i.irp.nih.gov/pi/0012009821.jpg</a></p>

</div>
<div id="talking-glossary-founder-effect-a-type-of-genetic-bottleneck-1-min" class="section level2" number="2.15">
<h2><span class="header-section-number">2.15</span> Talking glossary: Founder effect (A type of genetic bottleneck; 1 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Founder-Effect" class="uri">https://www.genome.gov/genetics-glossary/Founder-Effect</a></p>
<p><strong>Abstract</strong>: “The founder effect is the reduction in genetic variation that results when a small subset of a large population is used to establish a new colony. The new population may be very different from the original population, both in terms of its genotypes and phenotypes. In some cases, the founder effect plays a role in the emergence of new species.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/founder_effect.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/founder_effect.mp3</a></p>
<p><strong>Transcript</strong>: “Founder effect is a phenomenon in the work that we do that refers to the migration of a small group of people from a larger population to go settle in another environment. And they carry along with them a subset of genetic information that existed in the larger population. And because of that, carrying the subset, they actually reduce the amount of genetic variation that exists within a new population now. And as a result of that, they may emphasize certain phenotypes or certain genes, and all this may be deemphasized. So a founder effect can sometimes impact a population in such a way that they may have less of a particular type of gene or more of a particular type of gene. And it can change what we call phenotype, which is the things that we look at like your height, you know, your weight, or having a particular disease or not having a particular disease. So in effect that is what founder effect is.”</p>
<p><a href="https://en.wikipedia.org/wiki/Charles_Rotimi">Charles N. Rotimi</a>, Ph.D. is the Director of the Trans-National Institutes of Health (NIH) center for research in genomics and global health. He works to ensure that population genetics include genomes from African populations and founded the African Society of Human Genetics in 2003. Rotimi was instrumental in the launch of the Human Heredity and Health in Africa (H3Africa) with the NIH and the Wellcome Trust. He was elected to the National Academy of Medicine in 2018. (<a href="https://en.wikipedia.org/wiki/Charles_Rotimi">Wikipedia</a>))</p>
<p>Image: <a href="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a5/Charles_Rotimi_in_NHGRI%27s_Oral_History_Collection.jpg/440px-Charles_Rotimi_in_NHGRI%27s_Oral_History_Collection.jpg" class="uri">https://upload.wikimedia.org/wikipedia/commons/thumb/a/a5/Charles_Rotimi_in_NHGRI%27s_Oral_History_Collection.jpg/440px-Charles_Rotimi_in_NHGRI%27s_Oral_History_Collection.jpg</a></p>
<p>For an <a href="https://www.genome.gov/player/H6tg-48aVP4/PL1ay9ko4A8sk0o9O-YhseFHzbU2I2HQQp">interview</a> with Dr. Rotimi see: <a href="https://www.genome.gov/player/H6tg-48aVP4/PL1ay9ko4A8sk0o9O-YhseFHzbU2I2HQQp" class="uri">https://www.genome.gov/player/H6tg-48aVP4/PL1ay9ko4A8sk0o9O-YhseFHzbU2I2HQQp</a></p>

</div>
<div id="talking-glossary-frameshift-mutation-1.5-min" class="section level2" number="2.16">
<h2><span class="header-section-number">2.16</span> Talking Glossary: Frameshift mutation (1.5 min)</h2>
<p>Abstract: “A frameshift mutation is a type of mutation involving the insertion or deletion of a nucleotide in which the number of deleted base pairs is not divisible by three.”Divisible by three” is important because the cell reads a gene in groups of three bases. Each group of three bases corresponds to one of 20 different amino acids used to build a protein. If a mutation disrupts this reading frame, then the entire DNA sequence following the mutation will be read incorrectly.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/frameshift_mutation.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/frameshift_mutation.mp3</a></p>
<p>Transcript</p>
<p>“A frameshift mutation is a particular type of mutation that involves either insertion or deletion of extra bases of DNA. Now, what’s important here is the number three. The number of bases that are either added or subtracted can’t be divisible by three. And that’s important because the cell reads a gene in groups of three bases. Every group of three bases corresponds to one of the 20 different amino acids that are used by your body to make proteins. And keep in mind your body has a lot of proteins; everything from the material that makes up your skin, to the material that makes up your hair, to the digestive juices that help you digest that yummy lunch you just had. If a mutation disrupts one of those reading frames, so that the wrong amino acid is put in place, then the entire DNA sequence following the mutation will be disrupted or read incorrectly. Very often, what we see is a premature termination. Instead of the encoded protein being of a certain particular size, it’ll end up being much shorter, and it won’t be able to accomplish the role that’s been set out for it.”</p>
<p><a href="https://irp.nih.gov/pi/elaine-ostrander">Elaine A. Ostrander</a>, Ph.D.
Photo: <a href="https://i.irp.nih.gov/pi/0011756295.jpg" class="uri">https://i.irp.nih.gov/pi/0011756295.jpg</a></p>
<p>(see below for an optional video on Dr. Ostrander’s work on dog genomics)</p>
<p>Elaine A. Ostrander</p>
<p>Image
<a href="https://www.genome.gov/sites/default/files/tg/en/illustration/frameshift_mutation.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/frameshift_mutation.jpg</a>
Normal versus frameshift mutations
Frameshift mutations</p>
<p>Here’s an optional video about Dr. Ostrander’s work on dog genomics:
<a href="https://youtu.be/Tma8RdXADmU" class="uri">https://youtu.be/Tma8RdXADmU</a></p>

</div>
<div id="talking-glossary-u03-gel-electrophoresis-3-min" class="section level2" number="2.17">
<h2><span class="header-section-number">2.17</span> Talking Glossary U03: Gel Electrophoresis (“3 min)</h2>
<p>AKA “running a gel</p>
<p>National Human Genome Research Institute
<a href="https://www.genome.gov/genetics-glossary/Electrophoresis" class="uri">https://www.genome.gov/genetics-glossary/Electrophoresis</a></p>
<p>Abstract: “Electrophoresis is a laboratory technique used to separate DNA, RNA, or protein molecules based on their size and electrical charge. An electric current is used to move molecules to be separated through a gel. Pores in the gel work like a sieve, allowing smaller molecules to move faster than larger molecules. The conditions used during electrophoresis can be adjusted to separate molecules in a desired size range.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/electrophoresis.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/electrophoresis.mp3</a></p>
<p>Image (Wikipedia): <a href="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a6/Gel_electrophoresis_apparatus.JPG/188px-Gel_electrophoresis_apparatus.JPG" class="uri">https://upload.wikimedia.org/wikipedia/commons/thumb/a/a6/Gel_electrophoresis_apparatus.JPG/188px-Gel_electrophoresis_apparatus.JPG</a></p>
<p><strong>Transcript</strong>: Electrophoresis is a very broadly used technique which, fundamentally, applies electric current to biological molecules, whether–they’re usually DNA, they can be protein or RNA, too…and separates these fragments into pieces which are larger or smaller. It’s used in a variety of applications… Everything from forensics for determining the identity of individuals that may have been involved in a crime, by linking their DNA pattern, their electrophoresis pattern, to one that’s in a database. The whole basis by which the human genome was done is by something called capillary electrophoresis, by separating DNA into shorter pieces and then running them on these electrophoresis gels which allow the patterns of As, Cs, Ts, and Gs to be elucidated. They’re also very important in protein research, and then genetic mutation research, because when proteins or DNA are mutated, they are frequently longer or shorter, and they therefore show up on an electrophoresis gel differently than normal, so many diagnostic tests are still done using electrophoresis, so it’s a very widely used basic research technique, was very important for the understanding of gene and protein function, but it’s now gotten into the area of clinical diagnostics and forensics as well. Electrophoresis is usually done in what looks like a box which has a positive charge at one end and a negative charge at the other. And as we all learned in basic physics, when you put a charged molecule into an environment like that, the negative molecules go to the positive charge, and vice versa. In looking at proteins in a gel, in one of these boxes, you usually take the entire protein, and you’re looking at the entire length of the protein and seeing how big it is, and the bigger it is, the shorter it will migrate into the gel, so that the small proteins will end up at the bottom of the gel, because they have migrated the farthest, and the biggest ones will wind up staying at the top. In the case of DNA, DNA’s a very long molecule, so you wouldn’t want to run, for the most part, a whole DNA molecule from a cell onto a gel. It’s just so big that it would never get into the gel, so what scientists do, and what people do in classrooms these days, is to chop up that DNA using things like inscription enzymes, which chop up the DNA into more manageable pieces in a reproducible way. And then those pieces, depending on how big the pieces are, migrate more or less into the gel father down the box from top to the bottom.</p>
<p>Christopher P. Austin, M.D.</p>

</div>
<div id="talking-glossary-gene-amplification-1.5-min" class="section level2" number="2.18">
<h2><span class="header-section-number">2.18</span> Talking Glossary: Gene amplification (1.5 min)</h2>
<p>National Human Genome Research Institute</p>
<p><strong>Introduction</strong>: “Gene amplification is an increase in the number of copies of a gene sequence. ….. The term also can refer to polymerase chain reaction (PCR), a laboratory technique that is used by scientists to amplify gene sequences in a test tube.”</p>
<p>Note: Cloning and gene amplification are related concepts and are used interchangeably by some people. In this course its most important to know generally what the terms are referring to (increasing the number of copies of gene); there will not be a question where, for example “cloning” is a correct answer but “gene amplification” is wrong.</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/gene_amplification.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/gene_amplification.mp3</a></p>
<p>Transcript: “Gene amplification is one of those clauses that means different things to different people, depending on how they look at it. At its simplest level, it means an increase in the number of copies of a gene sequence. Now, cancer cells sometimes make multiple copies of genes in response to erroneous signals from other cells or because of signals they’re getting from the environment. So in that sense we think of it as a pretty negative term; it’s heralding something that’s not really supposed to have happened in the body that’s clearly going on. But it’s also just a very scientific term that doesn’t have either a positive or a negative connotation. Because it refers to a laboratory technique called polymerase chain reaction, or PCR, and that’s a technique which scientists can amplify a single gene in a test tube against the background of the three billion other bases that are floating around inside the test tube. That technique, PCR, or polymerase chain reaction, is really the backbone of a great deal of molecular biology that’s being done today. As a matter of fact, the individual who discovered it ended up winning a Nobel Prize for it. So when you hear the phrase”gene amplification”, think first about the context, and then think about the specifics.”</p>
<p><a href="https://www.genome.gov/staff/Elaine-A-Ostrander-PhD">Elaine A. Ostrander</a>, Ph.D.</p>

</div>
<div id="talking-glossary-gene-pool-0.75-min" class="section level2" number="2.19">
<h2><span class="header-section-number">2.19</span> Talking Glossary: Gene Pool (0.75 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Gene-Pool" class="uri">https://www.genome.gov/genetics-glossary/Gene-Pool</a> (Links to an external site.)</p>
<p><strong>Abstract</strong>: “A gene pool is the total genetic diversity found within a population or a species. A large gene pool has extensive genetic diversity and is better able to withstand the challenges posed by environmental stresses. Inbreeding contributes to the creation of a small gene pool and makes populations or species more likely to go extinct when faced with some type of stress.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/gene_pool.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/gene_pool.mp3</a></p>
<p><strong>Transcript</strong>: “Gene pool refers to thorough genetic diversity that exists within a population [all of the genetic diversity]. The larger the genetic pool, the more the diversity and the more opportunity this population will have to survive environmental stress that may impact on them. So gene pool is really good in the sense that the larger the gene pool, the more the survival of that particular population in terms of withstanding different things that may come in terms of environment.”</p>
<p><a href="https://www.genome.gov/staff/Charles-N-Rotimi-PhD">Charles N. Rotimi</a>, Ph.D</p>

</div>
<div id="gene-prediction" class="section level2" number="2.20">
<h2><span class="header-section-number">2.20</span> Gene prediction</h2>
<p><a href="https://en.wikipedia.org/wiki/Gene_prediction" class="uri">https://en.wikipedia.org/wiki/Gene_prediction</a></p>
<p>In computational biology, gene prediction or gene finding refers to the process of identifying the regions of genomic DNA that encode genes. This includes protein-coding genes as well as RNA genes , but may also include prediction of other functional elements such as regulatory regions. Gene finding is one of the first and most important steps in understanding the genome of a species once it has been sequenced.</p>
<p>In its earliest days, “gene finding” was based on painstaking experimentation on living cells and organisms. Statistical analysis of the rates of homologous recombination of several different genes could determine their order on a certain chromosome , and information from many such experiments could be combined to create a genetic map specifying the rough location of known genes relative to each other. Today, with comprehensive genome sequence and powerful computational resources at the disposal of the research community, gene finding has been redefined as a largely computational problem.</p>
<p>Determining that a sequence is functional should be distinguished from determining the function of the gene or its product. Predicting the function of a gene and confirming that the gene prediction is accurate still demands <em>in vivo</em> experimentation through gene knockout and other assays, although frontiers of bioinformatics research are making it increasingly possible to predict the function of a gene based on its sequence alone.</p>
<p>Gene prediction is one of the key steps in genome annotation , following sequence assembly , the filtering of non-coding regions and repeat masking.</p>
<p>Gene prediction is closely related to the so-called ‘target search problem’ investigating how DNA-binding proteins (transcription factors ) locate specific binding sites within the genome. Many aspects of structural gene prediction are based on current understanding of underlying biochemical processes in the cell such as gene transcription , translation , protein–protein interactions and regulation processes , which are subject of active research in the various omics fields such as transcriptomics , proteomics , metabolomics , and more generally structural and functional genomics.</p>

</div>
<div id="genbank" class="section level2" number="2.21">
<h2><span class="header-section-number">2.21</span> GenBank</h2>
<p>From the blog post “[Introduction to GenBank and Bioinformatics with Python](<a href="https://medium.com/@wvsharber/introduction-to-genbank-and-bioinformatics-with-python-8a25a0f15e3f" class="uri">https://medium.com/@wvsharber/introduction-to-genbank-and-bioinformatics-with-python-8a25a0f15e3f</a>” (Sharber 2015)</p>
<p>“GenBank [is] arguably one of the most important repositories of biological data in the world. GenBank, funded and operated by the National Institutes of Health (NIH) through the National Center for Biotechnology Information (NCBI), is a genetic sequence database. Researchers around the world submit their DNA sequences to GenBank to store and distribute them. Think of it as a huge library of DNA. Almost every DNA sequence that has ever been used in scientific research is stored in GenBank. Most academic publishers won’t accept your manuscript without you submitting your DNA to GenBank. Talk about open-source data! In case you’re curious, NCBI also hosts and produces other databases and tools, such as PubMed, which holds publication records, and BLAST, which is basically Google for sequence data.</p>
<p>“A DNA sequence stored in GenBank consists of a record holding more than just the DNA itself. There will be lots of metadata along with it, such as an unique accession number, what organism the DNA originated from, who sequenced it, what publications reference the data, the translation of the DNA into a protein sequence (if it’s a coding region), etc. These data are critically important for using the sequence data with the appropriate biological context.”</p>

</div>
<div id="genetic-distance" class="section level2" number="2.22">
<h2><span class="header-section-number">2.22</span> Genetic distance</h2>
<p>From Wikipedia, the free encyclopedia
<a href="https://en.wikipedia.org/wiki/Genetic_distance" class="uri">https://en.wikipedia.org/wiki/Genetic_distance</a> (Links to an external site.)</p>
<p>Genetic distance is a measure of the genetic (Links to an external site.) divergence between species (Links to an external site.) or between populations (Links to an external site.) within a species, whether the distance measures time from common ancestor (Links to an external site.) or degree of differentiation.[2] (Links to an external site.) Populations with many similar alleles (Links to an external site.) have small genetic distances. This indicates that they are closely related and have a recent common ancestor.</p>
<p>Genetic distance is useful for reconstructing the history of populations. For example, sequences can be compared and genetic distances calculated. These distances can then be used to build “distance-based” phylogenetic trees.</p>
<p>[This section is interesting but you do not need to know the details].</p>
<p>Evidence from genetic distance suggests that Sub-Saharan African and Eurasian people diverged about 100,000 years ago.[3] (Links to an external site.) Genetic distance is also used for understanding the origin of biodiversity (Links to an external site.). For example, the genetic distances between different breeds of domesticated animals are often investigated in order to determine which breeds should be protected to maintain genetic diversity.[4]</p>
<p><strong>References</strong>
1. Cavalli-Sforza, L.L., Menozzi, P. &amp; Piazza, A. (1994). The History and Geography of Human Genes. New Jersey: Princeton University Press.
1. Nei, M. (1987). Molecular Evolutionary Genetics. (Chapter 9). New York: Columbia University Press.
1. Ramachandran S, Deshpande O, Roseman CC, Rosenberg NA, Feldman MW, Cavalli-Sforza LL (November 2005). “Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in Africa”. Proc Natl Acad Sci U S A. 102 (44): 15942–7. <a href="doi:10.1073/pnas.0507611102" class="uri">doi:10.1073/pnas.0507611102</a>. PMC 1276087. PMID 16243969.
1. Ruane J (1999). “A critical review of the value of genetic distance studies in conservation of animal genetic resources”. Journal of Animal Breeding and Genetics. 116 (5): 317–323. <a href="doi:10.1046/j.1439-0388.1999.00205.x" class="uri">doi:10.1046/j.1439-0388.1999.00205.x</a>.
1. Nei, M. (1972). “Genetic distance between populations”. Am. Nat. 106 (949): 283–292. <a href="doi:10.1086/282771" class="uri">doi:10.1086/282771</a>. S2CID 55212907.</p>

</div>
<div id="talking-glossary-genetic-drift-0.75-min" class="section level2" number="2.23">
<h2><span class="header-section-number">2.23</span> Talking Glossary: Genetic drift (0.75 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Genetic-Drift" class="uri">https://www.genome.gov/genetics-glossary/Genetic-Drift</a> (Links to an external site.)</p>
<p><strong>Abstract</strong>: “Genetic drift is a mechanism of evolution. It refers to random fluctuations in the frequencies of alleles from generation to generation due to chance events. Genetic drift can cause traits to be dominant or disappear from a population. The effects of genetic drift are most pronounced in small populations.”</p>
<p>Audio: <a href="https://www.genome.gov/genetics-glossary/Genetic-Drift" class="uri">https://www.genome.gov/genetics-glossary/Genetic-Drift</a></p>
<p><strong>Transcript</strong>: “Genetic drift. I think the best way to look at it is to look at genetic drift as an evolutionary process. It is really the shuffling of our DNA frequency from one generation to the other. And as a result of this chance event, you know, you can emphasize it in things, you know, in the population. So it is in things. Something that may cause a particular type of gene to be more dominant or a particular type disease to disappear from a population. So again, it is a chance event, and it is an evolutionary process.”</p>
<p><a href="https://www.genome.gov/staff/Charles-N-Rotimi-PhD">Charles N. Rotimi</a>, Ph.D.</p>

</div>
<div id="talking-glossary-genetic-engineering-1.5-min" class="section level2" number="2.24">
<h2><span class="header-section-number">2.24</span> Talking Glossary: Genetic engineering (1.5 min)</h2>
<p>National Human Genome Research Institute
<a href="https://www.genome.gov/genetics-glossary/Genetic-Engineering" class="uri">https://www.genome.gov/genetics-glossary/Genetic-Engineering</a></p>
<p><strong>Introduction</strong>: “Genetic engineering is the process of using recombinant DNA (rDNA) technology to alter the genetic makeup of an organism. Traditionally, humans have manipulated genomes indirectly by controlling breeding and selecting offspring with desired traits. Genetic engineering involves the direct manipulation of one or more genes. Most often, a gene from another species is added to an organism’s genome to give it a desired phenotype.”</p>
<p>NOTE: The use of the term genetic engineering varies a lot. In general, academic researchers seem to refer to things as “recombinant” when DNA from one organism has been added via a plasmid or another route to another organism. “Genetically engineered” seems to be preferred by biotech companies, the media – science fiction Any time you change the genetic composition of an organism (e.g. putting syphilis protein into E. coli via a plasmid) you could refer to it as genetically engineered. I will use the term recombinant though.</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/genetic_engineering.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/genetic_engineering.mp3</a></p>
<p><strong>Transcript</strong>: “Genetic engineering is a term that was first introduced into our language in the 1970s to describe the emerging field of recombinant DNA technology and some of the things that were going on. As most people who read textbooks and things know, recombinant DNA technology started with pretty simple things–cloning very small pieces of DNA and growing them in bacteria–and has evolved to an enormous field where whole genomes can be cloned and moved from cell to cell, to cell using variations of techniques that all would come under genetic engineering as a very broad definition. To me, genetic engineering, broadly defined, means that you are taking pieces of DNA and combining them with other pieces of DNA. [This] doesn’t really happen in nature, but is something that you engineer in your own laboratory and test tubes. And then taking what you have engineered and propagating that in any number of different organisms that range from bacterial cells to yeast cells, to plants and animals. So while there isn’t a precise definition of genetic engineering, I think it more defines an entire field of recombinant DNA technology, genomics, and genetics in the 2000s.”</p>

</div>
<div id="genome-assembly" class="section level2" number="2.25">
<h2><span class="header-section-number">2.25</span> Genome assembly</h2>
<p>Adapted from Wikipedia
<a href="https://en.wikipedia.org/wiki/Sequence_assembly" class="uri">https://en.wikipedia.org/wiki/Sequence_assembly</a></p>
<p>In bioinformatics, genome assembly refers to aligning and merging fragments from a longer DNA sequence such as a whole genome in order to reconstruct the original sequence. This is needed as DNA sequencing technology cannot read whole genomes in one go, but rather reads small pieces, called reads.</p>
<p>The problem of sequence assembly can be compared to taking many copies of a book, passing each of them through different paper shredders with different cutters, and piecing the text of the book back together just by looking at the shredded pieces.</p>
<!-- Figure 1: Assemblers need copious sequencing data and informatic exertion to put the genome back together. Credit: Kelly Howe, Lawrence Berkeley Laboratory -->
<p>In sequence assembly, two different types can be distinguished:</p>
<p>1, de-novo: assembling short reads to create full-length (sometimes novel) sequences, without using a template (see de novo sequence assemblers , de novo transcriptome assembly )
1. using a reference sequence: assembling reads against an existing reference sequence, building a sequence that is similar but not necessarily identical to the backbone sequence</p>
<p>In terms of complexity and time requirements, de-novo assemblies takes much more work. Referring to the comparison drawn to shredded books in the introduction: when a reference sequence is available, one would have a very similar book as a template (perhaps with the names of the main characters and a few locations changed), de-novo assemblies present a more daunting challenge in that one would not know beforehand whether this would become a science book, a novel, a catalog, or even several books. Also, every shred would be compared with every other shred.</p>

</div>
<div id="talking-glossary-genomics" class="section level2" number="2.26">
<h2><span class="header-section-number">2.26</span> Talking Glossary: Genomics</h2>
<p><strong>Abstract</strong>: “Genomics refers to the study of the entire genome of an organism whereas genetics refers to the study of a particular gene.”</p>
<p><strong>Transcript</strong>: “Genomics refers to the study of the entire genome, essentially all the genes that can be found in an organism. It’s in contrast to genetics which can study individual genes one at a time. A genomicist, or someone who studies genomes, studies all of the DNA and all of the sequence in an organism and makes conclusions based on all of it. Whereas a geneticist can study all of the DNA in an organism, but can also study one gene at a time. For example, you might study the genetics of a single gene and sequence that gene. If you wanted to study the genomics of an organism or a person you could sequence all of their genes and all of their DNA and look for changes and make comparisons with other individual’s genomes.”</p>
<p>Lawrence C. Brody, Ph.D.</p>

</div>
<div id="talking-glossary-genotype" class="section level2" number="2.27">
<h2><span class="header-section-number">2.27</span> Talking Glossary: Genotype</h2>
<p><strong>Abstract</strong>: “A genotype is an individual’s collection of genes. The term also can refer to the two alleles inherited for a particular gene. The genotype is expressed when the information encoded in the genes’ DNA is used to make protein and RNA molecules. The expression of the genotype contributes to the individual’s observable traits, called the phenotype.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/genotype.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/genotype.mp3</a></p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/genotype.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/genotype.jpg</a></p>
<p><strong>Transcript</strong>: “Genotype, very simply, is the version of a DNA sequence that an individual has. There’s a large amount of DNA that we all have in common–of course, that’s why we’re all humans–but there’s also a large amount of variation in sequence among individuals. And those specific differences in sequence, when usually applied to an individual gene, are called a genotype. These days, with the ability to test for many different sequence differences between individuals, genotype has taken on a connotation which frequently refers to a difference in sequence in a specific place in a specific gene. When used in that way, it’s usually related to another term, called phenotype, which is the change in sequence to which the genotype refers. It is frequently, not always, but is frequently related to a change in an external trait; something that’s observable, like height, hair color, or occurrence of disease. And so in that case, we talk about a genotype-phenotype correlation. Then what we’re talking about is, well, here’s a change in DNA sequence; why is it important? It’s important because it leads to an observable change in a trait in a person. And that change in trait can be positive, it can be negative, or it could just be a difference.”</p>
<p>Christopher P. Austin, M.D.</p>

</div>
<div id="talking-glossary-germline-1.25-min" class="section level2" number="2.28">
<h2><span class="header-section-number">2.28</span> Talking Glossary: Germline (1.25 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/germ-line" class="uri">https://www.genome.gov/genetics-glossary/germ-line</a></p>
<p>Abstract: “A germ line is the sex cells (eggs and sperm) that are used by sexually reproducing organisms to pass on genes from generation to generation. Egg and sperm cells are called germ cells, in contrast to the other cells of the body that are called somatic cells.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/germ_line.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/germ_line.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/germ_line.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/germ_line.mp3</a></p>
<p>Transcript: “Germ line actually refers to the sex cells of an organism. And it does seem like a bit of a strange term, but it actually refers not just to mammals, but to any living organism that uses sex to reproduce, and that includes plants. And so it’s a rather generic term which refers to the cells in the sexual organ, and those cells will either be making sperm or eggs. And that sperm be could, of course, be pollen instead of what you traditionally think of as sperm. And the key important aspect of germ line is this is where genetic information is transferred from one generation to the next. And it’s inherited both from the female and the male of the species, so this is where all the action in terms of genetics is happening. And when those organisms grow up they, of course, are not comprised primarily of germ cells. Most of the body is consisted of somatic cells. But the key issue in terms of evolution, in terms of inheritance, all go through the germ line.”</p>
<p>Shawn Burgess, Ph.D.</p>

</div>
<div id="guild" class="section level2" number="2.29">
<h2><span class="header-section-number">2.29</span> Guild</h2>
<p><a href="https://en.wikipedia.org/wiki/Guild_(ecology)" class="uri">https://en.wikipedia.org/wiki/Guild_(ecology)</a>.</p>
<p>A guild (or ecological guild) is any group of species that exploit the same resources, or that exploit different resources in related ways.[1].[2].[3]. It is not necessary that the species within a guild occupy the same, or even similar, ecological niches.. An ecological niche is defined as the role an organism plays in its community, i.e. decomposer, primary producer, etc.[4]. Guilds are defined according to the locations, attributes, or activities of their component species. For example, the mode of acquiring nutrients, the mobility, and the habitat zones that the species occupy or exploit can be used to define a guild. The number of guilds occupying an ecosystem is termed its disparity. Members of a guild within a given ecosystem could be competing for resources, such as space or light, while cooperating in resisting wind stresses, attracting pollinators, or detecting predators, such as happens among savannah-dwelling antelope. and zebra..</p>
<p>A guild does not typically have strict, or even clearly defined boundaries, nor does it need to be taxonomically cohesive. A broadly defined guild will almost always have constituent guilds; for example, grazing guilds will have some species that concentrate on coarse, plentiful forage, while others concentrate on low-growing, finer plants. Each of those two sub-guilds may be regarded as guilds in appropriate contexts, and they might, in turn, have sub-guilds in more closely selective contexts. Some authorities even speak of guilds in terms of a fractal. resource model.[5]. This concept arises in several related contexts, such as the metabolic theory of ecology., the scaling pattern of occupancy., and spatial analysis. in ecology, all of which are fundamental concepts in defining guilds.</p>
<p><strong>Examples of guilds</strong>:
1. Graminoids (grasses, rushes and sedges)
1. Plankton
1. Shrubs
1. Trees
1. Vines
1. Scavengers
1. Browsers and terrestrial folivores (animals that eat leaves other than grass)
1. Forest canopy folivores (flower-eaters)
1. Grazers (grass-eating animals)
1. Forbs (herbaceous plants that aren’t grasses)
1. Saprophytes (decomposers, such as fungi)
1. Piscivores (fish-eaters)</p>
<p><strong>References</strong>:
1. Simberloff, D; Dayan, T (1991). “The Guild Concept and the Structure of Ecological Communities”. Annual Review of Ecology and Systematics. 22: 115–143. <a href="doi:10.1146/annurev.es.22.110191.000555" class="uri">doi:10.1146/annurev.es.22.110191.000555</a>.
1. Encyclopædia Britannica article on guilds
1. Williams, SE; Hero, JM (1998). “Rainforest frogs of the Australian Wet Tropics: guild classification and the ecological similarity of declining species”. Proceedings: Biological Sciences. 265 (1396): 597–602. <a href="doi:10.1098/rspb.1998.0336" class="uri">doi:10.1098/rspb.1998.0336</a>. PMC 1689015. PMID 9881468.
1. “the definition of ecological niche”. Dictionary.com. Retrieved 2017-05-02.
1. Guillerme; et al. (2020). “Disparities in the analysis of morphological disparity”. Biology Letters. 16 (7). <a href="doi:10.1098/rsbl.2020.0199" class="uri">doi:10.1098/rsbl.2020.0199</a>.
1. Ritchie, Mark E. (2010). Scale, Heterogeneity, and the Structure and Diversity of Ecological Communities. Volume 45 of Monographs in population biology. Princeton: Princeton University Press. ISBN 978-0-691-09070-2.</p>

</div>
<div id="talking-glossary-haploid-1.25-min" class="section level2" number="2.30">
<h2><span class="header-section-number">2.30</span> Talking Glossary: Haploid (1.25 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/haploid" class="uri">https://www.genome.gov/genetics-glossary/haploid</a></p>
<p><strong>Abstract</strong>: “Haploid is the quality of a cell or organism having a single set of chromosomes. Organisms that reproduce asexually are haploid. Sexually reproducing organisms are diploid (having two sets of chromosomes, one from each parent). In humans, only their egg and sperm cells are haploid.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/haploid.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/haploid.mp3</a></p>
<p><strong>Transcript</strong>: “Haploid refers to a cell or an organism that has only a single set of chromosomes. This is to be contrasted with diploid.”Di” means two, of course. So most animal cells and plant cells are diploid. Then they’re diploid in part because they got one chromosome from their mother and one chromosome from their father, therefore making them diploid. A haploid cell only has one set of chromosomes, and most of the time that refers to the so-called sex cells, either eggs or sperm. And these are [a] critical transition from a diploid cell to a haploid cell to allow normal reproduction to occur, so that when these two haploid cells come together with a single set of genetic information–single chromosomes–they can come together into a so-called zygote made of when the egg cell and the sperm cell come together that then reconstitutes a diploid cell, which can then become a new individual.”</p>
<p>Christopher P. Austin, M.D.</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/haploid.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/haploid.jpg</a></p>

</div>
<div id="talking-glossary-heterozygous" class="section level2" number="2.31">
<h2><span class="header-section-number">2.31</span> Talking Glossary: Heterozygous</h2>
<p><a href="https://www.genome.gov/genetics-glossary/heterozygous" class="uri">https://www.genome.gov/genetics-glossary/heterozygous</a> (Links to an external site.)</p>
<p><strong>Abstract</strong>: “Heterozygous refers to having inherited different forms of a particular gene from each parent. A heterozygous genotype stands in contrast to a homozygous genotype, where an individual inherits identical forms of a particular gene from each parent.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/heterozygous.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/heterozygous.mp3</a></p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/heterozygous.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/heterozygous.jpg</a></p>
<p><strong>Transcript</strong>: “Heterozygous is a state of having inherited different forms of a particular gene from each one of your biological parents. Now, by different forms we generally mean that there are different portions of the gene where the sequence is different. They may be inconsequential portions of the gene, or they may in fact be pretty important portions of the gene. That doesn’t really matter for our discussion today. The word”heterozygous” simply means that your biological mother and your biological father, when they contributed their copies of a particular gene to you, they did so in a way so that the DNA sequence is slightly different. It can be different at one point in the gene, or it can be different at dozens and dozens of different points in the gene. Now, a heterozygous genotype stands in contrast to a homozygous genotype. And in the case of a homozygous genotype, we’re talking about a case where we’ve gotten identical forms of a particular gene from each biological parent. That is, if we were to read along the DNA sequence that mom gave you and the DNA sequence that dad gave you, we would find absolutely, positively no differences in that gene or in the region of the gene that we’re concerned about. “Heterozygous” meaning different, “homozygous” meaning the same.”</p>
<p>Amalia S. Dutra, Ph.D.</p>
<p>Amalia Dutra, Ph.D, is a Uruguayan genetic biologist known for being part of the team that mapped the human genome</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="when-things-are-homolgous-and-analogous.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["openbio.pdf", "openbio.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
