<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title> 5 D | Foundations of Biology and Environmental Science: An Open-Access Encyclopedia</title>
  <meta name="description" content="A collection of open-access and/or free resources on biology and environmental science." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content=" 5 D | Foundations of Biology and Environmental Science: An Open-Access Encyclopedia" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content=" 5 D | Foundations of Biology and Environmental Science: An Open-Access Encyclopedia" />
  
  <meta name="twitter:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

<meta name="author" content="Compiled and edited by Nathan Brouwer" />


<meta name="date" content="2022-01-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="c.html"/>
<link rel="next" href="h.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Biology for everyone</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="a.html"><a href="a.html"><i class="fa fa-check"></i><b>2</b> A</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a.html"><a href="a.html#alignments"><i class="fa fa-check"></i><b>2.1</b> Alignments</a></li>
<li class="chapter" data-level="2.2" data-path="a.html"><a href="a.html#talking-glossary-allele-0.5-min"><i class="fa fa-check"></i><b>2.2</b> Talking Glossary: Allele (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="b.html"><a href="b.html"><i class="fa fa-check"></i><b>3</b> B</a>
<ul>
<li class="chapter" data-level="3.1" data-path="b.html"><a href="b.html#talking-glossary-bioinformatics"><i class="fa fa-check"></i><b>3.1</b> Talking glossary: Bioinformatics</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="c.html"><a href="c.html"><i class="fa fa-check"></i><b>4</b> C</a>
<ul>
<li class="chapter" data-level="4.1" data-path="c.html"><a href="c.html#talking-glossary-candidate-gene"><i class="fa fa-check"></i><b>4.1</b> Talking Glossary: Candidate gene</a></li>
<li class="chapter" data-level="4.2" data-path="c.html"><a href="c.html#cath-database"><i class="fa fa-check"></i><b>4.2</b> CATH database</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="c.html"><a href="c.html#references"><i class="fa fa-check"></i><b>4.2.1</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="c.html"><a href="c.html#clade"><i class="fa fa-check"></i><b>4.3</b> Clade</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="c.html"><a href="c.html#clades-and-phylogenetic-trees"><i class="fa fa-check"></i><b>4.3.1</b> Clades and phylogenetic trees</a></li>
<li class="chapter" data-level="4.3.2" data-path="c.html"><a href="c.html#terminology"><i class="fa fa-check"></i><b>4.3.2</b> Terminology</a></li>
<li class="chapter" data-level="4.3.3" data-path="c.html"><a href="c.html#in-popular-culture"><i class="fa fa-check"></i><b>4.3.3</b> In popular culture</a></li>
<li class="chapter" data-level="4.3.4" data-path="c.html"><a href="c.html#references-1"><i class="fa fa-check"></i><b>4.3.4</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="c.html"><a href="c.html#talking-glossary-cloning-0.5-min"><i class="fa fa-check"></i><b>4.4</b> Talking Glossary: Cloning (0.5 min)</a></li>
<li class="chapter" data-level="4.5" data-path="c.html"><a href="c.html#talking-glossary-codominance-0.5-min"><i class="fa fa-check"></i><b>4.5</b> Talking Glossary: Codominance (0.5 min)</a></li>
<li class="chapter" data-level="4.6" data-path="c.html"><a href="c.html#comparative-genomics"><i class="fa fa-check"></i><b>4.6</b> Comparative genomics</a></li>
<li class="chapter" data-level="4.7" data-path="c.html"><a href="c.html#talking-glossary-contig-0.75-min"><i class="fa fa-check"></i><b>4.7</b> Talking Glossary: Contig (0.75 min)</a></li>
<li class="chapter" data-level="4.8" data-path="c.html"><a href="c.html#talking-glossary---foundations-1-review-crossing-over"><i class="fa fa-check"></i><b>4.8</b> Talking Glossary - Foundations 1 Review: Crossing over</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="d.html"><a href="d.html"><i class="fa fa-check"></i><b>5</b> D</a>
<ul>
<li class="chapter" data-level="5.1" data-path="d.html"><a href="d.html#talking-glossary-deletion-mutation-1.5-min"><i class="fa fa-check"></i><b>5.1</b> Talking Glossary: Deletion mutation (1.5 min)</a></li>
<li class="chapter" data-level="5.2" data-path="d.html"><a href="d.html#talking-glossary-dna-sequencing-0.75-min"><i class="fa fa-check"></i><b>5.2</b> Talking Glossary: DNA sequencing (0.75 min)</a></li>
<li class="chapter" data-level="5.3" data-path="d.html"><a href="d.html#sex-chromosome-dosage-compensation"><i class="fa fa-check"></i><b>5.3</b> Sex-Chromosome Dosage compensation</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="d.html"><a href="d.html#random-inactivation-of-one-x-in-xx-individuals"><i class="fa fa-check"></i><b>5.3.1</b> Random inactivation of one X in XX individuals</a></li>
<li class="chapter" data-level="5.3.2" data-path="d.html"><a href="d.html#results"><i class="fa fa-check"></i><b>5.3.2</b> Results</a></li>
<li class="chapter" data-level="5.3.3" data-path="d.html"><a href="d.html#clinical-applications"><i class="fa fa-check"></i><b>5.3.3</b> Clinical applications</a></li>
<li class="chapter" data-level="5.3.4" data-path="d.html"><a href="d.html#agricultural-applications"><i class="fa fa-check"></i><b>5.3.4</b> Agricultural applications</a></li>
<li class="chapter" data-level="5.3.5" data-path="d.html"><a href="d.html#limitations"><i class="fa fa-check"></i><b>5.3.5</b> Limitations</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="d.html"><a href="d.html#fine-mapping"><i class="fa fa-check"></i><b>5.4</b> Fine-mapping</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="h.html"><a href="h.html"><i class="fa fa-check"></i><b>6</b> H</a>
<ul>
<li class="chapter" data-level="6.1" data-path="h.html"><a href="h.html#talking-glossary-haploid-1.25-min"><i class="fa fa-check"></i><b>6.1</b> Talking Glossary: Haploid (1.25 min)</a></li>
<li class="chapter" data-level="6.2" data-path="h.html"><a href="h.html#talking-glossary-halplotype"><i class="fa fa-check"></i><b>6.2</b> Talking Glossary: Halplotype</a></li>
<li class="chapter" data-level="6.3" data-path="h.html"><a href="h.html#talking-glossary-heterozygous"><i class="fa fa-check"></i><b>6.3</b> Talking Glossary: Heterozygous</a></li>
<li class="chapter" data-level="6.4" data-path="h.html"><a href="h.html#homologyl-when-things-are-homolgous-and-analogous"><i class="fa fa-check"></i><b>6.4</b> Homology:L When things are homolgous AND analogous!</a></li>
<li class="chapter" data-level="6.5" data-path="h.html"><a href="h.html#talking-glossary-homozygous"><i class="fa fa-check"></i><b>6.5</b> Talking Glossary: Homozygous</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="i.html"><a href="i.html"><i class="fa fa-check"></i><b>7</b> I</a>
<ul>
<li class="chapter" data-level="7.1" data-path="i.html"><a href="i.html#talking-glossary-insertion-mutation-1.25-min"><i class="fa fa-check"></i><b>7.1</b> Talking Glossary: Insertion mutation (1.25 min)</a></li>
<li class="chapter" data-level="7.2" data-path="i.html"><a href="i.html#intrinscially-disordered-protein"><i class="fa fa-check"></i><b>7.2</b> Intrinscially disordered protein</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="i.html"><a href="i.html#abundance"><i class="fa fa-check"></i><b>7.2.1</b> Abundance</a></li>
<li class="chapter" data-level="7.2.2" data-path="i.html"><a href="i.html#disorder-annotation"><i class="fa fa-check"></i><b>7.2.2</b> Disorder annotation</a></li>
<li class="chapter" data-level="7.2.3" data-path="i.html"><a href="i.html#disorder-databases"><i class="fa fa-check"></i><b>7.2.3</b> Disorder databases</a></li>
<li class="chapter" data-level="7.2.4" data-path="i.html"><a href="i.html#predicting-idps-by-sequence"><i class="fa fa-check"></i><b>7.2.4</b> Predicting IDPs by sequence</a></li>
<li class="chapter" data-level="7.2.5" data-path="i.html"><a href="i.html#prediction-methods"><i class="fa fa-check"></i><b>7.2.5</b> Prediction methods</a></li>
<li class="chapter" data-level="7.2.6" data-path="i.html"><a href="i.html#disorder-and-disease"><i class="fa fa-check"></i><b>7.2.6</b> Disorder and disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="j.html"><a href="j.html"><i class="fa fa-check"></i><b>8</b> J</a>
<ul>
<li class="chapter" data-level="8.1" data-path="j.html"><a href="j.html#jove-videos"><i class="fa fa-check"></i><b>8.1</b> JoVe Videos</a>
<ul>
<li class="chapter" data-level="8.1.1" data-path="j.html"><a href="j.html#biotechnology"><i class="fa fa-check"></i><b>8.1.1</b> Biotechnology</a></li>
<li class="chapter" data-level="8.1.2" data-path="j.html"><a href="j.html#macromolecules"><i class="fa fa-check"></i><b>8.1.2</b> Macromolecules</a></li>
<li class="chapter" data-level="8.1.3" data-path="j.html"><a href="j.html#cellular-respiration"><i class="fa fa-check"></i><b>8.1.3</b> Cellular respiration</a></li>
<li class="chapter" data-level="8.1.4" data-path="j.html"><a href="j.html#population-and-community-ecology"><i class="fa fa-check"></i><b>8.1.4</b> Population and community ecology</a></li>
<li class="chapter" data-level="8.1.5" data-path="j.html"><a href="j.html#population-genetics"><i class="fa fa-check"></i><b>8.1.5</b> Population genetics</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="k.html"><a href="k.html"><i class="fa fa-check"></i><b>9</b> K</a>
<ul>
<li class="chapter" data-level="9.1" data-path="k.html"><a href="k.html#talking-glossary-knockout-genetic-knockout-gene-knockout"><i class="fa fa-check"></i><b>9.1</b> Talking glossary: Knockout (Genetic knockout, gene knockout)</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="l.html"><a href="l.html"><i class="fa fa-check"></i><b>10</b> L</a>
<ul>
<li class="chapter" data-level="10.1" data-path="l.html"><a href="l.html#talking-glossary-locus-1-min"><i class="fa fa-check"></i><b>10.1</b> Talking Glossary: Locus (1 min)</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="m.html"><a href="m.html"><i class="fa fa-check"></i><b>11</b> M</a>
<ul>
<li class="chapter" data-level="11.1" data-path="m.html"><a href="m.html#manhattan-plot"><i class="fa fa-check"></i><b>11.1</b> Manhattan Plot</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="m.html"><a href="m.html#gwas"><i class="fa fa-check"></i><b>11.1.1</b> GWAS</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="m.html"><a href="m.html#talking-glossary-microarray"><i class="fa fa-check"></i><b>11.2</b> Talking Glossary: Microarray</a></li>
<li class="chapter" data-level="11.3" data-path="m.html"><a href="m.html#talking-glossary-missense-mutation-1.25-min"><i class="fa fa-check"></i><b>11.3</b> Talking Glossary: Missense Mutation (1.25 min)</a></li>
<li class="chapter" data-level="11.4" data-path="m.html"><a href="m.html#talking-glossary-mitochondrial-dna-1.5-min"><i class="fa fa-check"></i><b>11.4</b> Talking Glossary: Mitochondrial DNA (1.5 min)</a></li>
<li class="chapter" data-level="11.5" data-path="m.html"><a href="m.html#talking-glossary-mutagen-0.5-min"><i class="fa fa-check"></i><b>11.5</b> Talking Glossary: Mutagen (0.5 min)</a></li>
<li class="chapter" data-level="11.6" data-path="m.html"><a href="m.html#talking-glossary-mutation-0.5-min"><i class="fa fa-check"></i><b>11.6</b> Talking Glossary: Mutation (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="n.html"><a href="n.html"><i class="fa fa-check"></i><b>12</b> N</a>
<ul>
<li class="chapter" data-level="12.1" data-path="n.html"><a href="n.html#talking-glossary-non-coding-dna-1-min"><i class="fa fa-check"></i><b>12.1</b> Talking Glossary: Non-coding DNA (1 min)</a></li>
<li class="chapter" data-level="12.2" data-path="n.html"><a href="n.html#talking-glossary-nonsene-mutation"><i class="fa fa-check"></i><b>12.2</b> Talking Glossary: Nonsene mutation</a></li>
<li class="chapter" data-level="12.3" data-path="n.html"><a href="n.html#nucleic-acids-review"><i class="fa fa-check"></i><b>12.3</b> Nucleic Acids - Overview</a>
<ul>
<li class="chapter" data-level="12.3.1" data-path="n.html"><a href="n.html#dna-and-rna"><i class="fa fa-check"></i><b>12.3.1</b> DNA and RNA</a></li>
<li class="chapter" data-level="12.3.2" data-path="n.html"><a href="n.html#dna-double-helix-structure"><i class="fa fa-check"></i><b>12.3.2</b> DNA Double-Helix Structure</a></li>
<li class="chapter" data-level="12.3.3" data-path="n.html"><a href="n.html#rna"><i class="fa fa-check"></i><b>12.3.3</b> RNA</a></li>
<li class="chapter" data-level="12.3.4" data-path="n.html"><a href="n.html#protein-function"><i class="fa fa-check"></i><b>12.3.4</b> Protein function</a></li>
<li class="chapter" data-level="12.3.5" data-path="n.html"><a href="n.html#summary"><i class="fa fa-check"></i><b>12.3.5</b> Summary</a></li>
<li class="chapter" data-level="12.3.6" data-path="n.html"><a href="n.html#glossary"><i class="fa fa-check"></i><b>12.3.6</b> Glossary</a></li>
<li class="chapter" data-level="12.3.7" data-path="n.html"><a href="n.html#contributors-and-attributions"><i class="fa fa-check"></i><b>12.3.7</b> Contributors and Attributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="13" data-path="o.html"><a href="o.html"><i class="fa fa-check"></i><b>13</b> O</a>
<ul>
<li class="chapter" data-level="13.1" data-path="o.html"><a href="o.html#talking-glossary-open-reading-frame-definition-3-min"><i class="fa fa-check"></i><b>13.1</b> Talking Glossary : Open reading frame definition (3 min)</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="p.html"><a href="p.html"><i class="fa fa-check"></i><b>14</b> P</a>
<ul>
<li class="chapter" data-level="14.1" data-path="p.html"><a href="p.html#talking-glossary-pcr---the-polymerase-chain-reaction-0.5-min"><i class="fa fa-check"></i><b>14.1</b> Talking Glossary: PCR - The Polymerase Chain Reaction (0.5 min)</a></li>
<li class="chapter" data-level="14.2" data-path="p.html"><a href="p.html#protein-data-bank"><i class="fa fa-check"></i><b>14.2</b> Protein Data Bank</a>
<ul>
<li class="chapter" data-level="14.2.1" data-path="p.html"><a href="p.html#history"><i class="fa fa-check"></i><b>14.2.1</b> History</a></li>
<li class="chapter" data-level="14.2.2" data-path="p.html"><a href="p.html#file-format"><i class="fa fa-check"></i><b>14.2.2</b> File format</a></li>
</ul></li>
<li class="chapter" data-level="14.3" data-path="p.html"><a href="p.html#pfam"><i class="fa fa-check"></i><b>14.3</b> PFam</a>
<ul>
<li class="chapter" data-level="14.3.1" data-path="p.html"><a href="p.html#uses"><i class="fa fa-check"></i><b>14.3.1</b> Uses</a></li>
</ul></li>
<li class="chapter" data-level="14.4" data-path="p.html"><a href="p.html#talking-glossary-phenotype"><i class="fa fa-check"></i><b>14.4</b> Talking Glossary: Phenotype</a></li>
<li class="chapter" data-level="14.5" data-path="p.html"><a href="p.html#phylogenetics-vocab"><i class="fa fa-check"></i><b>14.5</b> Phylogenetics vocab</a></li>
<li class="chapter" data-level="14.6" data-path="p.html"><a href="p.html#talking-glossary-plasmid-1-min"><i class="fa fa-check"></i><b>14.6</b> Talking Glossary: Plasmid (1 min)</a></li>
<li class="chapter" data-level="14.7" data-path="p.html"><a href="p.html#talking-glossary-point-mutation-0.5-min"><i class="fa fa-check"></i><b>14.7</b> Talking Glossary: Point mutation (0.5 min)</a></li>
<li class="chapter" data-level="14.8" data-path="p.html"><a href="p.html#talking-glossary-polymorphism-1-min"><i class="fa fa-check"></i><b>14.8</b> Talking Glossary: Polymorphism (1 min)</a></li>
<li class="chapter" data-level="14.9" data-path="p.html"><a href="p.html#talking-glossary-u03-primer-0.5-min"><i class="fa fa-check"></i><b>14.9</b> Talking Glossary U03: Primer (0.5 min)</a></li>
<li class="chapter" data-level="14.10" data-path="p.html"><a href="p.html#talking-glossary-promoter-1.5-min"><i class="fa fa-check"></i><b>14.10</b> Talking Glossary: Promoter (1.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="q.html"><a href="q.html"><i class="fa fa-check"></i><b>15</b> Q</a></li>
<li class="chapter" data-level="16" data-path="r.html"><a href="r.html"><i class="fa fa-check"></i><b>16</b> R</a>
<ul>
<li class="chapter" data-level="16.1" data-path="r.html"><a href="r.html#talking-glossary-recessive"><i class="fa fa-check"></i><b>16.1</b> Talking Glossary: Recessive</a></li>
<li class="chapter" data-level="16.2" data-path="r.html"><a href="r.html#talking-glossary-recombinant-dna-0.5-min"><i class="fa fa-check"></i><b>16.2</b> Talking Glossary: Recombinant DNA (0.5 min)</a></li>
<li class="chapter" data-level="16.3" data-path="r.html"><a href="r.html#talking-glossary-repressor-1-min"><i class="fa fa-check"></i><b>16.3</b> Talking Glossary: Repressor (1 min)</a></li>
<li class="chapter" data-level="16.4" data-path="r.html"><a href="r.html#talking-glossary-restriction-enzyme-0.5-min"><i class="fa fa-check"></i><b>16.4</b> Talking Glossary: Restriction Enzyme (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="s.html"><a href="s.html"><i class="fa fa-check"></i><b>17</b> S</a>
<ul>
<li class="chapter" data-level="17.1" data-path="s.html"><a href="s.html#structural-classification-of-proteins-database-scop"><i class="fa fa-check"></i><b>17.1</b> Structural Classification of Proteins database (SCOP)</a>
<ul>
<li class="chapter" data-level="17.1.1" data-path="s.html"><a href="s.html#hierarchical-organisation"><i class="fa fa-check"></i><b>17.1.1</b> Hierarchical organisation</a></li>
<li class="chapter" data-level="17.1.2" data-path="s.html"><a href="s.html#superfamilies"><i class="fa fa-check"></i><b>17.1.2</b> Superfamilies</a></li>
<li class="chapter" data-level="17.1.3" data-path="s.html"><a href="s.html#families"><i class="fa fa-check"></i><b>17.1.3</b> Families</a></li>
<li class="chapter" data-level="17.1.4" data-path="s.html"><a href="s.html#pdb-entry-domains"><i class="fa fa-check"></i><b>17.1.4</b> PDB entry domains</a></li>
</ul></li>
<li class="chapter" data-level="17.2" data-path="s.html"><a href="s.html#sequence-alignment"><i class="fa fa-check"></i><b>17.2</b> Sequence alignment</a>
<ul>
<li class="chapter" data-level="17.2.1" data-path="s.html"><a href="s.html#interpretation"><i class="fa fa-check"></i><b>17.2.1</b> Interpretation</a></li>
<li class="chapter" data-level="17.2.2" data-path="s.html"><a href="s.html#alignment-methods"><i class="fa fa-check"></i><b>17.2.2</b> Alignment methods</a></li>
<li class="chapter" data-level="17.2.3" data-path="s.html"><a href="s.html#representations"><i class="fa fa-check"></i><b>17.2.3</b> Representations</a></li>
<li class="chapter" data-level="17.2.4" data-path="s.html"><a href="s.html#global-and-local-alignments"><i class="fa fa-check"></i><b>17.2.4</b> Global and local alignments</a></li>
<li class="chapter" data-level="17.2.5" data-path="s.html"><a href="s.html#pairwise-alignment"><i class="fa fa-check"></i><b>17.2.5</b> Pairwise alignment</a></li>
<li class="chapter" data-level="17.2.6" data-path="s.html"><a href="s.html#dot-matrix-methods"><i class="fa fa-check"></i><b>17.2.6</b> Dot-matrix methods</a></li>
<li class="chapter" data-level="17.2.7" data-path="s.html"><a href="s.html#dynamic-programming"><i class="fa fa-check"></i><b>17.2.7</b> Dynamic programming</a></li>
<li class="chapter" data-level="17.2.8" data-path="s.html"><a href="s.html#word-methods"><i class="fa fa-check"></i><b>17.2.8</b> Word methods</a></li>
<li class="chapter" data-level="17.2.9" data-path="s.html"><a href="s.html#multiple-sequence-alignment"><i class="fa fa-check"></i><b>17.2.9</b> Multiple sequence alignment</a></li>
<li class="chapter" data-level="17.2.10" data-path="s.html"><a href="s.html#phylogenetic-analysis"><i class="fa fa-check"></i><b>17.2.10</b> Phylogenetic analysis</a></li>
<li class="chapter" data-level="17.2.11" data-path="s.html"><a href="s.html#assessment-of-significance"><i class="fa fa-check"></i><b>17.2.11</b> Assessment of significance</a></li>
<li class="chapter" data-level="17.2.12" data-path="s.html"><a href="s.html#other-biological-uses"><i class="fa fa-check"></i><b>17.2.12</b> Other biological uses</a></li>
<li class="chapter" data-level="17.2.13" data-path="s.html"><a href="s.html#references-2"><i class="fa fa-check"></i><b>17.2.13</b> References</a></li>
</ul></li>
<li class="chapter" data-level="17.3" data-path="s.html"><a href="s.html#talking-glossary-u03-shotgun-sequencing-1-min"><i class="fa fa-check"></i><b>17.3</b> Talking Glossary U03: Shotgun sequencing (1 min)</a></li>
<li class="chapter" data-level="17.4" data-path="s.html"><a href="s.html#talking-glossary-sickle-cell-disease-4-min"><i class="fa fa-check"></i><b>17.4</b> Talking Glossary: Sickle Cell Disease (4 min)</a></li>
<li class="chapter" data-level="17.5" data-path="s.html"><a href="s.html#medline-snps-single-nucleotide-polymorphism"><i class="fa fa-check"></i><b>17.5</b> Medline: SNPs (single nucleotide polymorphism)</a></li>
<li class="chapter" data-level="17.6" data-path="s.html"><a href="s.html#talking-glossary-somatic-cells-0.75-min"><i class="fa fa-check"></i><b>17.6</b> Talking glossary: Somatic cells (0.75 min)</a></li>
<li class="chapter" data-level="17.7" data-path="s.html"><a href="s.html#sequence-annotation"><i class="fa fa-check"></i><b>17.7</b> Sequence annotation</a>
<ul>
<li class="chapter" data-level="17.7.1" data-path="s.html"><a href="s.html#process"><i class="fa fa-check"></i><b>17.7.1</b> Process</a></li>
</ul></li>
<li class="chapter" data-level="17.8" data-path="s.html"><a href="s.html#talking-glossary-substitution-mutation-1.75-min"><i class="fa fa-check"></i><b>17.8</b> Talking Glossary: Substitution mutation (1.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="t.html"><a href="t.html"><i class="fa fa-check"></i><b>18</b> T</a>
<ul>
<li class="chapter" data-level="18.1" data-path="t.html"><a href="t.html#taxa-taxon-and-clades-a-brief-primer"><i class="fa fa-check"></i><b>18.1</b> Taxa, Taxon, and clades: A Brief Primer</a>
<ul>
<li class="chapter" data-level="18.1.1" data-path="t.html"><a href="t.html#what-is-a-taxon"><i class="fa fa-check"></i><b>18.1.1</b> What is a “taxon”</a></li>
<li class="chapter" data-level="18.1.2" data-path="t.html"><a href="t.html#what-is-a-clade"><i class="fa fa-check"></i><b>18.1.2</b> What is a clade?</a></li>
<li class="chapter" data-level="18.1.3" data-path="t.html"><a href="t.html#further-reading"><i class="fa fa-check"></i><b>18.1.3</b> Further Reading:</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="t.html"><a href="t.html#talking-glossary-trait-0.75-min"><i class="fa fa-check"></i><b>18.2</b> Talking Glossary: Trait (0.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="u.html"><a href="u.html"><i class="fa fa-check"></i><b>19</b> U</a>
<ul>
<li class="chapter" data-level="19.1" data-path="u.html"><a href="u.html#uniprot"><i class="fa fa-check"></i><b>19.1</b> Uniprot</a>
<ul>
<li class="chapter" data-level="19.1.1" data-path="u.html"><a href="u.html#the-uniprot-consortium"><i class="fa fa-check"></i><b>19.1.1</b> The UniProt consortium</a></li>
<li class="chapter" data-level="19.1.2" data-path="u.html"><a href="u.html#the-roots-of-uniprot-databases"><i class="fa fa-check"></i><b>19.1.2</b> The roots of UniProt databases</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="20" data-path="v.html"><a href="v.html"><i class="fa fa-check"></i><b>20</b> V</a>
<ul>
<li class="chapter" data-level="20.1" data-path="v.html"><a href="v.html#talking-glossary-vector-2-min"><i class="fa fa-check"></i><b>20.1</b> Talking Glossary: Vector (2 min)</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="w.html"><a href="w.html"><i class="fa fa-check"></i><b>21</b> W</a>
<ul>
<li class="chapter" data-level="21.1" data-path="w.html"><a href="w.html#talking-glossary-x-chromosome-1.25-min"><i class="fa fa-check"></i><b>21.1</b> Talking Glossary: X-chromosome (1.25 min)</a></li>
<li class="chapter" data-level="21.2" data-path="w.html"><a href="w.html#talking-glossary-x-inactivation-lyonization-1.45-min"><i class="fa fa-check"></i><b>21.2</b> Talking Glossary: X-inactivation (“Lyonization”; 1.45 min))</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="x.html"><a href="x.html"><i class="fa fa-check"></i><b>22</b> X</a></li>
<li class="chapter" data-level="23" data-path="v-1.html"><a href="v-1.html"><i class="fa fa-check"></i><b>23</b> V</a>
<ul>
<li class="chapter" data-level="23.1" data-path="v-1.html"><a href="v-1.html#additional-glossaries"><i class="fa fa-check"></i><b>23.1</b> Additional Glossaries</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="z.html"><a href="z.html"><i class="fa fa-check"></i><b>24</b> Z</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Foundations of Biology and Environmental Science: An Open-Access Encyclopedia</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="d" class="section level1" number="5">
<h1><span class="header-section-number"> 5</span> D</h1>

<div id="talking-glossary-deletion-mutation-1.5-min" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Talking Glossary: Deletion mutation (1.5 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Deletion" class="uri">https://www.genome.gov/genetics-glossary/Deletion</a></p>
<p><strong>Abstract</strong>: “Deletion is a type of mutation involving the loss of genetic material. It can be small, involving a single missing DNA base pair, or large, involving a piece of a chromosome.”</p>
<p>Note: Among evolutionary biologists, “deletion” usually means deletion of a single base; larger deletions are specified, e.g. “chromosomal deletion.” In order to recognize a deletion, there’s needs to be some frame of reference, such as a consensus sequence, a reference genomic sequence, or other individuals in a multiple sequence alignment (MSA). Deletions of all sizes can be useful for constructing phylogenies.</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/deletion.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/deletion.jpg</a>
Example of single base and chromosomal deletion.</p>
<p><strong>Transcript:</strong> “Deletion really means that something is missing. And as a geneticist talking about deletion it means something is missing of the genetic material. And it can be something small, just a base pair; it can be something larger; it can be part of a gene; it can be even larger; it can be an entire gene; or yet larger again, it can be part of the chromosome. And depending upon what it is, you have to look at it in different ways. You can find a deletion in a chromosome just by doing a cytogenetic or chromosome analysis, or a deletion in a gene you can find out by sequencing the DNA. So when you have a deletion, depending upon the size, it can have different effects. What was the most surprising to me was that just by having a deletion of one base pair, you can have the most severe birth defect, and sometimes by missing an entire chromosome, you don’t even see all that much compared to just having a deletion of a small base pair. Different deletions can lead to different findings, and they can affect just behavior; they can affect how a child, how a person looks; they can affect a very severe problem that the child may die at birth; or they can affect something that just has to do with eye color, hair color, with weight or height of the person.”</p>
<p>Maximilian Muenke, M.D.</p>

</div>
<div id="talking-glossary-dna-sequencing-0.75-min" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> Talking Glossary: DNA sequencing (0.75 min)</h2>
<p><strong>Introduction</strong>: “DNA sequencing is a laboratory technique used to determine the exact sequence of bases (A, C, G, and T) in a DNA molecule. The DNA base sequence carries the information a cell needs to assemble protein and RNA molecules. DNA sequence information is important to scientists investigating the functions of genes. The technology of DNA sequencing was made faster and less expensive as a part of the Human Genome Project.”</p>
<p><strong>Transcript:</strong> “DNA consists of a linear string of nucleotides, or bases, for simplicity, referred to by the first letters of their chemical names–A, T, C and G. The process of deducing the order of nucleotides in DNA is called DNA sequencing. Since the DNA sequence confers information that the cell uses to make RNA molecules and proteins, establishing the sequence of DNA is key for understanding how genomes work. The technology for DNA sequencing was made faster and less expensive as a part of the Human Genome Project. And recent developments have profoundly increased the efficiency of DNA sequencing even further.”</p>
<p>Eric D. Green, M.D., Ph.D.</p>

</div>
<div id="sex-chromosome-dosage-compensation" class="section level2" number="5.3">
<h2><span class="header-section-number">5.3</span> Sex-Chromosome Dosage compensation</h2>
<p><a href="https://en.wikipedia.org/wiki/Sex-Chromosome_Dosage_compensation">Adapted from Wikipedia</a></p>
<p>Dosage compensation is the process by which organisms equalize the expression of genes between individuals with different sex chromosome karyotypes (e.g. XX versus XY). Across species, different “sexes” are often characterized by different types and numbers of sex chromosomes . In order to neutralize the large difference in gene dosage produced by differing numbers of sex chromosomes, various evolutionary branches have acquired various methods to equalize gene expression . Because sex chromosomes contain different numbers of genes , different species of organisms have developed different mechanisms to cope with this inequality. Replicating the actual gene is impossible; thus organisms instead equalize the expression from each gene. For example, in humans , XX individuals silence the transcription of one X chromosome of each pair, and transcribe all information from the other, expressed X chromosome. Thus, human XX individuals have the same number of expressed X-linked genes as do human XY individuals, with both having essentially one X chromosome per cell, from which to transcribe and express genes. This is called X-inactivation. In each XX cell, one of the two X chromosomes is randomly inactivated.</p>
<p>Other lineages have evolved different mechanisms to cope with the differences in gene copy numbers between the sexes that are observed on sex chromosomes. Some lineages have evolved upregulation mechanisms which restores expression of X-specific genes in the heterogametic sex (e.g. XY) to the same levels observed in the ancestor prior to the evolution of the sex chromosome.</p>
<div id="random-inactivation-of-one-x-in-xx-individuals" class="section level3" number="5.3.1">
<h3><span class="header-section-number">5.3.1</span> Random inactivation of one X in XX individuals</h3>
<p>One logical way to equalize gene expression amongst XX and XY that follow a XX/XY sex differentiation scheme would be to decrease or altogether eliminate the expression of one of the X chromosomes in an XX, (homogametic) individual, such that both XX and XY individuals express only one X chromosome. This is the case in many mammalian organisms, including humans and mice.[1 This process involves histone tail modifications, DNA methylation patterns, and reorganization of large-scale chromatin structure. In spite of these extensive modifications, not all genes along the X chromosome are subject to X-inactivation.] Because so many variants are tested, it is standard practice to require the p-value to be lower than 5×10−8 to consider a variant significant.</p>
<p>There are several variations to this case-control approach. A common alternative to case-control GWAS is the analysis of quantitative phenotypic data, e.g. height or biomarker concentrations or even gene expression. Likewise, alternative statistics designed for dominance or recessive penetrance patterns can be used (16] Calculations are typically done using bioinformatics software such as SNPTEST and PLINK, which also include support for many of these alternative statistics (15, 17). GWAS focuses on the effect of individual SNPs. However, it is also possible that complex interactions among two or more SNPs (epistasis) might contribute to complex diseases. Due to the potentially exponential number of interactions, detecting statistically significant interactions in GWAS data is both computationally and statistically challenging. This task has been tackled in existing publications that use algorithms inspired from data mining (18) Moreover, the researchers try to integrate GWA data with other biological data such as protein-protein interaction network to extract more informative results (19, 20).</p>
<!-- A key step in the majority of GWAS is the imputation of genotypes at SNPs not on the genotype chip used in the study (21] This process greatly increases the number of SNPs that can be tested for association, increases the power of the study, and facilitates meta-analysis of GWAS across distinct cohorts. Genotype imputation is carried out by statistical methods that combine the GWAS data together with a reference panel of haplotypes. These methods take advantage of sharing of haplotypes between individuals over short stretches of sequence to impute alleles. Existing software packages for genotype imputation include IMPUTE2,[22] Minimac, Beagle[23] and MaCH (24] -->
<p>In addition to the calculation of association, it is common to take into account any variables that could potentially confound the results. Sex and age are common examples of confounding variables. Moreover, it is also known that many genetic variations are associated with the geographical and historical populations in which the mutations first arose (25). Because of this association, studies must take account of the geographic and ethnic background of participants by controlling for what is called population stratification. If they fail to do so, these studies can produce false positive results (26)</p>
<p>After odds ratios and P-values have been calculated for all SNPs, a common approach is to create a Manhattan plot. In the context of GWAS, this plot shows the negative logarithm of the P-value as a function of genomic location. Thus the SNPs with the most significant association stand out on the plot, usually as stacks of points because of haploblock structure. Importantly, the P-value threshold for significance is corrected for multiple testing issues. The exact threshold varies by study,[27] but the conventional threshold is 5×10−8 to be significant in the face of hundreds of thousands to millions of tested SNPs (7, 16, 28] GWAS typically perform the first analysis in a discovery cohort, followed by validation of the most significant SNPs in an independent validation cohort (29]</p>
</div>
<div id="results" class="section level3" number="5.3.2">
<h3><span class="header-section-number">5.3.2</span> Results</h3>
<!-- Regional association plot, showing individual SNPs in the LDL receptor region and their association to LDL-cholesterol levels. This type of plot is similar to the Manhattan plot in the lead section, but for a more limited section of the genome. The haploblock structure is visualized with colour scale and the association level is given by the left Y-axis. The dot representing the rs73015013 SNP (in the top-middle) has a high Y-axis location because this SNP explains some of the variation in LDL-cholesterol (30] -->
<!-- Relationship between the minor allele frequency and the effect size of genome wide significant variants in a GWAS of height. -->
<!-- Attempts have been made at creating comprehensive catalogues of SNPs that have been identified from GWAS (31] As of 2009, SNPs associated with diseases are numbered in the thousands (32] -->
<p>An early GWAS, conducted in 2005, compared 96 patients with the eye disorder age-related macular degeneration (ARMD) with 50 healthy controls (33). It identified two SNPs with significantly altered allele frequencies between the two groups. These SNPs were located in the gene encoding complement factor H, which was an unexpected finding in the research of ARMD. These findings prompted further functional research towards therapeutical manipulation of the complement system in ARMD (34). Another landmark publication in the history of GWAS was the Wellcome Trust Case Control Consortium (WTCCC) study, the largest GWAS ever conducted at the time of its publication in 2007. The WTCCC included 14,000 cases of seven common diseases (~2,000 individuals for each of coronary heart disease, type 1 diabetes, type 2 diabetes, rheumatoid arthritis, Crohn’s disease, bipolar disorder, and hypertension) and 3,000 shared controls (15). This study was successful in uncovering many new disease genes underlying these diseases (15, 35).</p>
<p>Since these first landmark GWAS, there have been two general trends (36). One has been towards larger and larger sample sizes. In 2018, several genome-wide association studies are reaching a total sample size of over 1 million participants, including a study of insomnia containing 1.3 million individuals (38). The reason is the drive towards reliably detecting risk-SNPs that have ever smaller effects and lower allele frequencies. Another trend has been towards the use of more narrowly defined continous phenotypes, such as concentrations of blood lipids, proinsulin or similar biomarkers (39, 40). These are called intermediate phenotypes, and their analyses may be of value to functional research into biomarkers (41).</p>
<!-- A variation of GWAS uses participants that are first-degree relatives of people with a disease. This type of study has been named genome-wide association study by proxy (GWAX) (42] -->
<p>A central point of debate on GWAS has been that most of the SNP variations found by GWAS are associated with only a small increased risk of the disease, and have only a small predictive value.</p>
<!-- The median odds ratio is 1.33 per risk-SNP, with only a few showing odds ratios above 3.0 (1, 43] -->
<p>The effects sizes are considered small because they do not explain much of the heritable variation. This heritable variation is estimated from heritability studies based on monozygotic twins (44). For example, it is known that 80-90% of variance in height can be explained by hereditary differences, but GWAS only account for a minority of this variance (44). This is sometimes referred to as the problem of the <strong>missing heritability</strong> of GWAS.</p>
</div>
<div id="clinical-applications" class="section level3" number="5.3.3">
<h3><span class="header-section-number">5.3.3</span> Clinical applications</h3>
<p><strong>THIS SECTION IS OPTIONAL</strong></p>
<p>A challenge for future successful GWAS is to apply the findings in a way that accelerates drug and diagnostics development, including better integration of genetic studies into the drug-development process and a focus on the role of genetic variation in maintaining health as a blueprint for designing new drugs and diagnostics (45). Several studies have looked into the use of risk-SNP markers as a means of directly improving the accuracy of prognosis. Some have found that the accuracy of prognosis improves (46), while others report only minor benefits from this use (47). Generally, a problem with this direct approach is the small magnitudes of the effects observed. A small effect ultimately translates into a poor separation of cases and controls and thus only a small improvement of prognosis accuracy. An alternative application is therefore the potential for GWAS to elucidate pathophysiology (48).</p>
<p>One such success is related to identifying the genetic variant associated with response to anti-hepatitis C virus treatment. For genotype 1 hepatitis C treated with Pegylated interferon-alpha-2a or Pegylated interferon-alpha-2b combined with ribavirin, a GWAS (49) has shown that SNPs near the human IL28B gene, encoding interferon lambda 3, are associated with significant differences in response to the treatment. A later report demonstrated that the same genetic variants are also associated with the natural clearance of the genotype 1 hepatitis C virus (50). These major findings facilitated the development of personalized medicine and allowed physicians to customize medical decisions based on the patient’s genotype (51).</p>
<p>The goal of elucidating pathophysiology has also led to increased interest in the association between risk-SNPs and the gene <em>expression</em> of nearby genes, the so-called <strong>expression quantitative trait loci (eQTL)</strong> studies (52). The reason is that GWAS studies identify risk-SNPs, but not risk-genes, and specification of genes is one step closer towards actionable drug targets. As a result, major GWAS by 2011 typically included extensive eQTL analysis (53, 54, 55). One of the strongest eQTL effects observed for a GWA-identified risk SNP is the SORT1 locus (39). Functional follow up studies of this locus using small interfering RNA and gene knock-out mice have shed light on the metabolism of low-density lipoproteins, which have important clinical implications for cardiovascular disease (39, 56, 57).</p>
<div id="atrial-fibrillation" class="section level4" number="5.3.3.1">
<h4><span class="header-section-number">5.3.3.1</span> Atrial fibrillation</h4>
<p><strong>This section is optional</strong></p>
<p>For example, a meta-analysis accomplished in 2018 revealed the discovery of 70 new loci associated with atrial fibrillation. It has been identified different variants associated with transcription factor coding-genes, such as TBX3 and TBX5, NKX2-5 o PITX2, which are involved in cardiac conduction regulation, in ionic channel modulation and cardiac development. It was also identified new genes involved in tachycardia (CASQ2) or associated with alteration of cardiac muscle cell communication (PKP2) (58)</p>
</div>
<div id="schizophrenia" class="section level4" number="5.3.3.2">
<h4><span class="header-section-number">5.3.3.2</span> Schizophrenia</h4>
<p><strong>This section is optional</strong></p>
<p>While there is some research using a High-Precision Protein Interaction Prediction (HiPPIP) computational model that discovered 504 new protein-protein interactions (PPIs) associated with genes linked to schizophrenia (59,60) the evidence supporting the genetic basis of schizophrenia is actually controversial and may suffer from some of the limitation of this method of study (61]</p>
</div>
</div>
<div id="agricultural-applications" class="section level3" number="5.3.4">
<h3><span class="header-section-number">5.3.4</span> Agricultural applications</h3>
<div id="plant-growth-stages-and-yield-components" class="section level4" number="5.3.4.1">
<h4><span class="header-section-number">5.3.4.1</span> Plant growth stages and yield components</h4>
<p><strong>This section is optional</strong></p>
<p>GWAS act as an important tool in plant breeding. With large genotyping and phenotyping data, GWAS are powerful in analyzing complex inheritance modes of traits that are important yield components such as number of grains per spike, weight of each grain and plant structure. In a study on GWAS in spring wheat, GWAS have revealed a strong correlation of grain production with booting data, biomass and number of grains per spike (62).</p>
</div>
<div id="plant-pathogens" class="section level4" number="5.3.4.2">
<h4><span class="header-section-number">5.3.4.2</span> Plant pathogens</h4>
<p><strong>This section is optional</strong></p>
<p>The emergences of plant pathogens have posed serious threats to plant health and biodiversity. Under this consideration, identification of wild types that have the natural resistance to certain pathogens could be of vital importance. Furthermore, we need to predict which alleles are associated with the resistance. GWAS is a powerful tool to detect the relationships of certain variants and the resistance to the plant pathogen, which is beneficial for developing new pathogen-resisted cultivars (63).</p>
</div>
</div>
<div id="limitations" class="section level3" number="5.3.5">
<h3><span class="header-section-number">5.3.5</span> Limitations</h3>
<div id="limitations---common-errors" class="section level4" number="5.3.5.1">
<h4><span class="header-section-number">5.3.5.1</span> Limitations - common errors</h4>
<p><strong>This section is optional</strong></p>
<p>GWAS have several issues and limitations that can be taken care of through proper quality control and study setup. Lack of well defined case and control groups, insufficient sample size, control for <strong>multiple testing</strong> and control for <strong>population stratification</strong> are common problems (2). Particularly the statistical issue of multiple testing wherein it has been noted that “the GWA approach can be problematic because the massive number of statistical tests performed presents an unprecedented potential for false-positive results” (2). Ignoring these correctible issues has been cited as contributing to a general sense of problems with the GWAS methodology (64). In addition to easily correctible problems such as these, some more subtle but important issues have surfaced. A high-profile GWA study that investigated individuals with very long life spans to identify SNPs associated with longevity is an example of this (65). The publication came under scrutiny because of a discrepancy between the type of genotyping array in the case and control group, which caused several SNPs to be falsely highlighted as associated with longevity (66). The study was subsequently retracted (67), but a modified manuscript was later published (68).</p>
</div>
<div id="limitations---fundamental-problems-read-this-ection" class="section level4" number="5.3.5.2">
<h4><span class="header-section-number">5.3.5.2</span> Limitations - fundamental problems (READ THIS ECTION)</h4>
<p><strong>READ THIS SECTION</strong></p>
<p>The fundamental assumptions of GWAS and have attracted fundamental criticism, mainly because of their assumption that common genetic variation plays a large role in explaining the heritable variation of common disease (69). Indeed, it has been estimated that for most conditions the SNP heritability attributable to common SNPs is &lt;0.05 (70). This aspect of GWAS has attracted the criticism that, although it could not have been known prospectively, GWAS were ultimately not worth the expenditure (48). GWAS also face criticism that the broad variation of individual responses or compensatory mechanisms to a disease state cancel out and mask potential genes or causal variants associated with the disease (71). Additionally, GWAS identify candidate risk variants for the population from which their analysis is performed, and with most GWAS stemming from European databases, there is a lack of translation of the identified risk variants to other non-European populations (72] Alternative strategies suggested involve linkage analysis (73, 74] More recently, the rapidly decreasing price of complete genome sequencing have also provided a realistic alternative to genotyping array-based GWAS. It can be discussed if the use of this new technique is still referred to as a GWA study, but high-throughput sequencing does have potential to side-step some of the shortcomings of non-sequencing GWA (75]</p>
</div>
</div>
</div>
<div id="fine-mapping" class="section level2" number="5.4">
<h2><span class="header-section-number">5.4</span> Fine-mapping</h2>
<p><strong>THIS SECTION IS OPTIONAL</strong></p>
<p>Genotyping arrays designed for GWAS rely on linkage disequilibrium to provide coverage of the entire genome by genotyping a subset of variants. Because of this, the reported associated variants are unlikely to be the actual causal variants. Associated regions can contain hundreds of variants spanning large regions and encompassing many different genes, making the biological interpretation of GWAS loci more difficult. Fine-mapping is a process to refine these lists of associated variants to a credible set most likely to include the causal variant.</p>
<p>Fine-mapping requires all variants in the associated region to have been genotyped or imputed (dense coverage), very stringent quality control resulting in high-quality genotypes, and large sample sizes sufficient in separating out highly correlated signals. There are several different methods to perform fine-mapping, and all methods produce a posterior probability that a variant in that locus is causal. Because the requirements are often difficult to satisfy, there are still limited examples of these methods being more generally applied.</p>
<!-- See also -->
<!-- Association mapping -->
<!-- Epidemiology -->
<!-- Gene–environment interaction -->
<!-- Genomics -->
<!-- Linkage disequilibrium -->
<!-- Molecular epidemiology -->
<!-- Polygenic score -->

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="c.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="h.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["openbio.pdf", "openbio.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
