<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title> 8 G | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</title>
  <meta name="description" content="A collection of open-access and/or free resources on biology and environmental science." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content=" 8 G | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content=" 8 G | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  
  <meta name="twitter:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

<meta name="author" content="Compiled and edited by Nathan Brouwer" />


<meta name="date" content="2021-09-03" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="f.html"/>
<link rel="next" href="h.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Biology for everyone</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="a.html"><a href="a.html"><i class="fa fa-check"></i><b>2</b> A</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a.html"><a href="a.html#alignments"><i class="fa fa-check"></i><b>2.1</b> Alignments</a></li>
<li class="chapter" data-level="2.2" data-path="a.html"><a href="a.html#talking-glossary-allele-0.5-min"><i class="fa fa-check"></i><b>2.2</b> Talking Glossary: Allele (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="b.html"><a href="b.html"><i class="fa fa-check"></i><b>3</b> B</a>
<ul>
<li class="chapter" data-level="3.1" data-path="b.html"><a href="b.html#talking-glossary-bioinformatics"><i class="fa fa-check"></i><b>3.1</b> Talking glossary: Bioinformatics</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="c.html"><a href="c.html"><i class="fa fa-check"></i><b>4</b> C</a>
<ul>
<li class="chapter" data-level="4.1" data-path="c.html"><a href="c.html#clade"><i class="fa fa-check"></i><b>4.1</b> Clade</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="c.html"><a href="c.html#clades-and-phylogenetic-trees"><i class="fa fa-check"></i><b>4.1.1</b> Clades and phylogenetic trees</a></li>
<li class="chapter" data-level="4.1.2" data-path="c.html"><a href="c.html#terminology"><i class="fa fa-check"></i><b>4.1.2</b> Terminology</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="c.html"><a href="c.html#in-popular-culture"><i class="fa fa-check"></i><b>4.2</b> In popular culture</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="c.html"><a href="c.html#references"><i class="fa fa-check"></i><b>4.2.1</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="c.html"><a href="c.html#talking-glossary-cloning-0.5-min"><i class="fa fa-check"></i><b>4.3</b> Talking Glossary: Cloning (0.5 min)</a></li>
<li class="chapter" data-level="4.4" data-path="c.html"><a href="c.html#talking-glossary-codominance-0.5-min"><i class="fa fa-check"></i><b>4.4</b> Talking Glossary: Codominance (0.5 min)</a></li>
<li class="chapter" data-level="4.5" data-path="c.html"><a href="c.html#comparative-genomics"><i class="fa fa-check"></i><b>4.5</b> Comparative genomics</a></li>
<li class="chapter" data-level="4.6" data-path="c.html"><a href="c.html#talking-glossary-contig-0.75-min"><i class="fa fa-check"></i><b>4.6</b> Talking Glossary: Contig (0.75 min)</a></li>
<li class="chapter" data-level="4.7" data-path="c.html"><a href="c.html#talking-glossary---foundations-1-review-crossing-over"><i class="fa fa-check"></i><b>4.7</b> Talking Glossary - Foundations 1 Review: Crossing over</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="d.html"><a href="d.html"><i class="fa fa-check"></i><b>5</b> D</a>
<ul>
<li class="chapter" data-level="5.1" data-path="d.html"><a href="d.html#talking-glossary-deletion-mutation-1.5-min"><i class="fa fa-check"></i><b>5.1</b> Talking Glossary: Deletion mutation (1.5 min)</a></li>
<li class="chapter" data-level="5.2" data-path="d.html"><a href="d.html#talking-glossary-dna-sequencing-0.75-min"><i class="fa fa-check"></i><b>5.2</b> Talking Glossary: DNA sequencing (0.75 min)</a></li>
<li class="chapter" data-level="5.3" data-path="d.html"><a href="d.html#sex-chromosome-dosage-compensation"><i class="fa fa-check"></i><b>5.3</b> Sex-Chromosome Dosage compensation</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="d.html"><a href="d.html#random-inactivation-of-one-x-in-xx-individuals"><i class="fa fa-check"></i><b>5.3.1</b> Random inactivation of one X in XX individuals</a></li>
<li class="chapter" data-level="5.3.2" data-path="d.html"><a href="d.html#two-fold-increased-transcription-of-a-single-x-in-xy-individuals"><i class="fa fa-check"></i><b>5.3.2</b> Two-fold increased transcription of a single X in XY individuals</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="d.html"><a href="d.html#talking-glossary-duplication-0.75-min"><i class="fa fa-check"></i><b>5.4</b> Talking Glossary: Duplication (0.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="b-1.html"><a href="b-1.html"><i class="fa fa-check"></i><b>6</b> B</a>
<ul>
<li class="chapter" data-level="6.1" data-path="b-1.html"><a href="b-1.html#talking-glossary-epigenetics"><i class="fa fa-check"></i><b>6.1</b> Talking Glossary: Epigenetics</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="f.html"><a href="f.html"><i class="fa fa-check"></i><b>7</b> F</a>
<ul>
<li class="chapter" data-level="7.1" data-path="f.html"><a href="f.html#talking-glossary-founder-effect-a-type-of-genetic-bottleneck-1-min"><i class="fa fa-check"></i><b>7.1</b> Talking glossary: Founder effect (A type of genetic bottleneck; 1 min)</a></li>
<li class="chapter" data-level="7.2" data-path="f.html"><a href="f.html#talking-glossary-frameshift-mutation-1.5-min"><i class="fa fa-check"></i><b>7.2</b> Talking Glossary: Frameshift mutation (1.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="g.html"><a href="g.html"><i class="fa fa-check"></i><b>8</b> G</a>
<ul>
<li class="chapter" data-level="8.1" data-path="g.html"><a href="g.html#talking-glossary-u03-gel-electrophoresis-3-min"><i class="fa fa-check"></i><b>8.1</b> Talking Glossary U03: Gel Electrophoresis (“3 min)</a></li>
<li class="chapter" data-level="8.2" data-path="g.html"><a href="g.html#talking-glossary-gene-amplification-1.5-min"><i class="fa fa-check"></i><b>8.2</b> Talking Glossary: Gene amplification (1.5 min)</a></li>
<li class="chapter" data-level="8.3" data-path="g.html"><a href="g.html#talking-glossary-gene-pool-0.75-min"><i class="fa fa-check"></i><b>8.3</b> Talking Glossary: Gene Pool (0.75 min)</a></li>
<li class="chapter" data-level="8.4" data-path="g.html"><a href="g.html#gene-prediction"><i class="fa fa-check"></i><b>8.4</b> Gene prediction</a></li>
<li class="chapter" data-level="8.5" data-path="g.html"><a href="g.html#genbank"><i class="fa fa-check"></i><b>8.5</b> GenBank</a></li>
<li class="chapter" data-level="8.6" data-path="g.html"><a href="g.html#genetic-code"><i class="fa fa-check"></i><b>8.6</b> The genetic code - Overview</a>
<ul>
<li class="chapter" data-level="8.6.1" data-path="g.html"><a href="g.html#the-genetics-code"><i class="fa fa-check"></i><b>8.6.1</b> The genetics code</a></li>
<li class="chapter" data-level="8.6.2" data-path="g.html"><a href="g.html#the-central-dogma-dna-encodes-rna-rna-encodes-protein"><i class="fa fa-check"></i><b>8.6.2</b> The Central Dogma: DNA Encodes RNA; RNA Encodes Protein</a></li>
<li class="chapter" data-level="8.6.3" data-path="g.html"><a href="g.html#the-genetic-code-is-degenerate-and-universal"><i class="fa fa-check"></i><b>8.6.3</b> The Genetic Code Is “Degenerate” and Universal</a></li>
<li class="chapter" data-level="8.6.4" data-path="g.html"><a href="g.html#start-and-stop-codons"><i class="fa fa-check"></i><b>8.6.4</b> Start and stop codons</a></li>
<li class="chapter" data-level="8.6.5" data-path="g.html"><a href="g.html#summary"><i class="fa fa-check"></i><b>8.6.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="8.7" data-path="g.html"><a href="g.html#genetic-distance"><i class="fa fa-check"></i><b>8.7</b> Genetic distance</a></li>
<li class="chapter" data-level="8.8" data-path="g.html"><a href="g.html#talking-glossary-genetic-drift-0.75-min"><i class="fa fa-check"></i><b>8.8</b> Talking Glossary: Genetic drift (0.75 min)</a></li>
<li class="chapter" data-level="8.9" data-path="g.html"><a href="g.html#talking-glossary-genetic-engineering-1.5-min"><i class="fa fa-check"></i><b>8.9</b> Talking Glossary: Genetic engineering (1.5 min)</a></li>
<li class="chapter" data-level="8.10" data-path="g.html"><a href="g.html#genome-assembly"><i class="fa fa-check"></i><b>8.10</b> Genome assembly</a></li>
<li class="chapter" data-level="8.11" data-path="g.html"><a href="g.html#talking-glossary-genomics"><i class="fa fa-check"></i><b>8.11</b> Talking Glossary: Genomics</a></li>
<li class="chapter" data-level="8.12" data-path="g.html"><a href="g.html#talking-glossary-genotype"><i class="fa fa-check"></i><b>8.12</b> Talking Glossary: Genotype</a></li>
<li class="chapter" data-level="8.13" data-path="g.html"><a href="g.html#talking-glossary-germline-1.25-min"><i class="fa fa-check"></i><b>8.13</b> Talking Glossary: Germline (1.25 min)</a></li>
<li class="chapter" data-level="8.14" data-path="g.html"><a href="g.html#guild"><i class="fa fa-check"></i><b>8.14</b> Guild</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="h.html"><a href="h.html"><i class="fa fa-check"></i><b>9</b> H</a>
<ul>
<li class="chapter" data-level="9.1" data-path="h.html"><a href="h.html#talking-glossary-haploid-1.25-min"><i class="fa fa-check"></i><b>9.1</b> Talking Glossary: Haploid (1.25 min)</a></li>
<li class="chapter" data-level="9.2" data-path="h.html"><a href="h.html#talking-glossary-heterozygous"><i class="fa fa-check"></i><b>9.2</b> Talking Glossary: Heterozygous</a></li>
<li class="chapter" data-level="9.3" data-path="h.html"><a href="h.html#homologyl-when-things-are-homolgous-and-analogous"><i class="fa fa-check"></i><b>9.3</b> Homology:L When things are homolgous AND analogous!</a></li>
<li class="chapter" data-level="9.4" data-path="h.html"><a href="h.html#talking-glossary-homozygous"><i class="fa fa-check"></i><b>9.4</b> Talking Glossary: Homozygous</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="i.html"><a href="i.html"><i class="fa fa-check"></i><b>10</b> I</a>
<ul>
<li class="chapter" data-level="10.1" data-path="i.html"><a href="i.html#talking-glossary-insertion-mutation-1.25-min"><i class="fa fa-check"></i><b>10.1</b> Talking Glossary: Insertion mutation (1.25 min)</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="j.html"><a href="j.html"><i class="fa fa-check"></i><b>11</b> J</a>
<ul>
<li class="chapter" data-level="11.1" data-path="j.html"><a href="j.html#jove-videos"><i class="fa fa-check"></i><b>11.1</b> JoVe Videos</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="j.html"><a href="j.html#biotechnology"><i class="fa fa-check"></i><b>11.1.1</b> Biotechnology</a></li>
<li class="chapter" data-level="11.1.2" data-path="j.html"><a href="j.html#macromolecules"><i class="fa fa-check"></i><b>11.1.2</b> Macromolecules</a></li>
<li class="chapter" data-level="11.1.3" data-path="j.html"><a href="j.html#cellular-respiration"><i class="fa fa-check"></i><b>11.1.3</b> Cellular respiration</a></li>
<li class="chapter" data-level="11.1.4" data-path="j.html"><a href="j.html#population-and-community-ecology"><i class="fa fa-check"></i><b>11.1.4</b> Population and community ecology</a></li>
<li class="chapter" data-level="11.1.5" data-path="j.html"><a href="j.html#population-genetics"><i class="fa fa-check"></i><b>11.1.5</b> Population genetics</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="12" data-path="k.html"><a href="k.html"><i class="fa fa-check"></i><b>12</b> K</a></li>
<li class="chapter" data-level="13" data-path="l.html"><a href="l.html"><i class="fa fa-check"></i><b>13</b> L</a>
<ul>
<li class="chapter" data-level="13.1" data-path="l.html"><a href="l.html#talking-glossary-locus-1-min"><i class="fa fa-check"></i><b>13.1</b> Talking Glossary: Locus (1 min)</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="m.html"><a href="m.html"><i class="fa fa-check"></i><b>14</b> M</a>
<ul>
<li class="chapter" data-level="14.1" data-path="m.html"><a href="m.html#talking-glossary-missense-mutation-1.25-min"><i class="fa fa-check"></i><b>14.1</b> Talking Glossary: Missense Mutation (1.25 min)</a></li>
<li class="chapter" data-level="14.2" data-path="m.html"><a href="m.html#talking-glossary-mitochondrial-dna-1.5-min"><i class="fa fa-check"></i><b>14.2</b> Talking Glossary: Mitochondrial DNA (1.5 min)</a></li>
<li class="chapter" data-level="14.3" data-path="m.html"><a href="m.html#talking-glossary-mutagen-0.5-min"><i class="fa fa-check"></i><b>14.3</b> Talking Glossary: Mutagen (0.5 min)</a></li>
<li class="chapter" data-level="14.4" data-path="m.html"><a href="m.html#talking-glossary-mutation-0.5-min"><i class="fa fa-check"></i><b>14.4</b> Talking Glossary: Mutation (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="n.html"><a href="n.html"><i class="fa fa-check"></i><b>15</b> N</a>
<ul>
<li class="chapter" data-level="15.1" data-path="n.html"><a href="n.html#talking-glossary-non-coding-dna-1-min"><i class="fa fa-check"></i><b>15.1</b> Talking Glossary: Non-coding DNA (1 min)</a></li>
<li class="chapter" data-level="15.2" data-path="n.html"><a href="n.html#talking-glossary-nonsene-mutation"><i class="fa fa-check"></i><b>15.2</b> Talking Glossary: Nonsene mutation</a></li>
<li class="chapter" data-level="15.3" data-path="n.html"><a href="n.html#nucleic-acids-review"><i class="fa fa-check"></i><b>15.3</b> Nucleic Acids - Overview</a>
<ul>
<li class="chapter" data-level="15.3.1" data-path="n.html"><a href="n.html#dna-and-rna"><i class="fa fa-check"></i><b>15.3.1</b> DNA and RNA</a></li>
<li class="chapter" data-level="15.3.2" data-path="n.html"><a href="n.html#dna-double-helix-structure"><i class="fa fa-check"></i><b>15.3.2</b> DNA Double-Helix Structure</a></li>
<li class="chapter" data-level="15.3.3" data-path="n.html"><a href="n.html#rna"><i class="fa fa-check"></i><b>15.3.3</b> RNA</a></li>
<li class="chapter" data-level="15.3.4" data-path="n.html"><a href="n.html#protein-function"><i class="fa fa-check"></i><b>15.3.4</b> Protein function</a></li>
<li class="chapter" data-level="15.3.5" data-path="n.html"><a href="n.html#summary-1"><i class="fa fa-check"></i><b>15.3.5</b> Summary</a></li>
<li class="chapter" data-level="15.3.6" data-path="n.html"><a href="n.html#glossary"><i class="fa fa-check"></i><b>15.3.6</b> Glossary</a></li>
<li class="chapter" data-level="15.3.7" data-path="n.html"><a href="n.html#contributors-and-attributions"><i class="fa fa-check"></i><b>15.3.7</b> Contributors and Attributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16" data-path="o.html"><a href="o.html"><i class="fa fa-check"></i><b>16</b> O</a>
<ul>
<li class="chapter" data-level="16.1" data-path="o.html"><a href="o.html#talking-glossary-open-reading-frame-definition-3-min"><i class="fa fa-check"></i><b>16.1</b> Talking Glossary : Open reading frame definition (3 min)</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="p.html"><a href="p.html"><i class="fa fa-check"></i><b>17</b> P</a>
<ul>
<li class="chapter" data-level="17.1" data-path="p.html"><a href="p.html#talking-glossary-pcr---the-polymerase-chain-reaction-0.5-min"><i class="fa fa-check"></i><b>17.1</b> Talking Glossary: PCR - The Polymerase Chain Reaction (0.5 min)</a></li>
<li class="chapter" data-level="17.2" data-path="p.html"><a href="p.html#talking-glossary-phenotype"><i class="fa fa-check"></i><b>17.2</b> Talking Glossary: Phenotype</a></li>
<li class="chapter" data-level="17.3" data-path="p.html"><a href="p.html#phylogenetics-vocab"><i class="fa fa-check"></i><b>17.3</b> Phylogenetics vocab</a></li>
<li class="chapter" data-level="17.4" data-path="p.html"><a href="p.html#talking-glossary-plasmid-1-min"><i class="fa fa-check"></i><b>17.4</b> Talking Glossary: Plasmid (1 min)</a></li>
<li class="chapter" data-level="17.5" data-path="p.html"><a href="p.html#talking-glossary-point-mutation-0.5-min"><i class="fa fa-check"></i><b>17.5</b> Talking Glossary: Point mutation (0.5 min)</a></li>
<li class="chapter" data-level="17.6" data-path="p.html"><a href="p.html#talking-glossary-polymorphism-1-min"><i class="fa fa-check"></i><b>17.6</b> Talking Glossary: Polymorphism (1 min)</a></li>
<li class="chapter" data-level="17.7" data-path="p.html"><a href="p.html#talking-glossary-u03-primer-0.5-min"><i class="fa fa-check"></i><b>17.7</b> Talking Glossary U03: Primer (0.5 min)</a></li>
<li class="chapter" data-level="17.8" data-path="p.html"><a href="p.html#talking-glossary-promoter-1.5-min"><i class="fa fa-check"></i><b>17.8</b> Talking Glossary: Promoter (1.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="q.html"><a href="q.html"><i class="fa fa-check"></i><b>18</b> Q</a></li>
<li class="chapter" data-level="19" data-path="r.html"><a href="r.html"><i class="fa fa-check"></i><b>19</b> R</a>
<ul>
<li class="chapter" data-level="19.1" data-path="r.html"><a href="r.html#talking-glossary-recessive"><i class="fa fa-check"></i><b>19.1</b> Talking Glossary: Recessive</a></li>
<li class="chapter" data-level="19.2" data-path="r.html"><a href="r.html#talking-glossary-recombinant-dna-0.5-min"><i class="fa fa-check"></i><b>19.2</b> Talking Glossary: Recombinant DNA (0.5 min)</a></li>
<li class="chapter" data-level="19.3" data-path="r.html"><a href="r.html#talking-glossary-repressor-1-min"><i class="fa fa-check"></i><b>19.3</b> Talking Glossary: Repressor (1 min)</a></li>
<li class="chapter" data-level="19.4" data-path="r.html"><a href="r.html#talking-glossary-restriction-enzyme-0.5-min"><i class="fa fa-check"></i><b>19.4</b> Talking Glossary: Restriction Enzyme (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="s.html"><a href="s.html"><i class="fa fa-check"></i><b>20</b> S</a>
<ul>
<li class="chapter" data-level="20.1" data-path="s.html"><a href="s.html#talking-glossary-u03-shotgun-sequencing-1-min"><i class="fa fa-check"></i><b>20.1</b> Talking Glossary U03: Shotgun sequencing (1 min)</a></li>
<li class="chapter" data-level="20.2" data-path="s.html"><a href="s.html#talking-glossary-sickle-cell-disease-4-min"><i class="fa fa-check"></i><b>20.2</b> Talking Glossary: Sickle Cell Disease (4 min)</a></li>
<li class="chapter" data-level="20.3" data-path="s.html"><a href="s.html#talking-glossary-somatic-cells-0.75-min"><i class="fa fa-check"></i><b>20.3</b> Talking glossary: Somatic cells (0.75 min)</a></li>
<li class="chapter" data-level="20.4" data-path="s.html"><a href="s.html#sequence-annotation"><i class="fa fa-check"></i><b>20.4</b> Sequence annotation</a>
<ul>
<li class="chapter" data-level="20.4.1" data-path="s.html"><a href="s.html#process"><i class="fa fa-check"></i><b>20.4.1</b> Process</a></li>
</ul></li>
<li class="chapter" data-level="20.5" data-path="s.html"><a href="s.html#talking-glossary-substitution-mutation-1.75-min"><i class="fa fa-check"></i><b>20.5</b> Talking Glossary: Substitution mutation (1.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="t.html"><a href="t.html"><i class="fa fa-check"></i><b>21</b> T</a>
<ul>
<li class="chapter" data-level="21.1" data-path="t.html"><a href="t.html#taxa-taxon-and-clades-a-brief-primer"><i class="fa fa-check"></i><b>21.1</b> Taxa, Taxon, and clades: A Brief Primer</a></li>
<li class="chapter" data-level="21.2" data-path="t.html"><a href="t.html#what-is-a-taxon"><i class="fa fa-check"></i><b>21.2</b> What is a “taxon”</a></li>
<li class="chapter" data-level="21.3" data-path="t.html"><a href="t.html#what-is-a-clade"><i class="fa fa-check"></i><b>21.3</b> What is a clade?</a></li>
<li class="chapter" data-level="21.4" data-path="t.html"><a href="t.html#further-reading"><i class="fa fa-check"></i><b>21.4</b> Further Reading:</a></li>
<li class="chapter" data-level="21.5" data-path="t.html"><a href="t.html#talking-glossary-trait-0.75-min"><i class="fa fa-check"></i><b>21.5</b> Talking Glossary: Trait (0.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="u.html"><a href="u.html"><i class="fa fa-check"></i><b>22</b> U</a></li>
<li class="chapter" data-level="23" data-path="v.html"><a href="v.html"><i class="fa fa-check"></i><b>23</b> V</a>
<ul>
<li class="chapter" data-level="23.1" data-path="v.html"><a href="v.html#talking-glossary-vector-2-min"><i class="fa fa-check"></i><b>23.1</b> Talking Glossary: Vector (2 min)</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="w.html"><a href="w.html"><i class="fa fa-check"></i><b>24</b> W</a>
<ul>
<li class="chapter" data-level="24.1" data-path="w.html"><a href="w.html#talking-glossary-x-chromosome-1.25-min"><i class="fa fa-check"></i><b>24.1</b> Talking Glossary: X-chromosome (1.25 min)</a></li>
<li class="chapter" data-level="24.2" data-path="w.html"><a href="w.html#talking-glossary-x-inactivation-lyonization-1.45-min"><i class="fa fa-check"></i><b>24.2</b> Talking Glossary: X-inactivation (“Lyonization”; 1.45 min))</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="x.html"><a href="x.html"><i class="fa fa-check"></i><b>25</b> X</a></li>
<li class="chapter" data-level="26" data-path="v-1.html"><a href="v-1.html"><i class="fa fa-check"></i><b>26</b> V</a>
<ul>
<li class="chapter" data-level="26.1" data-path="v-1.html"><a href="v-1.html#additional-glossaries"><i class="fa fa-check"></i><b>26.1</b> Additional Glossaries</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="z.html"><a href="z.html"><i class="fa fa-check"></i><b>27</b> Z</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="g" class="section level1" number="8">
<h1><span class="header-section-number"> 8</span> G</h1>

<div id="talking-glossary-u03-gel-electrophoresis-3-min" class="section level2" number="8.1">
<h2><span class="header-section-number">8.1</span> Talking Glossary U03: Gel Electrophoresis (“3 min)</h2>
<p>AKA “running a gel</p>
<p>National Human Genome Research Institute
<a href="https://www.genome.gov/genetics-glossary/Electrophoresis" class="uri">https://www.genome.gov/genetics-glossary/Electrophoresis</a></p>
<p>Abstract: “Electrophoresis is a laboratory technique used to separate DNA, RNA, or protein molecules based on their size and electrical charge. An electric current is used to move molecules to be separated through a gel. Pores in the gel work like a sieve, allowing smaller molecules to move faster than larger molecules. The conditions used during electrophoresis can be adjusted to separate molecules in a desired size range.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/electrophoresis.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/electrophoresis.mp3</a></p>
<p>Image (Wikipedia): <a href="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a6/Gel_electrophoresis_apparatus.JPG/188px-Gel_electrophoresis_apparatus.JPG" class="uri">https://upload.wikimedia.org/wikipedia/commons/thumb/a/a6/Gel_electrophoresis_apparatus.JPG/188px-Gel_electrophoresis_apparatus.JPG</a></p>
<p><strong>Transcript</strong>: Electrophoresis is a very broadly used technique which, fundamentally, applies electric current to biological molecules, whether–they’re usually DNA, they can be protein or RNA, too…and separates these fragments into pieces which are larger or smaller. It’s used in a variety of applications… Everything from forensics for determining the identity of individuals that may have been involved in a crime, by linking their DNA pattern, their electrophoresis pattern, to one that’s in a database. The whole basis by which the human genome was done is by something called capillary electrophoresis, by separating DNA into shorter pieces and then running them on these electrophoresis gels which allow the patterns of As, Cs, Ts, and Gs to be elucidated. They’re also very important in protein research, and then genetic mutation research, because when proteins or DNA are mutated, they are frequently longer or shorter, and they therefore show up on an electrophoresis gel differently than normal, so many diagnostic tests are still done using electrophoresis, so it’s a very widely used basic research technique, was very important for the understanding of gene and protein function, but it’s now gotten into the area of clinical diagnostics and forensics as well. Electrophoresis is usually done in what looks like a box which has a positive charge at one end and a negative charge at the other. And as we all learned in basic physics, when you put a charged molecule into an environment like that, the negative molecules go to the positive charge, and vice versa. In looking at proteins in a gel, in one of these boxes, you usually take the entire protein, and you’re looking at the entire length of the protein and seeing how big it is, and the bigger it is, the shorter it will migrate into the gel, so that the small proteins will end up at the bottom of the gel, because they have migrated the farthest, and the biggest ones will wind up staying at the top. In the case of DNA, DNA’s a very long molecule, so you wouldn’t want to run, for the most part, a whole DNA molecule from a cell onto a gel. It’s just so big that it would never get into the gel, so what scientists do, and what people do in classrooms these days, is to chop up that DNA using things like inscription enzymes, which chop up the DNA into more manageable pieces in a reproducible way. And then those pieces, depending on how big the pieces are, migrate more or less into the gel father down the box from top to the bottom.</p>
<p>Christopher P. Austin, M.D.</p>

</div>
<div id="talking-glossary-gene-amplification-1.5-min" class="section level2" number="8.2">
<h2><span class="header-section-number">8.2</span> Talking Glossary: Gene amplification (1.5 min)</h2>
<p>National Human Genome Research Institute</p>
<p><strong>Introduction</strong>: “Gene amplification is an increase in the number of copies of a gene sequence. ….. The term also can refer to polymerase chain reaction (PCR), a laboratory technique that is used by scientists to amplify gene sequences in a test tube.”</p>
<p>Note: Cloning and gene amplification are related concepts and are used interchangeably by some people. In this course its most important to know generally what the terms are referring to (increasing the number of copies of gene); there will not be a question where, for example “cloning” is a correct answer but “gene amplification” is wrong.</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/gene_amplification.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/gene_amplification.mp3</a></p>
<p>Transcript: “Gene amplification is one of those clauses that means different things to different people, depending on how they look at it. At its simplest level, it means an increase in the number of copies of a gene sequence. Now, cancer cells sometimes make multiple copies of genes in response to erroneous signals from other cells or because of signals they’re getting from the environment. So in that sense we think of it as a pretty negative term; it’s heralding something that’s not really supposed to have happened in the body that’s clearly going on. But it’s also just a very scientific term that doesn’t have either a positive or a negative connotation. Because it refers to a laboratory technique called polymerase chain reaction, or PCR, and that’s a technique which scientists can amplify a single gene in a test tube against the background of the three billion other bases that are floating around inside the test tube. That technique, PCR, or polymerase chain reaction, is really the backbone of a great deal of molecular biology that’s being done today. As a matter of fact, the individual who discovered it ended up winning a Nobel Prize for it. So when you hear the phrase”gene amplification”, think first about the context, and then think about the specifics.”</p>
<p><a href="https://www.genome.gov/staff/Elaine-A-Ostrander-PhD">Elaine A. Ostrander</a>, Ph.D.</p>

</div>
<div id="talking-glossary-gene-pool-0.75-min" class="section level2" number="8.3">
<h2><span class="header-section-number">8.3</span> Talking Glossary: Gene Pool (0.75 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Gene-Pool" class="uri">https://www.genome.gov/genetics-glossary/Gene-Pool</a> (Links to an external site.)</p>
<p><strong>Abstract</strong>: “A gene pool is the total genetic diversity found within a population or a species. A large gene pool has extensive genetic diversity and is better able to withstand the challenges posed by environmental stresses. Inbreeding contributes to the creation of a small gene pool and makes populations or species more likely to go extinct when faced with some type of stress.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/gene_pool.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/gene_pool.mp3</a></p>
<p><strong>Transcript</strong>: “Gene pool refers to thorough genetic diversity that exists within a population [all of the genetic diversity]. The larger the genetic pool, the more the diversity and the more opportunity this population will have to survive environmental stress that may impact on them. So gene pool is really good in the sense that the larger the gene pool, the more the survival of that particular population in terms of withstanding different things that may come in terms of environment.”</p>
<p><a href="https://www.genome.gov/staff/Charles-N-Rotimi-PhD">Charles N. Rotimi</a>, Ph.D</p>

</div>
<div id="gene-prediction" class="section level2" number="8.4">
<h2><span class="header-section-number">8.4</span> Gene prediction</h2>
<p><a href="https://en.wikipedia.org/wiki/Gene_prediction" class="uri">https://en.wikipedia.org/wiki/Gene_prediction</a></p>
<p>In computational biology, gene prediction or gene finding refers to the process of identifying the regions of genomic DNA that encode genes. This includes protein-coding genes as well as RNA genes , but may also include prediction of other functional elements such as regulatory regions. Gene finding is one of the first and most important steps in understanding the genome of a species once it has been sequenced.</p>
<p>In its earliest days, “gene finding” was based on painstaking experimentation on living cells and organisms. Statistical analysis of the rates of homologous recombination of several different genes could determine their order on a certain chromosome , and information from many such experiments could be combined to create a genetic map specifying the rough location of known genes relative to each other. Today, with comprehensive genome sequence and powerful computational resources at the disposal of the research community, gene finding has been redefined as a largely computational problem.</p>
<p>Determining that a sequence is functional should be distinguished from determining the function of the gene or its product. Predicting the function of a gene and confirming that the gene prediction is accurate still demands <em>in vivo</em> experimentation through gene knockout and other assays, although frontiers of bioinformatics research are making it increasingly possible to predict the function of a gene based on its sequence alone.</p>
<p>Gene prediction is one of the key steps in genome annotation , following sequence assembly , the filtering of non-coding regions and repeat masking.</p>
<p>Gene prediction is closely related to the so-called ‘target search problem’ investigating how DNA-binding proteins (transcription factors ) locate specific binding sites within the genome. Many aspects of structural gene prediction are based on current understanding of underlying biochemical processes in the cell such as gene transcription , translation , protein–protein interactions and regulation processes , which are subject of active research in the various omics fields such as transcriptomics , proteomics , metabolomics , and more generally structural and functional genomics.</p>

</div>
<div id="genbank" class="section level2" number="8.5">
<h2><span class="header-section-number">8.5</span> GenBank</h2>
<p>From the blog post “[Introduction to GenBank and Bioinformatics with Python](<a href="https://medium.com/@wvsharber/introduction-to-genbank-and-bioinformatics-with-python-8a25a0f15e3f" class="uri">https://medium.com/@wvsharber/introduction-to-genbank-and-bioinformatics-with-python-8a25a0f15e3f</a>” (Sharber 2015)</p>
<p>“GenBank [is] arguably one of the most important repositories of biological data in the world. GenBank, funded and operated by the National Institutes of Health (NIH) through the National Center for Biotechnology Information (NCBI), is a genetic sequence database. Researchers around the world submit their DNA sequences to GenBank to store and distribute them. Think of it as a huge library of DNA. Almost every DNA sequence that has ever been used in scientific research is stored in GenBank. Most academic publishers won’t accept your manuscript without you submitting your DNA to GenBank. Talk about open-source data! In case you’re curious, NCBI also hosts and produces other databases and tools, such as PubMed, which holds publication records, and BLAST, which is basically Google for sequence data.</p>
<p>“A DNA sequence stored in GenBank consists of a record holding more than just the DNA itself. There will be lots of metadata along with it, such as an unique accession number, what organism the DNA originated from, who sequenced it, what publications reference the data, the translation of the DNA into a protein sequence (if it’s a coding region), etc. These data are critically important for using the sequence data with the appropriate biological context.”</p>

</div>
<div id="genetic-code" class="section level2" number="8.6">
<h2><span class="header-section-number">8.6</span> The genetic code - Overview</h2>
<p><strong>Authors</strong>: OpenStax / Libretext Formatted in RMarkdown by Nathan
Brouwer under the Creative Commons Attribution License 4.0 license.</p>
<p>This chapter was adapted from <a href="https://bio.libretexts.org/Bookshelves/Introductory_and_General_Biology/Book%3A_General_Biology_(OpenStax)">LibreText General
Biology</a>,
Chapter 15, <a href="https://bio.libretexts.org/Bookshelves/Introductory_and_General_Biology/Book%3A_General_Biology_(OpenStax)/3%3A_Genetics/15%3A_Genes_and_Proteins/15.0%3A_Prelude_to_Genes_and_Proteins">Section 15.0: Prelude</a> and <a href="https://bio.libretexts.org/Bookshelves/Introductory_and_General_Biology/Book%3A_General_Biology_(OpenStax)/3%3A_Genetics/15%3A_Genes_and_Proteins/15.1%3A_The_Genetic_Code">15.1: The Genetic Code</a>.
The LibreText book is based on <a href="https://openstax.org/details/books/biology-2e">OpenStax Biology 2nd
edition</a>.</p>
<p>Genes are composed of <strong>DNA</strong> and are linearly arranged on <strong>chromosomes</strong> within cells. Genes specify the sequences of <strong>amino acids</strong>, which are the building blocks of <strong>proteins</strong>. In turn, proteins are responsible for orchestrating nearly every function of the cell.</p>
<div id="the-genetics-code" class="section level3" number="8.6.1">
<h3><span class="header-section-number">8.6.1</span> The genetics code</h3>
<p>The cellular process of <strong>transcription</strong> generates <strong>messenger RNA (mRNA)</strong>, a mobile molecular copy of a gene with an alphabet of A, C, G, and uracil (U). <strong>Translation</strong> of the mRNA template converts nucleotide-based genetic information into a protein product. Protein sequences consist of 20 possible <strong>amino acids</strong>; therefore, it can be said that the protein alphabet consists of 20 letters.</p>
<p>Each amino acid is coded for by a three-nucleotide sequence called the <strong>triplet codon</strong> (often just called a <strong>codon</strong>). Variation in amino acid sequence gives rise to enormous variation in protein structure and function.</p>
</div>
<div id="the-central-dogma-dna-encodes-rna-rna-encodes-protein" class="section level3" number="8.6.2">
<h3><span class="header-section-number">8.6.2</span> The Central Dogma: DNA Encodes RNA; RNA Encodes Protein</h3>
<p>The flow of genetic information in cells from DNA to mRNA to protein is characterized by the <strong>Central Dogma</strong> of molecular biology, which states that genes specify the sequence of mRNAs, which in turn specify the sequence of proteins. The decoding of one molecule to another is performed by specific proteins. Because the information stored in
The copying of DNA to RNA is relatively straightforward, with one nucleotide being added to the mRNA strand for every nucleotide read in the DNA strand. The translation to protein is a bit more complex because <em>three</em> mRNA nucleotides correspond to one amino acid in the growing sequence. However, the translation to protein is still systematic, such that nucleotides 1 to 3 correspond to amino acid 1, nucleotides 4 to 6 correspond to amino acid 2, and so on.</p>
</div>
<div id="the-genetic-code-is-degenerate-and-universal" class="section level3" number="8.6.3">
<h3><span class="header-section-number">8.6.3</span> The Genetic Code Is “Degenerate” and Universal</h3>
<p>Given the different numbers of “letters” in the mRNA and protein “alphabets,” scientists theorized that combinations of nucleotides corresponded to single amino acids. Nucleotide doublets (set of 2) would not be sufficient to specify every amino acid because there are only 16 possible two-nucleotide combinations (4^2). In contrast, there are 64 possible nucleotide triplets (4^3), which is far more than the number of amino acids.</p>
<p>Scientists theorized that amino acids were encoded by nucleotide triplets and that the genetic code was <strong>degenerate</strong>. In other words, a given amino acid could be encoded by more than one nucleotide triplet. This was later confirmed experimentally. When one or two nucleotides were inserted into a gene, protein synthesis was completely prevented. When three nucleotides were inserted, the protein was synthesized and usually functional, but with an additional amino acid in it. This demonstrated that three nucleotides specify each amino acid. These nucleotide triplets are called <strong>codons</strong>. The insertion of one or two nucleotides completely changed the triplet <strong>reading frame</strong>, thereby altering the message for every subsequent amino acid. These are now called <strong>frameshift mutations</strong>. Though insertion of three nucleotides caused an extra amino acid to be inserted during translation, the integrity of the rest of the protein was maintained.</p>
<p>Degeneracy is believed to be a mechanism to reduce the negative impact of random mutations. Codons that specify the same amino acid typically only differ by one nucleotide. In addition, amino acids that are chemically similar (e.g. similar mass or similar electrical charge) are encoded by similar codons. This nuance of the genetic code ensures that a single-nucleotide substitution mutation might either specify the same amino acid but have no effect or specify a similar amino acid, preventing the protein from being rendered completely nonfunctional.</p>
<p>With a few exceptions, virtually all species use the same genetic code for protein synthesis, hence it is the universal code of life on earth. Conservation of codons means that a purified mRNA encoding the globin protein in horses could be transferred to a tulip cell, and the tulip would synthesize horse globin. That there is only one genetic code is powerful evidence that all of life on Earth shares a common origin.</p>
</div>
<div id="start-and-stop-codons" class="section level3" number="8.6.4">
<h3><span class="header-section-number">8.6.4</span> Start and stop codons</h3>
<p>In addition to instructing the addition of a specific amino acid to a polypeptide chain, three of the 64 codons terminate protein synthesis. These triplets are called <strong>stop codons</strong>. Another codon also has a special function. In addition to specifying the amino acid methionine, it also serves as the <strong>start codon</strong> to initiate translation of mRNA into protein. If a mutation removes or alters a start codon no protein can be made. Removal or alteration of a stop codon similarly results a non-functional protein.</p>
</div>
<div id="summary" class="section level3" number="8.6.5">
<h3><span class="header-section-number">8.6.5</span> Summary</h3>
<p>The genetic code refers to the DNA alphabet (A, T, C, G), the RNA alphabet (A, U, C, G), and the protein alphabet (20 amino acids). The Central Dogma describes the flow of genetic information in the cell from genes to mRNA to proteins. Genes serve as the templates for mRNA via the process of transcription; mRNA is then used as the template for proteins during translation. The genetic code is degenerate because 64 triplet codons in mRNA specify only 20 amino acids and three nonsense codons. Almost every species on the planet uses the same genetic code.</p>

</div>
</div>
<div id="genetic-distance" class="section level2" number="8.7">
<h2><span class="header-section-number">8.7</span> Genetic distance</h2>
<p>From Wikipedia, the free encyclopedia
<a href="https://en.wikipedia.org/wiki/Genetic_distance" class="uri">https://en.wikipedia.org/wiki/Genetic_distance</a> (Links to an external site.)</p>
<p>Genetic distance is a measure of the genetic (Links to an external site.) divergence between species (Links to an external site.) or between populations (Links to an external site.) within a species, whether the distance measures time from common ancestor (Links to an external site.) or degree of differentiation.[2] (Links to an external site.) Populations with many similar alleles (Links to an external site.) have small genetic distances. This indicates that they are closely related and have a recent common ancestor.</p>
<p>Genetic distance is useful for reconstructing the history of populations. For example, sequences can be compared and genetic distances calculated. These distances can then be used to build “distance-based” phylogenetic trees.</p>
<p>[This section is interesting but you do not need to know the details].</p>
<p>Evidence from genetic distance suggests that Sub-Saharan African and Eurasian people diverged about 100,000 years ago.[3] (Links to an external site.) Genetic distance is also used for understanding the origin of biodiversity (Links to an external site.). For example, the genetic distances between different breeds of domesticated animals are often investigated in order to determine which breeds should be protected to maintain genetic diversity.[4]</p>
<p><strong>References</strong>
1. Cavalli-Sforza, L.L., Menozzi, P. &amp; Piazza, A. (1994). The History and Geography of Human Genes. New Jersey: Princeton University Press.
1. Nei, M. (1987). Molecular Evolutionary Genetics. (Chapter 9). New York: Columbia University Press.
1. Ramachandran S, Deshpande O, Roseman CC, Rosenberg NA, Feldman MW, Cavalli-Sforza LL (November 2005). “Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in Africa”. Proc Natl Acad Sci U S A. 102 (44): 15942–7. <a href="doi:10.1073/pnas.0507611102" class="uri">doi:10.1073/pnas.0507611102</a>. PMC 1276087. PMID 16243969.
1. Ruane J (1999). “A critical review of the value of genetic distance studies in conservation of animal genetic resources”. Journal of Animal Breeding and Genetics. 116 (5): 317–323. <a href="doi:10.1046/j.1439-0388.1999.00205.x" class="uri">doi:10.1046/j.1439-0388.1999.00205.x</a>.
1. Nei, M. (1972). “Genetic distance between populations”. Am. Nat. 106 (949): 283–292. <a href="doi:10.1086/282771" class="uri">doi:10.1086/282771</a>. S2CID 55212907.</p>

</div>
<div id="talking-glossary-genetic-drift-0.75-min" class="section level2" number="8.8">
<h2><span class="header-section-number">8.8</span> Talking Glossary: Genetic drift (0.75 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Genetic-Drift" class="uri">https://www.genome.gov/genetics-glossary/Genetic-Drift</a> (Links to an external site.)</p>
<p><strong>Abstract</strong>: “Genetic drift is a mechanism of evolution. It refers to random fluctuations in the frequencies of alleles from generation to generation due to chance events. Genetic drift can cause traits to be dominant or disappear from a population. The effects of genetic drift are most pronounced in small populations.”</p>
<p>Audio: <a href="https://www.genome.gov/genetics-glossary/Genetic-Drift" class="uri">https://www.genome.gov/genetics-glossary/Genetic-Drift</a></p>
<p><strong>Transcript</strong>: “Genetic drift. I think the best way to look at it is to look at genetic drift as an evolutionary process. It is really the shuffling of our DNA frequency from one generation to the other. And as a result of this chance event, you know, you can emphasize it in things, you know, in the population. So it is in things. Something that may cause a particular type of gene to be more dominant or a particular type disease to disappear from a population. So again, it is a chance event, and it is an evolutionary process.”</p>
<p><a href="https://www.genome.gov/staff/Charles-N-Rotimi-PhD">Charles N. Rotimi</a>, Ph.D.</p>

</div>
<div id="talking-glossary-genetic-engineering-1.5-min" class="section level2" number="8.9">
<h2><span class="header-section-number">8.9</span> Talking Glossary: Genetic engineering (1.5 min)</h2>
<p>National Human Genome Research Institute
<a href="https://www.genome.gov/genetics-glossary/Genetic-Engineering" class="uri">https://www.genome.gov/genetics-glossary/Genetic-Engineering</a></p>
<p><strong>Introduction</strong>: “Genetic engineering is the process of using recombinant DNA (rDNA) technology to alter the genetic makeup of an organism. Traditionally, humans have manipulated genomes indirectly by controlling breeding and selecting offspring with desired traits. Genetic engineering involves the direct manipulation of one or more genes. Most often, a gene from another species is added to an organism’s genome to give it a desired phenotype.”</p>
<p>NOTE: The use of the term genetic engineering varies a lot. In general, academic researchers seem to refer to things as “recombinant” when DNA from one organism has been added via a plasmid or another route to another organism. “Genetically engineered” seems to be preferred by biotech companies, the media – science fiction Any time you change the genetic composition of an organism (e.g. putting syphilis protein into E. coli via a plasmid) you could refer to it as genetically engineered. I will use the term recombinant though.</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/genetic_engineering.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/genetic_engineering.mp3</a></p>
<p><strong>Transcript</strong>: “Genetic engineering is a term that was first introduced into our language in the 1970s to describe the emerging field of recombinant DNA technology and some of the things that were going on. As most people who read textbooks and things know, recombinant DNA technology started with pretty simple things–cloning very small pieces of DNA and growing them in bacteria–and has evolved to an enormous field where whole genomes can be cloned and moved from cell to cell, to cell using variations of techniques that all would come under genetic engineering as a very broad definition. To me, genetic engineering, broadly defined, means that you are taking pieces of DNA and combining them with other pieces of DNA. [This] doesn’t really happen in nature, but is something that you engineer in your own laboratory and test tubes. And then taking what you have engineered and propagating that in any number of different organisms that range from bacterial cells to yeast cells, to plants and animals. So while there isn’t a precise definition of genetic engineering, I think it more defines an entire field of recombinant DNA technology, genomics, and genetics in the 2000s.”</p>

</div>
<div id="genome-assembly" class="section level2" number="8.10">
<h2><span class="header-section-number">8.10</span> Genome assembly</h2>
<p>Adapted from Wikipedia
<a href="https://en.wikipedia.org/wiki/Sequence_assembly" class="uri">https://en.wikipedia.org/wiki/Sequence_assembly</a></p>
<p>In bioinformatics, genome assembly refers to aligning and merging fragments from a longer DNA sequence such as a whole genome in order to reconstruct the original sequence. This is needed as DNA sequencing technology cannot read whole genomes in one go, but rather reads small pieces, called reads.</p>
<p>The problem of sequence assembly can be compared to taking many copies of a book, passing each of them through different paper shredders with different cutters, and piecing the text of the book back together just by looking at the shredded pieces.</p>
<!-- Figure 1: Assemblers need copious sequencing data and informatic exertion to put the genome back together. Credit: Kelly Howe, Lawrence Berkeley Laboratory -->
<p>In sequence assembly, two different types can be distinguished:</p>
<p>1, de-novo: assembling short reads to create full-length (sometimes novel) sequences, without using a template (see de novo sequence assemblers , de novo transcriptome assembly )
1. using a reference sequence: assembling reads against an existing reference sequence, building a sequence that is similar but not necessarily identical to the backbone sequence</p>
<p>In terms of complexity and time requirements, de-novo assemblies takes much more work. Referring to the comparison drawn to shredded books in the introduction: when a reference sequence is available, one would have a very similar book as a template (perhaps with the names of the main characters and a few locations changed), de-novo assemblies present a more daunting challenge in that one would not know beforehand whether this would become a science book, a novel, a catalog, or even several books. Also, every shred would be compared with every other shred.</p>

</div>
<div id="talking-glossary-genomics" class="section level2" number="8.11">
<h2><span class="header-section-number">8.11</span> Talking Glossary: Genomics</h2>
<p><strong>Abstract</strong>: “Genomics refers to the study of the entire genome of an organism whereas genetics refers to the study of a particular gene.”</p>
<p><strong>Transcript</strong>: “Genomics refers to the study of the entire genome, essentially all the genes that can be found in an organism. It’s in contrast to genetics which can study individual genes one at a time. A genomicist, or someone who studies genomes, studies all of the DNA and all of the sequence in an organism and makes conclusions based on all of it. Whereas a geneticist can study all of the DNA in an organism, but can also study one gene at a time. For example, you might study the genetics of a single gene and sequence that gene. If you wanted to study the genomics of an organism or a person you could sequence all of their genes and all of their DNA and look for changes and make comparisons with other individual’s genomes.”</p>
<p>Lawrence C. Brody, Ph.D.</p>

</div>
<div id="talking-glossary-genotype" class="section level2" number="8.12">
<h2><span class="header-section-number">8.12</span> Talking Glossary: Genotype</h2>
<p><strong>Abstract</strong>: “A genotype is an individual’s collection of genes. The term also can refer to the two alleles inherited for a particular gene. The genotype is expressed when the information encoded in the genes’ DNA is used to make protein and RNA molecules. The expression of the genotype contributes to the individual’s observable traits, called the phenotype.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/genotype.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/genotype.mp3</a></p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/genotype.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/genotype.jpg</a></p>
<p><strong>Transcript</strong>: “Genotype, very simply, is the version of a DNA sequence that an individual has. There’s a large amount of DNA that we all have in common–of course, that’s why we’re all humans–but there’s also a large amount of variation in sequence among individuals. And those specific differences in sequence, when usually applied to an individual gene, are called a genotype. These days, with the ability to test for many different sequence differences between individuals, genotype has taken on a connotation which frequently refers to a difference in sequence in a specific place in a specific gene. When used in that way, it’s usually related to another term, called phenotype, which is the change in sequence to which the genotype refers. It is frequently, not always, but is frequently related to a change in an external trait; something that’s observable, like height, hair color, or occurrence of disease. And so in that case, we talk about a genotype-phenotype correlation. Then what we’re talking about is, well, here’s a change in DNA sequence; why is it important? It’s important because it leads to an observable change in a trait in a person. And that change in trait can be positive, it can be negative, or it could just be a difference.”</p>
<p>Christopher P. Austin, M.D.</p>

</div>
<div id="talking-glossary-germline-1.25-min" class="section level2" number="8.13">
<h2><span class="header-section-number">8.13</span> Talking Glossary: Germline (1.25 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/germ-line" class="uri">https://www.genome.gov/genetics-glossary/germ-line</a></p>
<p>Abstract: “A germ line is the sex cells (eggs and sperm) that are used by sexually reproducing organisms to pass on genes from generation to generation. Egg and sperm cells are called germ cells, in contrast to the other cells of the body that are called somatic cells.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/germ_line.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/germ_line.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/germ_line.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/germ_line.mp3</a></p>
<p>Transcript: “Germ line actually refers to the sex cells of an organism. And it does seem like a bit of a strange term, but it actually refers not just to mammals, but to any living organism that uses sex to reproduce, and that includes plants. And so it’s a rather generic term which refers to the cells in the sexual organ, and those cells will either be making sperm or eggs. And that sperm be could, of course, be pollen instead of what you traditionally think of as sperm. And the key important aspect of germ line is this is where genetic information is transferred from one generation to the next. And it’s inherited both from the female and the male of the species, so this is where all the action in terms of genetics is happening. And when those organisms grow up they, of course, are not comprised primarily of germ cells. Most of the body is consisted of somatic cells. But the key issue in terms of evolution, in terms of inheritance, all go through the germ line.”</p>
<p>Shawn Burgess, Ph.D.</p>

</div>
<div id="guild" class="section level2" number="8.14">
<h2><span class="header-section-number">8.14</span> Guild</h2>
<p><a href="https://en.wikipedia.org/wiki/Guild_(ecology)" class="uri">https://en.wikipedia.org/wiki/Guild_(ecology)</a>.</p>
<p>A guild (or ecological guild) is any group of species that exploit the same resources, or that exploit different resources in related ways.[1].[2].[3]. It is not necessary that the species within a guild occupy the same, or even similar, ecological niches.. An ecological niche is defined as the role an organism plays in its community, i.e. decomposer, primary producer, etc.[4]. Guilds are defined according to the locations, attributes, or activities of their component species. For example, the mode of acquiring nutrients, the mobility, and the habitat zones that the species occupy or exploit can be used to define a guild. The number of guilds occupying an ecosystem is termed its disparity. Members of a guild within a given ecosystem could be competing for resources, such as space or light, while cooperating in resisting wind stresses, attracting pollinators, or detecting predators, such as happens among savannah-dwelling antelope. and zebra..</p>
<p>A guild does not typically have strict, or even clearly defined boundaries, nor does it need to be taxonomically cohesive. A broadly defined guild will almost always have constituent guilds; for example, grazing guilds will have some species that concentrate on coarse, plentiful forage, while others concentrate on low-growing, finer plants. Each of those two sub-guilds may be regarded as guilds in appropriate contexts, and they might, in turn, have sub-guilds in more closely selective contexts. Some authorities even speak of guilds in terms of a fractal. resource model.[5]. This concept arises in several related contexts, such as the metabolic theory of ecology., the scaling pattern of occupancy., and spatial analysis. in ecology, all of which are fundamental concepts in defining guilds.</p>
<p><strong>Examples of guilds</strong>:
1. Graminoids (grasses, rushes and sedges)
1. Plankton
1. Shrubs
1. Trees
1. Vines
1. Scavengers
1. Browsers and terrestrial folivores (animals that eat leaves other than grass)
1. Forest canopy folivores (flower-eaters)
1. Grazers (grass-eating animals)
1. Forbs (herbaceous plants that aren’t grasses)
1. Saprophytes (decomposers, such as fungi)
1. Piscivores (fish-eaters)</p>
<p><strong>References</strong>:
1. Simberloff, D; Dayan, T (1991). “The Guild Concept and the Structure of Ecological Communities”. Annual Review of Ecology and Systematics. 22: 115–143. <a href="doi:10.1146/annurev.es.22.110191.000555" class="uri">doi:10.1146/annurev.es.22.110191.000555</a>.
1. Encyclopædia Britannica article on guilds
1. Williams, SE; Hero, JM (1998). “Rainforest frogs of the Australian Wet Tropics: guild classification and the ecological similarity of declining species”. Proceedings: Biological Sciences. 265 (1396): 597–602. <a href="doi:10.1098/rspb.1998.0336" class="uri">doi:10.1098/rspb.1998.0336</a>. PMC 1689015. PMID 9881468.
1. “the definition of ecological niche”. Dictionary.com. Retrieved 2017-05-02.
1. Guillerme; et al. (2020). “Disparities in the analysis of morphological disparity”. Biology Letters. 16 (7). <a href="doi:10.1098/rsbl.2020.0199" class="uri">doi:10.1098/rsbl.2020.0199</a>.
1. Ritchie, Mark E. (2010). Scale, Heterogeneity, and the Structure and Diversity of Ecological Communities. Volume 45 of Monographs in population biology. Princeton: Princeton University Press. ISBN 978-0-691-09070-2.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="f.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="h.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["openbio.pdf", "openbio.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
