<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title> 7 I | Foundations of Biology and Environmental Science: An Open-Access Encyclopedia</title>
  <meta name="description" content="A collection of open-access and/or free resources on biology and environmental science." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content=" 7 I | Foundations of Biology and Environmental Science: An Open-Access Encyclopedia" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content=" 7 I | Foundations of Biology and Environmental Science: An Open-Access Encyclopedia" />
  
  <meta name="twitter:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

<meta name="author" content="Compiled and edited by Nathan Brouwer" />


<meta name="date" content="2022-01-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="h.html"/>
<link rel="next" href="j.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Biology for everyone</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="a.html"><a href="a.html"><i class="fa fa-check"></i><b>2</b> A</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a.html"><a href="a.html#alignments"><i class="fa fa-check"></i><b>2.1</b> Alignments</a></li>
<li class="chapter" data-level="2.2" data-path="a.html"><a href="a.html#talking-glossary-allele-0.5-min"><i class="fa fa-check"></i><b>2.2</b> Talking Glossary: Allele (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="b.html"><a href="b.html"><i class="fa fa-check"></i><b>3</b> B</a>
<ul>
<li class="chapter" data-level="3.1" data-path="b.html"><a href="b.html#talking-glossary-bioinformatics"><i class="fa fa-check"></i><b>3.1</b> Talking glossary: Bioinformatics</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="c.html"><a href="c.html"><i class="fa fa-check"></i><b>4</b> C</a>
<ul>
<li class="chapter" data-level="4.1" data-path="c.html"><a href="c.html#talking-glossary-candidate-gene"><i class="fa fa-check"></i><b>4.1</b> Talking Glossary: Candidate gene</a></li>
<li class="chapter" data-level="4.2" data-path="c.html"><a href="c.html#cath-database"><i class="fa fa-check"></i><b>4.2</b> CATH database</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="c.html"><a href="c.html#references"><i class="fa fa-check"></i><b>4.2.1</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="c.html"><a href="c.html#clade"><i class="fa fa-check"></i><b>4.3</b> Clade</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="c.html"><a href="c.html#clades-and-phylogenetic-trees"><i class="fa fa-check"></i><b>4.3.1</b> Clades and phylogenetic trees</a></li>
<li class="chapter" data-level="4.3.2" data-path="c.html"><a href="c.html#terminology"><i class="fa fa-check"></i><b>4.3.2</b> Terminology</a></li>
<li class="chapter" data-level="4.3.3" data-path="c.html"><a href="c.html#in-popular-culture"><i class="fa fa-check"></i><b>4.3.3</b> In popular culture</a></li>
<li class="chapter" data-level="4.3.4" data-path="c.html"><a href="c.html#references-1"><i class="fa fa-check"></i><b>4.3.4</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="c.html"><a href="c.html#talking-glossary-cloning-0.5-min"><i class="fa fa-check"></i><b>4.4</b> Talking Glossary: Cloning (0.5 min)</a></li>
<li class="chapter" data-level="4.5" data-path="c.html"><a href="c.html#talking-glossary-codominance-0.5-min"><i class="fa fa-check"></i><b>4.5</b> Talking Glossary: Codominance (0.5 min)</a></li>
<li class="chapter" data-level="4.6" data-path="c.html"><a href="c.html#comparative-genomics"><i class="fa fa-check"></i><b>4.6</b> Comparative genomics</a></li>
<li class="chapter" data-level="4.7" data-path="c.html"><a href="c.html#talking-glossary-contig-0.75-min"><i class="fa fa-check"></i><b>4.7</b> Talking Glossary: Contig (0.75 min)</a></li>
<li class="chapter" data-level="4.8" data-path="c.html"><a href="c.html#talking-glossary---foundations-1-review-crossing-over"><i class="fa fa-check"></i><b>4.8</b> Talking Glossary - Foundations 1 Review: Crossing over</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="d.html"><a href="d.html"><i class="fa fa-check"></i><b>5</b> D</a>
<ul>
<li class="chapter" data-level="5.1" data-path="d.html"><a href="d.html#talking-glossary-deletion-mutation-1.5-min"><i class="fa fa-check"></i><b>5.1</b> Talking Glossary: Deletion mutation (1.5 min)</a></li>
<li class="chapter" data-level="5.2" data-path="d.html"><a href="d.html#talking-glossary-dna-sequencing-0.75-min"><i class="fa fa-check"></i><b>5.2</b> Talking Glossary: DNA sequencing (0.75 min)</a></li>
<li class="chapter" data-level="5.3" data-path="d.html"><a href="d.html#sex-chromosome-dosage-compensation"><i class="fa fa-check"></i><b>5.3</b> Sex-Chromosome Dosage compensation</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="d.html"><a href="d.html#random-inactivation-of-one-x-in-xx-individuals"><i class="fa fa-check"></i><b>5.3.1</b> Random inactivation of one X in XX individuals</a></li>
<li class="chapter" data-level="5.3.2" data-path="d.html"><a href="d.html#results"><i class="fa fa-check"></i><b>5.3.2</b> Results</a></li>
<li class="chapter" data-level="5.3.3" data-path="d.html"><a href="d.html#clinical-applications"><i class="fa fa-check"></i><b>5.3.3</b> Clinical applications</a></li>
<li class="chapter" data-level="5.3.4" data-path="d.html"><a href="d.html#agricultural-applications"><i class="fa fa-check"></i><b>5.3.4</b> Agricultural applications</a></li>
<li class="chapter" data-level="5.3.5" data-path="d.html"><a href="d.html#limitations"><i class="fa fa-check"></i><b>5.3.5</b> Limitations</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="d.html"><a href="d.html#fine-mapping"><i class="fa fa-check"></i><b>5.4</b> Fine-mapping</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="h.html"><a href="h.html"><i class="fa fa-check"></i><b>6</b> H</a>
<ul>
<li class="chapter" data-level="6.1" data-path="h.html"><a href="h.html#talking-glossary-haploid-1.25-min"><i class="fa fa-check"></i><b>6.1</b> Talking Glossary: Haploid (1.25 min)</a></li>
<li class="chapter" data-level="6.2" data-path="h.html"><a href="h.html#talking-glossary-halplotype"><i class="fa fa-check"></i><b>6.2</b> Talking Glossary: Halplotype</a></li>
<li class="chapter" data-level="6.3" data-path="h.html"><a href="h.html#talking-glossary-heterozygous"><i class="fa fa-check"></i><b>6.3</b> Talking Glossary: Heterozygous</a></li>
<li class="chapter" data-level="6.4" data-path="h.html"><a href="h.html#homologyl-when-things-are-homolgous-and-analogous"><i class="fa fa-check"></i><b>6.4</b> Homology:L When things are homolgous AND analogous!</a></li>
<li class="chapter" data-level="6.5" data-path="h.html"><a href="h.html#talking-glossary-homozygous"><i class="fa fa-check"></i><b>6.5</b> Talking Glossary: Homozygous</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="i.html"><a href="i.html"><i class="fa fa-check"></i><b>7</b> I</a>
<ul>
<li class="chapter" data-level="7.1" data-path="i.html"><a href="i.html#talking-glossary-insertion-mutation-1.25-min"><i class="fa fa-check"></i><b>7.1</b> Talking Glossary: Insertion mutation (1.25 min)</a></li>
<li class="chapter" data-level="7.2" data-path="i.html"><a href="i.html#intrinscially-disordered-protein"><i class="fa fa-check"></i><b>7.2</b> Intrinscially disordered protein</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="i.html"><a href="i.html#abundance"><i class="fa fa-check"></i><b>7.2.1</b> Abundance</a></li>
<li class="chapter" data-level="7.2.2" data-path="i.html"><a href="i.html#disorder-annotation"><i class="fa fa-check"></i><b>7.2.2</b> Disorder annotation</a></li>
<li class="chapter" data-level="7.2.3" data-path="i.html"><a href="i.html#disorder-databases"><i class="fa fa-check"></i><b>7.2.3</b> Disorder databases</a></li>
<li class="chapter" data-level="7.2.4" data-path="i.html"><a href="i.html#predicting-idps-by-sequence"><i class="fa fa-check"></i><b>7.2.4</b> Predicting IDPs by sequence</a></li>
<li class="chapter" data-level="7.2.5" data-path="i.html"><a href="i.html#prediction-methods"><i class="fa fa-check"></i><b>7.2.5</b> Prediction methods</a></li>
<li class="chapter" data-level="7.2.6" data-path="i.html"><a href="i.html#disorder-and-disease"><i class="fa fa-check"></i><b>7.2.6</b> Disorder and disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="j.html"><a href="j.html"><i class="fa fa-check"></i><b>8</b> J</a>
<ul>
<li class="chapter" data-level="8.1" data-path="j.html"><a href="j.html#jove-videos"><i class="fa fa-check"></i><b>8.1</b> JoVe Videos</a>
<ul>
<li class="chapter" data-level="8.1.1" data-path="j.html"><a href="j.html#biotechnology"><i class="fa fa-check"></i><b>8.1.1</b> Biotechnology</a></li>
<li class="chapter" data-level="8.1.2" data-path="j.html"><a href="j.html#macromolecules"><i class="fa fa-check"></i><b>8.1.2</b> Macromolecules</a></li>
<li class="chapter" data-level="8.1.3" data-path="j.html"><a href="j.html#cellular-respiration"><i class="fa fa-check"></i><b>8.1.3</b> Cellular respiration</a></li>
<li class="chapter" data-level="8.1.4" data-path="j.html"><a href="j.html#population-and-community-ecology"><i class="fa fa-check"></i><b>8.1.4</b> Population and community ecology</a></li>
<li class="chapter" data-level="8.1.5" data-path="j.html"><a href="j.html#population-genetics"><i class="fa fa-check"></i><b>8.1.5</b> Population genetics</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="k.html"><a href="k.html"><i class="fa fa-check"></i><b>9</b> K</a>
<ul>
<li class="chapter" data-level="9.1" data-path="k.html"><a href="k.html#talking-glossary-knockout-genetic-knockout-gene-knockout"><i class="fa fa-check"></i><b>9.1</b> Talking glossary: Knockout (Genetic knockout, gene knockout)</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="l.html"><a href="l.html"><i class="fa fa-check"></i><b>10</b> L</a>
<ul>
<li class="chapter" data-level="10.1" data-path="l.html"><a href="l.html#talking-glossary-locus-1-min"><i class="fa fa-check"></i><b>10.1</b> Talking Glossary: Locus (1 min)</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="m.html"><a href="m.html"><i class="fa fa-check"></i><b>11</b> M</a>
<ul>
<li class="chapter" data-level="11.1" data-path="m.html"><a href="m.html#manhattan-plot"><i class="fa fa-check"></i><b>11.1</b> Manhattan Plot</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="m.html"><a href="m.html#gwas"><i class="fa fa-check"></i><b>11.1.1</b> GWAS</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="m.html"><a href="m.html#talking-glossary-microarray"><i class="fa fa-check"></i><b>11.2</b> Talking Glossary: Microarray</a></li>
<li class="chapter" data-level="11.3" data-path="m.html"><a href="m.html#talking-glossary-missense-mutation-1.25-min"><i class="fa fa-check"></i><b>11.3</b> Talking Glossary: Missense Mutation (1.25 min)</a></li>
<li class="chapter" data-level="11.4" data-path="m.html"><a href="m.html#talking-glossary-mitochondrial-dna-1.5-min"><i class="fa fa-check"></i><b>11.4</b> Talking Glossary: Mitochondrial DNA (1.5 min)</a></li>
<li class="chapter" data-level="11.5" data-path="m.html"><a href="m.html#talking-glossary-mutagen-0.5-min"><i class="fa fa-check"></i><b>11.5</b> Talking Glossary: Mutagen (0.5 min)</a></li>
<li class="chapter" data-level="11.6" data-path="m.html"><a href="m.html#talking-glossary-mutation-0.5-min"><i class="fa fa-check"></i><b>11.6</b> Talking Glossary: Mutation (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="n.html"><a href="n.html"><i class="fa fa-check"></i><b>12</b> N</a>
<ul>
<li class="chapter" data-level="12.1" data-path="n.html"><a href="n.html#talking-glossary-non-coding-dna-1-min"><i class="fa fa-check"></i><b>12.1</b> Talking Glossary: Non-coding DNA (1 min)</a></li>
<li class="chapter" data-level="12.2" data-path="n.html"><a href="n.html#talking-glossary-nonsene-mutation"><i class="fa fa-check"></i><b>12.2</b> Talking Glossary: Nonsene mutation</a></li>
<li class="chapter" data-level="12.3" data-path="n.html"><a href="n.html#nucleic-acids-review"><i class="fa fa-check"></i><b>12.3</b> Nucleic Acids - Overview</a>
<ul>
<li class="chapter" data-level="12.3.1" data-path="n.html"><a href="n.html#dna-and-rna"><i class="fa fa-check"></i><b>12.3.1</b> DNA and RNA</a></li>
<li class="chapter" data-level="12.3.2" data-path="n.html"><a href="n.html#dna-double-helix-structure"><i class="fa fa-check"></i><b>12.3.2</b> DNA Double-Helix Structure</a></li>
<li class="chapter" data-level="12.3.3" data-path="n.html"><a href="n.html#rna"><i class="fa fa-check"></i><b>12.3.3</b> RNA</a></li>
<li class="chapter" data-level="12.3.4" data-path="n.html"><a href="n.html#protein-function"><i class="fa fa-check"></i><b>12.3.4</b> Protein function</a></li>
<li class="chapter" data-level="12.3.5" data-path="n.html"><a href="n.html#summary"><i class="fa fa-check"></i><b>12.3.5</b> Summary</a></li>
<li class="chapter" data-level="12.3.6" data-path="n.html"><a href="n.html#glossary"><i class="fa fa-check"></i><b>12.3.6</b> Glossary</a></li>
<li class="chapter" data-level="12.3.7" data-path="n.html"><a href="n.html#contributors-and-attributions"><i class="fa fa-check"></i><b>12.3.7</b> Contributors and Attributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="13" data-path="o.html"><a href="o.html"><i class="fa fa-check"></i><b>13</b> O</a>
<ul>
<li class="chapter" data-level="13.1" data-path="o.html"><a href="o.html#talking-glossary-open-reading-frame-definition-3-min"><i class="fa fa-check"></i><b>13.1</b> Talking Glossary : Open reading frame definition (3 min)</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="p.html"><a href="p.html"><i class="fa fa-check"></i><b>14</b> P</a>
<ul>
<li class="chapter" data-level="14.1" data-path="p.html"><a href="p.html#talking-glossary-pcr---the-polymerase-chain-reaction-0.5-min"><i class="fa fa-check"></i><b>14.1</b> Talking Glossary: PCR - The Polymerase Chain Reaction (0.5 min)</a></li>
<li class="chapter" data-level="14.2" data-path="p.html"><a href="p.html#protein-data-bank"><i class="fa fa-check"></i><b>14.2</b> Protein Data Bank</a>
<ul>
<li class="chapter" data-level="14.2.1" data-path="p.html"><a href="p.html#history"><i class="fa fa-check"></i><b>14.2.1</b> History</a></li>
<li class="chapter" data-level="14.2.2" data-path="p.html"><a href="p.html#file-format"><i class="fa fa-check"></i><b>14.2.2</b> File format</a></li>
</ul></li>
<li class="chapter" data-level="14.3" data-path="p.html"><a href="p.html#pfam"><i class="fa fa-check"></i><b>14.3</b> PFam</a>
<ul>
<li class="chapter" data-level="14.3.1" data-path="p.html"><a href="p.html#uses"><i class="fa fa-check"></i><b>14.3.1</b> Uses</a></li>
</ul></li>
<li class="chapter" data-level="14.4" data-path="p.html"><a href="p.html#talking-glossary-phenotype"><i class="fa fa-check"></i><b>14.4</b> Talking Glossary: Phenotype</a></li>
<li class="chapter" data-level="14.5" data-path="p.html"><a href="p.html#phylogenetics-vocab"><i class="fa fa-check"></i><b>14.5</b> Phylogenetics vocab</a></li>
<li class="chapter" data-level="14.6" data-path="p.html"><a href="p.html#talking-glossary-plasmid-1-min"><i class="fa fa-check"></i><b>14.6</b> Talking Glossary: Plasmid (1 min)</a></li>
<li class="chapter" data-level="14.7" data-path="p.html"><a href="p.html#talking-glossary-point-mutation-0.5-min"><i class="fa fa-check"></i><b>14.7</b> Talking Glossary: Point mutation (0.5 min)</a></li>
<li class="chapter" data-level="14.8" data-path="p.html"><a href="p.html#talking-glossary-polymorphism-1-min"><i class="fa fa-check"></i><b>14.8</b> Talking Glossary: Polymorphism (1 min)</a></li>
<li class="chapter" data-level="14.9" data-path="p.html"><a href="p.html#talking-glossary-u03-primer-0.5-min"><i class="fa fa-check"></i><b>14.9</b> Talking Glossary U03: Primer (0.5 min)</a></li>
<li class="chapter" data-level="14.10" data-path="p.html"><a href="p.html#talking-glossary-promoter-1.5-min"><i class="fa fa-check"></i><b>14.10</b> Talking Glossary: Promoter (1.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="q.html"><a href="q.html"><i class="fa fa-check"></i><b>15</b> Q</a></li>
<li class="chapter" data-level="16" data-path="r.html"><a href="r.html"><i class="fa fa-check"></i><b>16</b> R</a>
<ul>
<li class="chapter" data-level="16.1" data-path="r.html"><a href="r.html#talking-glossary-recessive"><i class="fa fa-check"></i><b>16.1</b> Talking Glossary: Recessive</a></li>
<li class="chapter" data-level="16.2" data-path="r.html"><a href="r.html#talking-glossary-recombinant-dna-0.5-min"><i class="fa fa-check"></i><b>16.2</b> Talking Glossary: Recombinant DNA (0.5 min)</a></li>
<li class="chapter" data-level="16.3" data-path="r.html"><a href="r.html#talking-glossary-repressor-1-min"><i class="fa fa-check"></i><b>16.3</b> Talking Glossary: Repressor (1 min)</a></li>
<li class="chapter" data-level="16.4" data-path="r.html"><a href="r.html#talking-glossary-restriction-enzyme-0.5-min"><i class="fa fa-check"></i><b>16.4</b> Talking Glossary: Restriction Enzyme (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="s.html"><a href="s.html"><i class="fa fa-check"></i><b>17</b> S</a>
<ul>
<li class="chapter" data-level="17.1" data-path="s.html"><a href="s.html#structural-classification-of-proteins-database-scop"><i class="fa fa-check"></i><b>17.1</b> Structural Classification of Proteins database (SCOP)</a>
<ul>
<li class="chapter" data-level="17.1.1" data-path="s.html"><a href="s.html#hierarchical-organisation"><i class="fa fa-check"></i><b>17.1.1</b> Hierarchical organisation</a></li>
<li class="chapter" data-level="17.1.2" data-path="s.html"><a href="s.html#superfamilies"><i class="fa fa-check"></i><b>17.1.2</b> Superfamilies</a></li>
<li class="chapter" data-level="17.1.3" data-path="s.html"><a href="s.html#families"><i class="fa fa-check"></i><b>17.1.3</b> Families</a></li>
<li class="chapter" data-level="17.1.4" data-path="s.html"><a href="s.html#pdb-entry-domains"><i class="fa fa-check"></i><b>17.1.4</b> PDB entry domains</a></li>
</ul></li>
<li class="chapter" data-level="17.2" data-path="s.html"><a href="s.html#sequence-alignment"><i class="fa fa-check"></i><b>17.2</b> Sequence alignment</a>
<ul>
<li class="chapter" data-level="17.2.1" data-path="s.html"><a href="s.html#interpretation"><i class="fa fa-check"></i><b>17.2.1</b> Interpretation</a></li>
<li class="chapter" data-level="17.2.2" data-path="s.html"><a href="s.html#alignment-methods"><i class="fa fa-check"></i><b>17.2.2</b> Alignment methods</a></li>
<li class="chapter" data-level="17.2.3" data-path="s.html"><a href="s.html#representations"><i class="fa fa-check"></i><b>17.2.3</b> Representations</a></li>
<li class="chapter" data-level="17.2.4" data-path="s.html"><a href="s.html#global-and-local-alignments"><i class="fa fa-check"></i><b>17.2.4</b> Global and local alignments</a></li>
<li class="chapter" data-level="17.2.5" data-path="s.html"><a href="s.html#pairwise-alignment"><i class="fa fa-check"></i><b>17.2.5</b> Pairwise alignment</a></li>
<li class="chapter" data-level="17.2.6" data-path="s.html"><a href="s.html#dot-matrix-methods"><i class="fa fa-check"></i><b>17.2.6</b> Dot-matrix methods</a></li>
<li class="chapter" data-level="17.2.7" data-path="s.html"><a href="s.html#dynamic-programming"><i class="fa fa-check"></i><b>17.2.7</b> Dynamic programming</a></li>
<li class="chapter" data-level="17.2.8" data-path="s.html"><a href="s.html#word-methods"><i class="fa fa-check"></i><b>17.2.8</b> Word methods</a></li>
<li class="chapter" data-level="17.2.9" data-path="s.html"><a href="s.html#multiple-sequence-alignment"><i class="fa fa-check"></i><b>17.2.9</b> Multiple sequence alignment</a></li>
<li class="chapter" data-level="17.2.10" data-path="s.html"><a href="s.html#phylogenetic-analysis"><i class="fa fa-check"></i><b>17.2.10</b> Phylogenetic analysis</a></li>
<li class="chapter" data-level="17.2.11" data-path="s.html"><a href="s.html#assessment-of-significance"><i class="fa fa-check"></i><b>17.2.11</b> Assessment of significance</a></li>
<li class="chapter" data-level="17.2.12" data-path="s.html"><a href="s.html#other-biological-uses"><i class="fa fa-check"></i><b>17.2.12</b> Other biological uses</a></li>
<li class="chapter" data-level="17.2.13" data-path="s.html"><a href="s.html#references-2"><i class="fa fa-check"></i><b>17.2.13</b> References</a></li>
</ul></li>
<li class="chapter" data-level="17.3" data-path="s.html"><a href="s.html#talking-glossary-u03-shotgun-sequencing-1-min"><i class="fa fa-check"></i><b>17.3</b> Talking Glossary U03: Shotgun sequencing (1 min)</a></li>
<li class="chapter" data-level="17.4" data-path="s.html"><a href="s.html#talking-glossary-sickle-cell-disease-4-min"><i class="fa fa-check"></i><b>17.4</b> Talking Glossary: Sickle Cell Disease (4 min)</a></li>
<li class="chapter" data-level="17.5" data-path="s.html"><a href="s.html#medline-snps-single-nucleotide-polymorphism"><i class="fa fa-check"></i><b>17.5</b> Medline: SNPs (single nucleotide polymorphism)</a></li>
<li class="chapter" data-level="17.6" data-path="s.html"><a href="s.html#talking-glossary-somatic-cells-0.75-min"><i class="fa fa-check"></i><b>17.6</b> Talking glossary: Somatic cells (0.75 min)</a></li>
<li class="chapter" data-level="17.7" data-path="s.html"><a href="s.html#sequence-annotation"><i class="fa fa-check"></i><b>17.7</b> Sequence annotation</a>
<ul>
<li class="chapter" data-level="17.7.1" data-path="s.html"><a href="s.html#process"><i class="fa fa-check"></i><b>17.7.1</b> Process</a></li>
</ul></li>
<li class="chapter" data-level="17.8" data-path="s.html"><a href="s.html#talking-glossary-substitution-mutation-1.75-min"><i class="fa fa-check"></i><b>17.8</b> Talking Glossary: Substitution mutation (1.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="t.html"><a href="t.html"><i class="fa fa-check"></i><b>18</b> T</a>
<ul>
<li class="chapter" data-level="18.1" data-path="t.html"><a href="t.html#taxa-taxon-and-clades-a-brief-primer"><i class="fa fa-check"></i><b>18.1</b> Taxa, Taxon, and clades: A Brief Primer</a>
<ul>
<li class="chapter" data-level="18.1.1" data-path="t.html"><a href="t.html#what-is-a-taxon"><i class="fa fa-check"></i><b>18.1.1</b> What is a “taxon”</a></li>
<li class="chapter" data-level="18.1.2" data-path="t.html"><a href="t.html#what-is-a-clade"><i class="fa fa-check"></i><b>18.1.2</b> What is a clade?</a></li>
<li class="chapter" data-level="18.1.3" data-path="t.html"><a href="t.html#further-reading"><i class="fa fa-check"></i><b>18.1.3</b> Further Reading:</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="t.html"><a href="t.html#talking-glossary-trait-0.75-min"><i class="fa fa-check"></i><b>18.2</b> Talking Glossary: Trait (0.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="u.html"><a href="u.html"><i class="fa fa-check"></i><b>19</b> U</a>
<ul>
<li class="chapter" data-level="19.1" data-path="u.html"><a href="u.html#uniprot"><i class="fa fa-check"></i><b>19.1</b> Uniprot</a>
<ul>
<li class="chapter" data-level="19.1.1" data-path="u.html"><a href="u.html#the-uniprot-consortium"><i class="fa fa-check"></i><b>19.1.1</b> The UniProt consortium</a></li>
<li class="chapter" data-level="19.1.2" data-path="u.html"><a href="u.html#the-roots-of-uniprot-databases"><i class="fa fa-check"></i><b>19.1.2</b> The roots of UniProt databases</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="20" data-path="v.html"><a href="v.html"><i class="fa fa-check"></i><b>20</b> V</a>
<ul>
<li class="chapter" data-level="20.1" data-path="v.html"><a href="v.html#talking-glossary-vector-2-min"><i class="fa fa-check"></i><b>20.1</b> Talking Glossary: Vector (2 min)</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="w.html"><a href="w.html"><i class="fa fa-check"></i><b>21</b> W</a>
<ul>
<li class="chapter" data-level="21.1" data-path="w.html"><a href="w.html#talking-glossary-x-chromosome-1.25-min"><i class="fa fa-check"></i><b>21.1</b> Talking Glossary: X-chromosome (1.25 min)</a></li>
<li class="chapter" data-level="21.2" data-path="w.html"><a href="w.html#talking-glossary-x-inactivation-lyonization-1.45-min"><i class="fa fa-check"></i><b>21.2</b> Talking Glossary: X-inactivation (“Lyonization”; 1.45 min))</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="x.html"><a href="x.html"><i class="fa fa-check"></i><b>22</b> X</a></li>
<li class="chapter" data-level="23" data-path="v-1.html"><a href="v-1.html"><i class="fa fa-check"></i><b>23</b> V</a>
<ul>
<li class="chapter" data-level="23.1" data-path="v-1.html"><a href="v-1.html#additional-glossaries"><i class="fa fa-check"></i><b>23.1</b> Additional Glossaries</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="z.html"><a href="z.html"><i class="fa fa-check"></i><b>24</b> Z</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Foundations of Biology and Environmental Science: An Open-Access Encyclopedia</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="i" class="section level1" number="7">
<h1><span class="header-section-number"> 7</span> I</h1>

<div id="talking-glossary-insertion-mutation-1.25-min" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> Talking Glossary: Insertion mutation (1.25 min)</h2>
<p>genome.gov/genetics-glossary/Insertion</p>
<p><strong>Abstract</strong>: “Insertion is a type of mutation involving the addition of genetic material. An insertion mutation can be small, involving a single extra DNA base pair, or large, involving a piece of a chromosome.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/insertion.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/insertion.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/insertion.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/insertion.mp3</a></p>
<p><strong>Transcript</strong> “Insertion really means that something has been stuck in there. And again, as a geneticist, when we think of an insertion, we think of a piece of DNA, and that can be small or large, being stuck in at a place where it really doesn’t belong. So an insertion of just one base pair could lead to something that we call a frameshift. It shifts the reading of the three-base pair code and by that can throw off the entire protein, and by that can lead, for example, to a birth defect. Insertion can be larger, that, for example, there is an insertion of three base pairs, and then it will not throw off the frame, or it will not lead to a frameshift, and potentially is less harmful than having the insertion of just one base pair. And of course you can have an insertion of huge pieces of DNA. They can be so large that you could actually see it on the chromosome analysis, where all of the smaller insertions you would see only by sequencing the stretch of DNA.”</p>
<p>Maximilian Muenke, M.D.</p>
<p>For an interview with Dr. Muenke, see: <a href="https://www.genome.gov/player/wyo8AF_3nz8/PL1ay9ko4A8sk0o9O-YhseFHzbU2I2HQQp" class="uri">https://www.genome.gov/player/wyo8AF_3nz8/PL1ay9ko4A8sk0o9O-YhseFHzbU2I2HQQp</a></p>

</div>
<div id="intrinscially-disordered-protein" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> Intrinscially disordered protein</h2>
<p>Adapted from Wikipedia
<a href="https://en.wikipedia.org/wiki/Intrinsically_disordered_proteins" class="uri">https://en.wikipedia.org/wiki/Intrinsically_disordered_proteins</a></p>
<!-- Conformational flexibility in SUMO-1 protein (PDB:1a5r). The central part shows relatively ordered structure. Conversely, the N- and C-terminal regions (left and right, respectively) show ‘intrinsic disorder’, although a short helical region persists in the N-terminal tail. Ten alternative NMR models were morphed. Secondary structure elements: α-helices (red), β-strands (blue arrows). [1] -->
<p>An <strong>intrinsically disordered protein</strong> (IDP) is a protein that lacks a fixed or ordered three-dimensional structure (2, 3, 4) typically in the absence of its macromolecular interaction partners, such as other proteins or RNA. IDPs range from fully unstructured to partially structured and include <strong>random coil</strong>, <strong>molten globule</strong>-like aggregates, or flexible linkers in large <strong>multi-domain</strong> proteins. They are sometimes considered as a separate class of proteins along with <strong>globular</strong>, <strong>fibrous</strong> and <strong>membrane</strong> proteins (5).</p>
<p>The discovery of IDPs offers support against the idea that three-dimensional structures of proteins must be fixed to accomplish their biological functions. The dogma of rigid protein structure has been questioned due to the increasing evidence of dynamics being necessary for the protein machines. Despite their lack of stable structure, IDPs are a very large and functionally important class of proteins. Many IDPs can adopt a fixed three-dimensional structure after binding to other macromolecules. Overall, IDPs are different from structured proteins in many ways and tend to have distinctive function, structure, sequence, interactions, evolution and regulation (6).</p>
<!-- Contents -->
<!-- 1  History -->
<!-- 2  Abundance -->
<!-- 3  Biological roles -->
<!-- 3.1    Flexible linkers -->
<!-- 3.2    Linear motifs -->
<!-- 3.3    Coupled folding and binding -->
<!-- 3.4    Disorder in the bound state (fuzzy complexes) -->
<!-- 4  Structural aspects -->
<!-- 5  Experimental validation -->
<!-- 5.1    In vivo approaches -->
<!-- 5.2    In vitro approaches -->
<!-- 6  Disorder annotation -->
<!-- 6.1    Disorder databases -->
<!-- 6.2    Predicting IDPs by sequence -->
<!-- 6.3    Prediction methods -->
<!-- 7  Disorder and disease -->
<!-- 8  Computer simulations -->
<!-- 9  See also -->
<!-- 10 References -->
<!-- An ensemble of NMR structures of the Thylakoid soluble phosphoprotein TSP9, which shows a largely flexible protein chain.[7] -->
<!-- In the 1930s-1950s, the first protein structures were solved by protein crystallography. These early structures suggested that a fixed three-dimensional structure might be generally required to mediate biological functions of proteins. These publications solidified the central dogma of molecular biology in that the amino acid sequence of a protein determines its structure which, in turn, determines its function.  -->
<!-- In 1950, Karush wrote about 'Configurational Adaptability' contradicting this assumption. He was convinced that proteins have more than one configuration at the same energy level and can choose one when binding to other substrates. In the 1960s, Levinthal's paradox suggested that the systematic conformational search of a long polypeptide is unlikely to yield a single folded protein structure on biologically relevant timescales (i.e. microseconds to minutes). Curiously, for many (small) proteins or protein domains, relatively rapid and efficient refolding can be observed in vitro. As stated in Anfinsen's Dogma from 1973, the fixed 3D structure of these proteins is uniquely encoded in its primary structure (the amino acid sequence), is kinetically accessible and stable under a range of (near) physiological conditions, and can therefore be considered as the native state of such "ordered" proteins.[8] -->
<!-- During the subsequent decades, however, many large protein regions could not be assigned in x-ray datasets, indicating that they occupy multiple positions, which average out in electron density maps. The lack of fixed, unique positions relative to the crystal lattice suggested that these regions were "disordered". Nuclear magnetic resonance spectroscopy of proteins also demonstrated the presence of large flexible linkers and termini in many solved structural ensembles. -->
<!-- In 2001, Dunker questioned whether the newly found information was ignored for 50 years[9] with more quantitative analyses becoming available in the 2000s.[10] In the 2010s it became clear that IDPs are common among disease-related proteins, such as alpha-synuclein and tau.[11] -->
<div id="abundance" class="section level3" number="7.2.1">
<h3><span class="header-section-number">7.2.1</span> Abundance</h3>
<p>It is now generally accepted that proteins exist as an ensemble of similar structures with some regions more constrained than others. IDPs occupy the extreme end of this spectrum of flexibility.</p>
<p>Bioinformatic predictions indicated that intrinsic disorder is more common in genomes and proteomes than in known structures in the protein database. Based on DISOPRED2 prediction, long (&gt;30 residue) disordered segments occur in 2.0% of archaean, 4.2% of eubacterial and 33.0% of eukaryotic proteins (10) including certain disease-related proteins (11).</p>
<!-- Biological roles -->
<!-- Highly dynamic disordered regions of proteins have been linked to functionally important phenomena such as allosteric regulation and enzyme catalysis.[12][13] Many disordered proteins have the binding affinity with their receptors regulated by post-translational modification, thus it has been proposed that the flexibility of disordered proteins facilitates the different conformational requirements for binding the modifying enzymes as well as their receptors.[14] Intrinsic disorder is particularly enriched in proteins implicated in cell signaling, transcription and chromatin remodeling functions.[15][16] Genes that have recently been born de novo tend to have higher disorder.[17][18] -->
<!-- Flexible linkers -->
<!-- Disordered regions are often found as flexible linkers or loops connecting domains. Linker sequences vary greatly in length but are typically rich in polar uncharged amino acids. Flexible linkers allow the connecting domains to freely twist and rotate to recruit their binding partners via protein domain dynamics. They also allow their binding partners to induce larger scale conformational changes by long-range allostery.[12][2] -->
<!-- Linear motifs -->
<!-- Linear motifs are short disordered segments of proteins that mediate functional interactions with other proteins or other biomolecules (RNA, DNA, sugars etc.). Many roles of linear motifs are associated with cell regulation, for instance in control of cell shape, subcellular localisation of individual proteins and regulated protein turnover. Often, post-translational modifications such as phosphorylation tune the affinity (not rarely by several orders of magnitude) of individual linear motifs for specific interactions. Relatively rapid evolution and a relatively small number of structural restraints for establishing novel (low-affinity) interfaces make it particularly challenging to detect linear motifs but their widespread biological roles and the fact that many viruses mimick/hijack linear motifs to efficiently recode infected cells underlines the timely urgency of research on this very challenging and exciting topic. Unlike globular proteins IDPs do not have spatially-disposed active pockets. Nevertheless, in 80% of IDPs (~3 dozens) subjected to detailed structural characterization by NMR there are linear motifs termed PreSMos (pre-structured motifs) that are transient secondary structural elements primed for target recognition. In several cases it has been demonstrated that these transient structures become full and stable secondary structures, e.g., helices, upon target binding. Hence, PreSMos are the putative active sites in IDPs.[19] -->
<!-- Coupled folding and binding -->
<!-- Many unstructured proteins undergo transitions to more ordered states upon binding to their targets (e.g. Molecular Recognition Features (MoRFs)[20]). The coupled folding and binding may be local, involving only a few interacting residues, or it might involve an entire protein domain. It was recently shown that the coupled folding and binding allows the burial of a large surface area that would be possible only for fully structured proteins if they were much larger.[21] Moreover, certain disordered regions might serve as "molecular switches" in regulating certain biological function by switching to ordered conformation upon molecular recognition like small molecule-binding, DNA/RNA binding, ion interactions etc.[22] -->
<!-- The ability of disordered proteins to bind, and thus to exert a function, shows that stability is not a required condition. Many short functional sites, for example Short Linear Motifs are over-represented in disordered proteins. Disordered proteins and short linear motifs are particularly abundant in many RNA viruses such as Hendra virus, HCV, HIV-1 and human papillomaviruses. This enables such viruses to overcome their informationally limited genomes by facilitating binding, and manipulation of, a large number of host cell proteins.[23][24] -->
<!-- Disorder in the bound state (fuzzy complexes) -->
<!-- Intrinsically disordered proteins can retain their conformational freedom even when they bind specifically to other proteins. The structural disorder in bound state can be static or dynamic. In fuzzy complexes structural multiplicity is required for function and the manipulation of the bound disordered region changes activity. The conformational ensemble of the complex is modulated via post-translational modifications or protein interactions.[25] Specificity of DNA binding proteins often depends on the length of fuzzy regions, which is varied by alternative splicing.[26] Some fuzzy complexes may exhibit high binding affinity,[27] although other studies showed different affinity values for the same system in a different concentration regime.[28] -->
<!-- Structural aspects -->
<!-- Intrinsically disordered proteins adapt many different structures in vivo according to the cell's conditions, creating a structural or conformational ensemble.[29][30] -->
<!-- Therefore, their structures are strongly function-related. However, only few proteins are fully disordered in their native state. Disorder is mostly found in intrinsically disordered regions (IDRs) within an otherwise well-structured protein. The term intrinsically disordered protein (IDP) therefore includes proteins that contain IDRs as well as fully disordered proteins. -->
<!-- The existence and kind of protein disorder is encoded in its amino acid sequence.[2] In general, IDPs are characterized by a low content of bulky hydrophobic amino acids and a high proportion of polar and charged amino acids, usually referred to as low hydrophobicity.[29] This property leads to good interactions with water. Furthermore, high net charges promote disorder because of electrostatic repulsion resulting from equally charged residues.[30] Thus disordered sequences cannot sufficiently bury a hydrophobic core to fold into stable globular proteins. In some cases, hydrophobic clusters in disordered sequences provide the clues for identifying the regions that undergo coupled folding and binding (refer to biological roles). Many disordered proteins reveal regions without any regular secondary structure. These regions can be termed as flexible, compared to structured loops. While the latter are rigid and contain only one set of Ramachandran angles, IDPs involve multiple sets of angles.[30] The term flexibility is also used for well-structured proteins, but describes a different phenomenon in the context of disordered proteins. Flexibility in structured proteins is bound to an equilibrium state, while it is not so in IDPs.[30] Many disordered proteins also reveal low complexity sequences, i.e. sequences with over-representation of a few residues. While low complexity sequences are a strong indication of disorder, the reverse is not necessarily true, that is, not all disordered proteins have low complexity sequences. Disordered proteins have a low content of predicted secondary structure. -->
<!-- Experimental validation -->
<!-- IDPs can be validated in several contexts. Most approaches for experimental validation of IDPs are restricted to extracted or purified proteins while some new experimental strategies aim to explore in vivo conformations and structural variations of IDPs inside intact living cells and systematic comparisons between their dynamics in vivo and in vitro. -->
<!-- In vivo approaches -->
<!-- The first direct evidence for in vivo persistence of intrinsic disorder has been achieved by in-cell NMR upon electroporation of a purified IDP and recovery of cells to an intact state.[31] -->
<!-- Larger-scale in vivo validation of IDR predictions is now possible using biotin 'painting'.[32][33] -->
<!-- In vitro approaches -->
<!-- Intrinsically unfolded proteins, once purified, can be identified by various experimental methods. The primary method to obtain information on disordered regions of a protein is NMR spectroscopy. The lack of electron density in X-ray crystallographic studies may also be a sign of disorder. -->
<!-- Folded proteins have a high density (partial specific volume of 0.72-0.74 mL/g) and commensurately small radius of gyration. Hence, unfolded proteins can be detected by methods that are sensitive to molecular size, density or hydrodynamic drag, such as size exclusion chromatography, analytical ultracentrifugation, small angle X-ray scattering (SAXS), and measurements of the diffusion constant. Unfolded proteins are also characterized by their lack of secondary structure, as assessed by far-UV (170-250 nm) circular dichroism (esp. a pronounced minimum at ~200 nm) or infrared spectroscopy. Unfolded proteins also have exposed backbone peptide groups exposed to solvent, so that they are readily cleaved by proteases, undergo rapid hydrogen-deuterium exchange and exhibit a small dispersion (<1 ppm) in their 1H amide chemical shifts as measured by NMR. (Folded proteins typically show dispersions as large as 5 ppm for the amide protons.) Recently, new methods including Fast parallel proteolysis (FASTpp) have been introduced, which allow to determine the fraction folded/disordered without the need for purification.[34][35] Even subtle differences in the stability of missense mutations, protein partner binding and (self)polymerisation-induced folding of (e.g.) coiled-coils can be detected using FASTpp as recently demonstrated using the tropomyosin-troponin protein interaction.[36] Fully unstructured protein regions can be experimentally validated by their hypersusceptibility to proteolysis using short digestion times and low protease concentrations.[37] -->
<!-- Bulk methods to study IDP structure and dynamics include SAXS for ensemble shape information, NMR for atomistic ensemble refinement, Fluorescence for visualising molecular interactions and conformational transitions, x-ray crystallography to highlight more mobile regions in otherwise rigid protein crystals, cryo-EM to reveal less fixed parts of proteins, light scattering to monitor size distributions of IDPs or their aggregation kinetics, NMR chemical shift and Circular Dichroism to monitor secondary structure of IDPs. -->
<!-- Single-molecule methods to study IDPs include spFRET[38] to study conformational flexibility of IDPs and the kinetics of structural transitions, optical tweezers[39] for high-resolution insights into the ensembles of IDPs and their oligomers or aggregates, nanopores[40] to reveal global shape distributions of IDPs, magnetic tweezers[41] to study structural transitions for long times at low forces, high-speed AFM[42] to visualise the spatio-temporal flexibility of IDPs directly. -->
</div>
<div id="disorder-annotation" class="section level3" number="7.2.2">
<h3><span class="header-section-number">7.2.2</span> Disorder annotation</h3>
<!-- REMARK465 - missing electron densities in X-ray structure representing protein disorder (PDB: 1a22, human growth hormone bound to receptor). Compilation of screenshots from PDB database and molecule representation via VMD. Blue and red arrows point to missing residues on receptor and growth hormone, respectively. -->
<p>Intrinsic disorder can be either annotated from experimental information or predicted with specialized software. Disorder prediction algorithms can predict Intrinsic Disorder (ID) propensity with high accuracy (approaching around 80%) based on primary sequence composition, similarity to unassigned segments in protein x-ray datasets, flexible regions in NMR studies and physico-chemical properties of amino acids.</p>
</div>
<div id="disorder-databases" class="section level3" number="7.2.3">
<h3><span class="header-section-number">7.2.3</span> Disorder databases</h3>
<p>Databases have been established to annotate protein sequences with intrinsic disorder information. The <strong>DisProt database</strong> contains a collection of <strong>manually curated</strong> protein segments which have been experimentally determined to be disordered. MobiDB is a database combining experimentally curated disorder annotations (e.g. from DisProt) with data derived from missing residues in X-ray crystallographic structures and flexible regions in NMR structures.</p>
</div>
<div id="predicting-idps-by-sequence" class="section level3" number="7.2.4">
<h3><span class="header-section-number">7.2.4</span> Predicting IDPs by sequence</h3>
<p>Separating disordered from ordered proteins is essential for disorder prediction. One of the first steps to find a factor that distinguishes IDPs from non-IDPs is to specify biases within the amino acid composition. The hydrophilic, charged amino acids (A, R, G, Q, S, P, E and K) have been characterized as disorder-promoting amino acids, while order-promoting amino acids (W, C, F, I, Y, V, L, and N) are hydrophobic and uncharged. The remaining amino acids (H, M, T and D) are ambiguous, found in both ordered and unstructured regions (2). A more recent analysis ranked amino acids by their propensity to form disordered regions as follows (order promoting to disorder promoting): W, F, Y, I, M, L, V, N, C, T, A, G, R, D, H, Q, K, S, E, P (43).</p>
<p>This information is the basis of most sequence-based predictors. Regions with little to no secondary structure, also known as NORS (NO Regular Secondary structure) regions (44) and low-complexity regions can easily be detected. However, not all disordered proteins contain such low complexity sequences.</p>
</div>
<div id="prediction-methods" class="section level3" number="7.2.5">
<h3><span class="header-section-number">7.2.5</span> Prediction methods</h3>
<p>Determining disordered regions from lab methods is very costly and time-consuming. Due to the variable nature of IDPs, only certain aspects of their structure can be detected, so that a full characterization requires a large number of different methods and experiments. This further increases the expense of IDP determination. In order to overcome this obstacle, computer-based methods are created for predicting protein structure and function. It is one of the main goals of bioinformatics to derive knowledge by prediction. Predictors for IDP function are also being developed, but mainly use structural information such as linear motif sites (4, 45). There are different approaches for predicting IDP structure, such as neural networks or matrix calculations, based on different structural and/or biophysical properties.</p>
<p>Many computational methods exploit sequence information to predict whether a protein is disordered (46). Notable examples of such software include IUPRED and Disopred. Different methods may use different definitions of disorder. Meta-predictors show a new concept, combining different primary predictors to create a more competent and exact predictor.</p>
<p>Due to the different approaches of predicting disordered proteins, estimating their relative accuracy is fairly difficult. For example, neural networks are often trained on different datasets. The disorder prediction category is a part of biannual CASP experiment that is designed to test methods according accuracy in finding regions with missing 3D structure (marked in PDB files as REMARK465, missing electron densities in X-ray structures).</p>
</div>
<div id="disorder-and-disease" class="section level3" number="7.2.6">
<h3><span class="header-section-number">7.2.6</span> Disorder and disease</h3>
<p><strong>This section optional</strong></p>
<p>Intrinsically unstructured proteins have been implicated in a number of diseases (47). Aggregation of misfolded proteins is the cause of many synucleinopathies and toxicity as those proteins start binding to each other randomly and can lead to cancer or cardiovascular diseases. Thereby, misfolding can happen spontaneously because millions of copies of proteins are made during the lifetime of an organism. The aggregation of the intrinsically unstructured protein α-synuclein is thought to be responsible. The structural flexibility of this protein together with its susceptibility to modification in the cell leads to misfolding and aggregation. Genetics, oxidative and nitrative stress as well as mitochondrial impairment impact the structural flexibility of the unstructured α-synuclein protein and associated disease mechanisms (48). Many key tumour suppressors have large intrinsically unstructured regions, for example p53 and BRCA1. These regions of the proteins are responsible for mediating many of their interactions. Taking the cell’s native defense mechanisms as a model drugs can be developed, trying to block the place of noxious substrates and inhibiting them, and thus counteracting the disease (49).</p>
<!-- Computer simulations -->
<!-- Owing to high structural heterogeneity, NMR/SAXS experimental parameters obtained will be an average over a large number of highly diverse and disordered states (an ensemble of disordered states). Hence, to understand the structural implications of these experimental parameters, there is a necessity for accurate representation of these ensembles by computer simulations. All-atom molecular dynamic simulations can be used for this purpose but their use is limited by the accuracy of current force-fields in representing disordered proteins. Nevertheless, some force-fields have been explicitly developed for studying disordered proteins by optimising force-field parameters using available NMR data for disordered proteins. (examples are CHARMM 22*, CHARMM 32,[50] Amber ff03* etc.) -->
<!-- MD simulations restrained by experimental parameters (restrained-MD) have also been used to characterise disordered proteins.[51][52][53] In principle, one can sample the whole conformational space given an MD simulation (with accurate Force-field) is run long enough. Because of very high structural heterogeneity, the time scales that needs to be run for this purpose are very large and are limited by computational power. However, other computational techniques such as accelerated-MD simulations,[54] replica exchange simulations,[55][56] metadynamics,[57][58] multicanonical MD simulations,[59] or methods using coarse-grained representation[60][61] have been used to sample broader conformational space in smaller time scales. -->
<!-- Moreover, various protocols and methods of analyzing IDPs, such as studies based on quantitative analysis of GC content in genes and their respective chromosomal bands, have been used to understand functional IDP segments.[62][63] -->
<!-- See also -->
<!-- IDPbyNMR -->
<!-- DisProt database -->
<!-- MobiDB database -->
<!-- Molten globule -->
<!-- Random coil -->
<p>References</p>
<ol style="list-style-type: decimal">
<li>Majorek K, Kozlowski L, Jakalski M, Bujnicki JM (December 18, 2008). “Chapter 2: First Steps of Protein Structure Prediction” (PDF). In Bujnicki J (ed.). Prediction of Protein Structures, Functions, and Interactions. John Wiley &amp; Sons, Ltd. pp. 39–62. <a href="doi:10.1002/9780470741894.ch2" class="uri">doi:10.1002/9780470741894.ch2</a>. ISBN 9780470517673.</li>
<li>Dunker AK, Lawson JD, Brown CJ, Williams RM, Romero P, Oh JS, Oldfield CJ, Campen AM, Ratliff CM, Hipps KW, Ausio J, Nissen MS, Reeves R, Kang C, Kissinger CR, Bailey RW, Griswold MD, Chiu W, Garner EC, Obradovic Z (2001). “Intrinsically disordered protein”. Journal of Molecular Graphics &amp; Modelling. 19 (1): 26–59. CiteSeerX 10.1.1.113.556. <a href="doi:10.1016/s1093-3263(00)00138-8" class="uri">doi:10.1016/s1093-3263(00)00138-8</a>. PMID 11381529.</li>
<li>Dyson HJ, Wright PE (March 2005). “Intrinsically unstructured proteins and their functions”. Nature Reviews Molecular Cell Biology. 6 (3): 197–208. <a href="doi:10.1038/nrm1589" class="uri">doi:10.1038/nrm1589</a>. PMID 15738986. S2CID 18068406.</li>
<li>Dunker AK, Silman I, Uversky VN, Sussman JL (December 2008). “Function and structure of inherently disordered proteins”. Current Opinion in Structural Biology. 18 (6): 756–64. <a href="doi:10.1016/j.sbi.2008.10.002" class="uri">doi:10.1016/j.sbi.2008.10.002</a>. PMID 18952168.</li>
<li>Andreeva A, Howorth D, Chothia C, Kulesha E, Murzin AG (January 2014). “SCOP2 prototype: a new approach to protein structure mining”. Nucleic Acids Research. 42 (Database issue): D310–4. <a href="doi:10.1093/nar/gkt1242" class="uri">doi:10.1093/nar/gkt1242</a>. PMC 3964979. PMID 24293656.</li>
<li>van der Lee R, Buljan M, Lang B, Weatheritt RJ, Daughdrill GW, Dunker AK, Fuxreiter M, Gough J, Gsponer J, Jones DT, Kim PM, Kriwacki RW, Oldfield CJ, Pappu RV, Tompa P, Uversky VN, Wright PE, Babu MM (2014). “Classification of intrinsically disordered regions and proteins”. Chemical Reviews. 114 (13): 6589–631. <a href="doi:10.1021/cr400525m" class="uri">doi:10.1021/cr400525m</a>. PMC 4095912. PMID 24773235.</li>
<li>Song J, Lee MS, Carlberg I, Vener AV, Markley JL (December 2006). “Micelle-induced folding of spinach thylakoid soluble phosphoprotein of 9 kDa and its functional implications”. Biochemistry. 45 (51): 15633–43. <a href="doi:10.1021/bi062148m" class="uri">doi:10.1021/bi062148m</a>. PMC 2533273. PMID 17176085.</li>
<li>Anfinsen, Christian B. (20 July 1973). “Principles that Govern the Folding of Protein Chains”. Science. 181 (4096): 223–230. <a href="doi:10.1126/science.181.4096.223" class="uri">doi:10.1126/science.181.4096.223</a>. PMID 4124164.</li>
<li>Dunker AK, Lawson JD, Brown CJ, Williams RM, Romero P, Oh JS, Oldfield CJ, Campen AM, Ratliff CM, Hipps KW, Ausio J, Nissen MS, Reeves R, Kang C, Kissinger CR, Bailey RW, Griswold MD, Chiu W, Garner EC, Obradovic Z (2001-01-01). “Intrinsically disordered protein”. Journal of Molecular Graphics &amp; Modelling. 19 (1): 26–59. CiteSeerX 10.1.1.113.556. <a href="doi:10.1016/s1093-3263(00)00138-8" class="uri">doi:10.1016/s1093-3263(00)00138-8</a>. PMID 11381529.</li>
<li>Ward JJ, Sodhi JS, McGuffin LJ, Buxton BF, Jones DT (March 2004). “Prediction and functional analysis of native disorder in proteins from the three kingdoms of life”. Journal of Molecular Biology. 337 (3): 635–45. CiteSeerX 10.1.1.120.5605. <a href="doi:10.1016/j.jmb.2004.02.002" class="uri">doi:10.1016/j.jmb.2004.02.002</a>. PMID 15019783.</li>
<li>Uversky VN, Oldfield CJ, Dunker AK (2008). “Intrinsically disordered proteins in human diseases: introducing the D2 concept”. Annual Review of Biophysics. 37: 215–46. <a href="doi:10.1146/annurev.biophys.37.032807.125924" class="uri">doi:10.1146/annurev.biophys.37.032807.125924</a>. PMID 18573080.</li>
<li>Bu Z, Callaway DJ (2011). “Proteins move! Protein dynamics and long-range allostery in cell signaling”. Protein Structure and Diseases. Advances in Protein Chemistry and Structural Biology. 83. pp. 163–221. <a href="doi:10.1016/B978-0-12-381262-9.00005-7" class="uri">doi:10.1016/B978-0-12-381262-9.00005-7</a>. ISBN 9780123812629. PMID 21570668.</li>
<li>Kamerlin SC, Warshel A (May 2010). “At the dawn of the 21st century: Is dynamics the missing link for understanding enzyme catalysis?”. Proteins. 78 (6): 1339–75. <a href="doi:10.1002/prot.22654" class="uri">doi:10.1002/prot.22654</a>. PMC 2841229. PMID 20099310.</li>
<li>Collins MO, Yu L, Campuzano I, Grant SG, Choudhary JS (July 2008). “Phosphoproteomic analysis of the mouse brain cytosol reveals a predominance of protein phosphorylation in regions of intrinsic sequence disorder” (PDF). Molecular &amp; Cellular Proteomics. 7 (7): 1331–48. <a href="doi:10.1074/mcp.M700564-MCP200" class="uri">doi:10.1074/mcp.M700564-MCP200</a>. PMID 18388127. S2CID 22193414.</li>
<li>Iakoucheva LM, Brown CJ, Lawson JD, Obradović Z, Dunker AK (October 2002). “Intrinsic disorder in cell-signaling and cancer-associated proteins”. Journal of Molecular Biology. 323 (3): 573–84. CiteSeerX 10.1.1.132.682. <a href="doi:10.1016/S0022-2836(02)00969-5" class="uri">doi:10.1016/S0022-2836(02)00969-5</a>. PMID 12381310.</li>
<li>Sandhu KS (2009). “Intrinsic disorder explains diverse nuclear roles of chromatin remodeling proteins”. Journal of Molecular Recognition. 22 (1): 1–8. <a href="doi:10.1002/jmr.915" class="uri">doi:10.1002/jmr.915</a>. PMID 18802931. S2CID 33010897.</li>
<li>Wilson, Benjamin A.; Foy, Scott G.; Neme, Rafik; Masel, Joanna (24 April 2017). “Young genes are highly disordered as predicted by the preadaptation hypothesis of de novo gene birth”. Nature Ecology &amp; Evolution. 1 (6): 0146–146. <a href="doi:10.1038/s41559-017-0146" class="uri">doi:10.1038/s41559-017-0146</a>. PMC 5476217. PMID 28642936.</li>
<li>Willis, Sara; Masel, Joanna (September 2018). “Gene Birth Contributes to Structural Disorder Encoded by Overlapping Genes”. Genetics. 210 (1): 303–313. <a href="doi:10.1534/genetics.118.301249" class="uri">doi:10.1534/genetics.118.301249</a>. PMC 6116962. PMID 30026186.
Lee SH, Kim DH, Han JJ, Cha EJ, Lim JE, Cho YJ, Lee C, Han KH (February 2012). “Understanding pre-structured motifs (PreSMos) in intrinsically unfolded proteins”. Current Protein &amp; Peptide Science. 13 (1): 34–54. <a href="doi:10.2174/138920312799277974" class="uri">doi:10.2174/138920312799277974</a>. PMID 22044148.</li>
<li>Mohan A, Oldfield CJ, Radivojac P, Vacic V, Cortese MS, Dunker AK, Uversky VN (October 2006). “Analysis of molecular recognition features (MoRFs)”. Journal of Molecular Biology. 362 (5): 1043–59. <a href="doi:10.1016/j.jmb.2006.07.087" class="uri">doi:10.1016/j.jmb.2006.07.087</a>. PMID 16935303.</li>
<li>Gunasekaran K, Tsai CJ, Kumar S, Zanuy D, Nussinov R (February 2003). “Extended disordered proteins: targeting function with less scaffold”. Trends in Biochemical Sciences. 28 (2): 81–5. <a href="doi:10.1016/S0968-0004(03)00003-3" class="uri">doi:10.1016/S0968-0004(03)00003-3</a>. PMID 12575995.</li>
<li>Sandhu KS, Dash D (July 2007). “Dynamic alpha-helices: conformations that do not conform”. Proteins. 68 (1): 109–22. <a href="doi:10.1002/prot.21328" class="uri">doi:10.1002/prot.21328</a>. PMID 17407165. S2CID 96719019.</li>
<li>Tarakhovsky A, Prinjha RK (July 2018). “Drawing on disorder: How viruses use histone mimicry to their advantage”. The Journal of Experimental Medicine. 215 (7): 1777–1787. <a href="doi:10.1084/jem.20180099" class="uri">doi:10.1084/jem.20180099</a>. PMC 6028506. PMID 29934321.</li>
<li>Atkinson SC, Audsley MD, Lieu KG, Marsh GA, Thomas DR, Heaton SM, Paxman JJ, Wagstaff KM, Buckle AM, Moseley GW, Jans DA, Borg NA (January 2018). “Recognition by host nuclear transport proteins drives disorder-to-order transition in Hendra virus V”. Scientific Reports. 8 (1): 358. Bibcode:2018NatSR…8..358A. <a href="doi:10.1038/s41598-017-18742-8" class="uri">doi:10.1038/s41598-017-18742-8</a>. PMC 5762688. PMID 29321677.</li>
<li>Fuxreiter M (January 2012). “Fuzziness: linking regulation to protein dynamics”. Molecular BioSystems. 8 (1): 168–77. <a href="doi:10.1039/c1mb05234a" class="uri">doi:10.1039/c1mb05234a</a>. PMID 21927770.</li>
<li>Fuxreiter M, Simon I, Bondos S (August 2011). “Dynamic protein-DNA recognition: beyond what can be seen”. Trends in Biochemical Sciences. 36 (8): 415–23. <a href="doi:10.1016/j.tibs.2011.04.006" class="uri">doi:10.1016/j.tibs.2011.04.006</a>. PMID 21620710.</li>
<li>Borgia A, Borgia MB, Bugge K, Kissling VM, Heidarsson PO, Fernandes CB, Sottini A, Soranno A, Buholzer KJ, Nettels D, Kragelund BB, Best RB, Schuler B (March 2018). “Extreme disorder in an ultrahigh-affinity protein complex”. Nature. 555 (7694): 61–66. Bibcode:2018Natur.555…61B. <a href="doi:10.1038/nature25762" class="uri">doi:10.1038/nature25762</a>. PMC 6264893. PMID 29466338.</li>
<li>Feng H, Zhou BR, Bai Y (November 2018). “Binding Affinity and Function of the Extremely Disordered Protein Complex Containing Human Linker Histone H1.0 and Its Chaperone ProTα”. Biochemistry. 57 (48): 6645–6648. <a href="doi:10.1021/acs.biochem.8b01075" class="uri">doi:10.1021/acs.biochem.8b01075</a>. PMC 7984725. PMID 30430826.
Uversky VN (August 2011). “Intrinsically disordered proteins from A to Z”. The International Journal of Biochemistry &amp; Cell Biology. 43 (8): 1090–103. <a href="doi:10.1016/j.biocel.2011.04.001" class="uri">doi:10.1016/j.biocel.2011.04.001</a>. PMID 21501695.</li>
<li>Oldfield, C. (2014). “Intrinsically Disordered Proteins and Intrinsically Disordered Protein Regions”. Annual Review of Biochemistry. 83: 553–584. <a href="doi:10.1146/annurev-biochem-072711-164947" class="uri">doi:10.1146/annurev-biochem-072711-164947</a>. PMID 24606139.</li>
<li>Theillet FX, Binolfi A, Bekei B, Martorana A, Rose HM, Stuiver M, Verzini S, Lorenz D, van Rossum M, Goldfarb D, Selenko P (2016). “Structural disorder of monomeric α-synuclein persists in mammalian cells”. Nature. 530 (7588): 45–50. Bibcode:2016Natur.530…45T. <a href="doi:10.1038/nature16531" class="uri">doi:10.1038/nature16531</a>. PMID 26808899. S2CID 4461465.</li>
<li>Minde DP, Ramakrishna M, Lilley KS (2018). “Biotinylation by proximity labelling favours unfolded proteins”. bioRxiv. <a href="doi:10.1101/274761" class="uri">doi:10.1101/274761</a>.</li>
<li>Minde DP, Ramakrishna M, Lilley KS (2020). “Biotin proximity tagging favours unfolded proteins and enables the study of intrinsically disordered regions”. Communications Biology. 3 (1): 38. <a href="doi:10.1038/s42003-020-0758-y" class="uri">doi:10.1038/s42003-020-0758-y</a>. PMC 6976632. PMID 31969649.</li>
<li>Minde DP, Maurice MM, Rüdiger SG (2012). Uversky VN (ed.). “Determining biophysical protein stability in lysates by a fast proteolysis assay, FASTpp”. PLOS ONE. 7 (10): e46147. Bibcode:2012PLoSO…746147M. <a href="doi:10.1371/journal.pone.0046147" class="uri">doi:10.1371/journal.pone.0046147</a>. PMC 3463568. PMID 23056252.</li>
<li>Park C, Marqusee S (March 2005). “Pulse proteolysis: a simple method for quantitative determination of protein stability and ligand binding”. Nature Methods. 2 (3): 207–12. <a href="doi:10.1038/nmeth740" class="uri">doi:10.1038/nmeth740</a>. PMID 15782190. S2CID 21364478.</li>
<li>Robaszkiewicz K, Ostrowska Z, Cyranka-Czaja A, Moraczewska J (May 2015). “Impaired tropomyosin-troponin interactions reduce activation of the actin thin filament”. Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics. 1854 (5): 381–90. <a href="doi:10.1016/j.bbapap.2015.01.004" class="uri">doi:10.1016/j.bbapap.2015.01.004</a>. PMID 25603119.</li>
<li>Minde DP, Radli M, Forneris F, Maurice MM, Rüdiger SG (2013). Buckle AM (ed.). “Large extent of disorder in Adenomatous Polyposis Coli offers a strategy to guard Wnt signalling against point mutations”. PLOS ONE. 8 (10): e77257. Bibcode:2013PLoSO…877257M. <a href="doi:10.1371/journal.pone.0077257" class="uri">doi:10.1371/journal.pone.0077257</a>. PMC 3793970. PMID 24130866.</li>
<li>Brucale M, Schuler B, Samorì B (March 2014). “Single-molecule studies of intrinsically disordered proteins”. Chemical Reviews. 114 (6): 3281–317. <a href="doi:10.1021/cr400297g" class="uri">doi:10.1021/cr400297g</a>. PMID 24432838.</li>
<li>Neupane K, Solanki A, Sosova I, Belov M, Woodside MT (2014). “Diverse metastable structures formed by small oligomers of α-synuclein probed by force spectroscopy”. PLOS ONE. 9 (1): e86495. Bibcode:2014PLoSO…986495N. <a href="doi:10.1371/journal.pone.0086495" class="uri">doi:10.1371/journal.pone.0086495</a>. PMC 3901707. PMID 24475132.</li>
<li>Japrung D, Dogan J, Freedman KJ, Nadzeyka A, Bauerdick S, Albrecht T, Kim MJ, Jemth P, Edel JB (February 2013). “Single-molecule studies of intrinsically disordered proteins using solid-state nanopores”. Analytical Chemistry. 85 (4): 2449–56. <a href="doi:10.1021/ac3035025" class="uri">doi:10.1021/ac3035025</a>. PMID 23327569.</li>
<li>Min D, Kim K, Hyeon C, Cho YH, Shin YK, Yoon TY (2013). “Mechanical unzipping and rezipping of a single SNARE complex reveals hysteresis as a force-generating mechanism”. Nature Communications. 4 (4): 1705. Bibcode:2013NatCo…4.1705M. <a href="doi:10.1038/ncomms2692" class="uri">doi:10.1038/ncomms2692</a>. PMC 3644077. PMID 23591872.</li>
<li>Miyagi A, Tsunaka Y, Uchihashi T, Mayanagi K, Hirose S, Morikawa K, Ando T (September 2008). “Visualization of intrinsically disordered regions of proteins by high-speed atomic force microscopy”. ChemPhysChem. 9 (13): 1859–66. <a href="doi:10.1002/cphc.200800210" class="uri">doi:10.1002/cphc.200800210</a>. PMID 18698566.</li>
<li>Campen, Andrew; Williams, Ryan M.; Brown, Celeste J.; Meng, Jingwei; Uversky, Vladimir N.; Dunker, A. Keith (2008). “TOP-IDP-scale: a new amino acid scale measuring propensity for intrinsic disorder”. Protein and Peptide Letters. 15 (9): 956–963. <a href="doi:10.2174/092986608785849164" class="uri">doi:10.2174/092986608785849164</a>. ISSN 0929-8665. PMC 2676888. PMID 18991772.</li>
<li>Schlessinger A, Schaefer C, Vicedo E, Schmidberger M, Punta M, Rost B (June 2011). “Protein disorder–a breakthrough invention of evolution?”. Current Opinion in Structural Biology. 21 (3): 412–8. <a href="doi:10.1016/j.sbi.2011.03.014" class="uri">doi:10.1016/j.sbi.2011.03.014</a>. PMID 21514145.</li>
<li>Tompa, P. (2011). “Unstructural biology coming of age”. Current Opinion in Structural Biology. 21 (3): 419–425. <a href="doi:10.1016/j.sbi.2011.03.012" class="uri">doi:10.1016/j.sbi.2011.03.012</a>. PMID 21514142.</li>
<li>Ferron F, Longhi S, Canard B, Karlin D (October 2006). “A practical overview of protein disorder prediction methods”. Proteins. 65 (1): 1–14. <a href="doi:10.1002/prot.21075" class="uri">doi:10.1002/prot.21075</a>. PMID 16856179. S2CID 30231497.</li>
<li>Uversky VN, Oldfield CJ, Dunker AK (2008). “Intrinsically disordered proteins in human diseases: introducing the D2 concept”. Annual Review of Biophysics. 37: 215–46. <a href="doi:10.1146/annurev.biophys.37.032807.125924" class="uri">doi:10.1146/annurev.biophys.37.032807.125924</a>. PMID 18573080.</li>
<li>Wise-Scira O, Dunn A, Aloglu AK, Sakallioglu IT, Coskuner O (March 2013). “Structures of the E46K mutant-type α-synuclein protein and impact of E46K mutation on the structures of the wild-type α-synuclein protein”. ACS Chemical Neuroscience. 4 (3): 498–508. <a href="doi:10.1021/cn3002027" class="uri">doi:10.1021/cn3002027</a>. PMC 3605821. PMID 23374074.</li>
<li>Dobson CM (December 2003). “Protein folding and misfolding”. Nature. 426 (6968): 884–90. Bibcode:2003Natur.426..884D. <a href="doi:10.1038/nature02261" class="uri">doi:10.1038/nature02261</a>. PMID 14685248. S2CID 1036192.</li>
<li>Best RB, Zhu X, Shim J, Lopes PE, Mittal J, Feig M, Mackerell AD (September 2012). “Optimization of the additive CHARMM all-atom protein force field targeting improved sampling of the backbone φ, ψ and side-chain χ(1) and χ(2) dihedral angles”. Journal of Chemical Theory and Computation. 8 (9): 3257–3273. <a href="doi:10.1021/ct300400x" class="uri">doi:10.1021/ct300400x</a>. PMC 3549273. PMID 23341755.</li>
<li>Best RB (February 2017). “Computational and theoretical advances in studies of intrinsically disordered proteins”. Current Opinion in Structural Biology. 42: 147–154. <a href="doi:10.1016/j.sbi.2017.01.006" class="uri">doi:10.1016/j.sbi.2017.01.006</a>. PMID 28259050.</li>
<li>Chong SH, Chatterjee P, Ham S (May 2017). “Computer Simulations of Intrinsically Disordered Proteins”. Annual Review of Physical Chemistry. 68: 117–134. Bibcode:2017ARPC…68..117C. <a href="doi:10.1146/annurev-physchem-052516-050843" class="uri">doi:10.1146/annurev-physchem-052516-050843</a>. PMID 28226222.</li>
<li>Fox SJ, Kannan S (September 2017). “Probing the dynamics of disorder”. Progress in Biophysics and Molecular Biology. 128: 57–62. <a href="doi:10.1016/j.pbiomolbio.2017.05.008" class="uri">doi:10.1016/j.pbiomolbio.2017.05.008</a>. PMID 28554553.</li>
<li>Terakawa T, Takada S (September 2011). “Multiscale ensemble modeling of intrinsically disordered proteins: p53 N-terminal domain”. Biophysical Journal. 101 (6): 1450–8. Bibcode:2011BpJ…101.1450T. <a href="doi:10.1016/j.bpj.2011.08.003" class="uri">doi:10.1016/j.bpj.2011.08.003</a>. PMC 3177054. PMID 21943426.</li>
<li>Fisher CK, Stultz CM (June 2011). “Constructing ensembles for intrinsically disordered proteins”. Current Opinion in Structural Biology. 21 (3): 426–31. <a href="doi:10.1016/j.sbi.2011.04.001" class="uri">doi:10.1016/j.sbi.2011.04.001</a>. PMC 3112268. PMID 21530234.</li>
<li>Apicella A, Marascio M, Colangelo V, Soncini M, Gautieri A, Plummer CJ (June 2017). “Molecular dynamics simulations of the intrinsically disordered protein amelogenin”. Journal of Biomolecular Structure &amp; Dynamics. 35 (8): 1813–1823. <a href="doi:10.1080/07391102.2016.1196151" class="uri">doi:10.1080/07391102.2016.1196151</a>. PMID 27366858. S2CID 205576649.</li>
<li>Zerze GH, Miller CM, Granata D, Mittal J (June 2015). “Free energy surface of an intrinsically disordered protein: comparison between temperature replica exchange molecular dynamics and bias-exchange metadynamics”. Journal of Chemical Theory and Computation. 11 (6): 2776–82. <a href="doi:10.1021/acs.jctc.5b00047" class="uri">doi:10.1021/acs.jctc.5b00047</a>. PMID 26575570.</li>
<li>Granata D, Baftizadeh F, Habchi J, Galvagnion C, De Simone A, Camilloni C, Laio A, Vendruscolo M (October 2015). “The inverted free energy landscape of an intrinsically disordered peptide by simulations and experiments”. Scientific Reports. 5: 15449. Bibcode:2015NatSR…515449G. <a href="doi:10.1038/srep15449" class="uri">doi:10.1038/srep15449</a>. PMC 4620491. PMID 26498066.</li>
<li>Iida, Shinji; Kawabata, Takeshi; Kasahara, Kota; Nakamura, Haruki; Higo, Junichi (2019-03-22). “Multimodal Structural Distribution of the p53 C-Terminal Domain upon Binding to S100B via a Generalized Ensemble Method: From Disorder to Extradisorder”. Journal of Chemical Theory and Computation. 15 (4): 2597–2607. <a href="doi:10.1021/acs.jctc.8b01042" class="uri">doi:10.1021/acs.jctc.8b01042</a>. ISSN 1549-9618. PMID 30855964.</li>
<li>Kurcinski M, Kolinski A, Kmiecik S (June 2014). “Mechanism of Folding and Binding of an Intrinsically Disordered Protein As Revealed by ab Initio Simulations”. Journal of Chemical Theory and Computation. 10 (6): 2224–31. <a href="doi:10.1021/ct500287c" class="uri">doi:10.1021/ct500287c</a>. PMID 26580746.</li>
<li>Ciemny, Maciej Pawel; Badaczewska-Dawid, Aleksandra Elzbieta; Pikuzinska, Monika; Kolinski, Andrzej; Kmiecik, Sebastian (2019). “Modeling of Disordered Protein Structures Using Monte Carlo Simulations and Knowledge-Based Statistical Force Fields”. International Journal of Molecular Sciences. 20 (3): 606. <a href="doi:10.3390/ijms20030606" class="uri">doi:10.3390/ijms20030606</a>. PMC 6386871. PMID 30708941.</li>
<li>Uversky VN (2013). “Digested disorder: Quarterly intrinsic disorder digest (January/February/March, 2013)”. Intrinsically Disordered Proteins. 1 (1): e25496. <a href="doi:10.4161/idp.25496" class="uri">doi:10.4161/idp.25496</a>. PMC 5424799. PMID 28516015.</li>
<li>Costantini S, Sharma A, Raucci R, Costantini M, Autiero I, Colonna G (March 2013). “Genealogy of an ancient protein family: the Sirtuins, a family of disordered members”. BMC Evolutionary Biology. 13: 60. <a href="doi:10.1186/1471-2148-13-60" class="uri">doi:10.1186/1471-2148-13-60</a>. PMC 3599600. PMID 23497088.</li>
</ol>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="h.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="j.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["openbio.pdf", "openbio.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
