<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title> 14 P | Foundations of Biology and Environmental Science: An Open-Access Encyclopedia</title>
  <meta name="description" content="A collection of open-access and/or free resources on biology and environmental science." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content=" 14 P | Foundations of Biology and Environmental Science: An Open-Access Encyclopedia" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content=" 14 P | Foundations of Biology and Environmental Science: An Open-Access Encyclopedia" />
  
  <meta name="twitter:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

<meta name="author" content="Compiled and edited by Nathan Brouwer" />


<meta name="date" content="2022-01-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="o.html"/>
<link rel="next" href="q.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Biology for everyone</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="a.html"><a href="a.html"><i class="fa fa-check"></i><b>2</b> A</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a.html"><a href="a.html#alignments"><i class="fa fa-check"></i><b>2.1</b> Alignments</a></li>
<li class="chapter" data-level="2.2" data-path="a.html"><a href="a.html#talking-glossary-allele-0.5-min"><i class="fa fa-check"></i><b>2.2</b> Talking Glossary: Allele (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="b.html"><a href="b.html"><i class="fa fa-check"></i><b>3</b> B</a>
<ul>
<li class="chapter" data-level="3.1" data-path="b.html"><a href="b.html#talking-glossary-bioinformatics"><i class="fa fa-check"></i><b>3.1</b> Talking glossary: Bioinformatics</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="c.html"><a href="c.html"><i class="fa fa-check"></i><b>4</b> C</a>
<ul>
<li class="chapter" data-level="4.1" data-path="c.html"><a href="c.html#talking-glossary-candidate-gene"><i class="fa fa-check"></i><b>4.1</b> Talking Glossary: Candidate gene</a></li>
<li class="chapter" data-level="4.2" data-path="c.html"><a href="c.html#cath-database"><i class="fa fa-check"></i><b>4.2</b> CATH database</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="c.html"><a href="c.html#references"><i class="fa fa-check"></i><b>4.2.1</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="c.html"><a href="c.html#clade"><i class="fa fa-check"></i><b>4.3</b> Clade</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="c.html"><a href="c.html#clades-and-phylogenetic-trees"><i class="fa fa-check"></i><b>4.3.1</b> Clades and phylogenetic trees</a></li>
<li class="chapter" data-level="4.3.2" data-path="c.html"><a href="c.html#terminology"><i class="fa fa-check"></i><b>4.3.2</b> Terminology</a></li>
<li class="chapter" data-level="4.3.3" data-path="c.html"><a href="c.html#in-popular-culture"><i class="fa fa-check"></i><b>4.3.3</b> In popular culture</a></li>
<li class="chapter" data-level="4.3.4" data-path="c.html"><a href="c.html#references-1"><i class="fa fa-check"></i><b>4.3.4</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="c.html"><a href="c.html#talking-glossary-cloning-0.5-min"><i class="fa fa-check"></i><b>4.4</b> Talking Glossary: Cloning (0.5 min)</a></li>
<li class="chapter" data-level="4.5" data-path="c.html"><a href="c.html#talking-glossary-codominance-0.5-min"><i class="fa fa-check"></i><b>4.5</b> Talking Glossary: Codominance (0.5 min)</a></li>
<li class="chapter" data-level="4.6" data-path="c.html"><a href="c.html#comparative-genomics"><i class="fa fa-check"></i><b>4.6</b> Comparative genomics</a></li>
<li class="chapter" data-level="4.7" data-path="c.html"><a href="c.html#talking-glossary-contig-0.75-min"><i class="fa fa-check"></i><b>4.7</b> Talking Glossary: Contig (0.75 min)</a></li>
<li class="chapter" data-level="4.8" data-path="c.html"><a href="c.html#talking-glossary---foundations-1-review-crossing-over"><i class="fa fa-check"></i><b>4.8</b> Talking Glossary - Foundations 1 Review: Crossing over</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="d.html"><a href="d.html"><i class="fa fa-check"></i><b>5</b> D</a>
<ul>
<li class="chapter" data-level="5.1" data-path="d.html"><a href="d.html#talking-glossary-deletion-mutation-1.5-min"><i class="fa fa-check"></i><b>5.1</b> Talking Glossary: Deletion mutation (1.5 min)</a></li>
<li class="chapter" data-level="5.2" data-path="d.html"><a href="d.html#talking-glossary-dna-sequencing-0.75-min"><i class="fa fa-check"></i><b>5.2</b> Talking Glossary: DNA sequencing (0.75 min)</a></li>
<li class="chapter" data-level="5.3" data-path="d.html"><a href="d.html#sex-chromosome-dosage-compensation"><i class="fa fa-check"></i><b>5.3</b> Sex-Chromosome Dosage compensation</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="d.html"><a href="d.html#random-inactivation-of-one-x-in-xx-individuals"><i class="fa fa-check"></i><b>5.3.1</b> Random inactivation of one X in XX individuals</a></li>
<li class="chapter" data-level="5.3.2" data-path="d.html"><a href="d.html#results"><i class="fa fa-check"></i><b>5.3.2</b> Results</a></li>
<li class="chapter" data-level="5.3.3" data-path="d.html"><a href="d.html#clinical-applications"><i class="fa fa-check"></i><b>5.3.3</b> Clinical applications</a></li>
<li class="chapter" data-level="5.3.4" data-path="d.html"><a href="d.html#agricultural-applications"><i class="fa fa-check"></i><b>5.3.4</b> Agricultural applications</a></li>
<li class="chapter" data-level="5.3.5" data-path="d.html"><a href="d.html#limitations"><i class="fa fa-check"></i><b>5.3.5</b> Limitations</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="d.html"><a href="d.html#fine-mapping"><i class="fa fa-check"></i><b>5.4</b> Fine-mapping</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="h.html"><a href="h.html"><i class="fa fa-check"></i><b>6</b> H</a>
<ul>
<li class="chapter" data-level="6.1" data-path="h.html"><a href="h.html#talking-glossary-haploid-1.25-min"><i class="fa fa-check"></i><b>6.1</b> Talking Glossary: Haploid (1.25 min)</a></li>
<li class="chapter" data-level="6.2" data-path="h.html"><a href="h.html#talking-glossary-halplotype"><i class="fa fa-check"></i><b>6.2</b> Talking Glossary: Halplotype</a></li>
<li class="chapter" data-level="6.3" data-path="h.html"><a href="h.html#talking-glossary-heterozygous"><i class="fa fa-check"></i><b>6.3</b> Talking Glossary: Heterozygous</a></li>
<li class="chapter" data-level="6.4" data-path="h.html"><a href="h.html#homologyl-when-things-are-homolgous-and-analogous"><i class="fa fa-check"></i><b>6.4</b> Homology:L When things are homolgous AND analogous!</a></li>
<li class="chapter" data-level="6.5" data-path="h.html"><a href="h.html#talking-glossary-homozygous"><i class="fa fa-check"></i><b>6.5</b> Talking Glossary: Homozygous</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="i.html"><a href="i.html"><i class="fa fa-check"></i><b>7</b> I</a>
<ul>
<li class="chapter" data-level="7.1" data-path="i.html"><a href="i.html#talking-glossary-insertion-mutation-1.25-min"><i class="fa fa-check"></i><b>7.1</b> Talking Glossary: Insertion mutation (1.25 min)</a></li>
<li class="chapter" data-level="7.2" data-path="i.html"><a href="i.html#intrinscially-disordered-protein"><i class="fa fa-check"></i><b>7.2</b> Intrinscially disordered protein</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="i.html"><a href="i.html#abundance"><i class="fa fa-check"></i><b>7.2.1</b> Abundance</a></li>
<li class="chapter" data-level="7.2.2" data-path="i.html"><a href="i.html#disorder-annotation"><i class="fa fa-check"></i><b>7.2.2</b> Disorder annotation</a></li>
<li class="chapter" data-level="7.2.3" data-path="i.html"><a href="i.html#disorder-databases"><i class="fa fa-check"></i><b>7.2.3</b> Disorder databases</a></li>
<li class="chapter" data-level="7.2.4" data-path="i.html"><a href="i.html#predicting-idps-by-sequence"><i class="fa fa-check"></i><b>7.2.4</b> Predicting IDPs by sequence</a></li>
<li class="chapter" data-level="7.2.5" data-path="i.html"><a href="i.html#prediction-methods"><i class="fa fa-check"></i><b>7.2.5</b> Prediction methods</a></li>
<li class="chapter" data-level="7.2.6" data-path="i.html"><a href="i.html#disorder-and-disease"><i class="fa fa-check"></i><b>7.2.6</b> Disorder and disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="j.html"><a href="j.html"><i class="fa fa-check"></i><b>8</b> J</a>
<ul>
<li class="chapter" data-level="8.1" data-path="j.html"><a href="j.html#jove-videos"><i class="fa fa-check"></i><b>8.1</b> JoVe Videos</a>
<ul>
<li class="chapter" data-level="8.1.1" data-path="j.html"><a href="j.html#biotechnology"><i class="fa fa-check"></i><b>8.1.1</b> Biotechnology</a></li>
<li class="chapter" data-level="8.1.2" data-path="j.html"><a href="j.html#macromolecules"><i class="fa fa-check"></i><b>8.1.2</b> Macromolecules</a></li>
<li class="chapter" data-level="8.1.3" data-path="j.html"><a href="j.html#cellular-respiration"><i class="fa fa-check"></i><b>8.1.3</b> Cellular respiration</a></li>
<li class="chapter" data-level="8.1.4" data-path="j.html"><a href="j.html#population-and-community-ecology"><i class="fa fa-check"></i><b>8.1.4</b> Population and community ecology</a></li>
<li class="chapter" data-level="8.1.5" data-path="j.html"><a href="j.html#population-genetics"><i class="fa fa-check"></i><b>8.1.5</b> Population genetics</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="k.html"><a href="k.html"><i class="fa fa-check"></i><b>9</b> K</a>
<ul>
<li class="chapter" data-level="9.1" data-path="k.html"><a href="k.html#talking-glossary-knockout-genetic-knockout-gene-knockout"><i class="fa fa-check"></i><b>9.1</b> Talking glossary: Knockout (Genetic knockout, gene knockout)</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="l.html"><a href="l.html"><i class="fa fa-check"></i><b>10</b> L</a>
<ul>
<li class="chapter" data-level="10.1" data-path="l.html"><a href="l.html#talking-glossary-locus-1-min"><i class="fa fa-check"></i><b>10.1</b> Talking Glossary: Locus (1 min)</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="m.html"><a href="m.html"><i class="fa fa-check"></i><b>11</b> M</a>
<ul>
<li class="chapter" data-level="11.1" data-path="m.html"><a href="m.html#manhattan-plot"><i class="fa fa-check"></i><b>11.1</b> Manhattan Plot</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="m.html"><a href="m.html#gwas"><i class="fa fa-check"></i><b>11.1.1</b> GWAS</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="m.html"><a href="m.html#talking-glossary-microarray"><i class="fa fa-check"></i><b>11.2</b> Talking Glossary: Microarray</a></li>
<li class="chapter" data-level="11.3" data-path="m.html"><a href="m.html#talking-glossary-missense-mutation-1.25-min"><i class="fa fa-check"></i><b>11.3</b> Talking Glossary: Missense Mutation (1.25 min)</a></li>
<li class="chapter" data-level="11.4" data-path="m.html"><a href="m.html#talking-glossary-mitochondrial-dna-1.5-min"><i class="fa fa-check"></i><b>11.4</b> Talking Glossary: Mitochondrial DNA (1.5 min)</a></li>
<li class="chapter" data-level="11.5" data-path="m.html"><a href="m.html#talking-glossary-mutagen-0.5-min"><i class="fa fa-check"></i><b>11.5</b> Talking Glossary: Mutagen (0.5 min)</a></li>
<li class="chapter" data-level="11.6" data-path="m.html"><a href="m.html#talking-glossary-mutation-0.5-min"><i class="fa fa-check"></i><b>11.6</b> Talking Glossary: Mutation (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="n.html"><a href="n.html"><i class="fa fa-check"></i><b>12</b> N</a>
<ul>
<li class="chapter" data-level="12.1" data-path="n.html"><a href="n.html#talking-glossary-non-coding-dna-1-min"><i class="fa fa-check"></i><b>12.1</b> Talking Glossary: Non-coding DNA (1 min)</a></li>
<li class="chapter" data-level="12.2" data-path="n.html"><a href="n.html#talking-glossary-nonsene-mutation"><i class="fa fa-check"></i><b>12.2</b> Talking Glossary: Nonsene mutation</a></li>
<li class="chapter" data-level="12.3" data-path="n.html"><a href="n.html#nucleic-acids-review"><i class="fa fa-check"></i><b>12.3</b> Nucleic Acids - Overview</a>
<ul>
<li class="chapter" data-level="12.3.1" data-path="n.html"><a href="n.html#dna-and-rna"><i class="fa fa-check"></i><b>12.3.1</b> DNA and RNA</a></li>
<li class="chapter" data-level="12.3.2" data-path="n.html"><a href="n.html#dna-double-helix-structure"><i class="fa fa-check"></i><b>12.3.2</b> DNA Double-Helix Structure</a></li>
<li class="chapter" data-level="12.3.3" data-path="n.html"><a href="n.html#rna"><i class="fa fa-check"></i><b>12.3.3</b> RNA</a></li>
<li class="chapter" data-level="12.3.4" data-path="n.html"><a href="n.html#protein-function"><i class="fa fa-check"></i><b>12.3.4</b> Protein function</a></li>
<li class="chapter" data-level="12.3.5" data-path="n.html"><a href="n.html#summary"><i class="fa fa-check"></i><b>12.3.5</b> Summary</a></li>
<li class="chapter" data-level="12.3.6" data-path="n.html"><a href="n.html#glossary"><i class="fa fa-check"></i><b>12.3.6</b> Glossary</a></li>
<li class="chapter" data-level="12.3.7" data-path="n.html"><a href="n.html#contributors-and-attributions"><i class="fa fa-check"></i><b>12.3.7</b> Contributors and Attributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="13" data-path="o.html"><a href="o.html"><i class="fa fa-check"></i><b>13</b> O</a>
<ul>
<li class="chapter" data-level="13.1" data-path="o.html"><a href="o.html#talking-glossary-open-reading-frame-definition-3-min"><i class="fa fa-check"></i><b>13.1</b> Talking Glossary : Open reading frame definition (3 min)</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="p.html"><a href="p.html"><i class="fa fa-check"></i><b>14</b> P</a>
<ul>
<li class="chapter" data-level="14.1" data-path="p.html"><a href="p.html#talking-glossary-pcr---the-polymerase-chain-reaction-0.5-min"><i class="fa fa-check"></i><b>14.1</b> Talking Glossary: PCR - The Polymerase Chain Reaction (0.5 min)</a></li>
<li class="chapter" data-level="14.2" data-path="p.html"><a href="p.html#protein-data-bank"><i class="fa fa-check"></i><b>14.2</b> Protein Data Bank</a>
<ul>
<li class="chapter" data-level="14.2.1" data-path="p.html"><a href="p.html#history"><i class="fa fa-check"></i><b>14.2.1</b> History</a></li>
<li class="chapter" data-level="14.2.2" data-path="p.html"><a href="p.html#file-format"><i class="fa fa-check"></i><b>14.2.2</b> File format</a></li>
</ul></li>
<li class="chapter" data-level="14.3" data-path="p.html"><a href="p.html#pfam"><i class="fa fa-check"></i><b>14.3</b> PFam</a>
<ul>
<li class="chapter" data-level="14.3.1" data-path="p.html"><a href="p.html#uses"><i class="fa fa-check"></i><b>14.3.1</b> Uses</a></li>
</ul></li>
<li class="chapter" data-level="14.4" data-path="p.html"><a href="p.html#talking-glossary-phenotype"><i class="fa fa-check"></i><b>14.4</b> Talking Glossary: Phenotype</a></li>
<li class="chapter" data-level="14.5" data-path="p.html"><a href="p.html#phylogenetics-vocab"><i class="fa fa-check"></i><b>14.5</b> Phylogenetics vocab</a></li>
<li class="chapter" data-level="14.6" data-path="p.html"><a href="p.html#talking-glossary-plasmid-1-min"><i class="fa fa-check"></i><b>14.6</b> Talking Glossary: Plasmid (1 min)</a></li>
<li class="chapter" data-level="14.7" data-path="p.html"><a href="p.html#talking-glossary-point-mutation-0.5-min"><i class="fa fa-check"></i><b>14.7</b> Talking Glossary: Point mutation (0.5 min)</a></li>
<li class="chapter" data-level="14.8" data-path="p.html"><a href="p.html#talking-glossary-polymorphism-1-min"><i class="fa fa-check"></i><b>14.8</b> Talking Glossary: Polymorphism (1 min)</a></li>
<li class="chapter" data-level="14.9" data-path="p.html"><a href="p.html#talking-glossary-u03-primer-0.5-min"><i class="fa fa-check"></i><b>14.9</b> Talking Glossary U03: Primer (0.5 min)</a></li>
<li class="chapter" data-level="14.10" data-path="p.html"><a href="p.html#talking-glossary-promoter-1.5-min"><i class="fa fa-check"></i><b>14.10</b> Talking Glossary: Promoter (1.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="q.html"><a href="q.html"><i class="fa fa-check"></i><b>15</b> Q</a></li>
<li class="chapter" data-level="16" data-path="r.html"><a href="r.html"><i class="fa fa-check"></i><b>16</b> R</a>
<ul>
<li class="chapter" data-level="16.1" data-path="r.html"><a href="r.html#talking-glossary-recessive"><i class="fa fa-check"></i><b>16.1</b> Talking Glossary: Recessive</a></li>
<li class="chapter" data-level="16.2" data-path="r.html"><a href="r.html#talking-glossary-recombinant-dna-0.5-min"><i class="fa fa-check"></i><b>16.2</b> Talking Glossary: Recombinant DNA (0.5 min)</a></li>
<li class="chapter" data-level="16.3" data-path="r.html"><a href="r.html#talking-glossary-repressor-1-min"><i class="fa fa-check"></i><b>16.3</b> Talking Glossary: Repressor (1 min)</a></li>
<li class="chapter" data-level="16.4" data-path="r.html"><a href="r.html#talking-glossary-restriction-enzyme-0.5-min"><i class="fa fa-check"></i><b>16.4</b> Talking Glossary: Restriction Enzyme (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="s.html"><a href="s.html"><i class="fa fa-check"></i><b>17</b> S</a>
<ul>
<li class="chapter" data-level="17.1" data-path="s.html"><a href="s.html#structural-classification-of-proteins-database-scop"><i class="fa fa-check"></i><b>17.1</b> Structural Classification of Proteins database (SCOP)</a>
<ul>
<li class="chapter" data-level="17.1.1" data-path="s.html"><a href="s.html#hierarchical-organisation"><i class="fa fa-check"></i><b>17.1.1</b> Hierarchical organisation</a></li>
<li class="chapter" data-level="17.1.2" data-path="s.html"><a href="s.html#superfamilies"><i class="fa fa-check"></i><b>17.1.2</b> Superfamilies</a></li>
<li class="chapter" data-level="17.1.3" data-path="s.html"><a href="s.html#families"><i class="fa fa-check"></i><b>17.1.3</b> Families</a></li>
<li class="chapter" data-level="17.1.4" data-path="s.html"><a href="s.html#pdb-entry-domains"><i class="fa fa-check"></i><b>17.1.4</b> PDB entry domains</a></li>
</ul></li>
<li class="chapter" data-level="17.2" data-path="s.html"><a href="s.html#sequence-alignment"><i class="fa fa-check"></i><b>17.2</b> Sequence alignment</a>
<ul>
<li class="chapter" data-level="17.2.1" data-path="s.html"><a href="s.html#interpretation"><i class="fa fa-check"></i><b>17.2.1</b> Interpretation</a></li>
<li class="chapter" data-level="17.2.2" data-path="s.html"><a href="s.html#alignment-methods"><i class="fa fa-check"></i><b>17.2.2</b> Alignment methods</a></li>
<li class="chapter" data-level="17.2.3" data-path="s.html"><a href="s.html#representations"><i class="fa fa-check"></i><b>17.2.3</b> Representations</a></li>
<li class="chapter" data-level="17.2.4" data-path="s.html"><a href="s.html#global-and-local-alignments"><i class="fa fa-check"></i><b>17.2.4</b> Global and local alignments</a></li>
<li class="chapter" data-level="17.2.5" data-path="s.html"><a href="s.html#pairwise-alignment"><i class="fa fa-check"></i><b>17.2.5</b> Pairwise alignment</a></li>
<li class="chapter" data-level="17.2.6" data-path="s.html"><a href="s.html#dot-matrix-methods"><i class="fa fa-check"></i><b>17.2.6</b> Dot-matrix methods</a></li>
<li class="chapter" data-level="17.2.7" data-path="s.html"><a href="s.html#dynamic-programming"><i class="fa fa-check"></i><b>17.2.7</b> Dynamic programming</a></li>
<li class="chapter" data-level="17.2.8" data-path="s.html"><a href="s.html#word-methods"><i class="fa fa-check"></i><b>17.2.8</b> Word methods</a></li>
<li class="chapter" data-level="17.2.9" data-path="s.html"><a href="s.html#multiple-sequence-alignment"><i class="fa fa-check"></i><b>17.2.9</b> Multiple sequence alignment</a></li>
<li class="chapter" data-level="17.2.10" data-path="s.html"><a href="s.html#phylogenetic-analysis"><i class="fa fa-check"></i><b>17.2.10</b> Phylogenetic analysis</a></li>
<li class="chapter" data-level="17.2.11" data-path="s.html"><a href="s.html#assessment-of-significance"><i class="fa fa-check"></i><b>17.2.11</b> Assessment of significance</a></li>
<li class="chapter" data-level="17.2.12" data-path="s.html"><a href="s.html#other-biological-uses"><i class="fa fa-check"></i><b>17.2.12</b> Other biological uses</a></li>
<li class="chapter" data-level="17.2.13" data-path="s.html"><a href="s.html#references-2"><i class="fa fa-check"></i><b>17.2.13</b> References</a></li>
</ul></li>
<li class="chapter" data-level="17.3" data-path="s.html"><a href="s.html#talking-glossary-u03-shotgun-sequencing-1-min"><i class="fa fa-check"></i><b>17.3</b> Talking Glossary U03: Shotgun sequencing (1 min)</a></li>
<li class="chapter" data-level="17.4" data-path="s.html"><a href="s.html#talking-glossary-sickle-cell-disease-4-min"><i class="fa fa-check"></i><b>17.4</b> Talking Glossary: Sickle Cell Disease (4 min)</a></li>
<li class="chapter" data-level="17.5" data-path="s.html"><a href="s.html#medline-snps-single-nucleotide-polymorphism"><i class="fa fa-check"></i><b>17.5</b> Medline: SNPs (single nucleotide polymorphism)</a></li>
<li class="chapter" data-level="17.6" data-path="s.html"><a href="s.html#talking-glossary-somatic-cells-0.75-min"><i class="fa fa-check"></i><b>17.6</b> Talking glossary: Somatic cells (0.75 min)</a></li>
<li class="chapter" data-level="17.7" data-path="s.html"><a href="s.html#sequence-annotation"><i class="fa fa-check"></i><b>17.7</b> Sequence annotation</a>
<ul>
<li class="chapter" data-level="17.7.1" data-path="s.html"><a href="s.html#process"><i class="fa fa-check"></i><b>17.7.1</b> Process</a></li>
</ul></li>
<li class="chapter" data-level="17.8" data-path="s.html"><a href="s.html#talking-glossary-substitution-mutation-1.75-min"><i class="fa fa-check"></i><b>17.8</b> Talking Glossary: Substitution mutation (1.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="t.html"><a href="t.html"><i class="fa fa-check"></i><b>18</b> T</a>
<ul>
<li class="chapter" data-level="18.1" data-path="t.html"><a href="t.html#taxa-taxon-and-clades-a-brief-primer"><i class="fa fa-check"></i><b>18.1</b> Taxa, Taxon, and clades: A Brief Primer</a>
<ul>
<li class="chapter" data-level="18.1.1" data-path="t.html"><a href="t.html#what-is-a-taxon"><i class="fa fa-check"></i><b>18.1.1</b> What is a “taxon”</a></li>
<li class="chapter" data-level="18.1.2" data-path="t.html"><a href="t.html#what-is-a-clade"><i class="fa fa-check"></i><b>18.1.2</b> What is a clade?</a></li>
<li class="chapter" data-level="18.1.3" data-path="t.html"><a href="t.html#further-reading"><i class="fa fa-check"></i><b>18.1.3</b> Further Reading:</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="t.html"><a href="t.html#talking-glossary-trait-0.75-min"><i class="fa fa-check"></i><b>18.2</b> Talking Glossary: Trait (0.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="u.html"><a href="u.html"><i class="fa fa-check"></i><b>19</b> U</a>
<ul>
<li class="chapter" data-level="19.1" data-path="u.html"><a href="u.html#uniprot"><i class="fa fa-check"></i><b>19.1</b> Uniprot</a>
<ul>
<li class="chapter" data-level="19.1.1" data-path="u.html"><a href="u.html#the-uniprot-consortium"><i class="fa fa-check"></i><b>19.1.1</b> The UniProt consortium</a></li>
<li class="chapter" data-level="19.1.2" data-path="u.html"><a href="u.html#the-roots-of-uniprot-databases"><i class="fa fa-check"></i><b>19.1.2</b> The roots of UniProt databases</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="20" data-path="v.html"><a href="v.html"><i class="fa fa-check"></i><b>20</b> V</a>
<ul>
<li class="chapter" data-level="20.1" data-path="v.html"><a href="v.html#talking-glossary-vector-2-min"><i class="fa fa-check"></i><b>20.1</b> Talking Glossary: Vector (2 min)</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="w.html"><a href="w.html"><i class="fa fa-check"></i><b>21</b> W</a>
<ul>
<li class="chapter" data-level="21.1" data-path="w.html"><a href="w.html#talking-glossary-x-chromosome-1.25-min"><i class="fa fa-check"></i><b>21.1</b> Talking Glossary: X-chromosome (1.25 min)</a></li>
<li class="chapter" data-level="21.2" data-path="w.html"><a href="w.html#talking-glossary-x-inactivation-lyonization-1.45-min"><i class="fa fa-check"></i><b>21.2</b> Talking Glossary: X-inactivation (“Lyonization”; 1.45 min))</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="x.html"><a href="x.html"><i class="fa fa-check"></i><b>22</b> X</a></li>
<li class="chapter" data-level="23" data-path="v-1.html"><a href="v-1.html"><i class="fa fa-check"></i><b>23</b> V</a>
<ul>
<li class="chapter" data-level="23.1" data-path="v-1.html"><a href="v-1.html#additional-glossaries"><i class="fa fa-check"></i><b>23.1</b> Additional Glossaries</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="z.html"><a href="z.html"><i class="fa fa-check"></i><b>24</b> Z</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Foundations of Biology and Environmental Science: An Open-Access Encyclopedia</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="p" class="section level1" number="14">
<h1><span class="header-section-number"> 14</span> P</h1>

<div id="talking-glossary-pcr---the-polymerase-chain-reaction-0.5-min" class="section level2" number="14.1">
<h2><span class="header-section-number">14.1</span> Talking Glossary: PCR - The Polymerase Chain Reaction (0.5 min)</h2>
<p>National Human Genome Research Institute</p>
<p><a href="https://www.genome.gov/genetics-glossary/Polymerase-Chain-Reaction" class="uri">https://www.genome.gov/genetics-glossary/Polymerase-Chain-Reaction</a></p>
<p><strong>Abstract</strong>: Polymerase chain reaction (PCR) is a laboratory technique used to amplify DNA sequences. The method involves using short DNA sequences called primers to select the portion of the genome to be amplified. The temperature of the sample is repeatedly raised and lowered to help a DNA replication enzyme copy the target DNA sequence. The technique can produce a billion copies of the target sequence in just a few hours.</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/polymerase_chain_reaction.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/polymerase_chain_reaction.mp3</a></p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/media/images/2021-01/polymerase_chain_reaction.jpg" class="uri">https://www.genome.gov/sites/default/files/media/images/2021-01/polymerase_chain_reaction.jpg</a></p>
<p><strong>Transcript</strong>: “PCR, or the polymerase chain reaction, is a chemical reaction that molecular biologists use to amplify pieces of DNA. This reaction allows a single or a few copies of DNA to be replicated into millions or billions of copies. And by amplifying that DNA, it allows us to study that DNA molecule in detail in the laboratory.”</p>
<p><a href="https://www.genome.gov/staff/Leslie-G-Biesecker-MD">Leslie G. Biesecker</a>, M.D.</p>

</div>
<div id="protein-data-bank" class="section level2" number="14.2">
<h2><span class="header-section-number">14.2</span> Protein Data Bank</h2>
<p>Adapated from Wikipedia</p>
<p>The Protein Data Bank (PDB)[1] is a database for the three-dimensional structural data of large biological molecules, such as proteins and nucleic acids. The data, typically obtained by <strong>X-ray crystallography</strong>, NMR spectroscopy, or, increasingly, cryo-electron microscopy, and submitted by biologists and biochemists from around the world, are freely accessible on the Internet via the websites of its member organisations (PDBe,[2] PDBj,[3] RCSB,[4] and BMRB[5]).</p>
<p>The PDB is a key in areas of structural biology, such as structural genomics. Most major scientific journals and some funding agencies now require scientists to submit their structure data to the PDB. Many other databases use protein structures deposited in the PDB. For example, SCOP and CATH classify protein structures, while PDBsum provides a graphic overview of PDB entries using information from other sources, such as Gene ontology.[6, 7]</p>
<p>Most structures are determined by X-ray diffraction, but about 10% of structures are determined by protein NMR. When using X-ray diffraction, approximations of the coordinates of the atoms of the protein are obtained, whereas using NMR, the distance between pairs of atoms of the protein is estimated. The final conformation of the protein is obtained from NMR by solving a distance geometry problem. After 2013, a growing number of proteins are determined by cryo-electron microscopy. Clicking on the numbers in the linked external table displays examples of structures determined by that method.</p>
<p>Historically, the number of structures in the PDB has grown at an approximately exponential rate, with 100 registered structures in 1982, 1,000 structures in 1993, 10,000 in 1999, and 100,000 in 2014.[19, 20]</p>
<div id="history" class="section level3" number="14.2.1">
<h3><span class="header-section-number">14.2.1</span> History</h3>
<p>** OPTIONAL **</p>
<p>Two forces converged to initiate the PDB: a small but growing collection of sets of protein structure data determined by X-ray diffraction; and the newly available (1968) molecular graphics display, the Brookhaven RAster Display (BRAD), to visualize these protein structures in 3-D. In 1969, with the sponsorship of Walter Hamilton at the Brookhaven National Laboratory, Edgar Meyer (Texas A&amp;M University) began to write software to store atomic coordinate files in a common format to make them available for geometric and graphical evaluation. By 1971, one of Meyer’s programs, SEARCH, enabled researchers to remotely access information from the database to study protein structures offline.[8] SEARCH was instrumental in enabling networking, thus marking the functional beginning of the PDB.</p>
<p>The Protein Data Bank was announced in October 1971 in Nature New Biology[9] as a joint venture between Cambridge Crystallographic Data Centre, UK and Brookhaven National Laboratory, US.</p>
<p>Upon Hamilton’s death in 1973, Tom Koeztle took over direction of the PDB for the subsequent 20 years. In January 1994, Joel Sussman of Israel’s Weizmann Institute of Science was appointed head of the PDB. In October 1998,[10] the PDB was transferred to the Research Collaboratory for Structural Bioinformatics (RCSB);[11] the transfer was completed in June 1999. The new director was Helen M. Berman of Rutgers University (one of the managing institutions of the RCSB, the other being the San Diego Supercomputer Center at UC San Diego).[12] In 2003, with the formation of the wwPDB, the PDB became an international organization. The founding members are PDBe (Europe),[2] RCSB (USA), and PDBj (Japan).[3] The BMRB[5] joined in 2006. Each of the four members of wwPDB can act as deposition, data processing and distribution centers for PDB data. The data processing refers to the fact that wwPDB staff review and annotate each submitted entry.[13] The data are then automatically checked for plausibility (the source code[14] for this validation software has been made available to the public at no charge).</p>
<p>For PDB structures determined by X-ray diffraction that have a structure factor file, their electron density map may be viewed. The data of such structures is stored on the “electron density server”.[17, 18]</p>
</div>
<div id="file-format" class="section level3" number="14.2.2">
<h3><span class="header-section-number">14.2.2</span> File format</h3>
<p><strong>OPTIONAL</strong>
The file format initially used by the PDB was called the PDB file format. The original format was restricted by the width of computer punch cards to 80 characters per line. Around 1996, the “macromolecular Crystallographic Information file” format, mmCIF, which is an extension of the CIF format was phased in. mmCIF became the standard format for the PDB archive in 2014.[21] In 2019, the wwPDB announced that depositions for crystallographic methods would only be accepted in mmCIF format.[22]</p>
<p>An XML version of PDB, called PDBML, was described in 2005.[23] The structure files can be downloaded in any of these three formats, though an increasing number of structures do not fit the legacy PDB format. Individual files are easily downloaded into graphics packages from Internet URLs:</p>
<!-- For PDB format files, use, e.g., http://www.pdb.org/pdb/files/4hhb.pdb.gz or http://pdbe.org/download/4hhb -->
<!-- For PDBML (XML) files, use, e.g., http://www.pdb.org/pdb/files/4hhb.xml.gz or http://pdbe.org/pdbml/4hhb -->
<!-- The "4hhb" is the PDB identifier. Each structure published in PDB receives a four-character alphanumeric identifier, its PDB ID. (This is not a unique identifier for biomolecules, because several structures for the same molecule—in different environments or conformations—may be contained in PDB with different PDB IDs.) -->
<!-- ### Viewing the data -->
<!-- The structure files may be viewed using one of several free and open source computer programs, including Jmol, Pymol, VMD, and Rasmol. Other non-free, shareware programs include ICM-Browser,[24] MDL Chime, UCSF Chimera, Swiss-PDB Viewer,[25] StarBiochem[26] (a Java-based interactive molecular viewer with integrated search of protein databank), Sirius, and VisProt3DS[27] (a tool for Protein Visualization in 3D stereoscopic view in anaglyph and other modes), and Discovery Studio. The RCSB PDB website contains an extensive list of both free and commercial molecule visualization programs and web browser plugins. -->
<!-- See also -->
<!-- Crystallographic database -->
<!-- Protein structure -->
<!-- Protein structure prediction -->
<!-- Protein structure database -->
<!-- PDBREPORT lists all anomalies (also errors) in PDB structures -->
<!-- PDBsum—extracts data from other databases about PDB structures -->
<!-- Proteopedia—a collaborative 3D encyclopedia of proteins and other molecules -->
<!-- References -->
<!--  wwPDB, Consortium (2019). "Protein Data Bank: the single global archive for 3D macromolecular structure data". Nucleic Acids Res. 47 (D1): 520–528. doi:10.1093/nar/gky949. PMC 6324056. PMID 30357364. -->
<!--  "PDBe home < Node < EMBL-EBI". pdbe.org. -->
<!--  "Protein Data Bank Japan - PDB Japan - PDBj". pdbj.org. -->
<!--  Bank, RCSB Protein Data. "RCSB PDB: Homepage". rcsb.org. -->
<!--  "Biological Magnetic Resonance Bank". bmrb.wisc.edu. -->
<!--  Berman, H. M. (January 2008). "The Protein Data Bank: a historical perspective" (PDF). Acta Crystallographica Section A. A64 (1): 88–95. doi:10.1107/S0108767307035623. PMID 18156675. -->
<!--  Laskowski RA, Hutchinson EG, Michie AD, Wallace AC, Jones ML, Thornton JM (December 1997). "PDBsum: a Web-based database of summaries and analyses of all PDB structures". Trends Biochem. Sci. 22 (12): 488–90. doi:10.1016/S0968-0004(97)01140-7. PMID 9433130. -->
<!--  Meyer EF (1997). "The first years of the Protein Data Bank". Protein Science. Cambridge University Press. 6 (7): 1591–1597. doi:10.1002/pro.5560060724. PMC 2143743. PMID 9232661. -->
<!--  "Protein Data Bank". Nature New Biology. 1971. doi:10.1038/newbio233223b0. -->
<!--  Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, Shindyalov IN, Bourne PE (January 2000). "The Protein Data Bank". Nucleic Acids Res. 28 (1): 235–242. doi:10.1093/nar/28.1.235. PMC 102472. PMID 10592235. -->
<!--  "Research Collaboratory for Structural Bioinformatics". RCSB.org. Research Collaboratory for Structural Bioinformatics. Archived from the original on 2007-02-05. -->
<!--  "RCSB PDB Newsletter Archive". RCSB Protein Data Bank. -->
<!--  Curry E, Freitas A, O'Riáin S (2010). "The Role of Community-Driven Data Curation for Enterprises". In D. Wood (ed.). Linking Enterprise Data. Boston: Springer US. pp. 25–47. ISBN 978-1-441-97664-2. -->
<!--  "PDB Validation Suite". sw-tools.pdb.org. -->
<!--  Burley SK, Berman HM, Bhikadiya C, Bi C, Chen L, Costanzo LD, et al. (wwPDB consortium) (January 2019). "Protein Data Bank: the single global archive for 3D macromolecular structure data". Nucleic Acids Research. 47 (D1): D520–D528. doi:10.1093/nar/gky949. PMC 6324056. PMID 30357364. -->
<!--  "PDB Current Holdings Breakdown". RCSB. -->
<!--  "The Uppsala Electron Density Server". Uppsala University. Retrieved 2013-04-06. -->
<!--  Kleywegt GJ, Harris MR, Zou JY, Taylor TC, Wählby A, Jones TA (Dec 2004). "The Uppsala Electron-Density Server". Acta Crystallogr D. 60 (Pt 12 Pt 1): 2240–2249. doi:10.1107/S0907444904013253. PMID 15572777. -->
<!--  Anon (2014). "Hard data: It has been no small feat for the Protein Data Bank to stay relevant for 100,000 structures". Nature. 509 (7500): 260. doi:10.1038/509260a. PMID 24834514. -->
<!--  "Content Growth Report". RCSB PDB. Archived from the original on 2007-04-28. Retrieved 2013-04-06. -->
<!--  "wwPDB: File Formats and the PDB". wwpdb.org. Retrieved April 1, 2020. -->
<!--  wwPDB.org. "wwPDB: 2019 News". wwpdb.org. -->
<!--  Westbrook J, Ito N, Nakamura H, Henrick K, Berman HM (April 2005). "PDBML: the representation of archival macromolecular structure data in XML". Bioinformatics. 21 (7): 988–992. doi:10.1093/bioinformatics/bti082. PMID 15509603. -->
<!--  "ICM-Browser". Molsoft L.L.C. Retrieved 2013-04-06. -->
<!--  "Swiss PDB Viewer". Swiss Institute of Bioinformatics. Retrieved 2013-04-06. -->
<!--  "STAR: Biochem - Home". web.mit.edu. -->
<!--  "VisProt3DS". Molecular Systems Ltd. Retrieved 2013-04-06. -->
<!-- External links -->
<!--    Wikidata has the property: -->
<!-- Disc Plain blue dark.svg PDB structure ID (P638) (see uses) -->
<!-- The Worldwide Protein Data Bank (wwPDB)—parent site to regional hosts (below) -->
<!-- RCSB Protein Data Bank (USA) -->
<!-- PDBe (Europe) -->
<!-- PDBj (Japan) -->
<!-- BMRB, Biological Magnetic Resonance Data Bank (USA) -->
<!-- wwPDB Documentation—documentation on both the PDB and PDBML file formats -->
<!-- Looking at Structures—The RCSB's introduction to crystallography -->
<!-- PDBsum Home Page—Extracts data from other databases about PDB structures. -->
<!-- Nucleic Acid Database, NDB—a PDB mirror especially for searching for nucleic acids -->
<!-- Introductory PDB tutorial sponsored by PDB -->

</div>
</div>
<div id="pfam" class="section level2" number="14.3">
<h2><span class="header-section-number">14.3</span> PFam</h2>
<p>Adapted from Wikipedia.</p>
<p>Pfam is a database of <strong>protein families</strong> that includes their annotations and multiple sequence alignments generated using bioinformatics tools known as hidden Markov models (HMM; 1, 2, 3. The most recent version, Pfam 34.0, was released in March 2021 and contains 19,179 families (4).</p>
<div id="uses" class="section level3" number="14.3.1">
<h3><span class="header-section-number">14.3.1</span> Uses</h3>
<p>The general purpose of the Pfam database is to provide a complete and accurate classification of protein families and domains.[5] Originally, the rationale behind creating the database was to have a semi-automated method of curating information on known protein families to improve the efficiency of annotating genomes.[6] The Pfam classification of protein families has been widely adopted by biologists because of its wide coverage of proteins and sensible naming conventions.[7]</p>
<p>It is used by experimental biologists researching specific proteins, by structural biologists to identify new targets for structure determination, by computational biologists to organise sequences and by evolutionary biologists tracing the origins of proteins.[8] Early genome projects, such as human and fly used Pfam extensively for functional annotation of genomic data.[9, 10, 11]</p>
<!-- The Pfam website allows users to submit protein or DNA sequences to search for matches to families in the database. If DNA is submitted, a six-frame translation is performed, then each frame is searched.[12] Rather than performing a typical BLAST search, Pfam uses profile hidden Markov models, which give greater weight to matches at conserved sites, allowing better remote homology detection, making them more suitable for annotating genomes of organisms with no well-annotated close relatives.[13] -->
<!-- Pfam has also been used in the creation of other resources such as iPfam, which catalogs domain-domain interactions within and between proteins, based on information in structure databases and mapping of Pfam domains onto these structures.[14] -->
<!-- Features -->
<!-- For each family in Pfam one can: -->
<!--   View a description of the family -->
<!-- Look at multiple alignments -->
<!-- View protein domain architectures -->
<!-- Examine species distribution -->
<!-- Follow links to other databases -->
<!-- View known protein structures -->
<!-- Entries can be of several types: family, domain, repeat or motif. Family is the default class, which simply indicates that members are related. Domains are defined as an autonomous structural unit or reusable sequence unit that can be found in multiple protein contexts. Repeats are not usually stable in isolation, but rather are usually required to form tandem repeats in order to form a domain or extended structure. Motifs are usually shorter sequence units found outside of globular domains.[9] -->
<!-- The descriptions of Pfam families are managed by the general public using Wikipedia (see History). -->
<!-- As of release 29.0, 76.1% of protein sequences in UniprotKB matched to at least one Pfam domain.[15] -->
<!-- Creation of new entries -->
<!-- New families come from a range of sources, primarily the PDB and analysis of complete proteomes to find genes with no Pfam hit.[16] -->
<!-- For each family, a representative subset of sequences are aligned into a high-quality seed alignment. Sequences for the seed alignment are taken primarily from pfamseq (a non-redundant database of reference proteomes) with some supplementation from UniprotKB.[15] This seed alignment is then used to build a profile hidden Markov model using HMMER. This HMM is then searched against sequence databases, and all hits that reach a curated gathering threshold are classified as members of the protein family. The resulting collection of members is then aligned to the profile HMM to generate a full alignment. -->
<!-- For each family, a manually curated gathering threshold is assigned that maximises the number of true matches to the family while excluding any false positive matches. False positives are estimated by observing overlaps between Pfam family hits that are not from the same clan. This threshold is used to assess whether a match to a family HMM should be included in the protein family. Upon each update of Pfam, gathering thresholds are reassessed to prevent overlaps between new and existing families.[16] -->
<!-- Domains of unknown function -->
<!-- Domains of unknown function (DUFs) represent a growing fraction of the Pfam database. The families are so named because they have been found to be conserved across species, but perform an unknown role. Each newly added DUF is named in order of addition. Names of these entries are updated as their functions are identified. Normally when the function of at least one protein belonging to a DUF has been determined, the function of the entire DUF is updated and the family is renamed. Some named families are still domains of unknown function, that are named after a representative protein, e.g. YbbR. Numbers of DUFs are expected to continue increasing as conserved sequences of unknown function continue to be identified in sequence data. It is expected that DUFs will eventually outnumber families of known function.[16] -->
<!-- Clans -->
<!-- Over time both sequence and residue coverage have increased, and as families have grown, more evolutionary relationships have been discovered, allowing the grouping of families into clans.[8] Clans were first introduced to the Pfam database in 2005. They are groupings of related families that share a single evolutionary origin, as confirmed by structural, functional, sequence and HMM comparisons.[5] As of release 29.0, approximately one third of protein families belonged to a clan.[15] This portion has grown to around three-fourths by 2019 (version 32.0).[17] -->
<!-- To identify possible clan relationships, Pfam curators use the Simple Comparison Of Outputs Program (SCOOP) as well as information from the ECOD database.[17] ECOD is a semi-automated hierarchical database of protein families with known structures, with families that map readily to Pfam entries and homology levels that usually map to Pfam clans.[18] -->
<!-- History -->
<!-- Pfam was founded in 1995 by Erik Sonhammer, Sean Eddy and Richard Durbin as a collection of commonly occurring protein domains that could be used to annotate the protein coding genes of multicellular animals.[6] One of its major aims at inception was to aid in the annotation of the C. elegans genome.[6] The project was partly driven by the assertion in ‘One thousand families for the molecular biologist’ by Cyrus Chothia that there were around 1500 different families of proteins and that the majority of proteins fell into just 1000 of these.[5][19] Counter to this assertion, the Pfam database currently contains 16,306 entries corresponding to unique protein domains and families. However, many of these families contain structural and functional similarities indicating a shared evolutionary origin (see Clans).[5] -->
<!-- A major point of difference between Pfam and other databases at the time of its inception was the use of two alignment types for entries: a smaller, manually checked seed alignment, as well as a full alignment built by aligning sequences to a profile hidden Markov model built from the seed alignment. This smaller seed alignment was easier to update as new releases of sequence databases came out, and thus represented a promising solution to the dilemma of how to keep the database up to date as genome sequencing became more efficient and more data needed to be processed over time. A further improvement to the speed at which the database could be updated came in version 24.0, with the introduction of HMMER3, which is ~100 times faster than HMMER2 and more sensitive.[8] -->
<!-- Because the entries in Pfam-A do not cover all known proteins, an automatically generated supplement was provided called Pfam-B. Pfam-B contained a large number of small families derived from clusters produced by an algorithm called ADDA.[20] Although of lower quality, Pfam-B families could be useful when no Pfam-A families were found. Pfam-B was discontinued as of release 28.0,[21] then reintroduced in release 33.1 using a new clustering algorithm, MMSeqs2.[22] -->
<!-- Pfam was originally hosted on three mirror sites around the world to preserve redundancy. However between 2012 and 2014, the Pfam resource was moved to EMBL-EBI, which allowed for hosting of the website from one domain (xfam.org), using duplicate independent data centres. This allowed for better centralisation of updates, and grouping with other Xfam projects such as Rfam, TreeFam, iPfam and others, whilst retaining critical resilience provided by hosting from multiple centres.[23] -->
<!-- Pfam has undergone a substantial reorganisation over the last two years to further reduce manual effort involved in curation and allow for more frequent updates.[15] -->
<!-- Community curation -->
<!-- Curation of such a large database presented issues in terms of keeping up with the volume of new families and updated information that needed to be added. To speed up releases of the database, the developers started a number of initiatives to allow greater community involvement in managing the database. -->
<!-- A critical step in improving the pace of updating and improving entries was to open up the functional annotation of Pfam domains to the Wikipedia community in release 26.0.[16] For entries that already had a Wikipedia entry, this was linked into the Pfam page, and for those that did not, the community were invited to create one and inform the curators, in order for it to be linked in. It is anticipated that while community involvement will greatly improve the level of annotation of these families, some will remain insufficiently notable for inclusion in Wikipedia, in which case they will retain their original Pfam description. Some Wikipedia articles cover multiple families, such as the Zinc finger article. An automated procedure for generating articles based on InterPro and Pfam data has also been implemented, which populates a page with information and links to databases as well as available images, then once an article has been reviewed by a curator it is moved from the Sandbox to Wikipedia proper. In order to guard against vandalism of articles, each Wikipedia revision is reviewed by curators before it is displayed on the Pfam website. Almost all cases of vandalism have been corrected by the community before they reach curators, however.[16] -->
<!-- Pfam is run by an international consortium of three groups. In the earlier releases of Pfam, family entries could only be modified at the Cambridge, UK site, limiting the ability of consortium members to contribute to site curation. In release 26.0, developers moved to a new system that allowed registered users anywhere in the world to add or modify Pfam families.[16] -->
<!-- References -->
<!-- Finn RD, Tate J, Mistry J, Coggill PC, Sammut SJ, Hotz HR, Ceric G, Forslund K, Eddy SR, Sonnhammer EL, Bateman A (2008). "The Pfam protein families database". Nucleic Acids Res. 36 (Database issue): D281–8. doi:10.1093/nar/gkm960. PMC 2238907. PMID 18039703. -->
<!-- Finn, R. D.; Mistry, J.; Schuster-Böckler, B.; Griffiths-Jones, S.; Hollich, V.; Lassmann, T.; Moxon, S.; Marshall, M.; Khanna, A.; Durbin, R.; Eddy, S. R.; Sonnhammer, E. L.; Bateman, A. (January 2006). "Pfam: clans, web tools and services" (Free full text). Nucleic Acids Research. 34 (Database issue): D247–D251. doi:10.1093/nar/gkj149. ISSN 0305-1048. PMC 1347511. PMID 16381856. -->
<!-- Bateman, A.; Coin, L.; Durbin, R.; Finn, R. D.; Hollich, V.; Griffiths-Jones, S.; Khanna, A.; Marshall, M.; Moxon, S.; Sonnhammer, E. L.; Studholme, D. J.; Yeats, C.; Eddy, S. R. (2004). "The Pfam protein families database". Nucleic Acids Research. 32 (Database issue): 138D–1141. doi:10.1093/nar/gkh121. ISSN 0305-1048. PMC 308855. PMID 14681378. open access -->
<!-- "Pfam 34.0 is released". Xfam Blog. Retrieved 2 July 2021. -->
<!-- Sammut, Stephen; Finn, Robert D.; Bateman, Alex (2008). "Pfam 10 years on: 10 000 families and still growing". Briefings in Bioinformatics. 9 (3): 210–219. doi:10.1093/bib/bbn010. PMID 18344544. -->
<!-- Sonnhammer, Erik L.L.; Eddy, Sean R.; Durbin, Richard (1997). "Pfam: A Comprehensive Database of Protein Domain Families Based on Seed Alignments". Proteins. 28 (3): 405–420. doi:10.1002/(sici)1097-0134(199707)28:3<405::aid-prot10>3.0.co;2-l. PMID 9223186. -->
<!-- Xu, Qifang; Dunbrack, Roland L. (2012). "Assignment of protein sequences to existing domain and family classification systems: Pfam and the PDB". Bioinformatics. 28 (21): 2763–2772. doi:10.1093/bioinformatics/bts533. PMC 3476341. PMID 22942020. -->
<!-- Finn, R. D.; Mistry, J.; Tate, J.; Coggill, P.; Heger, A.; Pollington, J. E.; Gavin, O. L.; Gunasekaran, P.; Ceric, G.; Forslund, K.; Holm, L.; Sonnhammer, E. L. L.; Eddy, S. R.; Bateman, A. (2009). "The Pfam protein families database". Nucleic Acids Research. 38 (Database): D211–D222. doi:10.1093/nar/gkp985. ISSN 0305-1048. PMC 2808889. PMID 19920124. -->
<!-- Bateman A, Birney E, Cerruti L, Durbin R, Etwiller L, Eddy SR, Griffiths-Jones S, Howe KL, Marshall M, Sonnhammer EL (2002). "The Pfam protein families database". Nucleic Acids Res. 30 (1): 276–80. doi:10.1093/nar/30.1.276. PMC 99071. PMID 11752314. -->
<!-- Adams MD, Celniker SE, Holt RA, Evans CA, Gocayne JD, et al. (2000). "The genome sequence of Drosophila melanogaster". Science. 287 (5461): 2185–95. Bibcode:2000Sci...287.2185.. CiteSeerX 10.1.1.549.8639. doi:10.1126/science.287.5461.2185. PMID 10731132. -->
<!-- Lander, Eric S.; Linton, Lauren M.; Birren, Bruce; Nusbaum, Chad; Zody, Michael C.; et al. (2001). "Initial sequencing and analysis of the human genome". Nature. 409 (6822): 860–921. doi:10.1038/35057062. ISSN 0028-0836. PMID 11237011. -->
<!-- Finn, Robert D.; Bateman, Alex; Clements, Jody; Coggill, Penelope; Eberhardt, Ruth Y.; Eddy, Sean R.; Heger, Andreas; Hetherington, Kirstie; Holm, Liisa; Mistry, Jaina; Sonnhammer, Erik L. L.; Tate, John; Punta, Marco (2014). "Pfam: the protein families database". Nucleic Acids Research. 42 (D1): D222–D230. doi:10.1093/nar/gkt1223. ISSN 0305-1048. PMC 3965110. PMID 24288371. -->
<!-- Sonnhammer EL, Eddy SR, Birney E, Bateman A, Durbin R (1998). "Pfam: multiple sequence alignments and HMM-profiles of protein domains". Nucleic Acids Res. 26 (1): 320–2. doi:10.1093/nar/26.1.320. PMC 147209. PMID 9399864. -->
<!-- Finn, R. D.; Marshall, M.; Bateman, A. (2004). "iPfam: visualization of protein-protein interactions in PDB at domain and amino acid resolutions". Bioinformatics. 21 (3): 410–412. doi:10.1093/bioinformatics/bti011. ISSN 1367-4803. PMID 15353450. -->
<!-- Finn, Robert D.; Coggill, Penelope; Eberhardt, Ruth Y.; Eddy, Sean R.; Mistry, Jaina; Mitchell, Alex L.; Potter, Simon C.; Punta, Marco; Qureshi, Matloob; Sangrador-Vegas, Amaia; Salazar, Gustavo A.; Tate, John; Bateman, Alex (2016). "The Pfam protein families database: towards a more sustainable future". Nucleic Acids Research. 44 (D1): D279–D285. doi:10.1093/nar/gkv1344. ISSN 0305-1048. PMC 4702930. PMID 26673716. -->
<!-- Punta, M.; Coggill, P. C.; Eberhardt, R. Y.; Mistry, J.; Tate, J.; Boursnell, C.; Pang, N.; Forslund, K.; Ceric, G.; Clements, J.; Heger, A.; Holm, L.; Sonnhammer, E. L. L.; Eddy, S. R.; Bateman, A.; Finn, R. D. (2011). "The Pfam protein families database". Nucleic Acids Research. 40 (D1): D290–D301. doi:10.1093/nar/gkr1065. ISSN 0305-1048. PMC 3245129. PMID 22127870. -->
<!-- El-Gebali, Sara; Mistry, Jaina; Bateman, Alex; Eddy, Sean R; Luciani, Aurélien; Potter, Simon C; Qureshi, Matloob; Richardson, Lorna J; Salazar, Gustavo A; Smart, Alfredo; Sonnhammer, Erik L L; Hirsh, Layla; Paladin, Lisanna; Piovesan, Damiano; Tosatto, Silvio C E; Finn, Robert D (8 January 2019). "The Pfam protein families database in 2019". Nucleic Acids Research. 47 (D1): D427–D432. doi:10.1093/nar/gky995. PMC 6324024. PMID 30357350. -->
<!-- "Evolutionary Classification of Protein Domains". prodata.swmed.edu. Retrieved 18 May 2019. -->
<!-- Chothia, Cyrus (1992). "One thousand families for the molecular biologist". Nature. 357 (6379): 543–544. Bibcode:1992Natur.357..543C. doi:10.1038/357543a0. ISSN 0028-0836. PMID 1608464. S2CID 4355476. -->
<!-- Heger, A.; Wilton, C. A.; Sivakumar, A.; Holm, L. (January 2005). "ADDA: a domain database with global coverage of the protein universe" (Free full text). Nucleic Acids Research. 33 (Database issue): D188–D191. doi:10.1093/nar/gki096. ISSN 0305-1048. PMC 540050. PMID 15608174. -->
<!-- "Pfam 28.0 release notes". Retrieved 30 June 2015. -->
<!-- "A new Pfam-B is released". Xfam Blog. 30 June 2020. -->
<!-- "Moving to xfam.org". Retrieved 25 November 2016. -->
<!-- Dunbrack, Roland. "PDBfam". Fox Chase Cancer Center. Retrieved 9 March 2013. -->
<!-- Xu, Qifang; Dunbrack, Roland (2012). "Assignment of protein sequences to existing domain and family classification systems: Pfam and the PDB". Bioinformatics. 28 (21): 2763–72. doi:10.1093/bioinformatics/bts533. PMC 3476341. PMID 22942020. -->

</div>
</div>
<div id="talking-glossary-phenotype" class="section level2" number="14.4">
<h2><span class="header-section-number">14.4</span> Talking Glossary: Phenotype</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Phenotype" class="uri">https://www.genome.gov/genetics-glossary/Phenotype</a></p>
<p><strong>Abstract</strong>: “A phenotype is an individual’s observable traits, such as height, eye color, and blood type. The genetic contribution to the phenotype is called the genotype. Some traits are largely determined by the genotype, while other traits are largely determined by environmental factors.”</p>
<p>Audio:</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/phenotype.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/phenotype.jpg</a></p>
<p><strong>Transcript:</strong> ““Phenotype” simply refers to an observable trait. “Pheno” simply means “observe” and comes from the same root as the word “phenomenon”. And so it’s an observable type of an organism, and it can refer to anything from a common trait, such as height or hair color, to presence or absence of a disease. Frequently, phenotypes are related and used–the term is used–to relate a difference in DNA sequence among individuals with a difference in trait, be it height or hair color, or disease, or what have you. But it’s important to remember that phenotypes are equally, or even sometimes more greatly influenced by environmental effects than genetic effects. So a phenotype can be directly related to a genotype, but not necessarily. There’s usually not a one-to-one correlation between a genotype and a phenotype. There are almost always environmental influences, such as what one eats, how much one exercises, how much one smokes, etc. All of those are environmental influences which will affect the phenotype as well.”</p>
<p>Christopher P. Austin, M.D.</p>

</div>
<div id="phylogenetics-vocab" class="section level2" number="14.5">
<h2><span class="header-section-number">14.5</span> Phylogenetics vocab</h2>
<p>Here are definitions of key terms related to phylogenetics.</p>
<p><strong>Apomorphy</strong>: A character present in one or more species in a clade that was not present in the clade’s common ancestor; an evolutionary novelty. Also known as a derived character. For a review of ancestral versus derived traits see this video . (Contrast with homplasy).</p>
<p><strong>Autapomorphy</strong>: A derived character state that is restricted to one taxon in a particular data set. Apomorphies are often biologically interesting (like wings in birds) but actually have no value for building phylogenetic trees because they don’t let you form clades. Stated another way, autapomorphis are unique to a single taxon and don’t allow you to group taxa into a clade.</p>
<p><strong>Convergence</strong>: The multiple and independent appearance in different lineages of similar evolutionary novelties (apomorphies). Often happens due to similar ecological conditions and evolutionary forces.</p>
<p><strong>Convergent evolution</strong>: Similarity between species that is caused by a similar, but evolutionary independent, response to a common environmental problem (Figure 1). Convergence can also occur at the molecular level where the same mutation occurs independently in two different lineages (Figure 2).</p>
<iframe src="https://wp.biologos.org/wp-content/uploads/2018/10/venema_23_1.jpg" width="672" height="400px" data-external="1">
</iframe>
<iframe src="https://wp.biologos.org/wp-content/uploads/2018/10/venema_22_7.jpg" width="672" height="400px" data-external="1">
</iframe>
<p><strong>Derived character</strong>: A character present in one or more species in a clade that was not present in the clade’s common ancestor; an evolutionary novelty; also known as an apomorphy. (Contrast with ancestral character)</p>
<p><strong>Homologous</strong>: Describes characters derived from a common ancestor. (Contrast with analogous).</p>
<p><strong>Homology</strong>: Similarity between species that results from inheritance of traits from a common ancestor. (Contrast with analogy). For more on homologies see <a href="https://evolution.berkeley.edu/evolibrary/article/0_0_0/lines_04">Understanding Evolution: Homologies</a>, <a href="https://evolution.berkeley.edu/evolibrary/article/0_0_0/lines_05">Homologies: anatomy</a> , <a href="https://evolution.berkeley.edu/evolibrary/article/0_0_0/lines_06">Homologies: comparative anatomy</a>, and <a href="https://evolution.berkeley.edu/evolibrary/article/0_0_0/lines_07">Homologies: development</a> .</p>
<p><strong>Homoplasy</strong>: Technical term which indicates similarity in the characters/traits found in different species that are unrelated to common ancestry. Instead, they can be due to convergent evolution or reversal - not common descent. An analogous trait is a type of homoplasy. For an in-depth discussion of homology versus homoplasy see <a href="https://www.youtube.com/watch?v=W-APHQ94gog">this video</a> (Contrast with homology).</p>
<p><strong>Lineage</strong>: A group of ancestral and descendant populations, species, or other taxa that are descended from a common ancestor. Synonymous with clade.</p>
<p><strong>Outgroup</strong>: A taxonomic group that diverged prior to the rest of the focal taxa (focal clade) in a phylogenetic analysis.</p>
<p><strong>Parsimony</strong>: A criterion for selecting among alternative patterns or explanations based on minimizing the total amount of change or complexity. Can be approximate as “simpler answers are to be preferred.”</p>
<p><strong>Reversal / evolutionary reversal</strong>: An event that results in the reversion of a derived trait (apomorphy) to the ancestral form. This happens frequently at the DNA level, where a mutation can change a base to something different, then later another mutation changes it back. This can be called a back mutation. Reversals can also occur at the protein or morphological level.</p>
<p><strong>Trait matrix (aka character matrix, character state matrix, data matrix)</strong>: A table showing the state of each trait occurring in each taxon. Generally, row represents taxa, columns represent characters, and numbers (usually just 0 or 1) represent character states.</p>

</div>
<div id="talking-glossary-plasmid-1-min" class="section level2" number="14.6">
<h2><span class="header-section-number">14.6</span> Talking Glossary: Plasmid (1 min)</h2>
<p>National Human Genome Research Institute
<a href="https://www.genome.gov/genetics-glossary/Plasmid" class="uri">https://www.genome.gov/genetics-glossary/Plasmid</a></p>
<p>Intro: “A plasmid is a small, often circular DNA molecule found in bacteria and other cells. Plasmids are separate from the bacterial chromosome and replicate independently of it. They generally carry only a small number of genes, notably some associated with antibiotic resistance. Plasmids may be passed between different bacterial cells.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/plasmid.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/plasmid.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/plasmid.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/plasmid.mp3</a></p>
<p>Transcription:
“Electrophoresis is a laboratory technique used to separate DNA, RNA, or protein molecules based on their size and electrical charge. An electric current is used to move molecules to be separated through a gel. Pores in the gel work like a sieve, allowing smaller molecules to move faster than larger molecules. The conditions used during electrophoresis can be adjusted to separate molecules in a desired size range.”</p>

</div>
<div id="talking-glossary-point-mutation-0.5-min" class="section level2" number="14.7">
<h2><span class="header-section-number">14.7</span> Talking Glossary: Point mutation (0.5 min)</h2>
<p>Point mutation</p>
<p><a href="https://www.genome.gov/genetics-glossary/Point-Mutation" class="uri">https://www.genome.gov/genetics-glossary/Point-Mutation</a></p>
<p>Abstract: “A point mutation is when a single base pair is altered. Point mutations can have one of three effects. First, the base substitution can be a silent mutation where the altered codon corresponds to the same amino acid. Second, the base substitution can be a missense mutation where the altered codon corresponds to a different amino acid. Or third, the base substitution can be a nonsense mutation where the altered codon corresponds to a stop signal.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/point_mutation.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/point_mutation.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/point_mutation.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/point_mutation.mp3</a></p>
<p>NOTE: Sources vary a little bit in how they define point mutations. Like this glossary entry, I include indels (insertions / deletions) as a type of point mutation.</p>
<p>Transcript:
“Point mutations are a large category of mutations that describe a change in single nucleotide of DNA, such that that nucleotide is switched for another nucleotide, or that nucleotide is deleted, or a single nucleotide is inserted into the DNA that causes that DNA to be different from the normal or wild type gene sequence.</p>
<p>Leslie G. Biesecker, M.D.”</p>

</div>
<div id="talking-glossary-polymorphism-1-min" class="section level2" number="14.8">
<h2><span class="header-section-number">14.8</span> Talking Glossary: Polymorphism (1 min)</h2>
<p><strong>Abstract</strong>: “Polymorphism involves one of two or more variants of a particular DNA sequence [and/or protein sequence]. The most common type of polymorphism involves variation at a single base pair [or amino acid]. Polymorphisms can also be much larger in size and involve long stretches of DNA. Called a single nucleotide polymorphism, or SNP (pronounced snip), scientists are studying how SNPs in the human genome correlate with disease, drug response, and other phenotypes.”</p>
<p><strong>Transcript</strong>: “Polymorphism, by strict definitions which hardly anybody pays attention to anymore, is a place in the DNA sequence where there is variation, and the less common variant is present in at least one percent of the people of who you test. That is to distinguish, therefore, polymorphism from a rare variant that might occur in only one in 1,000 people. A polymorphism, it has to occur in at least one in 100 people. Polymorphisms could be not just single-letter changes like a C instead of T. They could also be something more elaborate, like a whole stretch of DNA, that is either present or absent. You might call that a copy number variant; those are all polymorphisms. But this is basically a general term to talk about diversity in genomes in a species.”</p>
<p>Francis S. Collins, M.D., Ph.D.</p>

</div>
<div id="talking-glossary-u03-primer-0.5-min" class="section level2" number="14.9">
<h2><span class="header-section-number">14.9</span> Talking Glossary U03: Primer (0.5 min)</h2>
<p>National Human Genome Research Institute</p>
<p>Introduction: “A primer is a short, single-stranded DNA sequence used in the polymerase chain reaction (PCR) technique. In the PCR method, a pair of primers is used to hybridize with the sample DNA and define the region of the DNA that will be amplified. Primers are also referred to as oligonucleotides.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/primer.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/primer.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/primer.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/primer.mp3</a></p>
<p>Transcript
“Primer refers to a small set of nucleotides of DNA, typically 18 to 24 base pairs in length. And a primer can be used for a multitude of other experimental processes. You can use primer in PCR to target a locus to allow for amplification for further analysis. You’d use a primer for sequencing a sequencing reaction where you want to target a very specific region and then do analysis in the extension of that DNA molecule.</p>
<p><a href="https://www.genome.gov/staff/Stacie-Loftus-PhD">Stacie Loftus</a>, Ph.D.</p>

</div>
<div id="talking-glossary-promoter-1.5-min" class="section level2" number="14.10">
<h2><span class="header-section-number">14.10</span> Talking Glossary: Promoter (1.5 min)</h2>
<p><strong>Abstract</strong>: “A promoter is a sequence of DNA needed to turn a gene on or off. The process of transcription is initiated at the promoter. Usually found near the beginning of a gene, the promoter has a binding site for the enzyme used to make a messenger RNA (mRNA) molecule.”</p>
<p><strong>Audio</strong>: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/promoter.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/promoter.mp3</a></p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/promoter.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/promoter.jpg</a></p>
<p><strong>Transcript</strong>: “The promoter region is the sequence typically referred to that’s right upstream or right next to where a gene is about to be transcribed. It’s the region where certain regulatory elements will bind; these are proteins that will bind to help RNA get transcribed. Now,”promoter”, the term “promoter”, can actually be a little bit of a nebulous term because it’s not very exact. There are specific sequences that are generally found within a promoter region, but sometimes people refer to even extended promoter region that might include sequences that are farther upstream of the gene that might help enhance or repress the particular gene that’s about to be transcribed in certain cell types. In general, if you think of the promoter as that piece of DNA that’s just upstream of the transcription start site of a gene, that’s pretty much what we refer to as promoters.”</p>
<p>Elliott Margulies, Ph.D.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="o.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="q.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["openbio.pdf", "openbio.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
