<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title> 4 Phylogenetics vocab | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</title>
  <meta name="description" content="A collection of open-access and/or free resources on biology and environmental science." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content=" 4 Phylogenetics vocab | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content=" 4 Phylogenetics vocab | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  
  <meta name="twitter:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

<meta name="author" content="Compiled and edited by Nathan Brouwer" />


<meta name="date" content="2021-09-03" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="when-things-are-homolgous-and-analogous.html"/>

<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Biology for everyone</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#alignments"><i class="fa fa-check"></i><b>1.1</b> Alignments</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#talking-glossary-allele-0.5-min"><i class="fa fa-check"></i><b>1.2</b> Talking Glossary: Allele (0.5 min)</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#talking-glossary-bioinformatics"><i class="fa fa-check"></i><b>1.3</b> Talking glossary: Bioinformatics</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="clade.html"><a href="clade.html"><i class="fa fa-check"></i><b>2</b> Clade</a>
<ul>
<li class="chapter" data-level="2.1" data-path="clade.html"><a href="clade.html#clades-and-phylogenetic-trees"><i class="fa fa-check"></i><b>2.1</b> Clades and phylogenetic trees</a></li>
<li class="chapter" data-level="2.2" data-path="clade.html"><a href="clade.html#terminology"><i class="fa fa-check"></i><b>2.2</b> Terminology</a></li>
<li class="chapter" data-level="2.3" data-path="clade.html"><a href="clade.html#in-popular-culture"><i class="fa fa-check"></i><b>2.3</b> In popular culture</a></li>
<li class="chapter" data-level="2.4" data-path="clade.html"><a href="clade.html#references"><i class="fa fa-check"></i><b>2.4</b> References</a></li>
<li class="chapter" data-level="2.5" data-path="clade.html"><a href="clade.html#talking-glossary-cloning-0.5-min"><i class="fa fa-check"></i><b>2.5</b> Talking Glossary: Cloning (0.5 min)</a></li>
<li class="chapter" data-level="2.6" data-path="clade.html"><a href="clade.html#talking-glossary-codominance-0.5-min"><i class="fa fa-check"></i><b>2.6</b> Talking Glossary: Codominance (0.5 min)</a></li>
<li class="chapter" data-level="2.7" data-path="clade.html"><a href="clade.html#comparative-genomics"><i class="fa fa-check"></i><b>2.7</b> Comparative genomics</a></li>
<li class="chapter" data-level="2.8" data-path="clade.html"><a href="clade.html#talking-glossary-contig-0.75-min"><i class="fa fa-check"></i><b>2.8</b> Talking Glossary: Contig (0.75 min)</a></li>
<li class="chapter" data-level="2.9" data-path="clade.html"><a href="clade.html#talking-glossary---foundations-1-review-crossing-over"><i class="fa fa-check"></i><b>2.9</b> Talking Glossary - Foundations 1 Review: Crossing over</a></li>
<li class="chapter" data-level="2.10" data-path="clade.html"><a href="clade.html#talking-glossary-deletion-mutation-1.5-min"><i class="fa fa-check"></i><b>2.10</b> Talking Glossary: Deletion mutation (1.5 min)</a></li>
<li class="chapter" data-level="2.11" data-path="clade.html"><a href="clade.html#talking-glossary-dna-sequencing-0.75-min"><i class="fa fa-check"></i><b>2.11</b> Talking Glossary: DNA sequencing (0.75 min)</a></li>
<li class="chapter" data-level="2.12" data-path="clade.html"><a href="clade.html#sex-chromosome-dosage-compensation"><i class="fa fa-check"></i><b>2.12</b> Sex-Chromosome Dosage compensation</a>
<ul>
<li class="chapter" data-level="2.12.1" data-path="clade.html"><a href="clade.html#random-inactivation-of-one-x-in-xx-individuals"><i class="fa fa-check"></i><b>2.12.1</b> Random inactivation of one X in XX individuals</a></li>
<li class="chapter" data-level="2.12.2" data-path="clade.html"><a href="clade.html#two-fold-increased-transcription-of-a-single-x-in-xy-individuals"><i class="fa fa-check"></i><b>2.12.2</b> Two-fold increased transcription of a single X in XY individuals</a></li>
</ul></li>
<li class="chapter" data-level="2.13" data-path="clade.html"><a href="clade.html#talking-glossary-duplication-0.75-min"><i class="fa fa-check"></i><b>2.13</b> Talking Glossary: Duplication (0.75 min)</a></li>
<li class="chapter" data-level="2.14" data-path="clade.html"><a href="clade.html#talking-glossary-epigenetics"><i class="fa fa-check"></i><b>2.14</b> Talking Glossary: Epigenetics</a></li>
<li class="chapter" data-level="2.15" data-path="clade.html"><a href="clade.html#talking-glossary-founder-effect-a-type-of-genetic-bottleneck-1-min"><i class="fa fa-check"></i><b>2.15</b> Talking glossary: Founder effect (A type of genetic bottleneck; 1 min)</a></li>
<li class="chapter" data-level="2.16" data-path="clade.html"><a href="clade.html#talking-glossary-frameshift-mutation-1.5-min"><i class="fa fa-check"></i><b>2.16</b> Talking Glossary: Frameshift mutation (1.5 min)</a></li>
<li class="chapter" data-level="2.17" data-path="clade.html"><a href="clade.html#talking-glossary-u03-gel-electrophoresis-3-min"><i class="fa fa-check"></i><b>2.17</b> Talking Glossary U03: Gel Electrophoresis (“3 min)</a></li>
<li class="chapter" data-level="2.18" data-path="clade.html"><a href="clade.html#talking-glossary-gene-amplification-1.5-min"><i class="fa fa-check"></i><b>2.18</b> Talking Glossary: Gene amplification (1.5 min)</a></li>
<li class="chapter" data-level="2.19" data-path="clade.html"><a href="clade.html#talking-glossary-gene-pool-0.75-min"><i class="fa fa-check"></i><b>2.19</b> Talking Glossary: Gene Pool (0.75 min)</a></li>
<li class="chapter" data-level="2.20" data-path="clade.html"><a href="clade.html#gene-prediction"><i class="fa fa-check"></i><b>2.20</b> Gene prediction</a></li>
<li class="chapter" data-level="2.21" data-path="clade.html"><a href="clade.html#genbank"><i class="fa fa-check"></i><b>2.21</b> GenBank</a></li>
<li class="chapter" data-level="2.22" data-path="clade.html"><a href="clade.html#genetic-distance"><i class="fa fa-check"></i><b>2.22</b> Genetic distance</a></li>
<li class="chapter" data-level="2.23" data-path="clade.html"><a href="clade.html#talking-glossary-genetic-drift-0.75-min"><i class="fa fa-check"></i><b>2.23</b> Talking Glossary: Genetic drift (0.75 min)</a></li>
<li class="chapter" data-level="2.24" data-path="clade.html"><a href="clade.html#talking-glossary-genetic-engineering-1.5-min"><i class="fa fa-check"></i><b>2.24</b> Talking Glossary: Genetic engineering (1.5 min)</a></li>
<li class="chapter" data-level="2.25" data-path="clade.html"><a href="clade.html#genome-assembly"><i class="fa fa-check"></i><b>2.25</b> Genome assembly</a></li>
<li class="chapter" data-level="2.26" data-path="clade.html"><a href="clade.html#talking-glossary-genomics"><i class="fa fa-check"></i><b>2.26</b> Talking Glossary: Genomics</a></li>
<li class="chapter" data-level="2.27" data-path="clade.html"><a href="clade.html#talking-glossary-genotype"><i class="fa fa-check"></i><b>2.27</b> Talking Glossary: Genotype</a></li>
<li class="chapter" data-level="2.28" data-path="clade.html"><a href="clade.html#talking-glossary-germline-1.25-min"><i class="fa fa-check"></i><b>2.28</b> Talking Glossary: Germline (1.25 min)</a></li>
<li class="chapter" data-level="2.29" data-path="clade.html"><a href="clade.html#guild"><i class="fa fa-check"></i><b>2.29</b> Guild</a></li>
<li class="chapter" data-level="2.30" data-path="clade.html"><a href="clade.html#talking-glossary-haploid-1.25-min"><i class="fa fa-check"></i><b>2.30</b> Talking Glossary: Haploid (1.25 min)</a></li>
<li class="chapter" data-level="2.31" data-path="clade.html"><a href="clade.html#talking-glossary-heterozygous"><i class="fa fa-check"></i><b>2.31</b> Talking Glossary: Heterozygous</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html"><i class="fa fa-check"></i><b>3</b> When things are homolgous AND analogous!</a>
<ul>
<li class="chapter" data-level="3.1" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-homozygous"><i class="fa fa-check"></i><b>3.1</b> Talking Glossary: Homozygous</a></li>
<li class="chapter" data-level="3.2" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-insertion-mutation-1.25-min"><i class="fa fa-check"></i><b>3.2</b> Talking Glossary: Insertion mutation (1.25 min)</a></li>
<li class="chapter" data-level="3.3" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#jove-videos"><i class="fa fa-check"></i><b>3.3</b> JoVe Videos</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#biotechnology"><i class="fa fa-check"></i><b>3.3.1</b> Biotechnology</a></li>
<li class="chapter" data-level="3.3.2" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#macromolecules"><i class="fa fa-check"></i><b>3.3.2</b> Macromolecules</a></li>
<li class="chapter" data-level="3.3.3" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#cellular-respiration"><i class="fa fa-check"></i><b>3.3.3</b> Cellular respiration</a></li>
<li class="chapter" data-level="3.3.4" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#population-and-community-ecology"><i class="fa fa-check"></i><b>3.3.4</b> Population and community ecology</a></li>
<li class="chapter" data-level="3.3.5" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#population-genetics"><i class="fa fa-check"></i><b>3.3.5</b> Population genetics</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-locus-1-min"><i class="fa fa-check"></i><b>3.4</b> Talking Glossary: Locus (1 min)</a></li>
<li class="chapter" data-level="3.5" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-missense-mutation-1.25-min"><i class="fa fa-check"></i><b>3.5</b> Talking Glossary: Missense Mutation (1.25 min)</a></li>
<li class="chapter" data-level="3.6" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-mitochondrial-dna-1.5-min"><i class="fa fa-check"></i><b>3.6</b> Talking Glossary: Mitochondrial DNA (1.5 min)</a></li>
<li class="chapter" data-level="3.7" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-mutagen-0.5-min"><i class="fa fa-check"></i><b>3.7</b> Talking Glossary: Mutagen (0.5 min)</a></li>
<li class="chapter" data-level="3.8" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-mutation-0.5-min"><i class="fa fa-check"></i><b>3.8</b> Talking Glossary: Mutation (0.5 min)</a></li>
<li class="chapter" data-level="3.9" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-non-coding-dna-1-min"><i class="fa fa-check"></i><b>3.9</b> Talking Glossary: Non-coding DNA (1 min)</a></li>
<li class="chapter" data-level="3.10" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-nonsene-mutation"><i class="fa fa-check"></i><b>3.10</b> Talking Glossary: Nonsene mutation</a></li>
<li class="chapter" data-level="3.11" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-open-reading-frame-definition-3-min"><i class="fa fa-check"></i><b>3.11</b> Talking Glossary : Open reading frame definition (3 min)</a></li>
<li class="chapter" data-level="3.12" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-pcr---the-polymerase-chain-reaction-0.5-min"><i class="fa fa-check"></i><b>3.12</b> Talking Glossary: PCR - The Polymerase Chain Reaction (0.5 min)</a></li>
<li class="chapter" data-level="3.13" data-path="when-things-are-homolgous-and-analogous.html"><a href="when-things-are-homolgous-and-analogous.html#talking-glossary-phenotype"><i class="fa fa-check"></i><b>3.13</b> Talking Glossary: Phenotype</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html"><i class="fa fa-check"></i><b>4</b> Phylogenetics vocab</a>
<ul>
<li class="chapter" data-level="4.1" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-plasmid-1-min"><i class="fa fa-check"></i><b>4.1</b> Talking Glossary: Plasmid (1 min)</a></li>
<li class="chapter" data-level="4.2" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-point-mutation-0.5-min"><i class="fa fa-check"></i><b>4.2</b> Talking Glossary: Point mutation (0.5 min)</a></li>
<li class="chapter" data-level="4.3" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-polymorphism-1-min"><i class="fa fa-check"></i><b>4.3</b> Talking Glossary: Polymorphism (1 min)</a></li>
<li class="chapter" data-level="4.4" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-u03-primer-0.5-min"><i class="fa fa-check"></i><b>4.4</b> Talking Glossary U03: Primer (0.5 min)</a></li>
<li class="chapter" data-level="4.5" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-promoter-1.5-min"><i class="fa fa-check"></i><b>4.5</b> Talking Glossary: Promoter (1.5 min)</a></li>
<li class="chapter" data-level="4.6" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-recessive"><i class="fa fa-check"></i><b>4.6</b> Talking Glossary: Recessive</a></li>
<li class="chapter" data-level="4.7" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-recombinant-dna-0.5-min"><i class="fa fa-check"></i><b>4.7</b> Talking Glossary: Recombinant DNA (0.5 min)</a></li>
<li class="chapter" data-level="4.8" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-repressor-1-min"><i class="fa fa-check"></i><b>4.8</b> Talking Glossary: Repressor (1 min)</a></li>
<li class="chapter" data-level="4.9" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-restriction-enzyme-0.5-min"><i class="fa fa-check"></i><b>4.9</b> Talking Glossary: Restriction Enzyme (0.5 min)</a></li>
<li class="chapter" data-level="4.10" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-u03-shotgun-sequencing-1-min"><i class="fa fa-check"></i><b>4.10</b> Talking Glossary U03: Shotgun sequencing (1 min)</a></li>
<li class="chapter" data-level="4.11" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-sickle-cell-disease-4-min"><i class="fa fa-check"></i><b>4.11</b> Talking Glossary: Sickle Cell Disease (4 min)</a></li>
<li class="chapter" data-level="4.12" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-somatic-cells-0.75-min"><i class="fa fa-check"></i><b>4.12</b> Talking glossary: Somatic cells (0.75 min)</a></li>
<li class="chapter" data-level="4.13" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#sequence-annotation"><i class="fa fa-check"></i><b>4.13</b> Sequence annotation</a>
<ul>
<li class="chapter" data-level="4.13.1" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#process"><i class="fa fa-check"></i><b>4.13.1</b> Process</a></li>
</ul></li>
<li class="chapter" data-level="4.14" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-substitution-mutation-1.75-min"><i class="fa fa-check"></i><b>4.14</b> Talking Glossary: Substitution mutation (1.75 min)</a></li>
<li class="chapter" data-level="4.15" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#taxa-taxon-and-clades-a-brief-primer"><i class="fa fa-check"></i><b>4.15</b> Taxa, Taxon, and clades: A Brief Primer</a></li>
<li class="chapter" data-level="4.16" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#what-is-a-taxon"><i class="fa fa-check"></i><b>4.16</b> What is a “taxon”</a></li>
<li class="chapter" data-level="4.17" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#what-is-a-clade"><i class="fa fa-check"></i><b>4.17</b> What is a clade?</a></li>
<li class="chapter" data-level="4.18" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#further-reading"><i class="fa fa-check"></i><b>4.18</b> Further Reading:</a></li>
<li class="chapter" data-level="4.19" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-trait-0.75-min"><i class="fa fa-check"></i><b>4.19</b> Talking Glossary: Trait (0.75 min)</a></li>
<li class="chapter" data-level="4.20" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-vector-2-min"><i class="fa fa-check"></i><b>4.20</b> Talking Glossary: Vector (2 min)</a></li>
<li class="chapter" data-level="4.21" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-x-chromosome-1.25-min"><i class="fa fa-check"></i><b>4.21</b> Talking Glossary: X-chromosome (1.25 min)</a></li>
<li class="chapter" data-level="4.22" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#talking-glossary-x-inactivation-lyonization-1.45-min"><i class="fa fa-check"></i><b>4.22</b> Talking Glossary: X-inactivation (“Lyonization”; 1.45 min))</a></li>
<li class="chapter" data-level="4.23" data-path="phylogenetics-vocab.html"><a href="phylogenetics-vocab.html#additional-glossaries"><i class="fa fa-check"></i><b>4.23</b> Additional Glossaries</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="phylogenetics-vocab" class="section level1" number="4">
<h1><span class="header-section-number"> 4</span> Phylogenetics vocab</h1>
<p>Here are definitions of key terms related to phylogenetics.</p>
<p><strong>Apomorphy</strong>: A character present in one or more species in a clade that was not present in the clade’s common ancestor; an evolutionary novelty. Also known as a derived character. For a review of ancestral versus derived traits see this video . (Contrast with homplasy).</p>
<p><strong>Autapomorphy</strong>: A derived character state that is restricted to one taxon in a particular data set. Apomorphies are often biologically interesting (like wings in birds) but actually have no value for building phylogenetic trees because they don’t let you form clades. Stated another way, autapomorphis are unique to a single taxon and don’t allow you to group taxa into a clade.</p>
<p><strong>Convergence</strong>: The multiple and independent appearance in different lineages of similar evolutionary novelties (apomorphies). Often happens due to similar ecological conditions and evolutionary forces.</p>
<p><strong>Convergent evolution</strong>: Similarity between species that is caused by a similar, but evolutionary independent, response to a common environmental problem (Figure 1). Convergence can also occur at the molecular level where the same mutation occurs independently in two different lineages (Figure 2).</p>
<iframe src="https://wp.biologos.org/wp-content/uploads/2018/10/venema_23_1.jpg" width="672" height="400px">
</iframe>
<iframe src="https://wp.biologos.org/wp-content/uploads/2018/10/venema_22_7.jpg" width="672" height="400px">
</iframe>
<p><strong>Derived character</strong>: A character present in one or more species in a clade that was not present in the clade’s common ancestor; an evolutionary novelty; also known as an apomorphy. (Contrast with ancestral character)</p>
<p><strong>Homologous</strong>: Describes characters derived from a common ancestor. (Contrast with analogous).</p>
<p><strong>Homology</strong>: Similarity between species that results from inheritance of traits from a common ancestor. (Contrast with analogy). For more on homologies see <a href="https://evolution.berkeley.edu/evolibrary/article/0_0_0/lines_04">Understanding Evolution: Homologies</a>, <a href="https://evolution.berkeley.edu/evolibrary/article/0_0_0/lines_05">Homologies: anatomy</a> , <a href="https://evolution.berkeley.edu/evolibrary/article/0_0_0/lines_06">Homologies: comparative anatomy</a>, and <a href="https://evolution.berkeley.edu/evolibrary/article/0_0_0/lines_07">Homologies: development</a> .</p>
<p><strong>Homoplasy</strong>: Technical term which indicates similarity in the characters/traits found in different species that are unrelated to common ancestry. Instead, they can be due to convergent evolution or reversal - not common descent. An analogous trait is a type of homoplasy. For an in-depth discussion of homology versus homoplasy see <a href="https://www.youtube.com/watch?v=W-APHQ94gog">this video</a> (Contrast with homology).</p>
<p><strong>Lineage</strong>: A group of ancestral and descendant populations, species, or other taxa that are descended from a common ancestor. Synonymous with clade.</p>
<p><strong>Outgroup</strong>: A taxonomic group that diverged prior to the rest of the focal taxa (focal clade) in a phylogenetic analysis.</p>
<p><strong>Parsimony</strong>: A criterion for selecting among alternative patterns or explanations based on minimizing the total amount of change or complexity. Can be approximate as “simpler answers are to be preferred.”</p>
<p><strong>Reversal / evolutionary reversal</strong>: An event that results in the reversion of a derived trait (apomorphy) to the ancestral form. This happens frequently at the DNA level, where a mutation can change a base to something different, then later another mutation changes it back. This can be called a back mutation. Reversals can also occur at the protein or morphological level.</p>
<p><strong>Trait matrix (aka character matrix, character state matrix, data matrix)</strong>: A table showing the state of each trait occurring in each taxon. Generally, row represents taxa, columns represent characters, and numbers (usually just 0 or 1) represent character states.</p>

<div id="talking-glossary-plasmid-1-min" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Talking Glossary: Plasmid (1 min)</h2>
<p>National Human Genome Research Institute
<a href="https://www.genome.gov/genetics-glossary/Plasmid" class="uri">https://www.genome.gov/genetics-glossary/Plasmid</a></p>
<p>Intro: “A plasmid is a small, often circular DNA molecule found in bacteria and other cells. Plasmids are separate from the bacterial chromosome and replicate independently of it. They generally carry only a small number of genes, notably some associated with antibiotic resistance. Plasmids may be passed between different bacterial cells.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/plasmid.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/plasmid.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/plasmid.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/plasmid.mp3</a></p>
<p>Transcription:
“Electrophoresis is a laboratory technique used to separate DNA, RNA, or protein molecules based on their size and electrical charge. An electric current is used to move molecules to be separated through a gel. Pores in the gel work like a sieve, allowing smaller molecules to move faster than larger molecules. The conditions used during electrophoresis can be adjusted to separate molecules in a desired size range.”</p>

</div>
<div id="talking-glossary-point-mutation-0.5-min" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> Talking Glossary: Point mutation (0.5 min)</h2>
<p>Point mutation</p>
<p><a href="https://www.genome.gov/genetics-glossary/Point-Mutation" class="uri">https://www.genome.gov/genetics-glossary/Point-Mutation</a></p>
<p>Abstract: “A point mutation is when a single base pair is altered. Point mutations can have one of three effects. First, the base substitution can be a silent mutation where the altered codon corresponds to the same amino acid. Second, the base substitution can be a missense mutation where the altered codon corresponds to a different amino acid. Or third, the base substitution can be a nonsense mutation where the altered codon corresponds to a stop signal.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/point_mutation.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/point_mutation.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/point_mutation.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/point_mutation.mp3</a></p>
<p>NOTE: Sources vary a little bit in how they define point mutations. Like this glossary entry, I include indels (insertions / deletions) as a type of point mutation.</p>
<p>Transcript:
“Point mutations are a large category of mutations that describe a change in single nucleotide of DNA, such that that nucleotide is switched for another nucleotide, or that nucleotide is deleted, or a single nucleotide is inserted into the DNA that causes that DNA to be different from the normal or wild type gene sequence.</p>
<p>Leslie G. Biesecker, M.D.”</p>

</div>
<div id="talking-glossary-polymorphism-1-min" class="section level2" number="4.3">
<h2><span class="header-section-number">4.3</span> Talking Glossary: Polymorphism (1 min)</h2>
<p><strong>Abstract</strong>: “Polymorphism involves one of two or more variants of a particular DNA sequence [and/or protein sequence]. The most common type of polymorphism involves variation at a single base pair [or amino acid]. Polymorphisms can also be much larger in size and involve long stretches of DNA. Called a single nucleotide polymorphism, or SNP (pronounced snip), scientists are studying how SNPs in the human genome correlate with disease, drug response, and other phenotypes.”</p>
<p><strong>Transcript</strong>: “Polymorphism, by strict definitions which hardly anybody pays attention to anymore, is a place in the DNA sequence where there is variation, and the less common variant is present in at least one percent of the people of who you test. That is to distinguish, therefore, polymorphism from a rare variant that might occur in only one in 1,000 people. A polymorphism, it has to occur in at least one in 100 people. Polymorphisms could be not just single-letter changes like a C instead of T. They could also be something more elaborate, like a whole stretch of DNA, that is either present or absent. You might call that a copy number variant; those are all polymorphisms. But this is basically a general term to talk about diversity in genomes in a species.”</p>
<p>Francis S. Collins, M.D., Ph.D.</p>

</div>
<div id="talking-glossary-u03-primer-0.5-min" class="section level2" number="4.4">
<h2><span class="header-section-number">4.4</span> Talking Glossary U03: Primer (0.5 min)</h2>
<p>National Human Genome Research Institute</p>
<p>Introduction: “A primer is a short, single-stranded DNA sequence used in the polymerase chain reaction (PCR) technique. In the PCR method, a pair of primers is used to hybridize with the sample DNA and define the region of the DNA that will be amplified. Primers are also referred to as oligonucleotides.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/primer.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/primer.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/primer.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/primer.mp3</a></p>
<p>Transcript
“Primer refers to a small set of nucleotides of DNA, typically 18 to 24 base pairs in length. And a primer can be used for a multitude of other experimental processes. You can use primer in PCR to target a locus to allow for amplification for further analysis. You’d use a primer for sequencing a sequencing reaction where you want to target a very specific region and then do analysis in the extension of that DNA molecule.</p>
<p><a href="https://www.genome.gov/staff/Stacie-Loftus-PhD">Stacie Loftus</a>, Ph.D.</p>

</div>
<div id="talking-glossary-promoter-1.5-min" class="section level2" number="4.5">
<h2><span class="header-section-number">4.5</span> Talking Glossary: Promoter (1.5 min)</h2>
<p><strong>Abstract</strong>: “A promoter is a sequence of DNA needed to turn a gene on or off. The process of transcription is initiated at the promoter. Usually found near the beginning of a gene, the promoter has a binding site for the enzyme used to make a messenger RNA (mRNA) molecule.”</p>
<p><strong>Audio</strong>: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/promoter.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/promoter.mp3</a></p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/promoter.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/promoter.jpg</a></p>
<p><strong>Transcript</strong>: “The promoter region is the sequence typically referred to that’s right upstream or right next to where a gene is about to be transcribed. It’s the region where certain regulatory elements will bind; these are proteins that will bind to help RNA get transcribed. Now,”promoter”, the term “promoter”, can actually be a little bit of a nebulous term because it’s not very exact. There are specific sequences that are generally found within a promoter region, but sometimes people refer to even extended promoter region that might include sequences that are farther upstream of the gene that might help enhance or repress the particular gene that’s about to be transcribed in certain cell types. In general, if you think of the promoter as that piece of DNA that’s just upstream of the transcription start site of a gene, that’s pretty much what we refer to as promoters.”</p>
<p>Elliott Margulies, Ph.D.</p>

</div>
<div id="talking-glossary-recessive" class="section level2" number="4.6">
<h2><span class="header-section-number">4.6</span> Talking Glossary: Recessive</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Recessive" class="uri">https://www.genome.gov/genetics-glossary/Recessive</a></p>
<p><strong>Abstract</strong>: “Recessive is a quality found in the relationship between two versions of a gene. Individuals receive one version of a gene, called an allele, from each parent. If the alleles are different, the dominant allele will be expressed, while the effect of the other allele, called recessive, is masked. In the case of a recessive genetic disorder, an individual must inherit two copies of the mutated allele in order for the disease to be present.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/recessive.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/recessive.jpg</a></p>
<p><strong>Transcript</strong>: Recessive refers to a type of allele which will not be manifested in an individual unless both of the individual’s copies of that gene have that particular genotype. It’s usually referred to in conjunction with a Punnett square, other types of Mendelian genetics, and frequently contrasted with a dominant pattern of inheritance wherein if one has one copy of the gene, regardless of what the other copy is, that dominant allele will show itself. In the case of a recessive allele, the individual will show the trait which corresponds to that genotype only if both alleles are the same and have that particular recessive characteristic. Now, that recessive characteristic can be one of no functional consequence. This results in differences between individuals such as in eye color or hair color, but it can also refer to a disease. For instance, in cystic fibrosis, which is a very common Mendelian disorder, that disease exists only when there’s a malfunction of both genes that correspond to cystic fibrosis. If there is only one mutation, then that recessive mutation can be compensated for by the normal allele. However, when the function of both are lost, then the disease manifests itself as a recessive disease where there is a loss of function and therefore observable disease.</p>
<p>Christopher P. Austin, M.D.</p>

</div>
<div id="talking-glossary-recombinant-dna-0.5-min" class="section level2" number="4.7">
<h2><span class="header-section-number">4.7</span> Talking Glossary: Recombinant DNA (0.5 min)</h2>
<p>National Human Genome Research Institute
<a href="https://www.genome.gov/genetics-glossary/Recombinant-DNA" class="uri">https://www.genome.gov/genetics-glossary/Recombinant-DNA</a></p>
<p><strong>Introduction</strong>: “Recombinant DNA (rDNA) is a technology that uses enzymes to cut and paste together DNA sequences of interest. The recombined DNA sequences can be placed into vehicles called vectors that ferry the DNA into a suitable host cell where it can be copied or expressed.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/recombinant_DNA.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/recombinant_DNA.mp3</a></p>
<p><strong>Transcript</strong>: “Pieces of DNA, such as human DNA, can be engineered in a fashion that allows them to be copied, or replicated, in bacteria or yeast. This involves attaching appropriate elements to a piece of DNA and then transferring into a bacterial or yeast cell, with those elements then instructing the bacterial or yeast cell to copy this DNA along with its own. This process is known as DNA cloning, with the resulting cloned DNA often referred to as recombinant DNA.”</p>
<p><a href="https://www.genome.gov/staff/Eric-D-Green-MD-PhD">Eric D. Green</a>, M.D., Ph.D.</p>

</div>
<div id="talking-glossary-repressor-1-min" class="section level2" number="4.8">
<h2><span class="header-section-number">4.8</span> Talking Glossary: Repressor (1 min)</h2>
<p><strong>Abstract</strong>: “A repressor is a protein that turns off the expression of one or more genes. The repressor protein works by binding to the gene’s promoter region, preventing the production of messenger RNA (mRNA).”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/repressor.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/repressor.mp3</a></p>
<p><strong>Transcript</strong>: “A repressor is a protein that has a negative effect on gene expression. So these usually are proteins that bind to DNA, and they either prevent the RNA transcription machinery from getting in there and transcribing that DNA, or they just slow it down. So repressors are present in cells where you don’t want a particular gene expressed. So if the repressor cell recognizes a sequence in that gene, it will travel to there and keep that gene off in that cell. And this is how you prevent hemoglobin from being expressed in neurons, and how you would prevent liver enzymes from being expressed in red blood cells. Repressors are very difficult to study because it’s much easier to study things that give you more of what you’re looking for. But I think as we go along we’re going to find they play as important a role in gene regulation as the activating proteins.”</p>
<p>David M. Bodine, Ph.D.</p>

</div>
<div id="talking-glossary-restriction-enzyme-0.5-min" class="section level2" number="4.9">
<h2><span class="header-section-number">4.9</span> Talking Glossary: Restriction Enzyme (0.5 min)</h2>
<p>National Human Genome Research Institute</p>
<p><a href="https://www.genome.gov/genetics-glossary/Restriction-Enzyme" class="uri">https://www.genome.gov/genetics-glossary/Restriction-Enzyme</a></p>
<p>Intro: “A restriction enzyme is an enzyme isolated from bacteria that cuts DNA molecules at specific sequences. The isolation of these enzymes was critical to the development of recombinant DNA (rDNA) technology and genetic engineering.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/restriction_enzymes.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/restriction_enzymes.mp3</a></p>
<p>Text “Restriction enzymes are proteins that bind to DNA in a very specific manner. So they actually recognize the base pairs within the DNA. And typically they will bind to a palindromic sequence, for instance, a sequence that is a mirror copy of itself–AGCCGA.”</p>
<p><a href="https://www.genome.gov/staff/Stacie-Loftus-PhD">Stacie Loftus</a>, Ph.D.</p>

</div>
<div id="talking-glossary-u03-shotgun-sequencing-1-min" class="section level2" number="4.10">
<h2><span class="header-section-number">4.10</span> Talking Glossary U03: Shotgun sequencing (1 min)</h2>
<p>Now just called “sequencing”</p>
<p><strong>Introduction</strong>: “Shotgun sequencing is a laboratory technique for determining the DNA sequence of an organism’s genome. The method involves breaking the genome into a collection of small DNA fragments that are sequenced individually. A computer program looks for overlaps in the DNA sequences and uses them to place the individual fragments in their correct order to reconstitute the genome.”</p>
<p><strong>Transcript</strong>: “The most efficient way to sequence a large piece of DNA involves a process known as shotgun sequencing. For this, the starting DNA is broken up randomly into many smaller pieces, sort of in a shotgun fashion, with each of those pieces then sequenced individually. The resulting sequence reads generated from the different pieces are then analyzed by a computer program, looking for stretches of sequence from different reads that are identical with one another. When identical regions are identified, they are overlapped with one another, allowing the two sequence reads to be stitched together. This computer process is repeated over and over and over again, eventually yielding the complete sequence of the starting piece of DNA. The initial random fragmenting and reading of the DNA gave this approach the name”shotgun sequencing”. ”</p>
<p>Eric D. Green, M.D., Ph.D.</p>

</div>
<div id="talking-glossary-sickle-cell-disease-4-min" class="section level2" number="4.11">
<h2><span class="header-section-number">4.11</span> Talking Glossary: Sickle Cell Disease (4 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Sickle-Cell-Disease" class="uri">https://www.genome.gov/genetics-glossary/Sickle-Cell-Disease</a></p>
<p><strong>Abstract</strong>: “Sickle cell disease is a hereditary disease seen most often among people of African ancestry. Caused by mutations in one of the genes that encode the hemoglobin protein, the disease is inherited as an autosomal recessive trait. The mutation causes the red blood cells to take on an unusual sickle shape. Individuals affected by sickle cell disease are chronically anemic and experience significant damage to their heart, lungs, and kidneys.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/sickle_cell_disease.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/sickle_cell_disease.jpg</a></p>
<p><strong>Transcript</strong>: “Sickle cell disease is the first human inherited disease that was understood at the protein and the DNA level. Sickle cell is a disease that’s primarily seen in people of African descent. In studying the genomics of people from Africa, it’s very clear now that three different times during the history of the human race the mutation in the beta-globin gene that changes an amino acid at the sixth position of the protein, the same mutation has happened. And this has been expanded throughout history in Africa in three different groups and then migrated all over the world. Now, the mutation does not really affect the ability of beta-globin to participate with alpha-globin and make hemoglobin, and it doesn’t affect the ability to carry oxygen. In matter of fact, people who inherit a hemoglobin S gene from their mother and a hemoglobin S gene from their father, so their homozygous for hemoglobin S, have perfectly normal oxygen-carrying capacity of hemoglobin. The problems happen when the red cell containing hemoglobin S gets into the muscles or into your brain and discharges the oxygen. And what happens is that hemoglobin S has a tendency when the oxygen goes off to stick to each other, and they form these polymers. Now, when they get oxygen back on, they go back into solution. The difference between hemoglobin S and wild-type hemoglobin is that the wild-type hemoglobin is soluble when it’s got oxygen, and it’s also soluble when it doesn’t have oxygen. Now, if everything is working well, these polymers in people with sickle cell disease don’t get very big. But that can be worsened when you have extreme oxygen stress, and that’s what causes the characteristic sickle cells. The polymers get very, very long, and they stretch the cell out of shape. Now your spleen is very good at raking these cells out of the peripheral blood as they come through. And that’s what causes the anemia, and the old name for sickle cell was sickle cell anemia. However, though, short polymers are very, very dangerous. If you think of a red blood cell going through your veins and arteries and capillaries as a water balloon, you’ll see how it can squish itself down into a long cylinder to get through a capillary and flatten out like a pancake to get through areas in the spleen that kind of rake out the bad-shape red blood cells, and then to form right back to a regular water balloon shape to go into the arteries and veins. But if you have a red blood cell full of these small hemoglobin S polymers, it’s like having a water balloon full of ice chips, and as that goes through the capillaries and the small veins, it tears up the lining of these things, just as if you had a cut, and it activates your clotting response and micro-clots will form. And sometimes these get to be bigger and bigger clots, and so the real lethality in sickle cell disease is not from the anemia, it’s from this vascular disease. And so it’s very unfortunate that about one-third of people who are homozygous for hemoglobin S will have one or more strokes before they’re 10 years old. And in those fortunate few that are able to get to be adolescents, these clots can build up in the lungs and give a very severe disease called acute chest syndrome, which is basically emphysema or lung destruction from these obstructions. About one-third of the patients will live healthily to adulthood, but they have many problems with iron-overloaded organs, and their life span is significantly shorter. And it is simply amazing to me that this all comes from one very small change, changes in amino acid, that really isn’t having any effect on the normal function of the protein, but has big effects on all of the other systems in the body that the red cells pass through.”</p>
<p>David M. Bodine, Ph.D.</p>

</div>
<div id="talking-glossary-somatic-cells-0.75-min" class="section level2" number="4.12">
<h2><span class="header-section-number">4.12</span> Talking glossary: Somatic cells (0.75 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Somatic-Cells" class="uri">https://www.genome.gov/genetics-glossary/Somatic-Cells</a></p>
<p><strong>Abstract</strong>: “A somatic cell is any cell of the body except sperm and egg cells. Somatic cells are diploid, meaning that they contain two sets of chromosomes, one inherited from each parent. Mutations in somatic cells can affect the individual, but they are not passed on to offspring.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/somatic_cells.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/somatic_cells.jpg</a></p>
<p><strong>Transcript</strong>: “Somatic cells is a fairly general term which refers to essentially all the cells of the body except for the germ line; the germ line being the cells in the sexual organs that produce sperm and eggs. So anything that doesn’t have the job of producing sperm or eggs is a somatic cell. It is very important, of course, for every living organism to be alive, but it contributes nothing in terms of inheritance through genetics, inheritance to the next generation. So it is only of use to the living organism and has no relation to anything that happens in the next generation of that organism.”</p>
<p>Shawn Burgess, Ph.D.</p>

</div>
<div id="sequence-annotation" class="section level2" number="4.13">
<h2><span class="header-section-number">4.13</span> Sequence annotation</h2>
<p>Adapted from Wikipedia, the free encyclopedia</p>
<p><a href="https://en.wikipedia.org/wiki/DNA_annotation" class="uri">https://en.wikipedia.org/wiki/DNA_annotation</a></p>
<p>Note: Wikipedia titles this “DNA annotation.” In my experience this phrase is not used; rather we refer to sequence annotation, genome annotation, or just annotation.</p>
<p>Genome annotation is the process of predicting the locations of genes and all of the coding regions in a genome and predicitng what those genes do. An annotation (irrespective of the context) is a note added by way of explanation or commentary. Once a genome is sequenced, it needs to be annotated to make sense of it.[1]
For DNA annotation, a previously unknown sequence representation of genetic material is enriched with information of intron -exon boundaries, regulatory sequences , gene names and protein products. This annotation is stored in genomic databases.</p>
<div id="process" class="section level3" number="4.13.1">
<h3><span class="header-section-number">4.13.1</span> Process</h3>
<p>Genome annotation consists of two main steps:</p>
<ol style="list-style-type: decimal">
<li>predicting coding elements on the genome , a process called gene prediction</li>
<li>attaching biological information to these elements</li>
</ol>
<p>Structural genomic annotation consists of the identification of genomic elements.</p>
<ol style="list-style-type: decimal">
<li>ORFs [open reading frames] and their localization</li>
<li>predicted coding regions</li>
<li>predicted location of regulatory motifs (e.g. promoters)</li>
<li>Functional annotation consists of attaching biological information to genomic elements.</li>
</ol>

</div>
</div>
<div id="talking-glossary-substitution-mutation-1.75-min" class="section level2" number="4.14">
<h2><span class="header-section-number">4.14</span> Talking Glossary: Substitution mutation (1.75 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Substitution" class="uri">https://www.genome.gov/genetics-glossary/Substitution</a></p>
<p><strong>Abstract</strong>: “Substitution is a type of mutation where one base pair is replaced by a different base pair. The term also refers to the replacement of one amino acid in a protein with a different amino acid.”</p>
<p><strong>Transcript:</strong> “Subsititution refers to the replacement of one amino acid with another amino acid in a protein or the replacement of one nucleotide with another in DNA or RNA. Substitutions generally give rise to–or they always give rise to–either a polymorphism, that is, a difference between one person, one individual, in a population or another, or a special kind of polymorphism that we call a mutation. In either case, all individuals in the population originally had the same sequence of a gene. There were substitution events that resulted in a change in DNA sequence, which resulted in a change in RNA sequence, which then could result in a change in amino acid sequence. When that happens, that change in DNA sequence or amino acid sequence, or both, could have no effect on the protein, in which case the substitution is benign and has no functional effect. In other cases, if it changes the function of the protein, then it will be observed as either a functional polymorphism, something which increases the effectiveness of the protein product, and therefore would be evolutionarily selected for, or is bad–deleterious–in which case the person might die early and get evolutionarily selected against. Substitutions which lead to mutations, which lead to a deleterious outcome, that is the organism having difficulty with living or dying early, those we call mutations, but they’re the result of a certain kind of a substitution.”</p>
<p>Christopher P. Austin, M.D.</p>

</div>
<div id="taxa-taxon-and-clades-a-brief-primer" class="section level2" number="4.15">
<h2><span class="header-section-number">4.15</span> Taxa, Taxon, and clades: A Brief Primer</h2>
<p>By Nathan Brouwer</p>
<p>The following reading discusses what biologists mean when we use the terms taxa, taxon and clade. A few advanced topics are discussed that will be further detailed in future lessons. Names of specific species, families, orders etc. are used as examples; you are not expected to memorize them. A google-doc version of this file can be found here (Links to an external site.).</p>
<p><strong>Key Vocab</strong>:
* Taxa
* Taxon
* Clade
* Order
* Taxonomic group
* Primates</p>
<p><strong>Advanced topics</strong>:**
* Monophyletic group
* Non-monophyletic group</p>
</div>
<div id="what-is-a-taxon" class="section level2" number="4.16">
<h2><span class="header-section-number">4.16</span> What is a “taxon”</h2>
<p><strong>Taxon</strong> and <strong>clade</strong> refer to different kinds of biological groups. <strong>Taxa</strong> is the plural of <strong>taxon</strong>. A taxon (or <strong>taxonomic group</strong>) can refer to any species <em>or</em> group of species. Humans, <em>Homo sapiens</em>, is a taxon. We are the genus <em>Homo</em>, a larger taxon composed of humans and our close relatives, including Neanderthals (genus: <em>Homo</em>, species: <em>Homo neanderthalensis</em>). We are part of the <strong>family</strong> Hominidae, a larger taxon which includes us, chimps and bonobos (genus <em>Pan</em>), gorillas (genus <em>Gorilla</em>), and orangutans (genus <em>Pongo</em>). All primates are part of an <strong>order</strong>, which includes all the taxa mentioned previously as well as taxa such as monkeys, gibbons, and lemurs. We can say all primates considered together are taxon (a group of species), which is composed of many taxa (many individual species).</p>
<p>The upshot: each level of the hierarchy is a taxon which contains multiple taxa:</p>
<ul>
<li>Order: Primates</li>
<li>Family: Hominidae</li>
<li>Genus: Homo</li>
<li>Species: <em>Homo sapiens</em></li>
</ul>
<p>The word “taxa” is often used as a generic stand-in for species, family, or order. I can say about my own research “I studied two taxa of plants” – that is, I studied two plant species, though it could also mean I studied two plant families, two plant orders, etc. Someone who studies the phenomena of punctuated equilibrium can say “the number of taxa on earth increased dramatically during the Cambrian explosion.” A key way to remember this: Each of the branch tips on a phylogenetic tree is a taxon. Often the tips of a tree are species, but not always.</p>
<p>When using “taxa” you don’t necessarily have to refer to the current evolutionary or phylogenetic understanding of the species you are talking about. Most taxonomic groups were thought to reflect evolutionary relationships when they were first proposed, but subsequent information has indicated that the group isn’t necessarily coherent. For historical reasons and convenience, these taxonomic groups are maintained.</p>
<p>For example, the taxonomic group <strong>reptiles</strong> includes lizards, snakes, crocodiles, and turtles. These were once thought to be a cohesive group. We now know that birds and crocodiles are <strong>sister taxa</strong> and that the group we call “reptiles” should include birds if it were to be evolutionarily consistent (see <a href="https://en.wikipedia.org/wiki/Reptile#Phylogenetics_and_modern_definition">here</a> for a summary). The field known as herpetology focuses on the biology of reptiles, as well as amphibians (frogs and salamanders). This is a group of taxa that excludes birds and evolutionarily isn’t coherent. Similarly, the taxonomic group of “fish” is problematic from an evolutionary perspective (The precise reasons for this are beyond the scope of this particular reading).</p>
<p>One key point is that “taxa” and “taxon” do not have to refer to the species level - it could refer to subspecies, populations, or other levels of the hierarchy of biological classification. It is most commonly used in reference to the level of species, but this isn’t necessary.</p>
</div>
<div id="what-is-a-clade" class="section level2" number="4.17">
<h2><span class="header-section-number">4.17</span> What is a clade?</h2>
<p>In contrast to taxon and taxa, <strong>clade</strong> has a very specific meaning - a clade is <strong>all</strong> of the taxa that descended from a common ancestor, <em>plus</em> the common ancestor. Clade is synonymous with <strong>monophyletic group</strong>. If you exclude one of the taxa that descended from a common ancestor you end up with what is called a <strong>non-monophyletic group</strong>. “Reptiles” and “fish” are taxonomic groups (taxa) that are not clades because they don’t contain all of the descendants of their common ancestor and therefore are not evolutionarily coherent.</p>
<p><strong>Glossary</strong>:</p>
<ul>
<li>Clade – a group composed of an ancestor and all of its descendants.</li>
<li>Monophyletic group – a group composed of an ancestor and all of its descendants (=clade). Example: Reptilia is a monophyletic group if (and only if) it includes birds</li>
<li>Non-monophyletic group – a group composed of an ancestor and only some of its descendants, where the missing ones have been placed in another group. Example: “Reptilia” is a paraphyletic group if birds are excluded from it. The names of paraphyletic groups are often placed in quotation marks by convention.</li>
<li>Taxon – a named group and its constituent members. Normally a taxon will be a named clade. Phylogenetic definition (of a group) – a definition for a group that is based on common ancestry. Example 1: Tetrapoda is the group composed of the last common ancestor of living amphibians and amniotes and all taxa more closely related to that clade than to lungfish.</li>
</ul>
</div>
<div id="further-reading" class="section level2" number="4.18">
<h2><span class="header-section-number">4.18</span> Further Reading:</h2>
<p>For more on clades: <a href="http://en.wikipedia.org/wiki/Clade">http://en.wikipedia.org/wiki/Clade</a>
For more on taxa: <a href="https://en.wikipedia.org/wiki/Taxon">https://en.wikipedia.org/wiki/Taxon</a></p>
<p>“In biology, a taxon (plural taxa), from taxonomy, is a group of one or more populations of an organism or organisms seen by taxonomists to form a unit.”
ICZN (1999) International Code of Zoological Nomenclature.</p>
<p>“A taxonomic unit, whether named or not: i.e. a population, or group of populations of organisms which are usually inferred to be phylogenetically related and which have characters in common which differentiate (q.v.) the unit (e.g. a geographic population, a genus, a family, an order) from other such units.”</p>
<p>For information on non-monophyletic groups and why fish aren’t a clade, see these sites</p>
<ul>
<li><a href="http://ess17.blogspot.com/2008/06/q-can-you-please-explain-question-3-on.html" class="uri">http://ess17.blogspot.com/2008/06/q-can-you-please-explain-question-3-on.html</a></li>
<li><a href="https://en.wikipedia.org/wiki/Paraphyly" class="uri">https://en.wikipedia.org/wiki/Paraphyly</a></li>
<li><a href="https://en.wikipedia.org/wiki/Fish" class="uri">https://en.wikipedia.org/wiki/Fish</a></li>
</ul>

</div>
<div id="talking-glossary-trait-0.75-min" class="section level2" number="4.19">
<h2><span class="header-section-number">4.19</span> Talking Glossary: Trait (0.75 min)</h2>
<p>Abstract: “A trait is a specific characteristic of an organism. Traits can be determined by genes or the environment, or more commonly by interactions between them. The genetic contribution to a trait is called the genotype. The outward expression of the genotype is called the phenotype.”</p>
<p>Comments:
In ecology and evolutionary biology, we often use the terms trait and character interchangeably. In phylogenetics the term character is common.</p>
<p>In phylogenetics, we organize our data in a grid called a trait matrix, or a character matrix. We can consider phenotypic traits/phenotypic characters or sequences (genetic traits/genetic characters). Often we refer to phenotypic traits as morphological traits if they have to do with the size, shape, structure, anatomy etc. of an organism.</p>
<p>The definition here in the Talking Glossary highlights the fact that phenotypic traits are due to both genes and the environment. When we do phylogenetics with phenotypic traits we strive to use consistent morphological differences that are due to genetics. Because phenotypic traits like height, weight, color etc can vary so much due to the environment we use traits that can take on distinct states, like tails: monkeys have tails while humans don’t; dogs long have tails but bears have very short ones. We therefore draw a distinction between characters (e.g. tails) and character states (tail present, tail absent).</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/trait.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/trait.mp3</a></p>
<p>Transcript
“Trait is a specific characteristic of an individual. For example, their hair color or their blood type. Traits are determined by genes, and also they are determined by the interaction with the environment with genes. And remember that genes are the messages in our DNA that define individual characteristics. So the trait is the manifestation of the product of a gene that is coded for by the DNA. The word”phenotype” is sometimes used interchangeably with the word trait, and “phenotype” may also define a whole compendium of traits together.”</p>
<p><a href="https://www.nigms.nih.gov/about/Pages/krasnewich.aspx">Donna Krasnewich</a>, M.D., Ph.D. Program director in the Division of <a href="https://www.nigms.nih.gov/about/overview/Pages/GMCDB.aspx">Genetics and Molecular, Cellular, and Developmental Biology</a>, National Institutes of Health</p>
<p>Photograph: <a href="https://www.nigms.nih.gov/about/PublishingImages/headshots/headshot-donna-krasnewich-small.jpg" class="uri">https://www.nigms.nih.gov/about/PublishingImages/headshots/headshot-donna-krasnewich-small.jpg</a></p>

</div>
<div id="talking-glossary-vector-2-min" class="section level2" number="4.20">
<h2><span class="header-section-number">4.20</span> Talking Glossary: Vector (2 min)</h2>
<p>National Human Genome Research Institute</p>
<p>Note: Usually refers to a plasmid, though in this entry they mention how it can also apply to viruses.</p>
<p>Introduction: “A vector is any vehicle, often a virus or a plasmid [usually a plasmid in practice, and that’s what we focus on in this class] that is used to ferry a desired DNA sequence into a host cell as part of a molecular cloning procedure. Depending on the purpose of the cloning procedure, the vector may assist in multiplying, isolating, or expressing the foreign DNA insert.”</p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/vector.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/vector.mp3</a></p>
<p>Transcript
“A vector is a way to take a sequence of DNA, usually, and introduce it into another place. So what vectors do is allow you to propagate the DNA you’re interested in, in the organism you’ve chosen to propagate it in. So the simplest one is the origins of recombinant DNA technology: They made copies of RNAs, and they were able to insert these into what is known as plasmids. Now, plasmids are kind of mini-bacterial chromosomes. They have a way to replicate themselves, and what makes it work is they also carry one or two genes on them that make them resistant to specific antibiotics. So if you can insert the gene you’re interested in into this plasmid, you can select for the bacteria that have picked up that plasmid by growing them on an antibiotic that, if they haven’t picked it up, would kill them. So that plasmid is a vector for taking a particular DNA sequence into a bacteria. And then you can isolate one colony of bacteria and clone that, grow that clone up, and that’s how you would propagate that. There are other vectors that are larger and will have multiple sites of origins of replication, and these are known as bacterial artificial chromosomes, and they can handle much larger pieces of DNA. There are yeast artificial chromosomes that allow very large fragments of DNA to be grown in yeast cells. And recently, human artificial chromosome have been developed that allow enormous pieces of DNA to be introduced and propagated into human cells. So vector is really just a means to take a piece of DNA that you’re interested in and insert it, and select for it, and identify it in the organism that you want to propagate it in.”</p>
<p><a href="https://www.genome.gov/staff/David-M-Bodine-PhD">David M. Bodine</a>, Ph.D</p>

</div>
<div id="talking-glossary-x-chromosome-1.25-min" class="section level2" number="4.21">
<h2><span class="header-section-number">4.21</span> Talking Glossary: X-chromosome (1.25 min)</h2>
<p>Length: 1.25 min</p>
<p><a href="https://www.genome.gov/genetics-glossary/X-Chromosome" class="uri">https://www.genome.gov/genetics-glossary/X-Chromosome</a></p>
<p>Abstract: The X chromosome is one of two sex chromosomes. Humans and most mammals have two sex chromosomes, the X and Y. Some individuals have two X chromosomes in their cells, while others have X and Y chromosomes in their cells. Egg cells all contain an X chromosome, while sperm cells contain an X or a Y chromosome. This arrangement means that during fertilization, it is the XY parent that determines the whether the offspring will be XY or XX.</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/x_chromosome.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/x_chromosome.jpg</a></p>
<p>Audio: <a href="https://www.genome.gov/sites/default/files/tg/en/narration/x-chromosome.mp3" class="uri">https://www.genome.gov/sites/default/files/tg/en/narration/x-chromosome.mp3</a></p>
<p>Transcript
“So this, because I’m a female, is truly one of my favorite chromosomes. As you know, females have two X chromosomes. They’re quite large in comparison to the male chromosomes. They are carried by the egg, and so consequently you pass on–if you have an egg–you can only pass on an X chromosome to your offspring. What’s also quite interesting is the number of genes that are found on the X chromosome. It is predicted that there are approximately 155 million base pairs, which translates to about 900 to 1,400 genes on the X chromosome. Meaning that it carries about five percent of the total DNA in the entire cell. Which is quite in contrast to the Y chromosome, which is considerably smaller. Again, if you look at the X chromosomes in the genes that it carries, often times you’ll see that sex-linked disorders are carried on the X chromosome. Which is why, as I stated before, they’re more predominant in male, because there’s not a protective mechanism against having a mutation on one of those genes because we don’t have the normal copy of that gene on the X chromosome.”</p>
<p>Carla Easter, Ph.D.</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/genome-old/images/content/easter_carla.jpg" class="uri">https://www.genome.gov/sites/default/files/genome-old/images/content/easter_carla.jpg</a></p>
<p>Biography</p>
<p>Carla Easter, Ph.D., is chief of the Education and Community Involvement Branch at the National Human Genome Research Institute (NHGRI). She played a major role in the development of the NHGRI/Smithsonian exhibition Genome: Unlocking Life’s Code, and its accompanying website, and serves as a liaison to the K-12 and university community as a speaker on genomic science and career preparation and pathways. Dr. Easter also serves as an adjunct faculty member at the University of the District of Columbia Department of Biology, Chemistry and Physics.</p>
<p>From 2003-2006, Dr. Easter was director of outreach for Washington University School of Medicine’s Genome Sequencing Center. Before assuming her role as outreach director, Dr. Easter was a research associate in the Department of Education at Washington University (2001-2003) where she explored the notions of science among secondary students. She served as pre-college coordinator for the NASA Summer High School Apprenticeship Research Plus Program and project associate for the Quality Education for Minorities Network. From 1997-2000, Dr. Easter conducted post-doctoral research at Washington University School of Medicine on the virulence factors associated with Streptococcus pyogenes.</p>
<p>Dr. Easter earned her bachelor’s degree in microbiology from the University of California, Los Angeles and her doctoral in biology with an emphasis on molecular genetics from the University of California, San Diego.</p>
<p>Video: <a href="https://youtu.be/S1-Y8cO-sMc" class="uri">https://youtu.be/S1-Y8cO-sMc</a>
Dr. Easter is a laboratory scientist who now specializes in science communication and outreach. The video below is a discussion between Dr. Easter and a highschool student sponsored by the Children’s Science Center Lab</p>
<p>The YouTube video below contains an audio-only interview with Dr. Easter
<a href="https://youtu.be/X_CX7FWSfoM" class="uri">https://youtu.be/X_CX7FWSfoM</a></p>

</div>
<div id="talking-glossary-x-inactivation-lyonization-1.45-min" class="section level2" number="4.22">
<h2><span class="header-section-number">4.22</span> Talking Glossary: X-inactivation (“Lyonization”; 1.45 min))</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Lyonization" class="uri">https://www.genome.gov/genetics-glossary/Lyonization</a> (Links to an external site.)</p>
<p>Abstract: “Lyonization is commonly known as X-inactivation. In mammals, males receive one copy of the X chromosome while females receive two copies. To prevent female cells from having twice as many gene products from the X chromosomes as males, one copy of the X chromosome in each female cell is inactivated. In placental mammals, the choice of which X chromosome is inactivated is random, whereas in marsupials it is always the paternal copy that is inactivated.”</p>
<p>Audio</p>
<p>Transcript: “Lyonization is named after Mary Lyon, who was a geneticist who first figured out that in females who have two copies of the X chromosome, that one copy of each gene is turned off permanently in one chromosome or another. So that females, who have two copies of the X chromosome, and males, who have one copy of the X chromosome, are both human, and they can both operate fairly normally. So this process of turning off one copy of one gene or another on the X chromosome is called lyonization, and it happens from a series of essentially irreversible chemical modifications to one copy of the gene. The fascinating thing about this is that in so-called X-linked diseases, if the female inherits a gene responsible for an X-linked disease, and has one copy that’s abnormal and one copy that’s normal, the abnormal gene is almost always the one that’s turned off. And the normal gene is almost always allowed to stay on. In X-linked diseases in males, of course, X-linked diseases manifest because they only have one X chromosome, and so those mutated genes have to show up. But the fascinating part about it, which we really don’t understand, is how it is that the body knows if the female inherits one copy of a gene that is abnormal and would otherwise cause an X-linked disease if the normal copy was lyonized and turned off. That doesn’t happen. The abnormal copy gets turned off, leaving the normal copy to function, and saving the female from having the disease.”</p>
<p>Christopher P. Austin, M.D.</p>
<p>History:</p>
<p>English geneticist Mary Lyons (<a href="https://en.wikipedia.org/wiki/Mary_F._Lyon" class="uri">https://en.wikipedia.org/wiki/Mary_F._Lyon</a>) first characterized X-inactivation. You can read a short article about her life here (<a href="https://www.nature.com/articles/518036a" class="uri">https://www.nature.com/articles/518036a</a>).</p>

</div>
<div id="additional-glossaries" class="section level2" number="4.23">
<h2><span class="header-section-number">4.23</span> Additional Glossaries</h2>
<p><a href="https://www.ncbi.nlm.nih.gov/books/NBK5191/" class="uri">https://www.ncbi.nlm.nih.gov/books/NBK5191/</a></p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="when-things-are-homolgous-and-analogous.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["openbio.pdf", "openbio.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
