<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title> 17 S | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</title>
  <meta name="description" content="A collection of open-access and/or free resources on biology and environmental science." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content=" 17 S | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content=" 17 S | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  
  <meta name="twitter:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

<meta name="author" content="Compiled and edited by Nathan Brouwer" />


<meta name="date" content="2021-11-11" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="r.html"/>
<link rel="next" href="t.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Biology for everyone</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="a.html"><a href="a.html"><i class="fa fa-check"></i><b>2</b> A</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a.html"><a href="a.html#alignments"><i class="fa fa-check"></i><b>2.1</b> Alignments</a></li>
<li class="chapter" data-level="2.2" data-path="a.html"><a href="a.html#talking-glossary-allele-0.5-min"><i class="fa fa-check"></i><b>2.2</b> Talking Glossary: Allele (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="b.html"><a href="b.html"><i class="fa fa-check"></i><b>3</b> B</a>
<ul>
<li class="chapter" data-level="3.1" data-path="b.html"><a href="b.html#talking-glossary-bioinformatics"><i class="fa fa-check"></i><b>3.1</b> Talking glossary: Bioinformatics</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="c.html"><a href="c.html"><i class="fa fa-check"></i><b>4</b> C</a>
<ul>
<li class="chapter" data-level="4.1" data-path="c.html"><a href="c.html#talking-glossary-candidate-gene"><i class="fa fa-check"></i><b>4.1</b> Talking Glossary: Candidate gene</a></li>
<li class="chapter" data-level="4.2" data-path="c.html"><a href="c.html#cath-database"><i class="fa fa-check"></i><b>4.2</b> CATH database</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="c.html"><a href="c.html#references"><i class="fa fa-check"></i><b>4.2.1</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="c.html"><a href="c.html#clade"><i class="fa fa-check"></i><b>4.3</b> Clade</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="c.html"><a href="c.html#clades-and-phylogenetic-trees"><i class="fa fa-check"></i><b>4.3.1</b> Clades and phylogenetic trees</a></li>
<li class="chapter" data-level="4.3.2" data-path="c.html"><a href="c.html#terminology"><i class="fa fa-check"></i><b>4.3.2</b> Terminology</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="c.html"><a href="c.html#in-popular-culture"><i class="fa fa-check"></i><b>4.4</b> In popular culture</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="c.html"><a href="c.html#references-1"><i class="fa fa-check"></i><b>4.4.1</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="c.html"><a href="c.html#talking-glossary-cloning-0.5-min"><i class="fa fa-check"></i><b>4.5</b> Talking Glossary: Cloning (0.5 min)</a></li>
<li class="chapter" data-level="4.6" data-path="c.html"><a href="c.html#talking-glossary-codominance-0.5-min"><i class="fa fa-check"></i><b>4.6</b> Talking Glossary: Codominance (0.5 min)</a></li>
<li class="chapter" data-level="4.7" data-path="c.html"><a href="c.html#comparative-genomics"><i class="fa fa-check"></i><b>4.7</b> Comparative genomics</a></li>
<li class="chapter" data-level="4.8" data-path="c.html"><a href="c.html#talking-glossary-contig-0.75-min"><i class="fa fa-check"></i><b>4.8</b> Talking Glossary: Contig (0.75 min)</a></li>
<li class="chapter" data-level="4.9" data-path="c.html"><a href="c.html#talking-glossary---foundations-1-review-crossing-over"><i class="fa fa-check"></i><b>4.9</b> Talking Glossary - Foundations 1 Review: Crossing over</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="d.html"><a href="d.html"><i class="fa fa-check"></i><b>5</b> D</a>
<ul>
<li class="chapter" data-level="5.1" data-path="d.html"><a href="d.html#talking-glossary-deletion-mutation-1.5-min"><i class="fa fa-check"></i><b>5.1</b> Talking Glossary: Deletion mutation (1.5 min)</a></li>
<li class="chapter" data-level="5.2" data-path="d.html"><a href="d.html#talking-glossary-dna-sequencing-0.75-min"><i class="fa fa-check"></i><b>5.2</b> Talking Glossary: DNA sequencing (0.75 min)</a></li>
<li class="chapter" data-level="5.3" data-path="d.html"><a href="d.html#sex-chromosome-dosage-compensation"><i class="fa fa-check"></i><b>5.3</b> Sex-Chromosome Dosage compensation</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="d.html"><a href="d.html#random-inactivation-of-one-x-in-xx-individuals"><i class="fa fa-check"></i><b>5.3.1</b> Random inactivation of one X in XX individuals</a></li>
<li class="chapter" data-level="5.3.2" data-path="d.html"><a href="d.html#results"><i class="fa fa-check"></i><b>5.3.2</b> Results</a></li>
<li class="chapter" data-level="5.3.3" data-path="d.html"><a href="d.html#clinical-applications"><i class="fa fa-check"></i><b>5.3.3</b> Clinical applications</a></li>
<li class="chapter" data-level="5.3.4" data-path="d.html"><a href="d.html#agricultural-applications"><i class="fa fa-check"></i><b>5.3.4</b> Agricultural applications</a></li>
<li class="chapter" data-level="5.3.5" data-path="d.html"><a href="d.html#limitations"><i class="fa fa-check"></i><b>5.3.5</b> Limitations</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="d.html"><a href="d.html#fine-mapping"><i class="fa fa-check"></i><b>5.4</b> Fine-mapping</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="h.html"><a href="h.html"><i class="fa fa-check"></i><b>6</b> H</a>
<ul>
<li class="chapter" data-level="6.1" data-path="h.html"><a href="h.html#talking-glossary-haploid-1.25-min"><i class="fa fa-check"></i><b>6.1</b> Talking Glossary: Haploid (1.25 min)</a></li>
<li class="chapter" data-level="6.2" data-path="h.html"><a href="h.html#talking-glossary-halplotype"><i class="fa fa-check"></i><b>6.2</b> Talking Glossary: Halplotype</a></li>
<li class="chapter" data-level="6.3" data-path="h.html"><a href="h.html#talking-glossary-heterozygous"><i class="fa fa-check"></i><b>6.3</b> Talking Glossary: Heterozygous</a></li>
<li class="chapter" data-level="6.4" data-path="h.html"><a href="h.html#homologyl-when-things-are-homolgous-and-analogous"><i class="fa fa-check"></i><b>6.4</b> Homology:L When things are homolgous AND analogous!</a></li>
<li class="chapter" data-level="6.5" data-path="h.html"><a href="h.html#talking-glossary-homozygous"><i class="fa fa-check"></i><b>6.5</b> Talking Glossary: Homozygous</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="i.html"><a href="i.html"><i class="fa fa-check"></i><b>7</b> I</a>
<ul>
<li class="chapter" data-level="7.1" data-path="i.html"><a href="i.html#talking-glossary-insertion-mutation-1.25-min"><i class="fa fa-check"></i><b>7.1</b> Talking Glossary: Insertion mutation (1.25 min)</a></li>
<li class="chapter" data-level="7.2" data-path="i.html"><a href="i.html#intrinscially-disordered-protein"><i class="fa fa-check"></i><b>7.2</b> Intrinscially disordered protein</a></li>
<li class="chapter" data-level="7.3" data-path="i.html"><a href="i.html#abundance"><i class="fa fa-check"></i><b>7.3</b> Abundance</a></li>
<li class="chapter" data-level="7.4" data-path="i.html"><a href="i.html#disorder-annotation"><i class="fa fa-check"></i><b>7.4</b> Disorder annotation</a></li>
<li class="chapter" data-level="7.5" data-path="i.html"><a href="i.html#disorder-databases"><i class="fa fa-check"></i><b>7.5</b> Disorder databases</a></li>
<li class="chapter" data-level="7.6" data-path="i.html"><a href="i.html#predicting-idps-by-sequence"><i class="fa fa-check"></i><b>7.6</b> Predicting IDPs by sequence</a></li>
<li class="chapter" data-level="7.7" data-path="i.html"><a href="i.html#prediction-methods"><i class="fa fa-check"></i><b>7.7</b> Prediction methods</a></li>
<li class="chapter" data-level="7.8" data-path="i.html"><a href="i.html#disorder-and-disease"><i class="fa fa-check"></i><b>7.8</b> Disorder and disease</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="j.html"><a href="j.html"><i class="fa fa-check"></i><b>8</b> J</a>
<ul>
<li class="chapter" data-level="8.1" data-path="j.html"><a href="j.html#jove-videos"><i class="fa fa-check"></i><b>8.1</b> JoVe Videos</a>
<ul>
<li class="chapter" data-level="8.1.1" data-path="j.html"><a href="j.html#biotechnology"><i class="fa fa-check"></i><b>8.1.1</b> Biotechnology</a></li>
<li class="chapter" data-level="8.1.2" data-path="j.html"><a href="j.html#macromolecules"><i class="fa fa-check"></i><b>8.1.2</b> Macromolecules</a></li>
<li class="chapter" data-level="8.1.3" data-path="j.html"><a href="j.html#cellular-respiration"><i class="fa fa-check"></i><b>8.1.3</b> Cellular respiration</a></li>
<li class="chapter" data-level="8.1.4" data-path="j.html"><a href="j.html#population-and-community-ecology"><i class="fa fa-check"></i><b>8.1.4</b> Population and community ecology</a></li>
<li class="chapter" data-level="8.1.5" data-path="j.html"><a href="j.html#population-genetics"><i class="fa fa-check"></i><b>8.1.5</b> Population genetics</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="k.html"><a href="k.html"><i class="fa fa-check"></i><b>9</b> K</a>
<ul>
<li class="chapter" data-level="9.1" data-path="k.html"><a href="k.html#talking-glossary-knockout-genetic-knockout-gene-knockout"><i class="fa fa-check"></i><b>9.1</b> Talking glossary: Knockout (Genetic knockout, gene knockout)</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="l.html"><a href="l.html"><i class="fa fa-check"></i><b>10</b> L</a>
<ul>
<li class="chapter" data-level="10.1" data-path="l.html"><a href="l.html#talking-glossary-locus-1-min"><i class="fa fa-check"></i><b>10.1</b> Talking Glossary: Locus (1 min)</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="m.html"><a href="m.html"><i class="fa fa-check"></i><b>11</b> M</a>
<ul>
<li class="chapter" data-level="11.1" data-path="m.html"><a href="m.html#manhattan-plot"><i class="fa fa-check"></i><b>11.1</b> Manhattan Plot</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="m.html"><a href="m.html#gwas"><i class="fa fa-check"></i><b>11.1.1</b> GWAS</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="m.html"><a href="m.html#talking-glossary-microarray"><i class="fa fa-check"></i><b>11.2</b> Talking Glossary: Microarray</a></li>
<li class="chapter" data-level="11.3" data-path="m.html"><a href="m.html#talking-glossary-missense-mutation-1.25-min"><i class="fa fa-check"></i><b>11.3</b> Talking Glossary: Missense Mutation (1.25 min)</a></li>
<li class="chapter" data-level="11.4" data-path="m.html"><a href="m.html#talking-glossary-mitochondrial-dna-1.5-min"><i class="fa fa-check"></i><b>11.4</b> Talking Glossary: Mitochondrial DNA (1.5 min)</a></li>
<li class="chapter" data-level="11.5" data-path="m.html"><a href="m.html#talking-glossary-mutagen-0.5-min"><i class="fa fa-check"></i><b>11.5</b> Talking Glossary: Mutagen (0.5 min)</a></li>
<li class="chapter" data-level="11.6" data-path="m.html"><a href="m.html#talking-glossary-mutation-0.5-min"><i class="fa fa-check"></i><b>11.6</b> Talking Glossary: Mutation (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="n.html"><a href="n.html"><i class="fa fa-check"></i><b>12</b> N</a>
<ul>
<li class="chapter" data-level="12.1" data-path="n.html"><a href="n.html#talking-glossary-non-coding-dna-1-min"><i class="fa fa-check"></i><b>12.1</b> Talking Glossary: Non-coding DNA (1 min)</a></li>
<li class="chapter" data-level="12.2" data-path="n.html"><a href="n.html#talking-glossary-nonsene-mutation"><i class="fa fa-check"></i><b>12.2</b> Talking Glossary: Nonsene mutation</a></li>
<li class="chapter" data-level="12.3" data-path="n.html"><a href="n.html#nucleic-acids-review"><i class="fa fa-check"></i><b>12.3</b> Nucleic Acids - Overview</a>
<ul>
<li class="chapter" data-level="12.3.1" data-path="n.html"><a href="n.html#dna-and-rna"><i class="fa fa-check"></i><b>12.3.1</b> DNA and RNA</a></li>
<li class="chapter" data-level="12.3.2" data-path="n.html"><a href="n.html#dna-double-helix-structure"><i class="fa fa-check"></i><b>12.3.2</b> DNA Double-Helix Structure</a></li>
<li class="chapter" data-level="12.3.3" data-path="n.html"><a href="n.html#rna"><i class="fa fa-check"></i><b>12.3.3</b> RNA</a></li>
<li class="chapter" data-level="12.3.4" data-path="n.html"><a href="n.html#protein-function"><i class="fa fa-check"></i><b>12.3.4</b> Protein function</a></li>
<li class="chapter" data-level="12.3.5" data-path="n.html"><a href="n.html#summary"><i class="fa fa-check"></i><b>12.3.5</b> Summary</a></li>
<li class="chapter" data-level="12.3.6" data-path="n.html"><a href="n.html#glossary"><i class="fa fa-check"></i><b>12.3.6</b> Glossary</a></li>
<li class="chapter" data-level="12.3.7" data-path="n.html"><a href="n.html#contributors-and-attributions"><i class="fa fa-check"></i><b>12.3.7</b> Contributors and Attributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="13" data-path="o.html"><a href="o.html"><i class="fa fa-check"></i><b>13</b> O</a>
<ul>
<li class="chapter" data-level="13.1" data-path="o.html"><a href="o.html#talking-glossary-open-reading-frame-definition-3-min"><i class="fa fa-check"></i><b>13.1</b> Talking Glossary : Open reading frame definition (3 min)</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="p.html"><a href="p.html"><i class="fa fa-check"></i><b>14</b> P</a>
<ul>
<li class="chapter" data-level="14.1" data-path="p.html"><a href="p.html#talking-glossary-pcr---the-polymerase-chain-reaction-0.5-min"><i class="fa fa-check"></i><b>14.1</b> Talking Glossary: PCR - The Polymerase Chain Reaction (0.5 min)</a></li>
<li class="chapter" data-level="14.2" data-path="p.html"><a href="p.html#talking-glossary-phenotype"><i class="fa fa-check"></i><b>14.2</b> Talking Glossary: Phenotype</a></li>
<li class="chapter" data-level="14.3" data-path="p.html"><a href="p.html#phylogenetics-vocab"><i class="fa fa-check"></i><b>14.3</b> Phylogenetics vocab</a></li>
<li class="chapter" data-level="14.4" data-path="p.html"><a href="p.html#talking-glossary-plasmid-1-min"><i class="fa fa-check"></i><b>14.4</b> Talking Glossary: Plasmid (1 min)</a></li>
<li class="chapter" data-level="14.5" data-path="p.html"><a href="p.html#talking-glossary-point-mutation-0.5-min"><i class="fa fa-check"></i><b>14.5</b> Talking Glossary: Point mutation (0.5 min)</a></li>
<li class="chapter" data-level="14.6" data-path="p.html"><a href="p.html#talking-glossary-polymorphism-1-min"><i class="fa fa-check"></i><b>14.6</b> Talking Glossary: Polymorphism (1 min)</a></li>
<li class="chapter" data-level="14.7" data-path="p.html"><a href="p.html#talking-glossary-u03-primer-0.5-min"><i class="fa fa-check"></i><b>14.7</b> Talking Glossary U03: Primer (0.5 min)</a></li>
<li class="chapter" data-level="14.8" data-path="p.html"><a href="p.html#talking-glossary-promoter-1.5-min"><i class="fa fa-check"></i><b>14.8</b> Talking Glossary: Promoter (1.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="q.html"><a href="q.html"><i class="fa fa-check"></i><b>15</b> Q</a></li>
<li class="chapter" data-level="16" data-path="r.html"><a href="r.html"><i class="fa fa-check"></i><b>16</b> R</a>
<ul>
<li class="chapter" data-level="16.1" data-path="r.html"><a href="r.html#talking-glossary-recessive"><i class="fa fa-check"></i><b>16.1</b> Talking Glossary: Recessive</a></li>
<li class="chapter" data-level="16.2" data-path="r.html"><a href="r.html#talking-glossary-recombinant-dna-0.5-min"><i class="fa fa-check"></i><b>16.2</b> Talking Glossary: Recombinant DNA (0.5 min)</a></li>
<li class="chapter" data-level="16.3" data-path="r.html"><a href="r.html#talking-glossary-repressor-1-min"><i class="fa fa-check"></i><b>16.3</b> Talking Glossary: Repressor (1 min)</a></li>
<li class="chapter" data-level="16.4" data-path="r.html"><a href="r.html#talking-glossary-restriction-enzyme-0.5-min"><i class="fa fa-check"></i><b>16.4</b> Talking Glossary: Restriction Enzyme (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="s.html"><a href="s.html"><i class="fa fa-check"></i><b>17</b> S</a>
<ul>
<li class="chapter" data-level="17.1" data-path="s.html"><a href="s.html#structural-classification-of-proteins-database-scop"><i class="fa fa-check"></i><b>17.1</b> Structural Classification of Proteins database (SCOP)</a>
<ul>
<li class="chapter" data-level="17.1.1" data-path="s.html"><a href="s.html#hierarchical-organisation"><i class="fa fa-check"></i><b>17.1.1</b> Hierarchical organisation</a></li>
<li class="chapter" data-level="17.1.2" data-path="s.html"><a href="s.html#superfamilies"><i class="fa fa-check"></i><b>17.1.2</b> Superfamilies</a></li>
<li class="chapter" data-level="17.1.3" data-path="s.html"><a href="s.html#families"><i class="fa fa-check"></i><b>17.1.3</b> Families</a></li>
<li class="chapter" data-level="17.1.4" data-path="s.html"><a href="s.html#pdb-entry-domains"><i class="fa fa-check"></i><b>17.1.4</b> PDB entry domains</a></li>
</ul></li>
<li class="chapter" data-level="17.2" data-path="s.html"><a href="s.html#sequence-alignment"><i class="fa fa-check"></i><b>17.2</b> Sequence alignment</a>
<ul>
<li class="chapter" data-level="17.2.1" data-path="s.html"><a href="s.html#interpretation"><i class="fa fa-check"></i><b>17.2.1</b> Interpretation</a></li>
<li class="chapter" data-level="17.2.2" data-path="s.html"><a href="s.html#alignment-methods"><i class="fa fa-check"></i><b>17.2.2</b> Alignment methods</a></li>
</ul></li>
<li class="chapter" data-level="17.3" data-path="s.html"><a href="s.html#representations"><i class="fa fa-check"></i><b>17.3</b> Representations</a>
<ul>
<li class="chapter" data-level="17.3.1" data-path="s.html"><a href="s.html#global-and-local-alignments"><i class="fa fa-check"></i><b>17.3.1</b> Global and local alignments</a></li>
<li class="chapter" data-level="17.3.2" data-path="s.html"><a href="s.html#pairwise-alignment"><i class="fa fa-check"></i><b>17.3.2</b> Pairwise alignment</a></li>
<li class="chapter" data-level="17.3.3" data-path="s.html"><a href="s.html#dot-matrix-methods"><i class="fa fa-check"></i><b>17.3.3</b> Dot-matrix methods</a></li>
<li class="chapter" data-level="17.3.4" data-path="s.html"><a href="s.html#dynamic-programming"><i class="fa fa-check"></i><b>17.3.4</b> Dynamic programming</a></li>
<li class="chapter" data-level="17.3.5" data-path="s.html"><a href="s.html#word-methods"><i class="fa fa-check"></i><b>17.3.5</b> Word methods</a></li>
<li class="chapter" data-level="17.3.6" data-path="s.html"><a href="s.html#multiple-sequence-alignment"><i class="fa fa-check"></i><b>17.3.6</b> Multiple sequence alignment</a></li>
<li class="chapter" data-level="17.3.7" data-path="s.html"><a href="s.html#phylogenetic-analysis"><i class="fa fa-check"></i><b>17.3.7</b> Phylogenetic analysis</a></li>
<li class="chapter" data-level="17.3.8" data-path="s.html"><a href="s.html#assessment-of-significance"><i class="fa fa-check"></i><b>17.3.8</b> Assessment of significance</a></li>
<li class="chapter" data-level="17.3.9" data-path="s.html"><a href="s.html#other-biological-uses"><i class="fa fa-check"></i><b>17.3.9</b> Other biological uses</a></li>
<li class="chapter" data-level="17.3.10" data-path="s.html"><a href="s.html#references-2"><i class="fa fa-check"></i><b>17.3.10</b> References</a></li>
</ul></li>
<li class="chapter" data-level="17.4" data-path="s.html"><a href="s.html#talking-glossary-u03-shotgun-sequencing-1-min"><i class="fa fa-check"></i><b>17.4</b> Talking Glossary U03: Shotgun sequencing (1 min)</a></li>
<li class="chapter" data-level="17.5" data-path="s.html"><a href="s.html#talking-glossary-sickle-cell-disease-4-min"><i class="fa fa-check"></i><b>17.5</b> Talking Glossary: Sickle Cell Disease (4 min)</a></li>
<li class="chapter" data-level="17.6" data-path="s.html"><a href="s.html#medline-snps-single-nucleotide-polymorphism"><i class="fa fa-check"></i><b>17.6</b> Medline: SNPs (single nucleotide polymorphism)</a></li>
<li class="chapter" data-level="17.7" data-path="s.html"><a href="s.html#talking-glossary-somatic-cells-0.75-min"><i class="fa fa-check"></i><b>17.7</b> Talking glossary: Somatic cells (0.75 min)</a></li>
<li class="chapter" data-level="17.8" data-path="s.html"><a href="s.html#sequence-annotation"><i class="fa fa-check"></i><b>17.8</b> Sequence annotation</a>
<ul>
<li class="chapter" data-level="17.8.1" data-path="s.html"><a href="s.html#process"><i class="fa fa-check"></i><b>17.8.1</b> Process</a></li>
</ul></li>
<li class="chapter" data-level="17.9" data-path="s.html"><a href="s.html#talking-glossary-substitution-mutation-1.75-min"><i class="fa fa-check"></i><b>17.9</b> Talking Glossary: Substitution mutation (1.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="t.html"><a href="t.html"><i class="fa fa-check"></i><b>18</b> T</a>
<ul>
<li class="chapter" data-level="18.1" data-path="t.html"><a href="t.html#taxa-taxon-and-clades-a-brief-primer"><i class="fa fa-check"></i><b>18.1</b> Taxa, Taxon, and clades: A Brief Primer</a></li>
<li class="chapter" data-level="18.2" data-path="t.html"><a href="t.html#what-is-a-taxon"><i class="fa fa-check"></i><b>18.2</b> What is a “taxon”</a></li>
<li class="chapter" data-level="18.3" data-path="t.html"><a href="t.html#what-is-a-clade"><i class="fa fa-check"></i><b>18.3</b> What is a clade?</a></li>
<li class="chapter" data-level="18.4" data-path="t.html"><a href="t.html#further-reading"><i class="fa fa-check"></i><b>18.4</b> Further Reading:</a></li>
<li class="chapter" data-level="18.5" data-path="t.html"><a href="t.html#talking-glossary-trait-0.75-min"><i class="fa fa-check"></i><b>18.5</b> Talking Glossary: Trait (0.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="u.html"><a href="u.html"><i class="fa fa-check"></i><b>19</b> U</a></li>
<li class="chapter" data-level="20" data-path="v.html"><a href="v.html"><i class="fa fa-check"></i><b>20</b> V</a>
<ul>
<li class="chapter" data-level="20.1" data-path="v.html"><a href="v.html#talking-glossary-vector-2-min"><i class="fa fa-check"></i><b>20.1</b> Talking Glossary: Vector (2 min)</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="w.html"><a href="w.html"><i class="fa fa-check"></i><b>21</b> W</a>
<ul>
<li class="chapter" data-level="21.1" data-path="w.html"><a href="w.html#talking-glossary-x-chromosome-1.25-min"><i class="fa fa-check"></i><b>21.1</b> Talking Glossary: X-chromosome (1.25 min)</a></li>
<li class="chapter" data-level="21.2" data-path="w.html"><a href="w.html#talking-glossary-x-inactivation-lyonization-1.45-min"><i class="fa fa-check"></i><b>21.2</b> Talking Glossary: X-inactivation (“Lyonization”; 1.45 min))</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="x.html"><a href="x.html"><i class="fa fa-check"></i><b>22</b> X</a></li>
<li class="chapter" data-level="23" data-path="v-1.html"><a href="v-1.html"><i class="fa fa-check"></i><b>23</b> V</a>
<ul>
<li class="chapter" data-level="23.1" data-path="v-1.html"><a href="v-1.html#additional-glossaries"><i class="fa fa-check"></i><b>23.1</b> Additional Glossaries</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="z.html"><a href="z.html"><i class="fa fa-check"></i><b>24</b> Z</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="s" class="section level1" number="17">
<h1><span class="header-section-number"> 17</span> S</h1>

<div id="structural-classification-of-proteins-database-scop" class="section level2" number="17.1">
<h2><span class="header-section-number">17.1</span> Structural Classification of Proteins database (SCOP)</h2>
<p>Adapted from Wikipedia
<a href="https://en.wikipedia.org/wiki/Structural_Classification_of_Proteins_database" class="uri">https://en.wikipedia.org/wiki/Structural_Classification_of_Proteins_database</a></p>
<p>The Structural Classification of Proteins (SCOP) database is a largely manual classification of protein <strong>structural domains</strong> based on similarities of their structures and amino acid sequences. A motivation for this classification is to determine the <strong>evolutionary relationship</strong> between proteins. Proteins with the same shapes but having little sequence or functional similarity are placed in different superfamilies, and are assumed to have only a very distant common ancestor. Proteins having the same shape and some similarity of sequence and/or function are placed in “families”, and are assumed to have a closer common ancestor.</p>
<p>Similar to <strong>CATH</strong> and <strong>Pfam</strong> databases, SCOP provides a classification of individual structural domains of proteins, rather than a classification of the entire proteins which may include a significant number of different domains.</p>
<p>The SCOP database is freely accessible on the internet. SCOP was created in 1994 in the Centre for Protein Engineering and the Laboratory of Molecular Biology (3). It was maintained by Alexey G. Murzin and his colleagues in the Centre for Protein Engineering until its closure in 2010 and subsequently at the Laboratory of Molecular Biology in Cambridge, England (4, 5, 6, 1).</p>
<div id="hierarchical-organisation" class="section level3" number="17.1.1">
<h3><span class="header-section-number">17.1.1</span> Hierarchical organisation</h3>
<p>The source of protein structures is the Protein Data Bank. The unit of classification of structure in SCOP is the protein domain. What the SCOP authors mean by “domain” is suggested by their statement that small proteins and most medium-sized ones have just one domain (8), and by the observation that human hemoglobin (9), which has an α2β2 structure, is assigned two SCOP domains, one for the α and one for the β subunit.</p>
<p>The shapes of domains are called “<strong>folds</strong>” in SCOP. Domains belonging to the same fold have the same major secondary structures in the same arrangement with the same topological connections. 1195 folds are given in SCOP version 1.75.</p>
<!-- Short descriptions of each fold are given. For example, the "globin-like" fold is described as core: 6 helices; folded leaf, partly opened. The fold to which a domain belongs is determined by inspection, rather than by software. -->
<!-- The levels of SCOP version 1.75 are as follows. -->
<!-- * Class: Types of folds, e.g., beta sheets. -->
<!-- * Fold: The different shapes of domains within a class. -->
<!-- * Superfamily: The domains in a fold are grouped into superfamilies, which have at least a distant common ancestor. -->
<!-- * Family: The domains in a superfamily are grouped into families, which have a more recent common ancestor. -->
<!-- * Protein domain: The domains in families are grouped into protein domains, which are essentially the same protein. -->
<!-- * Species: The domains in "protein domains" are grouped according to species. -->
<!-- * Domain: part of a protein. For simple proteins, it can be the entire protein -->
<!-- ### Classes -->
<!-- The broadest groups on SCOP version 1.75 are the protein fold classes. These classes group structures with similar secondary structure composition, but different overall tertiary structures and evolutionarily origins. This is the top level "root" of the SCOP hierarchical classification. -->
<!-- * All alpha proteins [46456] (284): Domains consisting of α-helices -->
<!-- * All beta proteins [48724] (174): Domains consisting of β-sheets -->
<!-- * Alpha and beta proteins (a/b) [51349] (147): Mainly parallel beta sheets (beta-alpha-beta units) -->
<!-- * Alpha and beta proteins (a+b) [53931] (376): Mainly antiparallel beta sheets (segregated alpha and beta regions) -->
<!-- * Multi-domain proteins (alpha and beta) [56572] (66): Folds consisting of two or more domains belonging to different classes membrane and cell surface proteins and peptides [56835] (58): Does not include proteins in the immune system -->
<!-- * Small proteins [56992] (90): Usually dominated by metal ligand, cofactor, and/or disulfide bridges -->
<!-- coiled-coil proteins [57942] (7): Not a true class -->
<!-- * Low resolution protein structures [58117] (26): Peptides and fragments. Not a true class -->
<!-- Peptides [58231] (121): peptides and fragments. Not a true class. -->
<!-- Designed proteins [58788] (44): Experimental structures of proteins with essentially non-natural sequences. Not a true class -->
<!-- The number in brackets, called a "sunid", is a SCOP unique integer identifier for each node in the SCOP hierarchy. The number in parentheses indicates how many elements are in each category. For example, there are 284 folds in the "All alpha proteins" class. Each member of the hierarchy is a link to the next level of the hierarchy. -->
<!-- ### Folds -->
<!-- Each class contains a number of distinct folds. This classification level indicates similar tertiary structure, but not necessarily evolutionary relatedness. For example, the "All-α proteins" class contains >280 distinct folds, including: Globin-like (core: 6 helices; folded leaf, partly opened), long alpha-hairpin (2 helices; antiparallel hairpin, left-handed twist) and Type I dockerin domains (tandem repeat of two calcium-binding loop-helix motifs, distinct from the EF-hand). -->
</div>
<div id="superfamilies" class="section level3" number="17.1.2">
<h3><span class="header-section-number">17.1.2</span> Superfamilies</h3>
<p>Domains within a fold are classified into <strong>superfamilies</strong>. This is a largest grouping of proteins for which structural similarity is sufficient to indicate evolutionary relatedness and therefore share a common ancestor. However, this ancestor is presumed to be distant, because the different members of a superfamily have low sequence identities. For example, the two superfamilies of the “Globin-like” fold are: the Globin superfamily and alpha-helical ferredoxin superfamily.</p>
</div>
<div id="families" class="section level3" number="17.1.3">
<h3><span class="header-section-number">17.1.3</span> Families</h3>
<p>Protein families are more closely related than superfamilies. Domains are placed in the same family if that have either:</p>
<ul>
<li>less than 30% sequence identity</li>
<li>some sequence identity (e.g., 15%) AND perform the same function</li>
</ul>
<p>The similarity in sequence and structure is evidence that these proteins have a closer evolutionary relationship than do proteins in the same superfamily. Sequence tools, such as BLAST, are used to assist in placing domains into superfamilies and families.</p>
<!-- For example, the four families in the "globin-like" superfamily of the "globin-like" fold are truncated hemoglobin (lack the first helix), nerve tissue mini-hemoglobin (lack the first helix but otherwise is more similar to conventional globins than the truncated ones), globins (Heme-binding protein), and phycocyanin-like phycobilisome proteins (oligomers of two different types of globin-like subunits containing two extra helices at the N-terminus binds a bilin chromophore). Families in SCOP are each assigned a concise classification string, sccs, where the letter identifies the class to which the domain belongs; the following integers identify the fold, superfamily, and family, respectively (e.g., a.1.1.2 for the "Globin" family).[10] -->
</div>
<div id="pdb-entry-domains" class="section level3" number="17.1.4">
<h3><span class="header-section-number">17.1.4</span> PDB entry domains</h3>
<p><strong>OPTIONAL</strong></p>
<p>A “TaxId” is the taxonomy ID number and links to the NCBI taxonomy browser, which provides more information about the species to which the protein belongs. Clicking on a species or isoform brings up a list of domains. For example, the “Hemoglobin, alpha-chain from Human (Homo sapiens)” protein has &gt;190 solved protein structures, such as 2dn3 (complexed with cmo), and 2dn1 (complexed with hem, mbn, oxy). Clicking on the PDB numbers is supposed to display the structure of the molecule, but the links are currently broken (links work in pre-SCOP).</p>
<!-- ### Example -->
<!-- Most pages in SCOP contain a search box. Entering "trypsin +human" retrieves several proteins, including the protein trypsinogen from humans. Selecting that entry displays a page that includes the "lineage", which is at the top of most SCOP pages. -->
<!-- Human trypsonogen lineage -->
<!-- Root: scop -->
<!-- Class: All beta proteins [48724] -->
<!-- Fold: Trypsin-like serine proteases [50493] -->
<!-- barrel, closed; n=6, S=8; greek-key -->
<!-- duplication: consists of two domains of the same fold -->
<!-- Superfamily: Trypsin-like serine proteases [50494] -->
<!-- Family: Eukaryotic proteases [50514] -->
<!-- Protein: Trypsin(ogen) [50515] -->
<!-- Species: Human (Homo sapiens) [TaxId: 9606] [50519] -->
<!-- Searching for "Subtilisin" returns the protein, "Subtilisin from Bacillus subtilis, carlsberg", with the following lineage. -->
<!-- Subtilisin from Bacillus subtilis, carlsberg lineage -->
<!-- Root: scop -->
<!-- Class: Alpha and beta proteins (a/b) [51349] -->
<!-- Mainly parallel beta sheets (beta-alpha-beta units) -->
<!-- Fold: Subtilisin-like [52742] -->
<!-- 3 layers: a/b/a, parallel beta-sheet of 7 strands, order 2314567; left-handed crossover connection between strands 2 & 3 -->
<!-- Superfamily: Subtilisin-like [52743] -->
<!-- Family: Subtilases [52744] -->
<!-- Protein: Subtilisin [52745] -->
<!-- Species: Bacillus subtilis, carlsberg [TaxId: 1423] [52746] -->
<!-- Although both of these proteins are proteases, they do not even belong to the same fold, which is consistent with them being an example of convergent evolution. -->
<!-- ### Comparison to other classification systems -->
<!-- SCOP classification is more dependent on manual decisions than the semi-automatic classification by CATH, its chief rival. Human expertise is used to decide whether certain proteins are evolutionary related and therefore should be assigned to the same superfamily, or their similarity is a result of structural constraints and therefore they belong to the same fold. Another database, FSSP, is purely automatically generated (including regular automatic updates) but offers no classification, allowing the user to draw their own conclusion as to the significance of structural relationships based on the pairwise comparisons of individual protein structures. -->
<!-- ### SCOP successors -->
<!-- By 2009, the original SCOP database manually classified 38,000 PDB entries into a strictly hierarchical structure. With the accelerating pace of protein structure publications, the limited automation of classification could not keep up, leading to a non-comprehensive dataset. The Structural Classification of Proteins extended (SCOPe) database was released in 2012 with far greater automation of the same hierarchical system and is full backwards compatible with SCOP version 1.75. In 2014, manual curation was reintroduced into SCOPe to maintain accurate structure assignment. As of February 2015, SCOPe 2.05 classified 71,000 of the 110,000 total PDB entries.[11] -->
<!-- SCOP2 prototype was a beta version of Structural classification of proteins and classification system that aimed to more the evolutionary complexity inherent in protein structure evolution.[12] It is therefore not a simple hierarchy, but a directed acyclic graph network connecting protein superfamilies representing structural and evolutionary relationships such as circular permutations, domain fusion and domain decay. Consequently, domains are not separated by strict fixed boundaries, but rather are defined by their relationships to the most similar other structures. The prototype was used for the development of the SCOP version 2 database.[7] The SCOP version 2, release January 2020, contains 5134 families and 2485 superfamilies compared to 3902 families and 1962 superfamilies in SCOP 1.75. The classification levels organise more than 41 000 non-redundant domains that represent more than 504 000 protein structures. -->
<!-- The Evolutionary Classification of Protein Domains (ECOD) database released in 2014 is a similar to SCOPe expansion of SCOP version 1.75. Unlike the compatible SCOPe, it renames the class-fold-superfamily-family hierarchy into an architecture-X-homology-topology-family (A-XHTF) grouping, with the last level mostly defined by Pfam and supplemented by HHsearch clustering for uncategorized sequences.[13] ECOD has the best PDB coverage of all three successors: it covers every PDB structure, and is updated biweekly.[14] The direct mapping to Pfam has proven useful to Pfam curators who use the homology-level category to supplement their "clan" grouping.[15] -->
<!-- See also -->
<p><!-- Structural alignment -->
<!-- CATH -->
<!-- FSSP -->
<!-- SUPERFAMILY -->
<!-- Pfam --></p>

</div>
</div>
<div id="sequence-alignment" class="section level2" number="17.2">
<h2><span class="header-section-number">17.2</span> Sequence alignment</h2>
<p>Adapated from Wikipedia
<a href="https://en.wikipedia.org/wiki/Sequence_alignment" class="uri">https://en.wikipedia.org/wiki/Sequence_alignment</a></p>
<p>In bioinformatics, a sequence alignment is a way of arranging the sequences of DNA, RNA, or protein to identify regions of similarity that may be a consequence of functional, structural, or evolutionary relationships between the sequences (1). Aligned sequences of nucleotide or amino acid residues are typically represented as rows within a grid or matrix. Gaps are inserted between the residues so that identical or similar characters are aligned in successive columns.</p>
<!-- A sequence alignment, produced by ClustalO, of mammalian histone proteins. -->
<!-- Sequences are the amino acids for residues 120-180 of the proteins. Residues that are conserved across all sequences are highlighted in grey. Below the protein sequences is a key denoting conserved sequence (*), conservative mutations (:), semi-conservative mutations (.), and non-conservative mutations ( ).[2] -->
<div id="interpretation" class="section level3" number="17.2.1">
<h3><span class="header-section-number">17.2.1</span> Interpretation</h3>
<p>If two sequences in an alignment share a common ancestor, mismatches can be interpreted as point mutations and gaps as indels (that is, insertion or deletion mutations) introduced in one or both lineages in the time since they diverged from one another. In sequence alignments of proteins, the degree of similarity between amino acids occupying a particular position in the sequence can be interpreted as a rough measure of how conserved a particular region or sequence motif is among lineages. The absence of substitutions, or the presence of only very conservative substitutions (that is, the substitution of amino acids whose side chains have similar biochemical properties) in a particular region of the sequence, suggest (3) that this region has structural or functional importance. Although DNA and RNA nucleotide bases are more similar to each other than are amino acids, the conservation of base pairs can indicate a similar functional or structural role.</p>
</div>
<div id="alignment-methods" class="section level3" number="17.2.2">
<h3><span class="header-section-number">17.2.2</span> Alignment methods</h3>
<p>Very short or very similar sequences can be aligned by hand. However, most interesting problems require the alignment of lengthy, highly variable or extremely numerous sequences that cannot be aligned solely by human effort. Instead, human knowledge is applied in constructing algorithms to produce high-quality sequence alignments, and occasionally in adjusting the final results to reflect patterns that are difficult to represent algorithmically (especially in the case of nucleotide sequences).</p>
<p>Computational approaches to sequence alignment generally fall into two categories: <strong>global alignments</strong> and <strong>local alignments</strong>. Calculating a global alignment is a form of global optimization that “forces” the alignment to span the <em>entire</em> length of all sequences. By contrast, local alignments identify regions of similarity within long sequences that are often widely divergent overall. Local alignments are often preferable, but can be more difficult to calculate because of the additional challenge of identifying isolated regions of high similarity (4). BLAST searches are an example of local alignment.</p>
</div>
</div>
<div id="representations" class="section level2" number="17.3">
<h2><span class="header-section-number">17.3</span> Representations</h2>
<p>Alignments are commonly represented both graphically and in text format. In almost all sequence alignment representations, sequences are written in rows arranged so that aligned residues appear in successive columns.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="s.html#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggmsa)</span></code></pre></div>
<pre><code>## Registered S3 methods overwritten by &#39;ggalt&#39;:
##   method                  from   
##   grid.draw.absoluteGrob  ggplot2
##   grobHeight.absoluteGrob ggplot2
##   grobWidth.absoluteGrob  ggplot2
##   grobX.absoluteGrob      ggplot2
##   grobY.absoluteGrob      ggplot2</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="s.html#cb4-1" aria-hidden="true" tabindex="-1"></a>fasta <span class="ot">&lt;-</span> <span class="fu">system.file</span>(<span class="st">&quot;extdata&quot;</span>, <span class="st">&quot;sample.fasta&quot;</span>, <span class="at">package =</span> <span class="st">&quot;ggmsa&quot;</span>)</span>
<span id="cb4-2"><a href="s.html#cb4-2" aria-hidden="true" tabindex="-1"></a><span class="fu">ggmsa</span>(fasta, <span class="dv">164</span>, <span class="dv">213</span>, <span class="at">color=</span><span class="st">&quot;Chemistry_AA&quot;</span>, <span class="at">none_bg =</span> T,</span>
<span id="cb4-3"><a href="s.html#cb4-3" aria-hidden="true" tabindex="-1"></a>      <span class="at">consensus_views =</span> T, </span>
<span id="cb4-4"><a href="s.html#cb4-4" aria-hidden="true" tabindex="-1"></a>      <span class="at">use_dot =</span> T)</span></code></pre></div>
<p><img src="openbio_files/figure-html/unnamed-chunk-8-1.png" width="672" />
Figure 1: Dots * for concensus</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="s.html#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggmsa)</span>
<span id="cb5-2"><a href="s.html#cb5-2" aria-hidden="true" tabindex="-1"></a>fasta <span class="ot">&lt;-</span> <span class="fu">system.file</span>(<span class="st">&quot;extdata&quot;</span>, <span class="st">&quot;sample.fasta&quot;</span>, <span class="at">package =</span> <span class="st">&quot;ggmsa&quot;</span>)</span>
<span id="cb5-3"><a href="s.html#cb5-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggmsa</span>(fasta, <span class="dv">164</span>, <span class="dv">213</span>, <span class="at">color=</span><span class="st">&quot;Chemistry_AA&quot;</span>, <span class="at">none_bg =</span> T,</span>
<span id="cb5-4"><a href="s.html#cb5-4" aria-hidden="true" tabindex="-1"></a>      <span class="at">consensus_views =</span> T, </span>
<span id="cb5-5"><a href="s.html#cb5-5" aria-hidden="true" tabindex="-1"></a>      <span class="at">use_dot =</span> F)</span></code></pre></div>
<p><img src="openbio_files/figure-html/unnamed-chunk-9-1.png" width="672" />
Figure 2: Pipes (|) for consensus</p>
<p>In text formats, aligned columns containing identical or similar characters are indicated with a system of conservation symbols. In the image above, an asterisk (*) is used to show identity between two columns. Some programs use the pipe symbol (|). Other symbols that may be used include a colon for <strong>conservative substitutions</strong> and a period for <strong>semiconservative substitutions</strong>.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="s.html#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggmsa)</span>
<span id="cb6-2"><a href="s.html#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="co"># fasta &lt;- system.file(&quot;extdata&quot;, &quot;sample.fasta&quot;, package = &quot;ggmsa&quot;)</span></span>
<span id="cb6-3"><a href="s.html#cb6-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggmsa</span>(fasta, <span class="dv">164</span>, <span class="dv">213</span>, <span class="at">color=</span><span class="st">&quot;Chemistry_AA&quot;</span></span>
<span id="cb6-4"><a href="s.html#cb6-4" aria-hidden="true" tabindex="-1"></a>      <span class="co">#none_bg = T</span></span>
<span id="cb6-5"><a href="s.html#cb6-5" aria-hidden="true" tabindex="-1"></a>      )</span></code></pre></div>
<p><img src="openbio_files/figure-html/unnamed-chunk-10-1.png" width="672" /></p>
<p>Many sequence visualization programs also use color to display information about the properties of the individual sequence elements; in DNA and RNA sequences, this equates to assigning each nucleotide its own color. In protein alignments, such as the one in the image below, color is often used to indicate amino acid properties to aid in judging the conservation of a given amino acid substitution. For multiple sequences the last row in each column is often the consensus sequence determined by the alignment; the consensus sequence is also often represented in graphical format with a <strong>sequence logo</strong> in which the size of each nucleotide or amino acid letter corresponds to its degree of conservation (5).</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="s.html#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggseqlogo)</span>
<span id="cb7-2"><a href="s.html#cb7-2" aria-hidden="true" tabindex="-1"></a><span class="fu">data</span>(ggseqlogo_sample)</span>
<span id="cb7-3"><a href="s.html#cb7-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggseqlogo</span>( seqs_dna[[<span class="dv">1</span>]] )</span></code></pre></div>
<pre><code>## Warning: `guides(&lt;scale&gt; = FALSE)` is deprecated. Please use `guides(&lt;scale&gt; =
## &quot;none&quot;)` instead.</code></pre>
<p><img src="openbio_files/figure-html/unnamed-chunk-11-1.png" width="672" />
Figure 3: Sequence log</p>
<p><strong>This section is optional</strong>:</p>
<p>Sequence alignments can be stored in a wide variety of text-based file formats, many of which were originally developed in conjunction with a specific alignment program or implementation. Most web-based tools allow a limited number of input and output formats, such as FASTA format and GenBank format and the output is not easily editable. Several conversion programs that provide graphical and/or command line interfaces are available, such as READSEQ and EMBOSS. There are also several programming packages which provide this conversion functionality, such as BioPython, BioRuby and BioPerl. The SAM/BAM files use the CIGAR (Compact Idiosyncratic Gapped Alignment Report) string format to represent an alignment of a sequence to a reference by encoding a sequence of events (e.g. match/mismatch, insertions, deletions).[6]</p>
<!-- CIGAR Format -->
<!-- Ref.  : GTCGTAGAATA -->
<!-- Read: CACGTAG—TA -->
<!-- CIGAR: 2S5M2D2M where: -->
<!-- 2S = 2 soft clipping (could be mismatches, or a read longer than the matched sequence) -->
<!-- 5M = 5 matches or mismatches -->
<!-- 2D = 2 deletions -->
<!-- 2M = 2 matches or mismatches -->
<!-- The original CIGAR format from the exonerate alignment program did not distinguish between mismatches or matches with the M character. -->
<!-- The SAMv1 spec document defines newer CIGAR codes. In most cases it is preferred to use the '=' and 'X' characters to denote matches or mismatches rather than the older 'M' character, which is ambiguous. -->
<!-- CIGAR Code BAM Integer Description Consumes query  Consumes reference -->
<!-- M  0   alignment match (can be a sequence match or mismatch)   yes yes -->
<!-- I  1   insertion to the reference  yes no -->
<!-- D  2   deletion from the reference no  yes -->
<!-- N  3   skipped region from the reference   no  yes -->
<!-- S  4   soft clipping (clipped sequences present in SEQ)    yes no -->
<!-- H  5   hard clipping (clipped sequences NOT present in SEQ)    no  no -->
<!-- P  6   padding (silent deletion from padded reference) no  no -->
<!-- =  7   sequence match  yes yes -->
<!-- X  8   sequence mismatch   yes yes -->
<!-- “Consumes query” and “consumes reference” indicate whether the CIGAR operation causes the alignment to step along the query sequence and the reference sequence respectively. -->
<!-- H can only be present as the first and/or last operation. -->
<!-- S may only have H operations between them and the ends of the CIGAR string. -->
<!-- For mRNA-to-genome alignment, an N operation represents an intron. For other types of alignments, the interpretation of N is not defined. -->
<!-- Sum of lengths of the M/I/S/=/X operations shall equal the length of SEQ -->
<div id="global-and-local-alignments" class="section level3" number="17.3.1">
<h3><span class="header-section-number">17.3.1</span> Global and local alignments</h3>
<p>Global alignments, which attempt to align every residue in every sequence, are most useful when the sequences in the query set are similar and of roughly equal size. (This does not mean global alignments cannot start and/or end in gaps.) A general global alignment technique is the <strong>Needleman–Wunsch algorithm</strong>, which is based on an approach known as dynamic programming. Local alignments are more useful for dissimilar sequences that are suspected to contain regions of similarity or similar sequence motifs within their larger sequence context. The <strong>Smith–Waterman algorithm</strong> is a general local alignment method based on the same dynamic programming scheme but with additional choices to start and end at any place (4).</p>
<!-- Hybrid methods, known as semi-global or "glocal" (short for global-local) methods, search for the best possible partial alignment of the two sequences (in other words, a combination of one or both starts and one or both ends is stated to be aligned). This can be especially useful when the downstream part of one sequence overlaps with the upstream part of the other sequence. In this case, neither global nor local alignment is entirely appropriate: a global alignment would attempt to force the alignment to extend beyond the region of overlap, while a local alignment might not fully cover the region of overlap.[7] Another case where semi-global alignment is useful is when one sequence is short (for example a gene sequence) and the other is very long (for example a chromosome sequence). In that case, the short sequence should be globally (fully) aligned but only a local (partial) alignment is desired for the long sequence. -->
<!-- Fast expansion of genetic data challenges speed of current DNA sequence alignment algorithms. Essential needs for an efficient and accurate method for DNA variant discovery demand innovative approaches for parallel processing in real time. Optical computing approaches have been suggested as promising alternatives to the current electrical implementations, yet their applicability remains to be tested [1]. -->
</div>
<div id="pairwise-alignment" class="section level3" number="17.3.2">
<h3><span class="header-section-number">17.3.2</span> Pairwise alignment</h3>
<p>Pairwise sequence alignment methods are used to find the best-matching piecewise (local or global) alignments of two query sequences. Pairwise alignments can only be used between two sequences at a time, but they are efficient to calculate and are often used for methods that do not require extreme precision (such as searching a database for sequences with high similarity to a query).</p>
<p>The three primary methods of producing pairwise alignments are (1)</p>
<ol style="list-style-type: decimal">
<li>graphic dot-matrix methods</li>
<li>dynamic programming,</li>
<li>and word methods</li>
</ol>
<p>Multiple sequence alignment techniques can also begin by aligning pairs of sequences. Although each method has its individual strengths and weaknesses, all three pairwise methods have difficulty with highly repetitive sequences of low information content - especially where the number of repetitions differ in the two sequences to be aligned.</p>
<!-- ## Maximal unique match -->
<!-- One way of quantifying the utility of a given pairwise alignment is the 'maximal unique match' (MUM), or the longest subsequence that occurs in both query sequences. Longer MUM sequences typically reflect closer relatedness.[8] in the multiple sequence alignment of genomes in computational biology. Identification of MUMs and other potential anchors, is the first step in larger alignment systems such as MUMmer. Anchors are the areas between two genomes where they are highly similar. To understand what a MUM is we can break down each word in the acronym. Match implies that the substring occurs in both sequences to be aligned. Unique means that the substring occurs only once in each sequence. Finally, maximal states that the substring is not part of another larger string that fulfills both prior requirements. The idea behind this, is that long sequences that match exactly and occur only once in each genome are almost certainly part of the global alignment. -->
<!-- More precisely: -->
<!-- "Given two genomes A and B, Maximal Unique Match (MUM) substring is a common substring of A and B of length longer than a specified minimum length d (by default d= 20) such that -->
<!-- it is maximal, that is, it cannot be extended on either end without incurring a mismatch; and -->
<!-- it is unique in both sequences"(9) -->
</div>
<div id="dot-matrix-methods" class="section level3" number="17.3.3">
<h3><span class="header-section-number">17.3.3</span> Dot-matrix methods</h3>
<!-- Self comparison of a part of a mouse strain genome. The dot-plot shows a patchwork of lines, demonstrating duplicated segments of DNA. -->
<!-- A DNA dot plot of a human zinc finger transcription factor (GenBank ID NM_002383), showing regional self-similarity. The main diagonal represents the sequence's alignment with itself; lines off the main diagonal represent similar or repetitive patterns within the sequence. This is a typical example of a recurrence plot. -->
<p>The dot-matrix approach, which implicitly produces a family of alignments for individual sequence regions, is qualitative and conceptually simple, though time-consuming to analyze on a large scale. In the absence of noise, it can be easy to visually identify certain sequence features—such as insertions, deletions, repeats, or inverted repeats—from a dot-matrix plot. To construct a dot-matrix plot, the two sequences are written along the top row and leftmost column of a two-dimensional matrix and a dot is placed at any point where the characters in the appropriate columns match—this is a typical recurrence plot. Some implementations vary the size or intensity of the dot depending on the degree of similarity of the two characters, to accommodate conservative substitutions. The dot plots of very closely related sequences will appear as a single line along the matrix’s main diagonal.</p>
<p>Problems with dot plots as an information display technique include: noise, lack of clarity, non-intuitiveness, difficulty extracting match summary statistics and match positions on the two sequences. There is also much wasted space where the match data is inherently duplicated across the diagonal and most of the actual area of the plot is taken up by either empty space or noise, and, finally, dot-plots are limited to two sequences. None of these limitations apply to Miropeats alignment diagrams but they have their own particular flaws.</p>
<p>Dot plots can also be used to assess repetitiveness in a single sequence. A sequence can be plotted against itself and regions that share significant similarities will appear as lines off the main diagonal. This effect can occur when a protein consists of multiple similar structural domains.</p>
</div>
<div id="dynamic-programming" class="section level3" number="17.3.4">
<h3><span class="header-section-number">17.3.4</span> Dynamic programming</h3>
<p>The technique of “dynamic programming” can be applied to produce global alignments via the <strong>Needleman-Wunsch algorithm</strong>, and local alignments via the <strong>Smith-Waterman algorithm</strong>. In typical usage, protein alignments use a <strong>substitution matrix</strong> to assign scores to amino-acid <strong>matches</strong> or <strong>mismatches</strong>, and a <strong>gap penalty</strong> for matching an amino acid in one sequence to a gap in the other.</p>
<p>Substitution matrices are symmetric matrices that assign high scores to amino acids that are more chemically similar or which substitutions are most common. Low values are assigned to chemically dissimilar amino acids / unlikely substitutions.</p>
<!-- (In standard dynamic programming, the score of each amino acid position is independent of the identity of its neighbors, and therefore base stacking effects are not taken into account. However, it is possible to account for such effects by modifying the algorithm.) -->
<pre><code>##    A  R  N  D  C  Q  E  G  H  I  L  K  M  F  P  S  T  W  Y  V  B  J
## A  5 NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA
## R -2  7 NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA
## N -1  0  6 NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA
## D -2 -1  2  7 NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA
## C -1 -3 -2 -3 12 NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA
## Q -1  1  0  0 -3  6 NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA
## E -1  0  0  2 -3  2  6 NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA
## G  0 -2  0 -1 -3 -2 -2  7 NA NA NA NA NA NA NA NA NA NA NA NA NA NA
## H -2  0  1  0 -3  1  0 -2 10 NA NA NA NA NA NA NA NA NA NA NA NA NA
## I -1 -3 -2 -4 -3 -2 -3 -4 -3  5 NA NA NA NA NA NA NA NA NA NA NA NA
## L -1 -2 -3 -3 -2 -2 -2 -3 -2  2  5 NA NA NA NA NA NA NA NA NA NA NA
## K -1  3  0  0 -3  1  1 -2 -1 -3 -3  5 NA NA NA NA NA NA NA NA NA NA
## M -1 -1 -2 -3 -2  0 -2 -2  0  2  2 -1  6 NA NA NA NA NA NA NA NA NA
## F -2 -2 -2 -4 -2 -4 -3 -3 -2  0  1 -3  0  8 NA NA NA NA NA NA NA NA
## P -1 -2 -2 -1 -4 -1  0 -2 -2 -2 -3 -1 -2 -3  9 NA NA NA NA NA NA NA
## S  1 -1  1  0 -1  0  0  0 -1 -2 -3 -1 -2 -2 -1  4 NA NA NA NA NA NA
## T  0 -1  0 -1 -1 -1 -1 -2 -2 -1 -1 -1 -1 -1 -1  2  5 NA NA NA NA NA
## W -2 -2 -4 -4 -5 -2 -3 -2 -3 -2 -2 -2 -2  1 -3 -4 -3 15 NA NA NA NA
## Y -2 -1 -2 -2 -3 -1 -2 -3  2  0  0 -1  0  3 -3 -2 -1  3  8 NA NA NA
## V  0 -2 -3 -3 -1 -3 -3 -3 -3  3  1 -2  1  0 -3 -1  0 -3 -1  5 NA NA
## B -1 -1  5  6 -2  0  1 -1  0 -3 -3  0 -2 -3 -2  0  0 -4 -2 -3  5 NA
## J -1 -3 -3 -3 -2 -2 -3 -4 -2  4  4 -3  2  1 -3 -2 -1 -2  0  2 -3  4</code></pre>
<p>Figure 1: The BLOSUM45 substitution matrix</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="s.html#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="fu">nucleotideSubstitutionMatrix</span>(<span class="at">match =</span> <span class="dv">1</span>, <span class="at">mismatch =</span> <span class="sc">-</span><span class="dv">1</span>, <span class="at">baseOnly =</span> T, <span class="at">type =</span> <span class="st">&quot;DNA&quot;</span>)</span></code></pre></div>
<pre><code>##    A  C  G  T
## A  1 -1 -1 -1
## C -1  1 -1 -1
## G -1 -1  1 -1
## T -1 -1 -1  1</code></pre>
<p>DNA and RNA alignments may use a <strong>scoring matrix</strong>, but in practice often simply assign a positive match score when both sequences have the same base, a negative mismatch score when they are different, and a negative gap penalty.</p>
<p>A common extension to standard linear gap costs, is the usage of two different gap penalties for <strong>opening</strong> a gap and for <strong>extending</strong> a gap. Typically the former is much larger than the latter, e.g. -10 for gap open and -2 for gap extension. Thus, the number of gaps in an alignment is usually reduced and residues and gaps are kept together, which typically makes more biological sense.</p>
<!-- The Gotoh algorithm implements affine gap costs by using three matrices. -->
<!-- Dynamic programming can be useful in aligning nucleotide to protein sequences, a task complicated by the need to take into account frameshift mutations (usually insertions or deletions). The framesearch method produces a series of global or local pairwise alignments between a query nucleotide sequence and a search set of protein sequences, or vice versa. Its ability to evaluate frameshifts offset by an arbitrary number of nucleotides makes the method useful for sequences containing large numbers of indels, which can be very difficult to align with more efficient heuristic methods. In practice, the method requires large amounts of computing power or a system whose architecture is specialized for dynamic programming. The BLAST and EMBOSS suites provide basic tools for creating translated alignments (though some of these approaches take advantage of side-effects of sequence searching capabilities of the tools). More general methods are available from open-source software such as GeneWise. -->
<!-- The dynamic programming method is guaranteed to find an optimal alignment given a particular scoring function; however, identifying a good scoring function is often an empirical rather than a theoretical matter. Although dynamic programming is extensible to more than two sequences, it is prohibitively slow for large numbers of sequences or extremely long sequences. -->
</div>
<div id="word-methods" class="section level3" number="17.3.5">
<h3><span class="header-section-number">17.3.5</span> Word methods</h3>
<p><strong>OPTIONAL</strong></p>
<p>Word methods are approximate methods that are not guaranteed to find an optimal alignment solution, but are significantly more efficient than dynamic programming. These methods are especially useful in large-scale <strong>database searches</strong> where it is understood that a large proportion of the candidate sequences will have essentially no significant match with the query sequence.</p>
<p>Word methods are best known for their implementation in the database search tools FASTA and the <strong>BLAST</strong> family (1). Word methods identify a series of short, nonoverlapping <strong>subsequences</strong> (“words”) in the query sequence that are then matched to candidate database sequences. The relative positions of the word in the two sequences being compared are subtracted to obtain an offset; this will indicate a region of alignment if multiple distinct words produce the same offset. Only if this region is detected do these methods apply more sensitive alignment criteria; thus, many unnecessary comparisons with sequences of no appreciable similarity are eliminated.</p>
<!-- In the FASTA method, the user defines a value k to use as the word length with which to search the database. The method is slower but more sensitive at lower values of k, which are also preferred for searches involving a very short query sequence. The BLAST family of search methods provides a number of algorithms optimized for particular types of queries, such as searching for distantly related sequence matches.  -->
<!-- BLAST was developed to provide a faster alternative to FASTA without sacrificing much accuracy; like FASTA,  -->
<p>BLAST uses a word search of length k and evaluates only the most significant word matches. Most BLAST implementations use a fixed default word length that is optimized for the query and database type, and that is changed only under special circumstances, such as when searching with repetitive or very short query sequences. Implementations can be found via a number of web portals, such as EMBL FASTA and NCBI BLAST.</p>
</div>
<div id="multiple-sequence-alignment" class="section level3" number="17.3.6">
<h3><span class="header-section-number">17.3.6</span> Multiple sequence alignment</h3>
<!-- Alignment of 27 avian influenza hemagglutinin protein sequences colored by residue conservation (top) and residue properties (bottom) -->
<p>Multiple sequence alignment (MSA) is an extension of pairwise alignment to incorporate more than two sequences at a time. Multiple alignment methods try to align all of the sequences in a given query set. Multiple alignments are often used in identifying <strong>conserved</strong> sequence regions across a group of sequences hypothesized to be evolutionarily related. Such conserved sequence <strong>motifs</strong> can be used in conjunction with structural and mechanistic information to locate the catalytic active sites of enzymes. MSA therefore occur regularly in molecular biology and biochemistry publications.</p>
<p>Alignments are also used to aid in establishing evolutionary relationships by constructing phylogenetic trees. Multiple sequence alignments are computationally difficult to produce (most formulations of the problem lead to NP-complete combinatorial optimization problems; 10, 11). Nevertheless, the utility of these alignments in bioinformatics has led to the development of a variety of methods suitable for aligning three or more sequences.</p>
<!-- ## Dynamic programming for mSA -->
<!-- The technique of dynamic programming is theoretically applicable to any number of sequences; however, because it is computationally expensive in both time and memory, it is rarely used for more than three or four sequences in its most basic form. This method requires constructing the n-dimensional equivalent of the sequence matrix formed from two sequences, where n is the number of sequences in the query. Standard dynamic programming is first used on all pairs of query sequences and then the "alignment space" is filled in by considering possible matches or gaps at intermediate positions, eventually constructing an alignment essentially between each two-sequence alignment. Although this technique is computationally expensive, its guarantee of a global optimum solution is useful in cases where only a few sequences need to be aligned accurately. One method for reducing the computational demands of dynamic programming, which relies on the "sum of pairs" objective function, has been implemented in the MSA software package.[12] -->
<!-- Progressive methods -->
<!-- Progressive, hierarchical, or tree methods generate a multiple sequence alignment by first aligning the most similar sequences and then adding successively less related sequences or groups to the alignment until the entire query set has been incorporated into the solution. The initial tree describing the sequence relatedness is based on pairwise comparisons that may include heuristic pairwise alignment methods similar to FASTA. Progressive alignment results are dependent on the choice of "most related" sequences and thus can be sensitive to inaccuracies in the initial pairwise alignments. Most progressive multiple sequence alignment methods additionally weight the sequences in the query set according to their relatedness, which reduces the likelihood of making a poor choice of initial sequences and thus improves alignment accuracy. -->
<!-- Many variations of the Clustal progressive implementation[13][14][15] are used for multiple sequence alignment, phylogenetic tree construction, and as input for protein structure prediction. A slower but more accurate variant of the progressive method is known as T-Coffee.[16] -->
<!-- Iterative methods -->
<!-- Iterative methods attempt to improve on the heavy dependence on the accuracy of the initial pairwise alignments, which is the weak point of the progressive methods. Iterative methods optimize an objective function based on a selected alignment scoring method by assigning an initial global alignment and then realigning sequence subsets. The realigned subsets are then themselves aligned to produce the next iteration's multiple sequence alignment. Various ways of selecting the sequence subgroups and objective function are reviewed in.[17] -->
<!-- Motif finding -->
<!-- Motif finding, also known as profile analysis, constructs global multiple sequence alignments that attempt to align short conserved sequence motifs among the sequences in the query set. This is usually done by first constructing a general global multiple sequence alignment, after which the highly conserved regions are isolated and used to construct a set of profile matrices. The profile matrix for each conserved region is arranged like a scoring matrix but its frequency counts for each amino acid or nucleotide at each position are derived from the conserved region's character distribution rather than from a more general empirical distribution. The profile matrices are then used to search other sequences for occurrences of the motif they characterize. In cases where the original data set contained a small number of sequences, or only highly related sequences, pseudocounts are added to normalize the character distributions represented in the motif. -->
<!-- Techniques inspired by computer science -->
<!-- A profile HMM modelling a multiple sequence alignment -->
<!-- A variety of general optimization algorithms commonly used in computer science have also been applied to the multiple sequence alignment problem. Hidden Markov models have been used to produce probability scores for a family of possible multiple sequence alignments for a given query set; although early HMM-based methods produced underwhelming performance, later applications have found them especially effective in detecting remotely related sequences because they are less susceptible to noise created by conservative or semiconservative substitutions.[18] Genetic algorithms and simulated annealing have also been used in optimizing multiple sequence alignment scores as judged by a scoring function like the sum-of-pairs method. More complete details and software packages can be found in the main article multiple sequence alignment. -->
<!-- The Burrows–Wheeler transform has been successfully applied to fast short read alignment in popular tools such as Bowtie and BWA. See FM-index. -->
<!-- Structural alignment -->
<!-- Main article: Structural alignment -->
<!-- Structural alignments, which are usually specific to protein and sometimes RNA sequences, use information about the secondary and tertiary structure of the protein or RNA molecule to aid in aligning the sequences. These methods can be used for two or more sequences and typically produce local alignments; however, because they depend on the availability of structural information, they can only be used for sequences whose corresponding structures are known (usually through X-ray crystallography or NMR spectroscopy). Because both protein and RNA structure is more evolutionarily conserved than sequence,[19] structural alignments can be more reliable between sequences that are very distantly related and that have diverged so extensively that sequence comparison cannot reliably detect their similarity. -->
<!-- Structural alignments are used as the "gold standard" in evaluating alignments for homology-based protein structure prediction[20] because they explicitly align regions of the protein sequence that are structurally similar rather than relying exclusively on sequence information. However, clearly structural alignments cannot be used in structure prediction because at least one sequence in the query set is the target to be modeled, for which the structure is not known. It has been shown that, given the structural alignment between a target and a template sequence, highly accurate models of the target protein sequence can be produced; a major stumbling block in homology-based structure prediction is the production of structurally accurate alignments given only sequence information.[20] -->
<!-- DALI -->
<!-- The DALI method, or distance matrix alignment, is a fragment-based method for constructing structural alignments based on contact similarity patterns between successive hexapeptides in the query sequences.[21] It can generate pairwise or multiple alignments and identify a query sequence's structural neighbors in the Protein Data Bank (PDB). It has been used to construct the FSSP structural alignment database (Fold classification based on Structure-Structure alignment of Proteins, or Families of Structurally Similar Proteins). A DALI webserver can be accessed at DALI and the FSSP is located at The Dali Database. -->
<!-- SSAP -->
<!-- SSAP (sequential structure alignment program) is a dynamic programming-based method of structural alignment that uses atom-to-atom vectors in structure space as comparison points. It has been extended since its original description to include multiple as well as pairwise alignments,[22] and has been used in the construction of the CATH (Class, Architecture, Topology, Homology) hierarchical database classification of protein folds.[23] The CATH database can be accessed at CATH Protein Structure Classification. -->
<!-- Combinatorial extension -->
<!-- The combinatorial extension method of structural alignment generates a pairwise structural alignment by using local geometry to align short fragments of the two proteins being analyzed and then assembles these fragments into a larger alignment.[24] Based on measures such as rigid-body root mean square distance, residue distances, local secondary structure, and surrounding environmental features such as residue neighbor hydrophobicity, local alignments called "aligned fragment pairs" are generated and used to build a similarity matrix representing all possible structural alignments within predefined cutoff criteria. A path from one protein structure state to the other is then traced through the matrix by extending the growing alignment one fragment at a time. The optimal such path defines the combinatorial-extension alignment. A web-based server implementing the method and providing a database of pairwise alignments of structures in the Protein Data Bank is located at the Combinatorial Extension website. -->
</div>
<div id="phylogenetic-analysis" class="section level3" number="17.3.7">
<h3><span class="header-section-number">17.3.7</span> Phylogenetic analysis</h3>
<p>Phylogenetics and sequence alignment are closely related fields due to the shared necessity of evaluating sequence relatedness (25). The field of phylogenetics makes extensive use of sequence alignments in the construction and interpretation of phylogenetic trees, which are used to classify the evolutionary relationships between homologous genes represented in the genomes of divergent species. The degree to which sequences in a query set differ is qualitatively related to the sequences’ <strong>evolutionary distance</strong> from one another. Roughly speaking, high <strong>sequence identity</strong> suggests that the sequences in question have a comparatively young <strong>most recent common ancestor</strong>, while low identity suggests that the divergence is more ancient.</p>
<p><strong>OPTIONAL</strong></p>
<p>This approximation, which reflects the “molecular clock” hypothesis that a roughly constant rate of evolutionary change can be used to extrapolate the elapsed time since two genes first diverged (that is, the coalescence time), assumes that the effects of mutation and selection are constant across sequence lineages. Therefore, it does not account for possible difference species in the possible functional conservation of specific regions in a sequence. (In the case of nucleotide sequences, the molecular clock hypothesis in its most basic form also discounts the difference in acceptance rates between silent mutations that do not alter the meaning of a given codon and other mutations that result in a different amino acid being incorporated into the protein). More statistically accurate methods allow the evolutionary rate on each branch of the phylogenetic tree to vary, thus producing better estimates of coalescence times for genes.</p>
<p>Progressive multiple alignment techniques produce a phylogenetic tree by necessity because they incorporate sequences into the growing alignment in order of relatedness. Other techniques that assemble multiple sequence alignments and phylogenetic trees score and sort trees first and calculate a multiple sequence alignment from the highest-scoring tree. Commonly used methods of phylogenetic tree construction are mainly heuristic because the problem of selecting the optimal tree, like the problem of selecting the optimal multiple sequence alignment, is NP-hard (26).</p>
</div>
<div id="assessment-of-significance" class="section level3" number="17.3.8">
<h3><span class="header-section-number">17.3.8</span> Assessment of significance</h3>
<p>Sequence alignments are useful in bioinformatics for identifying sequence similarity, producing phylogenetic trees, and developing homology models of protein structures. However, the biological relevance of sequence alignments is not always clear. Alignments are often assumed to reflect a degree of evolutionary change between sequences descended from a common ancestor; however, it is formally possible that <strong>convergent evolution</strong> can occur to produce apparent similarity between proteins that are evolutionarily unrelated but perform similar functions and have similar structures.</p>
<p>In database searches such as BLAST, statistical methods can determine the likelihood of a particular alignment between sequences or sequence regions arising by chance given the size and composition of the database being searched. These values can vary significantly depending on the <strong>search space</strong>. In particular, the likelihood of finding a given alignment by chance increases if the database consists only of sequences from the same organism as the query sequence. Repetitive sequences in the database or query can also distort both the search results and the assessment of statistical significance; BLAST automatically filters such repetitive sequences in the query to avoid apparent hits that are statistical artifacts.</p>
<div id="assessment-of-credibility" class="section level4" number="17.3.8.1">
<h4><span class="header-section-number">17.3.8.1</span> Assessment of credibility</h4>
<p>Statistical significance indicates the probability that an alignment of a given quality could arise by chance, but does not indicate how much superior a given alignment is to alternative alignments of the same sequences. Measures of alignment credibility indicate the extent to which the best scoring alignments for a given pair of sequences are substantially similar.</p>
</div>
<div id="scoring-functions" class="section level4" number="17.3.8.2">
<h4><span class="header-section-number">17.3.8.2</span> Scoring functions</h4>
<p>The choice of a scoring function that reflects biological or statistical observations about known sequences is important to producing good alignments. Protein sequences are frequently aligned using substitution matrices that reflect the probabilities of given character-to-character substitutions. A series of matrices called <strong>PAM matrices</strong> (Point Accepted Mutation matrices, originally defined by Margaret Dayhoff and sometimes referred to as “Dayhoff matrices”) explicitly encode evolutionary approximations regarding the rates and probabilities of particular amino acid mutations. Another common series of scoring matrices, known as <strong>BLOSUM</strong> (Blocks Substitution Matrix), encodes empirically derived substitution probabilities. Variants of both types of matrices are used to detect sequences with differing levels of divergence, thus allowing users of BLAST or FASTA to restrict searches to more closely related matches or expand to detect more divergent sequences. Gap penalties account for the introduction of a gap - on the evolutionary model, an insertion or deletion mutation - in both nucleotide and protein sequences, and therefore the penalty values should be proportional to the expected rate of such mutations. The quality of the alignments produced therefore depends on the quality of the scoring function.</p>
<p>It can be very useful and instructive to try the same alignment several times with different choices for scoring matrix and/or gap penalty values and compare the results. Regions where the solution is weak or non-unique can often be identified by observing which regions of the alignment are robust to variations in alignment parameters.</p>
</div>
</div>
<div id="other-biological-uses" class="section level3" number="17.3.9">
<h3><span class="header-section-number">17.3.9</span> Other biological uses</h3>
<p>Sequenced RNA, such as expressed sequence tags and full-length mRNAs, can be aligned to a sequenced genome to find where there are genes and get information about alternative splicing (35) and RNA editing (36). Sequence alignment is also a part of genome assembly, where sequences are aligned to find overlap so that contigs (long stretches of sequence) can be formed (37). Another use is SNP analysis, where sequences from different individuals are aligned to find single basepairs that are often different in a population (38).</p>
<!-- Non-biological uses -->
<!-- The methods used for biological sequence alignment have also found applications in other fields, most notably in natural language processing and in social sciences, where the Needleman-Wunsch algorithm is usually referred to as Optimal matching.[39] Techniques that generate the set of elements from which words will be selected in natural-language generation algorithms have borrowed multiple sequence alignment techniques from bioinformatics to produce linguistic versions of computer-generated mathematical proofs.[40] In the field of historical and comparative linguistics, sequence alignment has been used to partially automate the comparative method by which linguists traditionally reconstruct languages.[41] Business and marketing research has also applied multiple sequence alignment techniques in analyzing series of purchases over time.[42] -->
<!-- Software -->
<!-- Main article: Sequence alignment software -->
<!-- A more complete list of available software categorized by algorithm and alignment type is available at sequence alignment software, but common software tools used for general sequence alignment tasks include ClustalW2[43] and T-coffee[44] for alignment, and BLAST[45] and FASTA3x[46] for database searching. Commercial tools such as DNASTAR Lasergene, Geneious, and PatternHunter are also available. Tools annotated as performing sequence alignment are listed in the bio.tools registry. -->
<!-- Alignment algorithms and software can be directly compared to one another using a standardized set of benchmark reference multiple sequence alignments known as BAliBASE.[47] The data set consists of structural alignments, which can be considered a standard against which purely sequence-based methods are compared. The relative performance of many common alignment methods on frequently encountered alignment problems has been tabulated and selected results published online at BAliBASE.[48][49] A comprehensive list of BAliBASE scores for many (currently 12) different alignment tools can be computed within the protein workbench STRAP.[50] -->
<!-- See also -->
<!-- Sequence homology -->
<!-- Sequence mining -->
<!-- BLAST -->
<!-- String searching algorithm -->
<!-- Alignment-free sequence analysis -->
<!-- UGENE -->
<!-- Needleman–Wunsch algorithm -->
<!-- Smith-Waterman alogrithm -->
</div>
<div id="references-2" class="section level3" number="17.3.10">
<h3><span class="header-section-number">17.3.10</span> References</h3>
<ol style="list-style-type: decimal">
<li>Mount DM. (2004). Bioinformatics: Sequence and Genome Analysis (2nd ed.). Cold Spring Harbor Laboratory Press: Cold Spring Harbor, NY. ISBN 978-0-87969-608-5.</li>
<li>“Clustal FAQ Symbols”. Clustal. Archived from the original on 24 October 2016. Retrieved 8 December 2014.</li>
<li>Ng PC; Henikoff S (May 2001). “Predicting deleterious amino acid substitutions”. Genome Res. 11 (5): 863–74. <a href="doi:10.1101/gr.176601" class="uri">doi:10.1101/gr.176601</a>. PMC 311071. PMID 11337480.</li>
<li>Polyanovsky, V. O.; Roytberg, M. A.; Tumanyan, V. G. (2011). “Comparative analysis of the quality of a global algorithm and a local algorithm for alignment of two sequences”. Algorithms for Molecular Biology. 6 (1): 25. <a href="doi:10.1186/1748-7188-6-25" class="uri">doi:10.1186/1748-7188-6-25</a>. PMC 3223492. PMID 22032267. S2CID 2658261.</li>
<li>Schneider TD; Stephens RM (1990). “Sequence logos: a new way to display consensus sequences”. Nucleic Acids Res. 18 (20): 6097–6100. <a href="doi:10.1093/nar/18.20.6097" class="uri">doi:10.1093/nar/18.20.6097</a>. PMC 332411. PMID 2172928.</li>
<li>“Sequence Alignment/Map Format Specification” (PDF).</li>
<li>Brudno M; Malde S; Poliakov A; Do CB; Couronne O; Dubchak I; Batzoglou S (2003). “Glocal alignment: finding rearrangements during alignment”. Bioinformatics. 19. Suppl 1 (90001): i54–62. <a href="doi:10.1093/bioinformatics/btg1005" class="uri">doi:10.1093/bioinformatics/btg1005</a>. PMID 12855437.</li>
<li>Delcher, A. L.; Kasif, S.; Fleishmann, R.D.; Peterson, J.; White, O.; Salzberg, S.L. (1999). “Alignment of whole genomes”. Nucleic Acids Research. 27 (11): 2369–2376. <a href="doi:10.1093/nar/30.11.2478" class="uri">doi:10.1093/nar/30.11.2478</a>. PMC 148804. PMID 10325427.</li>
<li>Wing-Kin, Sung (2010). Algorithms in Bioinformatics: A Practical Introduction (First ed.). Boca Raton: Chapman &amp; Hall/CRC Press. ISBN 978-1420070330.</li>
<li>Wang L; Jiang T. (1994). “On the complexity of multiple sequence alignment”. J Comput Biol. 1 (4): 337–48. CiteSeerX 10.1.1.408.894. <a href="doi:10.1089/cmb.1994.1.337" class="uri">doi:10.1089/cmb.1994.1.337</a>. PMID 8790475.</li>
<li>Elias, Isaac (2006). “Settling the intractability of multiple alignment”. J Comput Biol. 13 (7): 1323–1339. CiteSeerX 10.1.1.6.256. <a href="doi:10.1089/cmb.2006.13.1323" class="uri">doi:10.1089/cmb.2006.13.1323</a>. PMID 17037961.</li>
<li>Lipman DJ; Altschul SF; Kececioglu JD (1989). “A tool for multiple sequence alignment”. Proc Natl Acad Sci USA. 86 (12): 4412–5. Bibcode:1989PNAS…86.4412L. <a href="doi:10.1073/pnas.86.12.4412" class="uri">doi:10.1073/pnas.86.12.4412</a>. PMC 287279. PMID 2734293.</li>
<li>Higgins DG, Sharp PM (1988). “CLUSTAL: a package for performing multiple sequence alignment on a microcomputer”. Gene. 73 (1): 237–44. <a href="doi:10.1016/0378-1119(88)90330-7" class="uri">doi:10.1016/0378-1119(88)90330-7</a>. PMID 3243435.</li>
<li>Thompson JD; Higgins DG; Gibson TJ. (1994). “CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice”. Nucleic Acids Res. 22 (22): 4673–80. <a href="doi:10.1093/nar/22.22.4673" class="uri">doi:10.1093/nar/22.22.4673</a>. PMC 308517. PMID 7984417.</li>
<li>Chenna R; Sugawara H; Koike T; Lopez R; Gibson TJ; Higgins DG; Thompson JD. (2003). “Multiple sequence alignment with the Clustal series of programs”. Nucleic Acids Res. 31 (13): 3497–500. <a href="doi:10.1093/nar/gkg500" class="uri">doi:10.1093/nar/gkg500</a>. PMC 168907. PMID 12824352.</li>
<li>Notredame C; Higgins DG; Heringa J. (2000). “T-Coffee: A novel method for fast and accurate multiple sequence alignment”. J Mol Biol. 302 (1): 205–17. <a href="doi:10.1006/jmbi.2000.4042" class="uri">doi:10.1006/jmbi.2000.4042</a>. PMID 10964570. S2CID 10189971.</li>
<li>Hirosawa M; Totoki Y; Hoshida M; Ishikawa M. (1995). “Comprehensive study on iterative algorithms of multiple sequence alignment”. Comput Appl Biosci. 11 (1): 13–8. <a href="doi:10.1093/bioinformatics/11.1.13" class="uri">doi:10.1093/bioinformatics/11.1.13</a>. PMID 7796270.</li>
<li>Karplus K; Barrett C; Hughey R. (1998). “Hidden Markov models for detecting remote protein homologies”. Bioinformatics. 14 (10): 846–856. <a href="doi:10.1093/bioinformatics/14.10.846" class="uri">doi:10.1093/bioinformatics/14.10.846</a>. PMID 9927713.</li>
<li>Chothia C; Lesk AM. (April 1986). “The relation between the divergence of sequence and structure in proteins”. EMBO J. 5 (4): 823–6. <a href="doi:10.1002/j.1460-2075.1986.tb04288.x" class="uri">doi:10.1002/j.1460-2075.1986.tb04288.x</a>. PMC 1166865. PMID 3709526.</li>
<li>Zhang Y; Skolnick J. (2005). “The protein structure prediction problem could be solved using the current PDB library”. Proc Natl Acad Sci USA. 102 (4): 1029–34. Bibcode:2005PNAS..102.1029Z. <a href="doi:10.1073/pnas.0407152101" class="uri">doi:10.1073/pnas.0407152101</a>. PMC 545829. PMID 15653774.</li>
<li>Holm L; Sander C (1996). “Mapping the protein universe”. Science. 273 (5275): 595–603. Bibcode:1996Sci…273..595H. <a href="doi:10.1126/science.273.5275.595" class="uri">doi:10.1126/science.273.5275.595</a>. PMID 8662544. S2CID 7509134.</li>
<li>Taylor WR; Flores TP; Orengo CA. (1994). “Multiple protein structure alignment”. Protein Sci. 3 (10): 1858–70. <a href="doi:10.1002/pro.5560031025" class="uri">doi:10.1002/pro.5560031025</a>. PMC 2142613. PMID 7849601.</li>
<li>Orengo CA; Michie AD; Jones S; Jones DT; Swindells MB; Thornton JM (1997). “CATH–a hierarchic classification of protein domain structures”. Structure. 5 (8): 1093–108. <a href="doi:10.1016/S0969-2126(97)00260-8" class="uri">doi:10.1016/S0969-2126(97)00260-8</a>. PMID 9309224.</li>
<li>Shindyalov IN; Bourne PE. (1998). “Protein structure alignment by incremental combinatorial extension (CE) of the optimal path”. Protein Eng. 11 (9): 739–47. <a href="doi:10.1093/protein/11.9.739" class="uri">doi:10.1093/protein/11.9.739</a>. PMID 9796821.</li>
<li>Ortet P; Bastien O (2010). “Where Does the Alignment Score Distribution Shape Come from?”. Evolutionary Bioinformatics. 6: 159–187. <a href="doi:10.4137/EBO.S5875" class="uri">doi:10.4137/EBO.S5875</a>. PMC 3023300. PMID 21258650.</li>
<li>Felsenstein J. (2004). Inferring Phylogenies. Sinauer Associates: Sunderland, MA. ISBN 978-0-87893-177-4.</li>
<li>Altschul SF; Gish W (1996). Local Alignment Statistics. Meth.Enz. Methods in Enzymology. 266. pp. 460–480. <a href="doi:10.1016/S0076-6879(96)66029-7" class="uri">doi:10.1016/S0076-6879(96)66029-7</a>. ISBN 9780121821678. PMID 8743700.</li>
<li>Hartmann AK (2002). “Sampling rare events: statistics of local sequence alignments”. Phys. Rev. E. 65 (5): 056102. arXiv:cond-mat/0108201. Bibcode:2002PhRvE..65e6102H. <a href="doi:10.1103/PhysRevE.65.056102" class="uri">doi:10.1103/PhysRevE.65.056102</a>. PMID 12059642. S2CID 193085.</li>
<li>Newberg LA (2008). “Significance of gapped sequence alignments”. J Comput Biol. 15 (9): 1187–1194. <a href="doi:10.1089/cmb.2008.0125" class="uri">doi:10.1089/cmb.2008.0125</a>. PMC 2737730. PMID 18973434.</li>
<li>Eddy SR; Rost, Burkhard (2008). Rost, Burkhard (ed.). “A probabilistic model of local sequence alignment that simplifies statistical significance estimation”. PLOS Comput Biol. 4 (5): e1000069. Bibcode:2008PLSCB…4E0069E. <a href="doi:10.1371/journal.pcbi.1000069" class="uri">doi:10.1371/journal.pcbi.1000069</a>. PMC 2396288. PMID 18516236. S2CID 15640896.</li>
<li>Bastien O; Aude JC; Roy S; Marechal E (2004). “Fundamentals of massive automatic pairwise alignments of protein sequences: theoretical significance of Z-value statistics”. Bioinformatics. 20 (4): 534–537. <a href="doi:10.1093/bioinformatics/btg440" class="uri">doi:10.1093/bioinformatics/btg440</a>. PMID 14990449.</li>
<li>Agrawal A; Huang X (2011). “Pairwise Statistical Significance of Local Sequence Alignment Using Sequence-Specific and Position-Specific Substitution Matrices”. IEEE/ACM Transactions on Computational Biology and Bioinformatics. 8 (1): 194–205. <a href="doi:10.1109/TCBB.2009.69" class="uri">doi:10.1109/TCBB.2009.69</a>. PMID 21071807. S2CID 6559731.</li>
<li>Agrawal A; Brendel VP; Huang X (2008). “Pairwise statistical significance and empirical determination of effective gap opening penalties for protein local sequence alignment”. International Journal of Computational Biology and Drug Design. 1 (4): 347–367. <a href="doi:10.1504/IJCBDD.2008.022207" class="uri">doi:10.1504/IJCBDD.2008.022207</a>. PMID 20063463. Archived from the original on 28 January 2013.</li>
<li>Newberg LA; Lawrence CE (2009). “Exact Calculation of Distributions on Integers, with Application to Sequence Alignment”. J Comput Biol. 16 (1): 1–18. <a href="doi:10.1089/cmb.2008.0137" class="uri">doi:10.1089/cmb.2008.0137</a>. PMC 2858568. PMID 19119992.</li>
<li>Kim N; Lee C (2008). Bioinformatics detection of alternative splicing. Methods in Molecular Biology. 452. pp. 179–97. <a href="doi:10.1007/978-1-60327-159-2_9" class="uri">doi:10.1007/978-1-60327-159-2_9</a>. ISBN 978-1-58829-707-5. PMID 18566765.</li>
<li>Li JB, Levanon EY, Yoon JK, et al. (May 2009). “Genome-wide identification of human RNA editing sites by parallel DNA capturing and sequencing”. Science. 324 (5931): 1210–3. Bibcode:2009Sci…324.1210L. <a href="doi:10.1126/science.1170995" class="uri">doi:10.1126/science.1170995</a>. PMID 19478186. S2CID 31148824.</li>
<li>Blazewicz J, Bryja M, Figlerowicz M, et al. (June 2009). “Whole genome assembly from 454 sequencing output via modified DNA graph concept”. Comput Biol Chem. 33 (3): 224–30. <a href="doi:10.1016/j.compbiolchem.2009.04.005" class="uri">doi:10.1016/j.compbiolchem.2009.04.005</a>. PMID 19477687.</li>
<li>Duran C; Appleby N; Vardy M; Imelfort M; Edwards D; Batley J (May 2009). “Single nucleotide polymorphism discovery in barley using autoSNPdb”. Plant Biotechnol. J. 7 (4): 326–33. <a href="doi:10.1111/j.1467-7652.2009.00407.x" class="uri">doi:10.1111/j.1467-7652.2009.00407.x</a>. PMID 19386041.</li>
<li>Abbott A.; Tsay A. (2000). “Sequence Analysis and Optimal Matching Methods in Sociology, Review and Prospect”. Sociological Methods and Research. 29 (1): 3–33. <a href="doi:10.1177/0049124100029001001" class="uri">doi:10.1177/0049124100029001001</a>. S2CID 121097811.</li>
<li>Barzilay R; Lee L. (2002). “Bootstrapping Lexical Choice via Multiple-Sequence Alignment” (PDF). Proceedings of the Conference on Empirical Methods in Natural Language Processing (EMNLP). 10: 164–171. arXiv:cs/0205065. Bibcode:2002cs……..5065B. <a href="doi:10.3115/1118693.1118715" class="uri">doi:10.3115/1118693.1118715</a>. S2CID 7521453.</li>
<li>Kondrak, Grzegorz (2002). “Algorithms for Language Reconstruction” (PDF). University of Toronto, Ontario. Archived from the original (PDF) on 17 December 2008. Retrieved 21 January 2007.</li>
<li>Prinzie A.; D. Van den Poel (2006). “Incorporating sequential information into traditional classification models by using an element/position-sensitive SAM”. Decision Support Systems. 42 (2): 508–526. <a href="doi:10.1016/j.dss.2005.02.004" class="uri">doi:10.1016/j.dss.2005.02.004</a>. See also Prinzie and Van den Poel’s paper Prinzie, A; Vandenpoel, D (2007). “Predicting home-appliance acquisition sequences: Markov/Markov for Discrimination and survival analysis for modeling sequential information in NPTB models”. Decision Support Systems. 44 (1): 28–45. <a href="doi:10.1016/j.dss.2007.02.008" class="uri">doi:10.1016/j.dss.2007.02.008</a>.</li>
<li>EMBL-EBI. “ClustalW2 &lt; Multiple Sequence Alignment &lt; EMBL-EBI”. www.EBI.ac.uk. Retrieved 12 June 2017.</li>
<li>T-coffee</li>
<li>“BLAST: Basic Local Alignment Search Tool”. blast.ncbi.nlm.NIH.gov. Retrieved 12 June 2017.</li>
<li>“UVA FASTA Server”. fasta.bioch.Virginia.edu. Retrieved 12 June 2017.</li>
<li>Thompson JD; Plewniak F; Poch O (1999). “BAliBASE: a benchmark alignment database for the evaluation of multiple alignment programs”. Bioinformatics. 15 (1): 87–8. <a href="doi:10.1093/bioinformatics/15.1.87" class="uri">doi:10.1093/bioinformatics/15.1.87</a>. PMID 10068696.</li>
<li>BAliBASE</li>
<li>Thompson JD; Plewniak F; Poch O. (1999). “A comprehensive comparison of multiple sequence alignment programs”. Nucleic Acids Res. 27 (13): 2682–90. <a href="doi:10.1093/nar/27.13.2682" class="uri">doi:10.1093/nar/27.13.2682</a>. PMC 148477. PMID 10373585.</li>
<li>“Multiple sequence alignment: Strap”. 3d-alignment.eu. Retrieved 12 June 2017.</li>
</ol>

</div>
</div>
<div id="talking-glossary-u03-shotgun-sequencing-1-min" class="section level2" number="17.4">
<h2><span class="header-section-number">17.4</span> Talking Glossary U03: Shotgun sequencing (1 min)</h2>
<p>Now just called “sequencing”</p>
<p><strong>Introduction</strong>: “Shotgun sequencing is a laboratory technique for determining the DNA sequence of an organism’s genome. The method involves breaking the genome into a collection of small DNA fragments that are sequenced individually. A computer program looks for overlaps in the DNA sequences and uses them to place the individual fragments in their correct order to reconstitute the genome.”</p>
<p><strong>Transcript</strong>: “The most efficient way to sequence a large piece of DNA involves a process known as shotgun sequencing. For this, the starting DNA is broken up randomly into many smaller pieces, sort of in a shotgun fashion, with each of those pieces then sequenced individually. The resulting sequence reads generated from the different pieces are then analyzed by a computer program, looking for stretches of sequence from different reads that are identical with one another. When identical regions are identified, they are overlapped with one another, allowing the two sequence reads to be stitched together. This computer process is repeated over and over and over again, eventually yielding the complete sequence of the starting piece of DNA. The initial random fragmenting and reading of the DNA gave this approach the name”shotgun sequencing”. ”</p>
<p>Eric D. Green, M.D., Ph.D.</p>

</div>
<div id="talking-glossary-sickle-cell-disease-4-min" class="section level2" number="17.5">
<h2><span class="header-section-number">17.5</span> Talking Glossary: Sickle Cell Disease (4 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Sickle-Cell-Disease" class="uri">https://www.genome.gov/genetics-glossary/Sickle-Cell-Disease</a></p>
<p><strong>Abstract</strong>: “Sickle cell disease is a hereditary disease seen most often among people of African ancestry. Caused by mutations in one of the genes that encode the hemoglobin protein, the disease is inherited as an autosomal recessive trait. The mutation causes the red blood cells to take on an unusual sickle shape. Individuals affected by sickle cell disease are chronically anemic and experience significant damage to their heart, lungs, and kidneys.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/sickle_cell_disease.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/sickle_cell_disease.jpg</a></p>
<p><strong>Transcript</strong>: “Sickle cell disease is the first human inherited disease that was understood at the protein and the DNA level. Sickle cell is a disease that’s primarily seen in people of African descent. In studying the genomics of people from Africa, it’s very clear now that three different times during the history of the human race the mutation in the beta-globin gene that changes an amino acid at the sixth position of the protein, the same mutation has happened. And this has been expanded throughout history in Africa in three different groups and then migrated all over the world. Now, the mutation does not really affect the ability of beta-globin to participate with alpha-globin and make hemoglobin, and it doesn’t affect the ability to carry oxygen. In matter of fact, people who inherit a hemoglobin S gene from their mother and a hemoglobin S gene from their father, so their homozygous for hemoglobin S, have perfectly normal oxygen-carrying capacity of hemoglobin. The problems happen when the red cell containing hemoglobin S gets into the muscles or into your brain and discharges the oxygen. And what happens is that hemoglobin S has a tendency when the oxygen goes off to stick to each other, and they form these polymers. Now, when they get oxygen back on, they go back into solution. The difference between hemoglobin S and wild-type hemoglobin is that the wild-type hemoglobin is soluble when it’s got oxygen, and it’s also soluble when it doesn’t have oxygen. Now, if everything is working well, these polymers in people with sickle cell disease don’t get very big. But that can be worsened when you have extreme oxygen stress, and that’s what causes the characteristic sickle cells. The polymers get very, very long, and they stretch the cell out of shape. Now your spleen is very good at raking these cells out of the peripheral blood as they come through. And that’s what causes the anemia, and the old name for sickle cell was sickle cell anemia. However, though, short polymers are very, very dangerous. If you think of a red blood cell going through your veins and arteries and capillaries as a water balloon, you’ll see how it can squish itself down into a long cylinder to get through a capillary and flatten out like a pancake to get through areas in the spleen that kind of rake out the bad-shape red blood cells, and then to form right back to a regular water balloon shape to go into the arteries and veins. But if you have a red blood cell full of these small hemoglobin S polymers, it’s like having a water balloon full of ice chips, and as that goes through the capillaries and the small veins, it tears up the lining of these things, just as if you had a cut, and it activates your clotting response and micro-clots will form. And sometimes these get to be bigger and bigger clots, and so the real lethality in sickle cell disease is not from the anemia, it’s from this vascular disease. And so it’s very unfortunate that about one-third of people who are homozygous for hemoglobin S will have one or more strokes before they’re 10 years old. And in those fortunate few that are able to get to be adolescents, these clots can build up in the lungs and give a very severe disease called acute chest syndrome, which is basically emphysema or lung destruction from these obstructions. About one-third of the patients will live healthily to adulthood, but they have many problems with iron-overloaded organs, and their life span is significantly shorter. And it is simply amazing to me that this all comes from one very small change, changes in amino acid, that really isn’t having any effect on the normal function of the protein, but has big effects on all of the other systems in the body that the red cells pass through.”</p>
<p>David M. Bodine, Ph.D.</p>

</div>
<div id="medline-snps-single-nucleotide-polymorphism" class="section level2" number="17.6">
<h2><span class="header-section-number">17.6</span> Medline: SNPs (single nucleotide polymorphism)</h2>
<p>Adapted from: Medline: “What are single nucleotide polymorphisms (SNPs)?”
<a href="https://medlineplus.gov/genetics/understanding/genomicresearch/snp/" class="uri">https://medlineplus.gov/genetics/understanding/genomicresearch/snp/</a></p>
<p>Single nucleotide polymorphisms, frequently called SNPs (pronounced “snips”), are the most common type of genetic variation among people. Each SNP represents a difference in a single DNA building block, called a nucleotide. For example, a SNP may replace the nucleotide cytosine (C) with the nucleotide thymine (T) in a certain stretch of DNA.</p>
<p>SNPs occur normally throughout a person’s DNA. They occur almost once in every 1,000 nucleotides on average, which means there are roughly 4 to 5 million SNPs in a person’s genome. These variations may be unique or occur in many individuals; scientists have found more than 100 million SNPs in populations around the world. Most commonly, these variations are found in the DNA between genes. They can act as biological markers, helping scientists locate genes that are associated with disease. When SNPs occur within a gene or in a regulatory region near a gene, they may play a more direct role in disease by affecting the gene’s function.</p>
<p>Most SNPs have no effect on health or development. Some of these genetic differences, however, have proven to be very important in the study of human health. Researchers have found SNPs that may help predict an individual’s response to certain drugs, susceptibility to environmental factors such as toxins, and risk of developing particular diseases. SNPs can also be used to track the inheritance of disease genes within families. Future studies will work to identify SNPs associated with complex diseases such as heart disease, diabetes, and cancer.</p>

</div>
<div id="talking-glossary-somatic-cells-0.75-min" class="section level2" number="17.7">
<h2><span class="header-section-number">17.7</span> Talking glossary: Somatic cells (0.75 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Somatic-Cells" class="uri">https://www.genome.gov/genetics-glossary/Somatic-Cells</a></p>
<p><strong>Abstract</strong>: “A somatic cell is any cell of the body except sperm and egg cells. Somatic cells are diploid, meaning that they contain two sets of chromosomes, one inherited from each parent. Mutations in somatic cells can affect the individual, but they are not passed on to offspring.”</p>
<p>Image: <a href="https://www.genome.gov/sites/default/files/tg/en/illustration/somatic_cells.jpg" class="uri">https://www.genome.gov/sites/default/files/tg/en/illustration/somatic_cells.jpg</a></p>
<p><strong>Transcript</strong>: “Somatic cells is a fairly general term which refers to essentially all the cells of the body except for the germ line; the germ line being the cells in the sexual organs that produce sperm and eggs. So anything that doesn’t have the job of producing sperm or eggs is a somatic cell. It is very important, of course, for every living organism to be alive, but it contributes nothing in terms of inheritance through genetics, inheritance to the next generation. So it is only of use to the living organism and has no relation to anything that happens in the next generation of that organism.”</p>
<p>Shawn Burgess, Ph.D.</p>

</div>
<div id="sequence-annotation" class="section level2" number="17.8">
<h2><span class="header-section-number">17.8</span> Sequence annotation</h2>
<p>Adapted from Wikipedia, the free encyclopedia</p>
<p><a href="https://en.wikipedia.org/wiki/DNA_annotation" class="uri">https://en.wikipedia.org/wiki/DNA_annotation</a></p>
<p>Note: Wikipedia titles this “DNA annotation.” In my experience this phrase is not used; rather we refer to sequence annotation, genome annotation, or just annotation.</p>
<p>Genome annotation is the process of predicting the locations of genes and all of the coding regions in a genome and predicitng what those genes do. An annotation (irrespective of the context) is a note added by way of explanation or commentary. Once a genome is sequenced, it needs to be annotated to make sense of it.[1]
For DNA annotation, a previously unknown sequence representation of genetic material is enriched with information of intron -exon boundaries, regulatory sequences , gene names and protein products. This annotation is stored in genomic databases.</p>
<div id="process" class="section level3" number="17.8.1">
<h3><span class="header-section-number">17.8.1</span> Process</h3>
<p>Genome annotation consists of two main steps:</p>
<ol style="list-style-type: decimal">
<li>predicting coding elements on the genome , a process called gene prediction</li>
<li>attaching biological information to these elements</li>
</ol>
<p>Structural genomic annotation consists of the identification of genomic elements.</p>
<ol style="list-style-type: decimal">
<li>ORFs [open reading frames] and their localization</li>
<li>predicted coding regions</li>
<li>predicted location of regulatory motifs (e.g. promoters)</li>
<li>Functional annotation consists of attaching biological information to genomic elements.</li>
</ol>

</div>
</div>
<div id="talking-glossary-substitution-mutation-1.75-min" class="section level2" number="17.9">
<h2><span class="header-section-number">17.9</span> Talking Glossary: Substitution mutation (1.75 min)</h2>
<p><a href="https://www.genome.gov/genetics-glossary/Substitution" class="uri">https://www.genome.gov/genetics-glossary/Substitution</a></p>
<p><strong>Abstract</strong>: “Substitution is a type of mutation where one base pair is replaced by a different base pair. The term also refers to the replacement of one amino acid in a protein with a different amino acid.”</p>
<p><strong>Transcript:</strong> “Subsititution refers to the replacement of one amino acid with another amino acid in a protein or the replacement of one nucleotide with another in DNA or RNA. Substitutions generally give rise to–or they always give rise to–either a polymorphism, that is, a difference between one person, one individual, in a population or another, or a special kind of polymorphism that we call a mutation. In either case, all individuals in the population originally had the same sequence of a gene. There were substitution events that resulted in a change in DNA sequence, which resulted in a change in RNA sequence, which then could result in a change in amino acid sequence. When that happens, that change in DNA sequence or amino acid sequence, or both, could have no effect on the protein, in which case the substitution is benign and has no functional effect. In other cases, if it changes the function of the protein, then it will be observed as either a functional polymorphism, something which increases the effectiveness of the protein product, and therefore would be evolutionarily selected for, or is bad–deleterious–in which case the person might die early and get evolutionarily selected against. Substitutions which lead to mutations, which lead to a deleterious outcome, that is the organism having difficulty with living or dying early, those we call mutations, but they’re the result of a certain kind of a substitution.”</p>
<p>Christopher P. Austin, M.D.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="r.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="t.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["openbio.pdf", "openbio.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
