<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title> 19 U | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</title>
  <meta name="description" content="A collection of open-access and/or free resources on biology and environmental science." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content=" 19 U | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content=" 19 U | Foundations of Biology and Environmental Science: An Open-Access Encyclepedia" />
  
  <meta name="twitter:description" content="A collection of open-access and/or free resources on biology and environmental science." />
  

<meta name="author" content="Compiled and edited by Nathan Brouwer" />


<meta name="date" content="2021-11-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="t.html"/>
<link rel="next" href="v.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Biology for everyone</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="a.html"><a href="a.html"><i class="fa fa-check"></i><b>2</b> A</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a.html"><a href="a.html#alignments"><i class="fa fa-check"></i><b>2.1</b> Alignments</a></li>
<li class="chapter" data-level="2.2" data-path="a.html"><a href="a.html#talking-glossary-allele-0.5-min"><i class="fa fa-check"></i><b>2.2</b> Talking Glossary: Allele (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="b.html"><a href="b.html"><i class="fa fa-check"></i><b>3</b> B</a>
<ul>
<li class="chapter" data-level="3.1" data-path="b.html"><a href="b.html#talking-glossary-bioinformatics"><i class="fa fa-check"></i><b>3.1</b> Talking glossary: Bioinformatics</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="c.html"><a href="c.html"><i class="fa fa-check"></i><b>4</b> C</a>
<ul>
<li class="chapter" data-level="4.1" data-path="c.html"><a href="c.html#talking-glossary-candidate-gene"><i class="fa fa-check"></i><b>4.1</b> Talking Glossary: Candidate gene</a></li>
<li class="chapter" data-level="4.2" data-path="c.html"><a href="c.html#cath-database"><i class="fa fa-check"></i><b>4.2</b> CATH database</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="c.html"><a href="c.html#references"><i class="fa fa-check"></i><b>4.2.1</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="c.html"><a href="c.html#clade"><i class="fa fa-check"></i><b>4.3</b> Clade</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="c.html"><a href="c.html#clades-and-phylogenetic-trees"><i class="fa fa-check"></i><b>4.3.1</b> Clades and phylogenetic trees</a></li>
<li class="chapter" data-level="4.3.2" data-path="c.html"><a href="c.html#terminology"><i class="fa fa-check"></i><b>4.3.2</b> Terminology</a></li>
<li class="chapter" data-level="4.3.3" data-path="c.html"><a href="c.html#in-popular-culture"><i class="fa fa-check"></i><b>4.3.3</b> In popular culture</a></li>
<li class="chapter" data-level="4.3.4" data-path="c.html"><a href="c.html#references-1"><i class="fa fa-check"></i><b>4.3.4</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="c.html"><a href="c.html#talking-glossary-cloning-0.5-min"><i class="fa fa-check"></i><b>4.4</b> Talking Glossary: Cloning (0.5 min)</a></li>
<li class="chapter" data-level="4.5" data-path="c.html"><a href="c.html#talking-glossary-codominance-0.5-min"><i class="fa fa-check"></i><b>4.5</b> Talking Glossary: Codominance (0.5 min)</a></li>
<li class="chapter" data-level="4.6" data-path="c.html"><a href="c.html#comparative-genomics"><i class="fa fa-check"></i><b>4.6</b> Comparative genomics</a></li>
<li class="chapter" data-level="4.7" data-path="c.html"><a href="c.html#talking-glossary-contig-0.75-min"><i class="fa fa-check"></i><b>4.7</b> Talking Glossary: Contig (0.75 min)</a></li>
<li class="chapter" data-level="4.8" data-path="c.html"><a href="c.html#talking-glossary---foundations-1-review-crossing-over"><i class="fa fa-check"></i><b>4.8</b> Talking Glossary - Foundations 1 Review: Crossing over</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="d.html"><a href="d.html"><i class="fa fa-check"></i><b>5</b> D</a>
<ul>
<li class="chapter" data-level="5.1" data-path="d.html"><a href="d.html#talking-glossary-deletion-mutation-1.5-min"><i class="fa fa-check"></i><b>5.1</b> Talking Glossary: Deletion mutation (1.5 min)</a></li>
<li class="chapter" data-level="5.2" data-path="d.html"><a href="d.html#talking-glossary-dna-sequencing-0.75-min"><i class="fa fa-check"></i><b>5.2</b> Talking Glossary: DNA sequencing (0.75 min)</a></li>
<li class="chapter" data-level="5.3" data-path="d.html"><a href="d.html#sex-chromosome-dosage-compensation"><i class="fa fa-check"></i><b>5.3</b> Sex-Chromosome Dosage compensation</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="d.html"><a href="d.html#random-inactivation-of-one-x-in-xx-individuals"><i class="fa fa-check"></i><b>5.3.1</b> Random inactivation of one X in XX individuals</a></li>
<li class="chapter" data-level="5.3.2" data-path="d.html"><a href="d.html#results"><i class="fa fa-check"></i><b>5.3.2</b> Results</a></li>
<li class="chapter" data-level="5.3.3" data-path="d.html"><a href="d.html#clinical-applications"><i class="fa fa-check"></i><b>5.3.3</b> Clinical applications</a></li>
<li class="chapter" data-level="5.3.4" data-path="d.html"><a href="d.html#agricultural-applications"><i class="fa fa-check"></i><b>5.3.4</b> Agricultural applications</a></li>
<li class="chapter" data-level="5.3.5" data-path="d.html"><a href="d.html#limitations"><i class="fa fa-check"></i><b>5.3.5</b> Limitations</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="d.html"><a href="d.html#fine-mapping"><i class="fa fa-check"></i><b>5.4</b> Fine-mapping</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="h.html"><a href="h.html"><i class="fa fa-check"></i><b>6</b> H</a>
<ul>
<li class="chapter" data-level="6.1" data-path="h.html"><a href="h.html#talking-glossary-haploid-1.25-min"><i class="fa fa-check"></i><b>6.1</b> Talking Glossary: Haploid (1.25 min)</a></li>
<li class="chapter" data-level="6.2" data-path="h.html"><a href="h.html#talking-glossary-halplotype"><i class="fa fa-check"></i><b>6.2</b> Talking Glossary: Halplotype</a></li>
<li class="chapter" data-level="6.3" data-path="h.html"><a href="h.html#talking-glossary-heterozygous"><i class="fa fa-check"></i><b>6.3</b> Talking Glossary: Heterozygous</a></li>
<li class="chapter" data-level="6.4" data-path="h.html"><a href="h.html#homologyl-when-things-are-homolgous-and-analogous"><i class="fa fa-check"></i><b>6.4</b> Homology:L When things are homolgous AND analogous!</a></li>
<li class="chapter" data-level="6.5" data-path="h.html"><a href="h.html#talking-glossary-homozygous"><i class="fa fa-check"></i><b>6.5</b> Talking Glossary: Homozygous</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="i.html"><a href="i.html"><i class="fa fa-check"></i><b>7</b> I</a>
<ul>
<li class="chapter" data-level="7.1" data-path="i.html"><a href="i.html#talking-glossary-insertion-mutation-1.25-min"><i class="fa fa-check"></i><b>7.1</b> Talking Glossary: Insertion mutation (1.25 min)</a></li>
<li class="chapter" data-level="7.2" data-path="i.html"><a href="i.html#intrinscially-disordered-protein"><i class="fa fa-check"></i><b>7.2</b> Intrinscially disordered protein</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="i.html"><a href="i.html#abundance"><i class="fa fa-check"></i><b>7.2.1</b> Abundance</a></li>
<li class="chapter" data-level="7.2.2" data-path="i.html"><a href="i.html#disorder-annotation"><i class="fa fa-check"></i><b>7.2.2</b> Disorder annotation</a></li>
<li class="chapter" data-level="7.2.3" data-path="i.html"><a href="i.html#disorder-databases"><i class="fa fa-check"></i><b>7.2.3</b> Disorder databases</a></li>
<li class="chapter" data-level="7.2.4" data-path="i.html"><a href="i.html#predicting-idps-by-sequence"><i class="fa fa-check"></i><b>7.2.4</b> Predicting IDPs by sequence</a></li>
<li class="chapter" data-level="7.2.5" data-path="i.html"><a href="i.html#prediction-methods"><i class="fa fa-check"></i><b>7.2.5</b> Prediction methods</a></li>
<li class="chapter" data-level="7.2.6" data-path="i.html"><a href="i.html#disorder-and-disease"><i class="fa fa-check"></i><b>7.2.6</b> Disorder and disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="j.html"><a href="j.html"><i class="fa fa-check"></i><b>8</b> J</a>
<ul>
<li class="chapter" data-level="8.1" data-path="j.html"><a href="j.html#jove-videos"><i class="fa fa-check"></i><b>8.1</b> JoVe Videos</a>
<ul>
<li class="chapter" data-level="8.1.1" data-path="j.html"><a href="j.html#biotechnology"><i class="fa fa-check"></i><b>8.1.1</b> Biotechnology</a></li>
<li class="chapter" data-level="8.1.2" data-path="j.html"><a href="j.html#macromolecules"><i class="fa fa-check"></i><b>8.1.2</b> Macromolecules</a></li>
<li class="chapter" data-level="8.1.3" data-path="j.html"><a href="j.html#cellular-respiration"><i class="fa fa-check"></i><b>8.1.3</b> Cellular respiration</a></li>
<li class="chapter" data-level="8.1.4" data-path="j.html"><a href="j.html#population-and-community-ecology"><i class="fa fa-check"></i><b>8.1.4</b> Population and community ecology</a></li>
<li class="chapter" data-level="8.1.5" data-path="j.html"><a href="j.html#population-genetics"><i class="fa fa-check"></i><b>8.1.5</b> Population genetics</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="k.html"><a href="k.html"><i class="fa fa-check"></i><b>9</b> K</a>
<ul>
<li class="chapter" data-level="9.1" data-path="k.html"><a href="k.html#talking-glossary-knockout-genetic-knockout-gene-knockout"><i class="fa fa-check"></i><b>9.1</b> Talking glossary: Knockout (Genetic knockout, gene knockout)</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="l.html"><a href="l.html"><i class="fa fa-check"></i><b>10</b> L</a>
<ul>
<li class="chapter" data-level="10.1" data-path="l.html"><a href="l.html#talking-glossary-locus-1-min"><i class="fa fa-check"></i><b>10.1</b> Talking Glossary: Locus (1 min)</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="m.html"><a href="m.html"><i class="fa fa-check"></i><b>11</b> M</a>
<ul>
<li class="chapter" data-level="11.1" data-path="m.html"><a href="m.html#manhattan-plot"><i class="fa fa-check"></i><b>11.1</b> Manhattan Plot</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="m.html"><a href="m.html#gwas"><i class="fa fa-check"></i><b>11.1.1</b> GWAS</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="m.html"><a href="m.html#talking-glossary-microarray"><i class="fa fa-check"></i><b>11.2</b> Talking Glossary: Microarray</a></li>
<li class="chapter" data-level="11.3" data-path="m.html"><a href="m.html#talking-glossary-missense-mutation-1.25-min"><i class="fa fa-check"></i><b>11.3</b> Talking Glossary: Missense Mutation (1.25 min)</a></li>
<li class="chapter" data-level="11.4" data-path="m.html"><a href="m.html#talking-glossary-mitochondrial-dna-1.5-min"><i class="fa fa-check"></i><b>11.4</b> Talking Glossary: Mitochondrial DNA (1.5 min)</a></li>
<li class="chapter" data-level="11.5" data-path="m.html"><a href="m.html#talking-glossary-mutagen-0.5-min"><i class="fa fa-check"></i><b>11.5</b> Talking Glossary: Mutagen (0.5 min)</a></li>
<li class="chapter" data-level="11.6" data-path="m.html"><a href="m.html#talking-glossary-mutation-0.5-min"><i class="fa fa-check"></i><b>11.6</b> Talking Glossary: Mutation (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="n.html"><a href="n.html"><i class="fa fa-check"></i><b>12</b> N</a>
<ul>
<li class="chapter" data-level="12.1" data-path="n.html"><a href="n.html#talking-glossary-non-coding-dna-1-min"><i class="fa fa-check"></i><b>12.1</b> Talking Glossary: Non-coding DNA (1 min)</a></li>
<li class="chapter" data-level="12.2" data-path="n.html"><a href="n.html#talking-glossary-nonsene-mutation"><i class="fa fa-check"></i><b>12.2</b> Talking Glossary: Nonsene mutation</a></li>
<li class="chapter" data-level="12.3" data-path="n.html"><a href="n.html#nucleic-acids-review"><i class="fa fa-check"></i><b>12.3</b> Nucleic Acids - Overview</a>
<ul>
<li class="chapter" data-level="12.3.1" data-path="n.html"><a href="n.html#dna-and-rna"><i class="fa fa-check"></i><b>12.3.1</b> DNA and RNA</a></li>
<li class="chapter" data-level="12.3.2" data-path="n.html"><a href="n.html#dna-double-helix-structure"><i class="fa fa-check"></i><b>12.3.2</b> DNA Double-Helix Structure</a></li>
<li class="chapter" data-level="12.3.3" data-path="n.html"><a href="n.html#rna"><i class="fa fa-check"></i><b>12.3.3</b> RNA</a></li>
<li class="chapter" data-level="12.3.4" data-path="n.html"><a href="n.html#protein-function"><i class="fa fa-check"></i><b>12.3.4</b> Protein function</a></li>
<li class="chapter" data-level="12.3.5" data-path="n.html"><a href="n.html#summary"><i class="fa fa-check"></i><b>12.3.5</b> Summary</a></li>
<li class="chapter" data-level="12.3.6" data-path="n.html"><a href="n.html#glossary"><i class="fa fa-check"></i><b>12.3.6</b> Glossary</a></li>
<li class="chapter" data-level="12.3.7" data-path="n.html"><a href="n.html#contributors-and-attributions"><i class="fa fa-check"></i><b>12.3.7</b> Contributors and Attributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="13" data-path="o.html"><a href="o.html"><i class="fa fa-check"></i><b>13</b> O</a>
<ul>
<li class="chapter" data-level="13.1" data-path="o.html"><a href="o.html#talking-glossary-open-reading-frame-definition-3-min"><i class="fa fa-check"></i><b>13.1</b> Talking Glossary : Open reading frame definition (3 min)</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="p.html"><a href="p.html"><i class="fa fa-check"></i><b>14</b> P</a>
<ul>
<li class="chapter" data-level="14.1" data-path="p.html"><a href="p.html#talking-glossary-pcr---the-polymerase-chain-reaction-0.5-min"><i class="fa fa-check"></i><b>14.1</b> Talking Glossary: PCR - The Polymerase Chain Reaction (0.5 min)</a></li>
<li class="chapter" data-level="14.2" data-path="p.html"><a href="p.html#protein-data-bank"><i class="fa fa-check"></i><b>14.2</b> Protein Data Bank</a>
<ul>
<li class="chapter" data-level="14.2.1" data-path="p.html"><a href="p.html#history"><i class="fa fa-check"></i><b>14.2.1</b> History</a></li>
<li class="chapter" data-level="14.2.2" data-path="p.html"><a href="p.html#file-format"><i class="fa fa-check"></i><b>14.2.2</b> File format</a></li>
</ul></li>
<li class="chapter" data-level="14.3" data-path="p.html"><a href="p.html#pfam"><i class="fa fa-check"></i><b>14.3</b> PFam</a>
<ul>
<li class="chapter" data-level="14.3.1" data-path="p.html"><a href="p.html#uses"><i class="fa fa-check"></i><b>14.3.1</b> Uses</a></li>
</ul></li>
<li class="chapter" data-level="14.4" data-path="p.html"><a href="p.html#talking-glossary-phenotype"><i class="fa fa-check"></i><b>14.4</b> Talking Glossary: Phenotype</a></li>
<li class="chapter" data-level="14.5" data-path="p.html"><a href="p.html#phylogenetics-vocab"><i class="fa fa-check"></i><b>14.5</b> Phylogenetics vocab</a></li>
<li class="chapter" data-level="14.6" data-path="p.html"><a href="p.html#talking-glossary-plasmid-1-min"><i class="fa fa-check"></i><b>14.6</b> Talking Glossary: Plasmid (1 min)</a></li>
<li class="chapter" data-level="14.7" data-path="p.html"><a href="p.html#talking-glossary-point-mutation-0.5-min"><i class="fa fa-check"></i><b>14.7</b> Talking Glossary: Point mutation (0.5 min)</a></li>
<li class="chapter" data-level="14.8" data-path="p.html"><a href="p.html#talking-glossary-polymorphism-1-min"><i class="fa fa-check"></i><b>14.8</b> Talking Glossary: Polymorphism (1 min)</a></li>
<li class="chapter" data-level="14.9" data-path="p.html"><a href="p.html#talking-glossary-u03-primer-0.5-min"><i class="fa fa-check"></i><b>14.9</b> Talking Glossary U03: Primer (0.5 min)</a></li>
<li class="chapter" data-level="14.10" data-path="p.html"><a href="p.html#talking-glossary-promoter-1.5-min"><i class="fa fa-check"></i><b>14.10</b> Talking Glossary: Promoter (1.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="q.html"><a href="q.html"><i class="fa fa-check"></i><b>15</b> Q</a></li>
<li class="chapter" data-level="16" data-path="r.html"><a href="r.html"><i class="fa fa-check"></i><b>16</b> R</a>
<ul>
<li class="chapter" data-level="16.1" data-path="r.html"><a href="r.html#talking-glossary-recessive"><i class="fa fa-check"></i><b>16.1</b> Talking Glossary: Recessive</a></li>
<li class="chapter" data-level="16.2" data-path="r.html"><a href="r.html#talking-glossary-recombinant-dna-0.5-min"><i class="fa fa-check"></i><b>16.2</b> Talking Glossary: Recombinant DNA (0.5 min)</a></li>
<li class="chapter" data-level="16.3" data-path="r.html"><a href="r.html#talking-glossary-repressor-1-min"><i class="fa fa-check"></i><b>16.3</b> Talking Glossary: Repressor (1 min)</a></li>
<li class="chapter" data-level="16.4" data-path="r.html"><a href="r.html#talking-glossary-restriction-enzyme-0.5-min"><i class="fa fa-check"></i><b>16.4</b> Talking Glossary: Restriction Enzyme (0.5 min)</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="s.html"><a href="s.html"><i class="fa fa-check"></i><b>17</b> S</a>
<ul>
<li class="chapter" data-level="17.1" data-path="s.html"><a href="s.html#structural-classification-of-proteins-database-scop"><i class="fa fa-check"></i><b>17.1</b> Structural Classification of Proteins database (SCOP)</a>
<ul>
<li class="chapter" data-level="17.1.1" data-path="s.html"><a href="s.html#hierarchical-organisation"><i class="fa fa-check"></i><b>17.1.1</b> Hierarchical organisation</a></li>
<li class="chapter" data-level="17.1.2" data-path="s.html"><a href="s.html#superfamilies"><i class="fa fa-check"></i><b>17.1.2</b> Superfamilies</a></li>
<li class="chapter" data-level="17.1.3" data-path="s.html"><a href="s.html#families"><i class="fa fa-check"></i><b>17.1.3</b> Families</a></li>
<li class="chapter" data-level="17.1.4" data-path="s.html"><a href="s.html#pdb-entry-domains"><i class="fa fa-check"></i><b>17.1.4</b> PDB entry domains</a></li>
</ul></li>
<li class="chapter" data-level="17.2" data-path="s.html"><a href="s.html#sequence-alignment"><i class="fa fa-check"></i><b>17.2</b> Sequence alignment</a>
<ul>
<li class="chapter" data-level="17.2.1" data-path="s.html"><a href="s.html#interpretation"><i class="fa fa-check"></i><b>17.2.1</b> Interpretation</a></li>
<li class="chapter" data-level="17.2.2" data-path="s.html"><a href="s.html#alignment-methods"><i class="fa fa-check"></i><b>17.2.2</b> Alignment methods</a></li>
<li class="chapter" data-level="17.2.3" data-path="s.html"><a href="s.html#representations"><i class="fa fa-check"></i><b>17.2.3</b> Representations</a></li>
<li class="chapter" data-level="17.2.4" data-path="s.html"><a href="s.html#global-and-local-alignments"><i class="fa fa-check"></i><b>17.2.4</b> Global and local alignments</a></li>
<li class="chapter" data-level="17.2.5" data-path="s.html"><a href="s.html#pairwise-alignment"><i class="fa fa-check"></i><b>17.2.5</b> Pairwise alignment</a></li>
<li class="chapter" data-level="17.2.6" data-path="s.html"><a href="s.html#dot-matrix-methods"><i class="fa fa-check"></i><b>17.2.6</b> Dot-matrix methods</a></li>
<li class="chapter" data-level="17.2.7" data-path="s.html"><a href="s.html#dynamic-programming"><i class="fa fa-check"></i><b>17.2.7</b> Dynamic programming</a></li>
<li class="chapter" data-level="17.2.8" data-path="s.html"><a href="s.html#word-methods"><i class="fa fa-check"></i><b>17.2.8</b> Word methods</a></li>
<li class="chapter" data-level="17.2.9" data-path="s.html"><a href="s.html#multiple-sequence-alignment"><i class="fa fa-check"></i><b>17.2.9</b> Multiple sequence alignment</a></li>
<li class="chapter" data-level="17.2.10" data-path="s.html"><a href="s.html#phylogenetic-analysis"><i class="fa fa-check"></i><b>17.2.10</b> Phylogenetic analysis</a></li>
<li class="chapter" data-level="17.2.11" data-path="s.html"><a href="s.html#assessment-of-significance"><i class="fa fa-check"></i><b>17.2.11</b> Assessment of significance</a></li>
<li class="chapter" data-level="17.2.12" data-path="s.html"><a href="s.html#other-biological-uses"><i class="fa fa-check"></i><b>17.2.12</b> Other biological uses</a></li>
<li class="chapter" data-level="17.2.13" data-path="s.html"><a href="s.html#references-2"><i class="fa fa-check"></i><b>17.2.13</b> References</a></li>
</ul></li>
<li class="chapter" data-level="17.3" data-path="s.html"><a href="s.html#talking-glossary-u03-shotgun-sequencing-1-min"><i class="fa fa-check"></i><b>17.3</b> Talking Glossary U03: Shotgun sequencing (1 min)</a></li>
<li class="chapter" data-level="17.4" data-path="s.html"><a href="s.html#talking-glossary-sickle-cell-disease-4-min"><i class="fa fa-check"></i><b>17.4</b> Talking Glossary: Sickle Cell Disease (4 min)</a></li>
<li class="chapter" data-level="17.5" data-path="s.html"><a href="s.html#medline-snps-single-nucleotide-polymorphism"><i class="fa fa-check"></i><b>17.5</b> Medline: SNPs (single nucleotide polymorphism)</a></li>
<li class="chapter" data-level="17.6" data-path="s.html"><a href="s.html#talking-glossary-somatic-cells-0.75-min"><i class="fa fa-check"></i><b>17.6</b> Talking glossary: Somatic cells (0.75 min)</a></li>
<li class="chapter" data-level="17.7" data-path="s.html"><a href="s.html#sequence-annotation"><i class="fa fa-check"></i><b>17.7</b> Sequence annotation</a>
<ul>
<li class="chapter" data-level="17.7.1" data-path="s.html"><a href="s.html#process"><i class="fa fa-check"></i><b>17.7.1</b> Process</a></li>
</ul></li>
<li class="chapter" data-level="17.8" data-path="s.html"><a href="s.html#talking-glossary-substitution-mutation-1.75-min"><i class="fa fa-check"></i><b>17.8</b> Talking Glossary: Substitution mutation (1.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="t.html"><a href="t.html"><i class="fa fa-check"></i><b>18</b> T</a>
<ul>
<li class="chapter" data-level="18.1" data-path="t.html"><a href="t.html#taxa-taxon-and-clades-a-brief-primer"><i class="fa fa-check"></i><b>18.1</b> Taxa, Taxon, and clades: A Brief Primer</a>
<ul>
<li class="chapter" data-level="18.1.1" data-path="t.html"><a href="t.html#what-is-a-taxon"><i class="fa fa-check"></i><b>18.1.1</b> What is a “taxon”</a></li>
<li class="chapter" data-level="18.1.2" data-path="t.html"><a href="t.html#what-is-a-clade"><i class="fa fa-check"></i><b>18.1.2</b> What is a clade?</a></li>
<li class="chapter" data-level="18.1.3" data-path="t.html"><a href="t.html#further-reading"><i class="fa fa-check"></i><b>18.1.3</b> Further Reading:</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="t.html"><a href="t.html#talking-glossary-trait-0.75-min"><i class="fa fa-check"></i><b>18.2</b> Talking Glossary: Trait (0.75 min)</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="u.html"><a href="u.html"><i class="fa fa-check"></i><b>19</b> U</a>
<ul>
<li class="chapter" data-level="19.1" data-path="u.html"><a href="u.html#uniprot"><i class="fa fa-check"></i><b>19.1</b> Uniprot</a>
<ul>
<li class="chapter" data-level="19.1.1" data-path="u.html"><a href="u.html#the-uniprot-consortium"><i class="fa fa-check"></i><b>19.1.1</b> The UniProt consortium</a></li>
<li class="chapter" data-level="19.1.2" data-path="u.html"><a href="u.html#the-roots-of-uniprot-databases"><i class="fa fa-check"></i><b>19.1.2</b> The roots of UniProt databases</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="20" data-path="v.html"><a href="v.html"><i class="fa fa-check"></i><b>20</b> V</a>
<ul>
<li class="chapter" data-level="20.1" data-path="v.html"><a href="v.html#talking-glossary-vector-2-min"><i class="fa fa-check"></i><b>20.1</b> Talking Glossary: Vector (2 min)</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="w.html"><a href="w.html"><i class="fa fa-check"></i><b>21</b> W</a>
<ul>
<li class="chapter" data-level="21.1" data-path="w.html"><a href="w.html#talking-glossary-x-chromosome-1.25-min"><i class="fa fa-check"></i><b>21.1</b> Talking Glossary: X-chromosome (1.25 min)</a></li>
<li class="chapter" data-level="21.2" data-path="w.html"><a href="w.html#talking-glossary-x-inactivation-lyonization-1.45-min"><i class="fa fa-check"></i><b>21.2</b> Talking Glossary: X-inactivation (“Lyonization”; 1.45 min))</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="x.html"><a href="x.html"><i class="fa fa-check"></i><b>22</b> X</a></li>
<li class="chapter" data-level="23" data-path="v-1.html"><a href="v-1.html"><i class="fa fa-check"></i><b>23</b> V</a>
<ul>
<li class="chapter" data-level="23.1" data-path="v-1.html"><a href="v-1.html#additional-glossaries"><i class="fa fa-check"></i><b>23.1</b> Additional Glossaries</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="z.html"><a href="z.html"><i class="fa fa-check"></i><b>24</b> Z</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Foundations of Biology and Environmental Science: An Open-Access Encyclepedia</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="u" class="section level1" number="19">
<h1><span class="header-section-number"> 19</span> U</h1>

<div id="uniprot" class="section level2" number="19.1">
<h2><span class="header-section-number">19.1</span> Uniprot</h2>
<p>Adapated from Wikipedia, the free encyclopedia</p>
<p>UniProt (www.uniprot.org/) is a freely accessible database of protein sequence and functional information, many entries being derived from genome sequencing projects. It contains a large amount of information about the biological function of proteins derived from the research literature. It is maintained by the UniProt consortium, which consists of several European bioinformatics organisations and a foundation from Washington, DC, United States.</p>
<div id="the-uniprot-consortium" class="section level3" number="19.1.1">
<h3><span class="header-section-number">19.1.1</span> The UniProt consortium</h3>
<p><strong>OPTIONAL</strong></p>
<p>The UniProt consortium comprises the European Bioinformatics Institute (EBI), the Swiss Institute of Bioinformatics (SIB), and the Protein Information Resource (PIR). EBI, located at the Wellcome Trust Genome Campus in Hinxton, UK, hosts a large resource of bioinformatics databases and services. SIB, located in Geneva, Switzerland, maintains the ExPASy (Expert Protein Analysis System) servers that are a central resource for proteomics tools and databases. PIR, hosted by the National Biomedical Research Foundation (NBRF) at the Georgetown University Medical Center in Washington, DC, US, is heir to the oldest protein sequence database, Margaret Dayhoff’s Atlas of Protein Sequence and Structure, first published in 1965.[2] In 2002, EBI, SIB, and PIR joined forces as the UniProt consortium.[3]</p>
</div>
<div id="the-roots-of-uniprot-databases" class="section level3" number="19.1.2">
<h3><span class="header-section-number">19.1.2</span> The roots of UniProt databases</h3>
<p><strong>OPTIONAL</strong></p>
<p>Each consortium member is heavily involved in protein database maintenance and annotation. Until recently, EBI and SIB together produced the Swiss-Prot and TrEMBL databases, while PIR produced the Protein Sequence Database (PIR-PSD).[4][5][6] These databases coexisted with differing protein sequence coverage and annotation priorities.</p>
<p>Swiss-Prot was created in 1986 by Amos Bairoch during his PhD and developed by the Swiss Institute of Bioinformatics and subsequently developed by Rolf Apweiler at the European Bioinformatics Institute.[7][8][9] Swiss-Prot aimed to provide reliable protein sequences associated with a high level of annotation (such as the description of the function of a protein, its domain structure, post-translational modifications, variants, etc.), a minimal level of redundancy and high level of integration with other databases. Recognizing that sequence data were being generated at a pace exceeding Swiss-Prot’s ability to keep up, TrEMBL (Translated EMBL Nucleotide Sequence Data Library) was created to provide automated annotations for those proteins not in Swiss-Prot. Meanwhile, PIR maintained the PIR-PSD and related databases, including iProClass, a database of protein sequences and curated families.</p>
<p>The consortium members pooled their overlapping resources and expertise, and launched UniProt in December 2003.[10]</p>
<!-- ### Organization of UniProt databases -->
<!-- UniProt provides four core databases: UniProtKB (with sub-parts Swiss-Prot and TrEMBL), UniParc, UniRef. -->
<!-- ### UniProtKB -->
<!-- UniProt Knowledgebase (UniProtKB) is a protein database partially curated by experts, consisting of two sections: UniProtKB/Swiss-Prot (containing reviewed, manually annotated entries) and UniProtKB/TrEMBL (containing unreviewed, automatically annotated entries).[11] As of 19 March 2014, release "2014_03" of UniProtKB/Swiss-Prot contains 542,782 sequence entries (comprising 193,019,802 amino acids abstracted from 226,896 references) and release "2014_03" of UniProtKB/TrEMBL contains 54,247,468 sequence entries (comprising 17,207,833,179 amino acids).[12][13] -->
<!-- ### UniProtKB/Swiss-Prot -->
<!-- UniProtKB/Swiss-Prot is a manually annotated, non-redundant protein sequence database. It combines information extracted from scientific literature and biocurator-evaluated computational analysis. The aim of UniProtKB/Swiss-Prot is to provide all known relevant information about a particular protein. Annotation is regularly reviewed to keep up with current scientific findings. The manual annotation of an entry involves detailed analysis of the protein sequence and of the scientific literature.[14] -->
<!-- Sequences from the same gene and the same species are merged into the same database entry. Differences between sequences are identified, and their cause documented (for example alternative splicing, natural variation, incorrect initiation sites, incorrect exon boundaries, frameshifts, unidentified conflicts). A range of sequence analysis tools is used in the annotation of UniProtKB/Swiss-Prot entries. Computer-predictions are manually evaluated, and relevant results selected for inclusion in the entry. These predictions include post-translational modifications, transmembrane domains and topology, signal peptides, domain identification, and protein family classification.[14][15] -->
<!-- Relevant publications are identified by searching databases such as PubMed. The full text of each paper is read, and information is extracted and added to the entry. Annotation arising from the scientific literature includes, but is not limited to:[10][14][15] -->
<!-- Protein and gene names -->
<!-- Function -->
<!-- Enzyme-specific information such as catalytic activity, cofactors and catalytic residues -->
<!-- Subcellular location -->
<!-- Protein-protein interactions -->
<!-- Pattern of expression -->
<!-- Locations and roles of significant domains and sites -->
<!-- Ion-, substrate- and cofactor-binding sites -->
<!-- Protein variant forms produced by natural genetic variation, RNA editing, alternative splicing, proteolytic processing, and post-translational modification -->
<!-- Annotated entries undergo quality assurance before inclusion into UniProtKB/Swiss-Prot. When new data becomes available, entries are updated. -->
<!-- UniProtKB/TrEMBL -->
<!-- UniProtKB/TrEMBL contains high-quality computationally analyzed records, which are enriched with automatic annotation. It was introduced in response to increased dataflow resulting from genome projects, as the time- and labour-consuming manual annotation process of UniProtKB/Swiss-Prot could not be broadened to include all available protein sequences.[10] The translations of annotated coding sequences in the EMBL-Bank/GenBank/DDBJ nucleotide sequence database are automatically processed and entered in UniProtKB/TrEMBL. UniProtKB/TrEMBL also contains sequences from PDB, and from gene prediction, including Ensembl, RefSeq and CCDS.[16] Since 22 July 2021 it also includes predicted with AlphaFold tertiary structures.[17] -->
<!-- UniParc -->
<!-- UniProt Archive (UniParc) is a comprehensive and non-redundant database, which contains all the protein sequences from the main, publicly available protein sequence databases.[18] Proteins may exist in several different source databases, and in multiple copies in the same database. In order to avoid redundancy, UniParc stores each unique sequence only once. Identical sequences are merged, regardless of whether they are from the same or different species. Each sequence is given a stable and unique identifier (UPI), making it possible to identify the same protein from different source databases. UniParc contains only protein sequences, with no annotation. Database cross-references in UniParc entries allow further information about the protein to be retrieved from the source databases. When sequences in the source databases change, these changes are tracked by UniParc and history of all changes is archived. -->
<!-- Source databases -->
<!-- Currently UniParc contains protein sequences from the following publicly available databases: -->
<!--   INSDC EMBL-Bank/DDBJ/GenBank nucleotide sequence databases -->
<!-- Ensembl -->
<!-- European Patent Office (EPO) -->
<!-- FlyBase: the primary repository of genetic and molecular data for the insect family Drosophilidae (FlyBase) -->
<!-- H-Invitational Database (H-Inv) -->
<!-- International Protein Index (IPI) -->
<!-- Japan Patent Office (JPO) -->
<!-- Protein Information Resource (PIR-PSD) -->
<!-- Protein Data Bank (PDB) -->
<!-- Protein Research Foundation (PRF)[19] -->
<!-- RefSeq -->
<!-- Saccharomyces Genome Database (SGD) -->
<!-- The Arabidopsis Information Resource (TAIR) -->
<!-- TROME[20] -->
<!-- US Patent Office (USPTO) -->
<!-- UniProtKB/Swiss-Prot, UniProtKB/Swiss-Prot protein isoforms, UniProtKB/TrEMBL -->
<!-- Vertebrate and Genome Annotation Database (VEGA) -->
<!-- WormBase -->
<!-- UniRef -->
<!-- The UniProt Reference Clusters (UniRef) consist of three databases of clustered sets of protein sequences from UniProtKB and selected UniParc records.[21] The UniRef100 database combines identical sequences and sequence fragments (from any organism) into a single UniRef entry. The sequence of a representative protein, the accession numbers of all the merged entries and links to the corresponding UniProtKB and UniParc records are displayed. UniRef100 sequences are clustered using the CD-HIT algorithm to build UniRef90 and UniRef50.[21][22] Each cluster is composed of sequences that have at least 90% or 50% sequence identity, respectively, to the longest sequence. Clustering sequences significantly reduces database size, enabling faster sequence searches. -->
<!-- UniRef is available from the UniProt FTP site. -->
<!-- Funding -->
<!-- UniProt is funded by grants from the National Human Genome Research Institute, the National Institutes of Health (NIH), the European Commission, the Swiss Federal Government through the Federal Office of Education and Science, NCI-caBIG, and the US Department of Defense.[11] -->
<!-- References -->
<!-- UniProt, Consortium. (January 2015). "UniProt: a hub for protein information". Nucleic Acids Research. 43 (Database issue): D204–12. doi:10.1093/nar/gku989. PMC 4384041. PMID 25348405. -->
<!-- Dayhoff, Margaret O. (1965). Atlas of protein sequence and structure. Silver Spring, Md: National Biomedical Research Foundation. -->
<!-- "2002 Release: NHGRI Funds Global Protein Database". National Human Genome Research Institute (NHGRI). Retrieved 14 April 2018. -->
<!-- O'Donovan, C.; Martin, M. J.; Gattiker, A.; Gasteiger, E.; Bairoch, A.; Apweiler, R. (2002). "High-quality protein knowledge resource: SWISS-PROT and TrEMBL". Briefings in Bioinformatics. 3 (3): 275–284. doi:10.1093/bib/3.3.275. PMID 12230036. -->
<!--  Wu, C. H.; Yeh, L. S.; Huang, H.; Arminski, L.; Castro-Alvear, J.; Chen, Y.; Hu, Z.; Kourtesis, P.; Ledley, R. S.; Suzek, B. E.; Vinayaka, C. R.; Zhang, J.; Barker, W. C. (2003). "The Protein Information Resource". Nucleic Acids Research. 31 (1): 345–347. doi:10.1093/nar/gkg040. PMC 165487. PMID 12520019. -->
<!--  Boeckmann, B.; Bairoch, A.; Apweiler, R.; Blatter, M. C.; Estreicher, A.; Gasteiger, E.; Martin, M. J.; Michoud, K.; O'Donovan, C.; Phan, I.; Pilbout, S.; Schneider, M. (2003). "The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003". Nucleic Acids Research. 31 (1): 365–370. doi:10.1093/nar/gkg095. PMC 165542. PMID 12520024. -->
<!-- Bairoch, A.; Apweiler, R. (1996). "The SWISS-PROT protein sequence data bank and its new supplement TREMBL". Nucleic Acids Research. 24 (1): 21–25. doi:10.1093/nar/24.1.21. PMC 145613. PMID 8594581. -->
<!-- Bairoch, A. (2000). "Serendipity in bioinformatics, the tribulations of a Swiss bioinformatician through exciting times!". Bioinformatics. 16 (1): 48–64. doi:10.1093/bioinformatics/16.1.48. PMID 10812477. -->
<!-- Séverine Altairac, "Naissance d’une banque de données: Interview du prof. Amos Bairoch". Protéines à la Une, August 2006. ISSN 1660-9824. -->
<!-- Apweiler, R.; Bairoch, A.; Wu, C. H. (2004). "Protein sequence databases". Current Opinion in Chemical Biology. 8 (1): 76–80. doi:10.1016/j.cbpa.2003.12.004. PMID 15036160. -->
<!-- Uniprot, C. (2009). "The Universal Protein Resource (UniProt) in 2010". Nucleic Acids Research. 38 (Database issue): D142–D148. doi:10.1093/nar/gkp846. PMC 2808944. PMID 19843607. -->
<!-- "UniProtKB/Swiss-Prot Release 2018_03 statistics". web.expasy.org. Retrieved 14 April 2018. -->
<!-- EMBL-EBI. "Current Release Statistics < Uniprot < EMBL-EBI". www.ebi.ac.uk. Retrieved 14 April 2018. -->
<!-- "How do we manually annotate a UniProtKB entry?". www.uniprot.org. Retrieved 14 April 2018. -->
<!-- Apweiler, R.; Bairoch, A.; Wu, C. H.; Barker, W. C.; Boeckmann, B.; Ferro, S.; Gasteiger, E.; Huang, H.; Lopez, R.; Magrane, M.; Martin, M. J.; Natale, D. A.; o’Donovan, C.; Redaschi, N.; Yeh, L. S. (2004). "UniProt: The Universal Protein knowledgebase". Nucleic Acids Research. 32 (90001): 115D–1119. doi:10.1093/nar/gkh131. PMC 308865. PMID 14681372. -->
<!-- "Where do the UniProtKB protein sequences come from?". www.uniprot.org. Retrieved 14 April 2018. -->
<!-- "Putting the power of AlphaFold into the world's hands". Deepmind. Retrieved 24 July 2021. -->
<!-- Leinonen, R.; Diez, F. G.; Binns, D.; Fleischmann, W.; Lopez, R.; Apweiler, R. (2004). "UniProt archive". Bioinformatics. 20 (17): 3236–3237. doi:10.1093/bioinformatics/bth191. PMID 15044231. -->
<!-- "Protein Research Foundation". -->
<!-- ftp://ftp.isrec.isb-sib.ch/pub/databases/trome -->
<!-- Suzek, B. E.; Huang, H.; McGarvey, P.; Mazumder, R.; Wu, C. H. (2007). "UniRef: Comprehensive and non-redundant UniProt reference clusters". Bioinformatics. 23 (10): 1282–1288. doi:10.1093/bioinformatics/btm098. PMID 17379688. -->
<!-- Li, W.; Jaroszewski, L.; Godzik, A. (2001). "Clustering of highly homologous sequences to reduce the size of large protein databases". Bioinformatics. 17 (3): 282–283. doi:10.1093/bioinformatics/17.3.282. PMID 11294794. -->

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="t.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="v.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["openbio.pdf", "openbio.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
